
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>A Network Pharmacology and Molecular-Docking-Based Approach to Identify the Probable Targets of Short-Chain Fatty-Acid-Producing Microbial Metabolites against Kidney Cancer and Inflammation - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A450000579376FA93CF.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="biomol" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC10669250/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10669250/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Biomolecules" /><meta name="citation_title" content="A Network Pharmacology and Molecular-Docking-Based Approach to Identify the Probable Targets of Short-Chain Fatty-Acid-Producing Microbial Metabolites against Kidney Cancer and Inflammation" /><meta name="citation_author" content="Md. Rezaul Karim" /><meta name="citation_author_institution" content="Department of Biopharmaceutical Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; rk.ca.uhk@lumihsluazer (M.R.K.); rk.ca.uhk@69dehsromjain (M.N.M.); rk.ca.uhk@nirodaifas (S.I.); rk.ca.uhk@gnaycd (D.C.Y.)" /><meta name="citation_author_institution" content="Department of Biotechnology and Genetic Engineering, Faculty of Biological Sciences, Islamic University, Kushtia 7003, Bangladesh" /><meta name="citation_author" content="Md. Niaj Morshed" /><meta name="citation_author_institution" content="Department of Biopharmaceutical Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; rk.ca.uhk@lumihsluazer (M.R.K.); rk.ca.uhk@69dehsromjain (M.N.M.); rk.ca.uhk@nirodaifas (S.I.); rk.ca.uhk@gnaycd (D.C.Y.)" /><meta name="citation_author" content="Safia Iqbal" /><meta name="citation_author_institution" content="Department of Biopharmaceutical Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; rk.ca.uhk@lumihsluazer (M.R.K.); rk.ca.uhk@69dehsromjain (M.N.M.); rk.ca.uhk@nirodaifas (S.I.); rk.ca.uhk@gnaycd (D.C.Y.)" /><meta name="citation_author_institution" content="Department of Microbiology, Varendra Institute of Biosciences, Affiliated University of Rajshahi, Natore, Rajshahi 6400, Bangladesh" /><meta name="citation_author" content="Shahnawaz Mohammad" /><meta name="citation_author_institution" content="Graduate School of Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; moc.oohay@dhomzwnhs (S.M.); moc.liamg@ofnibaymar (R.M.); rk.ca.uhk@mikujnoey (Y.J.K.)" /><meta name="citation_author" content="Ramya Mathiyalagan" /><meta name="citation_author_institution" content="Graduate School of Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; moc.oohay@dhomzwnhs (S.M.); moc.liamg@ofnibaymar (R.M.); rk.ca.uhk@mikujnoey (Y.J.K.)" /><meta name="citation_author" content="Deok Chun Yang" /><meta name="citation_author_institution" content="Department of Biopharmaceutical Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; rk.ca.uhk@lumihsluazer (M.R.K.); rk.ca.uhk@69dehsromjain (M.N.M.); rk.ca.uhk@nirodaifas (S.I.); rk.ca.uhk@gnaycd (D.C.Y.)" /><meta name="citation_author_institution" content="Hanbangbio Inc., Yongin-si 17104, Republic of Korea" /><meta name="citation_author" content="Yeon Ju Kim" /><meta name="citation_author_institution" content="Graduate School of Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; moc.oohay@dhomzwnhs (S.M.); moc.liamg@ofnibaymar (R.M.); rk.ca.uhk@mikujnoey (Y.J.K.)" /><meta name="citation_author" content="Joon Hyun Song" /><meta name="citation_author_institution" content="Department of Veterinary International Medicine, College of Veterinary Medicine, Chungnam National University, Daejeon 34134, Republic of Korea" /><meta name="citation_author" content="Dong Uk Yang" /><meta name="citation_author_institution" content="Department of Biopharmaceutical Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; rk.ca.uhk@lumihsluazer (M.R.K.); rk.ca.uhk@69dehsromjain (M.N.M.); rk.ca.uhk@nirodaifas (S.I.); rk.ca.uhk@gnaycd (D.C.Y.)" /><meta name="citation_author_institution" content="AIBIOME, 6, Jeonmin-ro 30beon-gil, Yuseong-gu 34052, Republic of Korea" /><meta name="citation_publication_date" content="2023/11" /><meta name="citation_issue" content="11" /><meta name="citation_volume" content="13" /><meta name="citation_doi" content="10.3390/biom13111678" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC10669250/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC10669250/" /><meta name="citation_pmid" content="38002360" /><meta name="DC.Title" content="A Network Pharmacology and Molecular-Docking-Based Approach to Identify the Probable Targets of Short-Chain Fatty-Acid-Producing Microbial Metabolites against Kidney Cancer and Inflammation" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Multidisciplinary Digital Publishing Institute  (MDPI)" /><meta name="DC.Contributor" content="Md. Rezaul Karim" /><meta name="DC.Contributor" content="Md. Niaj Morshed" /><meta name="DC.Contributor" content="Safia Iqbal" /><meta name="DC.Contributor" content="Shahnawaz Mohammad" /><meta name="DC.Contributor" content="Ramya Mathiyalagan" /><meta name="DC.Contributor" content="Deok Chun Yang" /><meta name="DC.Contributor" content="Yeon Ju Kim" /><meta name="DC.Contributor" content="Joon Hyun Song" /><meta name="DC.Contributor" content="Dong Uk Yang" /><meta name="DC.Date" content="2023 Nov" /><meta name="DC.Identifier" content="10.3390/biom13111678" /><meta name="DC.Language" content="en" /><meta property="og:title" content="A Network Pharmacology and Molecular-Docking-Based Approach to Identify the Probable Targets of Short-Chain Fatty-Acid-Producing Microbial Metabolites against Kidney Cancer and Inflammation" /><meta property="og:type" content="article" /><meta property="og:description" content="(1) Background: A large and diverse microbial population exists in the human intestinal tract, which supports gut homeostasis and the health of the host. Short-chain fatty acid (SCFA)-secreting microbes also generate several metabolites with favorable ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10669250/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC10669250/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Biomolecules%22[journal]" class="navlink">Biomolecules</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10669250
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10669250/pdf/biomolecules-13-01678.pdf" class="int-view">PDF (3.4M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10669250/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10669250/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10669250" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10669250%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="VXU0SHIBVOQbNlodCGewOPYoX8t6lb5ydxyA4H9n69Te4Ju9RYawKZ1Rqpsz0KHW">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10669250%2F&amp;text=A%20Network%20Pharmacology%20and%20Molecular-Docking-Based%20Approach%20to%20Identify%20the%20Probable%20Targets%20of%20Short-Chain%20Fatty-Acid-Producing%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10669250%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10669250/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10669250/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38002360/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10669250/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38002360/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10669250/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10669250/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=%22Biomolecules%22[journal]" class="navlink">Biomolecules</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC10669250
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-biomol.png" alt="Logo of biomol" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.mdpi.com/journal/biomolecules" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Biomolecules.</span> 2023 Nov; 13(11): 1678. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2023 Nov 20. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.3390%2Fbiom13111678" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.3390/biom13111678</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC10669250</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38002360">38002360</a></div></div></div><h1 class="content-title">A Network Pharmacology and Molecular-Docking-Based Approach to Identify the Probable Targets of Short-Chain Fatty-Acid-Producing Microbial Metabolites against Kidney Cancer and Inflammation</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Karim%20MR%5BAuthor%5D" class="affpopup" co-rid="_co_idm140445107345968" co-class="co-affbox">Md. Rezaul Karim</a>, <span class="fm-role">Formal analysis</span>, <span class="fm-role">Writing &#x02013; original draft</span>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>,<sup>1,</sup><sup>2</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Morshed%20MN%5BAuthor%5D" class="affpopup" co-rid="_co_idm140445096186032" co-class="co-affbox">Md. Niaj Morshed</a>, <span class="fm-role">Formal analysis</span>, <span class="fm-role">Writing &#x02013; original draft</span>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Iqbal%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140445104110704" co-class="co-affbox">Safia Iqbal</a>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>,<sup>1,</sup><sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mohammad%20S%5BAuthor%5D" class="affpopup" co-rid="_co_idm140445105189040" co-class="co-affbox">Shahnawaz Mohammad</a>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mathiyalagan%20R%5BAuthor%5D" class="affpopup" co-rid="_co_idm140445105185840" co-class="co-affbox">Ramya Mathiyalagan</a>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yang%20DC%5BAuthor%5D" class="affpopup" co-rid="_co_idm140445145080912" co-class="co-affbox">Deok Chun Yang</a>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>,<sup>1,</sup><sup>5</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kim%20YJ%5BAuthor%5D" class="affpopup" co-rid="_co_idm140445088883552" co-class="co-affbox">Yeon Ju Kim</a>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Song%20JH%5BAuthor%5D" class="affpopup" co-rid="_co_idm140445088880352" co-class="co-affbox">Joon Hyun Song</a>, <span class="fm-role">Writing &#x02013; review &#x00026; editing</span>, <span class="fm-role">Supervision</span>,<sup>6,</sup><sup>*</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yang%20DU%5BAuthor%5D" class="affpopup" co-rid="_co_idm140445094993808" co-class="co-affbox">Dong Uk Yang</a><sup>1,</sup><sup>7,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140445107345968"><h3 class="no_margin">Md. Rezaul Karim</h3><p><sup>1</sup>Department of Biopharmaceutical Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; <a href="mailto:dev@null" data-email="rk.ca.uhk@lumihsluazer" class="oemail">rk.ca.uhk@lumihsluazer</a> (M.R.K.); <a href="mailto:dev@null" data-email="rk.ca.uhk@69dehsromjain" class="oemail">rk.ca.uhk@69dehsromjain</a> (M.N.M.); <a href="mailto:dev@null" data-email="rk.ca.uhk@nirodaifas" class="oemail">rk.ca.uhk@nirodaifas</a> (S.I.); <a href="mailto:dev@null" data-email="rk.ca.uhk@gnaycd" class="oemail">rk.ca.uhk@gnaycd</a> (D.C.Y.)</p><p><sup>2</sup>Department of Biotechnology and Genetic Engineering, Faculty of Biological Sciences, Islamic University, Kushtia 7003, Bangladesh</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Karim%20MR%5BAuthor%5D">Md. Rezaul Karim</a></div></div><div id="_co_idm140445096186032"><h3 class="no_margin">Md. Niaj Morshed</h3><p><sup>1</sup>Department of Biopharmaceutical Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; <a href="mailto:dev@null" data-email="rk.ca.uhk@lumihsluazer" class="oemail">rk.ca.uhk@lumihsluazer</a> (M.R.K.); <a href="mailto:dev@null" data-email="rk.ca.uhk@69dehsromjain" class="oemail">rk.ca.uhk@69dehsromjain</a> (M.N.M.); <a href="mailto:dev@null" data-email="rk.ca.uhk@nirodaifas" class="oemail">rk.ca.uhk@nirodaifas</a> (S.I.); <a href="mailto:dev@null" data-email="rk.ca.uhk@gnaycd" class="oemail">rk.ca.uhk@gnaycd</a> (D.C.Y.)</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Morshed%20MN%5BAuthor%5D">Md. Niaj Morshed</a></div></div><div id="_co_idm140445104110704"><h3 class="no_margin">Safia Iqbal</h3><p><sup>1</sup>Department of Biopharmaceutical Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; <a href="mailto:dev@null" data-email="rk.ca.uhk@lumihsluazer" class="oemail">rk.ca.uhk@lumihsluazer</a> (M.R.K.); <a href="mailto:dev@null" data-email="rk.ca.uhk@69dehsromjain" class="oemail">rk.ca.uhk@69dehsromjain</a> (M.N.M.); <a href="mailto:dev@null" data-email="rk.ca.uhk@nirodaifas" class="oemail">rk.ca.uhk@nirodaifas</a> (S.I.); <a href="mailto:dev@null" data-email="rk.ca.uhk@gnaycd" class="oemail">rk.ca.uhk@gnaycd</a> (D.C.Y.)</p><p><sup>3</sup>Department of Microbiology, Varendra Institute of Biosciences, Affiliated University of Rajshahi, Natore, Rajshahi 6400, Bangladesh</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Iqbal%20S%5BAuthor%5D">Safia Iqbal</a></div></div><div id="_co_idm140445105189040"><h3 class="no_margin">Shahnawaz Mohammad</h3><p><sup>4</sup>Graduate School of Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; <a href="mailto:dev@null" data-email="moc.oohay@dhomzwnhs" class="oemail">moc.oohay@dhomzwnhs</a> (S.M.); <a href="mailto:dev@null" data-email="moc.liamg@ofnibaymar" class="oemail">moc.liamg@ofnibaymar</a> (R.M.); <a href="mailto:dev@null" data-email="rk.ca.uhk@mikujnoey" class="oemail">rk.ca.uhk@mikujnoey</a> (Y.J.K.)</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mohammad%20S%5BAuthor%5D">Shahnawaz Mohammad</a></div></div><div id="_co_idm140445105185840"><h3 class="no_margin">Ramya Mathiyalagan</h3><p><sup>4</sup>Graduate School of Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; <a href="mailto:dev@null" data-email="moc.oohay@dhomzwnhs" class="oemail">moc.oohay@dhomzwnhs</a> (S.M.); <a href="mailto:dev@null" data-email="moc.liamg@ofnibaymar" class="oemail">moc.liamg@ofnibaymar</a> (R.M.); <a href="mailto:dev@null" data-email="rk.ca.uhk@mikujnoey" class="oemail">rk.ca.uhk@mikujnoey</a> (Y.J.K.)</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mathiyalagan%20R%5BAuthor%5D">Ramya Mathiyalagan</a></div></div><div id="_co_idm140445145080912"><h3 class="no_margin">Deok Chun Yang</h3><p><sup>1</sup>Department of Biopharmaceutical Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; <a href="mailto:dev@null" data-email="rk.ca.uhk@lumihsluazer" class="oemail">rk.ca.uhk@lumihsluazer</a> (M.R.K.); <a href="mailto:dev@null" data-email="rk.ca.uhk@69dehsromjain" class="oemail">rk.ca.uhk@69dehsromjain</a> (M.N.M.); <a href="mailto:dev@null" data-email="rk.ca.uhk@nirodaifas" class="oemail">rk.ca.uhk@nirodaifas</a> (S.I.); <a href="mailto:dev@null" data-email="rk.ca.uhk@gnaycd" class="oemail">rk.ca.uhk@gnaycd</a> (D.C.Y.)</p><p><sup>5</sup>Hanbangbio Inc., Yongin-si 17104, Republic of Korea</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yang%20DC%5BAuthor%5D">Deok Chun Yang</a></div></div><div id="_co_idm140445088883552"><h3 class="no_margin">Yeon Ju Kim</h3><p><sup>4</sup>Graduate School of Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; <a href="mailto:dev@null" data-email="moc.oohay@dhomzwnhs" class="oemail">moc.oohay@dhomzwnhs</a> (S.M.); <a href="mailto:dev@null" data-email="moc.liamg@ofnibaymar" class="oemail">moc.liamg@ofnibaymar</a> (R.M.); <a href="mailto:dev@null" data-email="rk.ca.uhk@mikujnoey" class="oemail">rk.ca.uhk@mikujnoey</a> (Y.J.K.)</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Kim%20YJ%5BAuthor%5D">Yeon Ju Kim</a></div></div><div id="_co_idm140445088880352"><h3 class="no_margin">Joon Hyun Song</h3><p><sup>6</sup>Department of Veterinary International Medicine, College of Veterinary Medicine, Chungnam National University, Daejeon 34134, Republic of Korea</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Song%20JH%5BAuthor%5D">Joon Hyun Song</a></div></div><div id="_co_idm140445094993808"><h3 class="no_margin">Dong Uk Yang</h3><p><sup>1</sup>Department of Biopharmaceutical Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; <a href="mailto:dev@null" data-email="rk.ca.uhk@lumihsluazer" class="oemail">rk.ca.uhk@lumihsluazer</a> (M.R.K.); <a href="mailto:dev@null" data-email="rk.ca.uhk@69dehsromjain" class="oemail">rk.ca.uhk@69dehsromjain</a> (M.N.M.); <a href="mailto:dev@null" data-email="rk.ca.uhk@nirodaifas" class="oemail">rk.ca.uhk@nirodaifas</a> (S.I.); <a href="mailto:dev@null" data-email="rk.ca.uhk@gnaycd" class="oemail">rk.ca.uhk@gnaycd</a> (D.C.Y.)</p><p><sup>7</sup>AIBIOME, 6, Jeonmin-ro 30beon-gil, Yuseong-gu 34052, Republic of Korea</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yang%20DU%5BAuthor%5D">Dong Uk Yang</a></div></div></div></div><div class="contrib-group half_rhythm fm-editor">Alessandro Paiardini, <span class="fm-role">Academic Editor</span></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140445111394496_ai idm140445107346096_ai idm140445102599680_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140445111394496_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm140445111394496_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140445111394496_ai" style="display:none"><div class="fm-affl" id="af1-biomolecules-13-01678"><sup>1</sup>Department of Biopharmaceutical Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; <a href="mailto:dev@null" data-email="rk.ca.uhk@lumihsluazer" class="oemail">rk.ca.uhk@lumihsluazer</a> (M.R.K.); <a href="mailto:dev@null" data-email="rk.ca.uhk@69dehsromjain" class="oemail">rk.ca.uhk@69dehsromjain</a> (M.N.M.); <a href="mailto:dev@null" data-email="rk.ca.uhk@nirodaifas" class="oemail">rk.ca.uhk@nirodaifas</a> (S.I.); <a href="mailto:dev@null" data-email="rk.ca.uhk@gnaycd" class="oemail">rk.ca.uhk@gnaycd</a> (D.C.Y.)</div><div class="fm-affl" id="af2-biomolecules-13-01678"><sup>2</sup>Department of Biotechnology and Genetic Engineering, Faculty of Biological Sciences, Islamic University, Kushtia 7003, Bangladesh</div><div class="fm-affl" id="af3-biomolecules-13-01678"><sup>3</sup>Department of Microbiology, Varendra Institute of Biosciences, Affiliated University of Rajshahi, Natore, Rajshahi 6400, Bangladesh</div><div class="fm-affl" id="af4-biomolecules-13-01678"><sup>4</sup>Graduate School of Biotechnology, College of Life Science, Kyung Hee University, Yongin-si 17104, Republic of Korea; <a href="mailto:dev@null" data-email="moc.oohay@dhomzwnhs" class="oemail">moc.oohay@dhomzwnhs</a> (S.M.); <a href="mailto:dev@null" data-email="moc.liamg@ofnibaymar" class="oemail">moc.liamg@ofnibaymar</a> (R.M.); <a href="mailto:dev@null" data-email="rk.ca.uhk@mikujnoey" class="oemail">rk.ca.uhk@mikujnoey</a> (Y.J.K.)</div><div class="fm-affl" id="af5-biomolecules-13-01678"><sup>5</sup>Hanbangbio Inc., Yongin-si 17104, Republic of Korea</div><div class="fm-affl" id="af6-biomolecules-13-01678"><sup>6</sup>Department of Veterinary International Medicine, College of Veterinary Medicine, Chungnam National University, Daejeon 34134, Republic of Korea</div><div class="fm-affl" id="af7-biomolecules-13-01678"><sup>7</sup>AIBIOME, 6, Jeonmin-ro 30beon-gil, Yuseong-gu 34052, Republic of Korea</div><div id="c1-biomolecules-13-01678"><sup>*</sup>Correspondence: <a href="mailto:dev@null" data-email="rk.ca.unc@gnos.hj" class="oemail">rk.ca.unc@gnos.hj</a> (J.H.S.); <a href="mailto:dev@null" data-email="moc.liamg@28gnaykugnod" class="oemail">moc.liamg@28gnaykugnod</a> (D.U.Y.)</div></div><div class="fm-article-notes hide half_rhythm" id="idm140445111394496_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Sep 23; Revised 2023 Oct 26; Accepted 2023 Nov 17.</div></div><div class="permissions half_rhythm hide" id="idm140445111394496_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2023 by the authors.</div><div class="license half_rhythm">Licensee MDPI, Basel, Switzerland. This article is an open access article distributed under the terms and conditions of the Creative Commons Attribution (CC BY) license (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">https://creativecommons.org/licenses/by/4.0/</a>).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="1" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div><a href="/pmc/articles/PMC10669250/bin/biomolecules-13-01678-s001.zip" data-ga-action="click_feat_suppl">biomolecules-13-01678-s001.zip</a><span style="color:gray"> (4.9M)</span></div><div class="small guid">GUID: 73B475D7-A611-4C59-892E-9DF29F141E05</div></div></dd></dl><dl data-length="130" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p>All data generated or analyzed during this study are included in this published article (and its <a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary Information files</a>).</p></dd></dl></div><div id="abstract-a.r.b.v" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abstract-a.r.b.vtitle">Abstract</h2><!--article-meta--><div><p class="p p-first-last">(1) Background: A large and diverse microbial population exists in the human intestinal tract, which supports gut homeostasis and the health of the host. Short-chain fatty acid (SCFA)-secreting microbes also generate several metabolites with favorable regulatory effects on various malignancies and immunological inflammations. The involvement of intestinal SCFAs in kidney diseases, such as various kidney malignancies and inflammations, has emerged as a fascinating area of study in recent years. However, the mechanisms of SCFAs and other metabolites produced by SCFA-producing bacteria against kidney cancer and inflammation have not yet been investigated. (2) Methods: We considered 177 different SCFA-producing microbial species and 114 metabolites from the gutMgene database. Further, we used different online-based database platforms to predict 1890 gene targets associated with metabolites. Moreover, DisGeNET, OMIM, and Genecard databases were used to consider 13,104 disease-related gene targets. We used a Venn diagram and various protein&#x02212;protein interactions (PPIs), KEGG pathways, and GO analyses for the functional analysis of gene targets. Moreover, the subnetwork of protein&#x02212;protein interactions (through string and cytoscape platforms) was used to select the top 20% of gene targets through degree centrality, betweenness centrality, and closeness centrality. To screen the possible candidate compounds, we performed an analysis of the ADMET (absorption, distribution, metabolism, excretion, and toxicity) properties of metabolites and then found the best binding affinity using molecular docking simulation. (3) Results: Finally, we found the key gene targets that interact with suitable compounds and function against kidney cancer and inflammation, such as MTOR (with glycocholic acid), PIK3CA (with 11-methoxycurvularin, glycocholic acid, and isoquercitrin), IL6 (with isoquercitrin), PTGS2 (with isoquercitrin), and IGF1R (with 2-amino-1-methyl-6-phenylimidazo[4,5-b] pyridine, isoquercitrin), showed a lower binding affinity. (4) Conclusions: This study provides evidence to support the positive effects of SCFA-producing microbial metabolites that function against kidney cancer and inflammation and makes integrative research proposals that may be used to guide future studies.</p></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">network pharmacology, gut microbial metabolites, SCFA-producing microbes, kidney cancer, kidney inflammation, molecular docking</span></div></div><div id="sec1-biomolecules-13-01678" class="tsec sec"><h2 class="head no_bottom_margin" id="sec1-biomolecules-13-01678title">1. Introduction</h2><p class="p p-first">As kidney cancer causes more than 131,000 fatalities and 342,000 incident cases worldwide each year, it is one of the most serious malignancies [<a href="#B1-biomolecules-13-01678" rid="B1-biomolecules-13-01678" class=" bibr popnode">1</a>]. According to 2020 GLOBOCAN data, globally, 2.2% of total incidences and 1.8% of total cancer deaths occur in kidney cancer annually [<a href="#B2-biomolecules-13-01678" rid="B2-biomolecules-13-01678" class=" bibr popnode">2</a>,<a href="#B3-biomolecules-13-01678" rid="B3-biomolecules-13-01678" class=" bibr popnode">3</a>]. Cancer research has recently shifted its attention to the link between cancer and inflammation [<a href="#B4-biomolecules-13-01678" rid="B4-biomolecules-13-01678" class=" bibr popnode">4</a>]. Numerous studies have demonstrated that the development of cancer, including breast, pancreatic, colorectal, colon, rectal, prostate, bladder, lung, and ovarian cancers, is strongly influenced by inflammatory chemicals and pathways [<a href="#B5-biomolecules-13-01678" rid="B5-biomolecules-13-01678" class=" bibr popnode">5</a>,<a href="#B6-biomolecules-13-01678" rid="B6-biomolecules-13-01678" class=" bibr popnode">6</a>]. Renal cell carcinoma (RCC) and inflammation are both closely related, and both contribute to the growth of RCC tumors, which are thought to be immunogenic [<a href="#B7-biomolecules-13-01678" rid="B7-biomolecules-13-01678" class=" bibr popnode">7</a>,<a href="#B8-biomolecules-13-01678" rid="B8-biomolecules-13-01678" class=" bibr popnode">8</a>]. Surgery is still the most effective treatment for both localized and locally progressed RCC, because 25&#x02013;30% of affected patients have metastatic disease and, therefore, a poor prognosis [<a href="#B9-biomolecules-13-01678" rid="B9-biomolecules-13-01678" class=" bibr popnode">9</a>]. Therefore, the current study focuses on inflammation and kidney cancer. We focused on network pharmacology analysis to find the safer and more effective therapeutic gene targets needed to treat inflammation and kidney cancer.</p><p>In recent years, there has been growing interest in the gut&#x02212;kidney interaction as it relates to chronic kidney disease (CKD), including kidney cancer and inflammation [<a href="#B10-biomolecules-13-01678" rid="B10-biomolecules-13-01678" class=" bibr popnode">10</a>]. Microbial metabolites can function as signaling substances when circulated throughout the body [<a href="#B11-biomolecules-13-01678" rid="B11-biomolecules-13-01678" class=" bibr popnode">11</a>]. Currently, short-chain fatty acids (SCFAs) and their receptors, as well as changes to the gut microbiome, are some of the suggested mechanisms linking dysbiotic gut microbiota to CKD and its consequences [<a href="#B11-biomolecules-13-01678" rid="B11-biomolecules-13-01678" class=" bibr popnode">11</a>,<a href="#B12-biomolecules-13-01678" rid="B12-biomolecules-13-01678" class=" bibr popnode">12</a>,<a href="#B13-biomolecules-13-01678" rid="B13-biomolecules-13-01678" class=" bibr popnode">13</a>]. As a class of metabolites, SCFAs exert advantageous regulatory effects on blood pressure, immunological inflammation, hormone production, and cancer [<a href="#B14-biomolecules-13-01678" rid="B14-biomolecules-13-01678" class=" bibr popnode">14</a>]. A lack of gut-microbiota-produced SCFAs has also been linked to disorders such as inflammatory bowel disease, obesity, type 1 and type 2 diabetes, autism, major depression, colon cancer, and renal diseases, which are the topic of this discussion [<a href="#B15-biomolecules-13-01678" rid="B15-biomolecules-13-01678" class=" bibr popnode">15</a>,<a href="#B16-biomolecules-13-01678" rid="B16-biomolecules-13-01678" class=" bibr popnode">16</a>,<a href="#B17-biomolecules-13-01678" rid="B17-biomolecules-13-01678" class=" bibr popnode">17</a>,<a href="#B18-biomolecules-13-01678" rid="B18-biomolecules-13-01678" class=" bibr popnode">18</a>]. However, the relationship between bacteria that produce SCFAs and the various metabolites they secrete, including SCFAs, which help treat kidney inflammation and kidney cancer, is still not entirely understood. Therefore, the current study focuses on microbes that produce SCFAs and their metabolites to find potential targets for the treatment of inflammation and kidney cancer using network pharmacology.</p><p class="p p-last">To combat the co-morbidity of these diseases, the goal of this research was to discover the most significant gut SCFAs-producing microbial compounds that can be used to control the expression of the hierarchical targets for treating kidney cancer and inflammation. We also discussed the important SCFAs-producing probiotics that the molecular docking test (MDT) determined to be the most stably bound metabolites on a significant target. As a result, our research may identify approaches to reduce kidney cancer and inflammation by using the effect of complex microbiome networks.</p></div><div id="sec2-biomolecules-13-01678" class="tsec sec"><h2 class="head no_bottom_margin" id="sec2-biomolecules-13-01678title">2. Materials and Methods</h2><div id="sec2dot1-biomolecules-13-01678" class="sec sec-first"><h3 id="sec2dot1-biomolecules-13-01678title">2.1. Target Gene Prediction of SCFAs Microbial Metabolites and Diseases (Kidney Cancer and Inflammation)</h3><p class="p p-first-last">The targets and metabolites of the gut SCFA-producing microbiota were obtained using the gutMGene v1.0 database (<a href="http://bio-annotation.cn/gutmgene/">http://bio-annotation.cn/gutmgene/</a>) (accessed on 10 July 2023) [<a href="#B19-biomolecules-13-01678" rid="B19-biomolecules-13-01678" class=" bibr popnode">19</a>]. To retrieve kidney cancer and inflammation-related targets, we used the DisgeNET v7.0(<a href="https://www.disgenet.org/">https://www.disgenet.org/</a>) (accessed on 12 July 2023) [<a href="#B20-biomolecules-13-01678" rid="B20-biomolecules-13-01678" class=" bibr popnode">20</a>], OMIM (<a href="https://www.omim.org/">https://www.omim.org/</a>) [<a href="#B21-biomolecules-13-01678" rid="B21-biomolecules-13-01678" class=" bibr popnode">21</a>] (accessed on 13 July 2023), and Genecard databases v5.8 (<a href="https://www.genecards.org/">https://www.genecards.org/</a>) (accessed on 14 July 2023) [<a href="#B22-biomolecules-13-01678" rid="B22-biomolecules-13-01678" class=" bibr popnode">22</a>] databases. The major targets among the metabolites and chosen disease-related targets were identified using Venny 2.1.0 (<a href="https://bioinfogp.cnb.csic.es/tools/venny/">https://bioinfogp.cnb.csic.es/tools/venny/</a> accessed on 15 July 2023), an online mapping tool. We used the overlapped genes for further analysis using GeneMANIA (<a href="https://genemania.org/">https://genemania.org/</a> V3.6.0 version accessed on 16 July 2023) [<a href="#B23-biomolecules-13-01678" rid="B23-biomolecules-13-01678" class=" bibr popnode">23</a>] to find the target genes that are co-expressed and that share the same protein and pathway. We then used those genes for further analysis.</p></div><div id="sec2dot2-biomolecules-13-01678" class="sec"><h3 id="sec2dot2-biomolecules-13-01678title">2.2. Target Gene Location in Chromosomes and Tissues</h3><p class="p p-first-last">To evaluate the pathophysiology of some genes and identify the potential therapeutic targets, the chromosomal location of the target genes needs to be determined. As a result, the location of the genes on the chromosomes was determined using the ShinyGO web tool v0.75 (<a href="http://bioinformatics.sdstate.edu/go/">http://bioinformatics.sdstate.edu/go/</a> accessed on 18 July 2023) [<a href="#B24-biomolecules-13-01678" rid="B24-biomolecules-13-01678" class=" bibr popnode">24</a>]. Additionally, the distribution of shared genes varied in other organs. So, using the Pa-GenBase dataset from the Metascape web server v3.5.2023.05.01 (<a href="https://metascape.org/gp/index.html#/main/">https://metascape.org/gp/index.html#/main/</a> accessed on 19 July 2023) [<a href="#B25-biomolecules-13-01678" rid="B25-biomolecules-13-01678" class=" bibr popnode">25</a>], we were able to determine the distribution of genes that affect different tissues and cell types.</p></div><div id="sec2dot3-biomolecules-13-01678" class="sec"><h3 id="sec2dot3-biomolecules-13-01678title">2.3. Analysis of Target Gene Pathways Using Gene Ontology (GO) and Kyoto Encyclopedia of Genes and Genomes (KEGG) Databases</h3><p class="p p-first-last">GO analysis, which was created to define the activities of the targets, included analyses of cellular components (CC), biological function, and molecular function. The KEGG pathway analysis shed light on the putative signaling pathways linked to the final targets against kidney cancer and renal inflammation. The gene ratio of the differentially expressed genes to the total number of targets in a signaling pathway serves as the foundation for enrichment plots, which are based on a <em>p</em>-value and adj. <em>p</em>-value [<a href="#B26-biomolecules-13-01678" rid="B26-biomolecules-13-01678" class=" bibr popnode">26</a>]. Gene ontology and pathway studies were performed using the Protein Analysis Through Evolutionary Relationships (PANTHER) program v18.0 (<a href="http://pantherdb.org/">http://pantherdb.org/</a> accessed on 20 July 2023) to determine how these frequently connected genes collectively influence the signaling pathways [<a href="#B27-biomolecules-13-01678" rid="B27-biomolecules-13-01678" class=" bibr popnode">27</a>].</p></div><div id="sec2dot4-biomolecules-13-01678" class="sec"><h3 id="sec2dot4-biomolecules-13-01678title">2.4. Protein&#x02212;Protein Interaction (PPI) Network Analysis of Targeted Gene</h3><p class="p p-first-last">Protein interactions are examined throughout the early stages of drug development because they provide an immense amount of information about the functions of proteins [<a href="#B28-biomolecules-13-01678" rid="B28-biomolecules-13-01678" class=" bibr popnode">28</a>]. The total number of intricate biological processes is estimated through a thorough PPI network investigation [<a href="#B29-biomolecules-13-01678" rid="B29-biomolecules-13-01678" class=" bibr popnode">29</a>]. By using the STRING dataset with NetworkAnalyst v3.0 (<a href="https://www.networkanalyst.ca/">https://www.networkanalyst.ca/</a> accessed on 22 July 2023), the PPI of common genes has been determined to examine the molecular mechanisms linked to major signaling pathways and cellular activities [<a href="#B30-biomolecules-13-01678" rid="B30-biomolecules-13-01678" class=" bibr popnode">30</a>]. The PPI network was created using the fundamental PPI configuration, which used H. sapiens as the organism, STRING as the database, and a confidence score cutoff of 900. After assessing the accuracy, we concluded that the common nodes were the most likely hubs. The creation of sub-PPI networks using CytoHubba v0.1 in Cytoscape version 3.10.1 to select the suitable gene targets using the highest degree centrality (DC) values, highest betweenness centrality (BC) values, and highest closeness centrality (CC) values in the top 20% of the PPI networks [<a href="#B31-biomolecules-13-01678" rid="B31-biomolecules-13-01678" class=" bibr popnode">31</a>].</p></div><div id="sec2dot5-biomolecules-13-01678" class="sec"><h3 id="sec2dot5-biomolecules-13-01678title">2.5. Analysis of the Physiochemical and ADMET Characteristics of Microbial Compounds</h3><p class="p p-first-last">The hit compounds were subjected to in silico physiochemical analysis. The SwissADME (<a href="http://www.swissadme.ch/index.php">http://www.swissadme.ch/index.php</a> accessed on 23&#x02013;28 July 2023) [<a href="#B32-biomolecules-13-01678" rid="B32-biomolecules-13-01678" class=" bibr popnode">32</a>], ADMETlab2.0 (<a href="https://admetmesh.scbdd.com/">https://admetmesh.scbdd.com/</a> accessed on 23&#x02013;28 July 2023) [<a href="#B33-biomolecules-13-01678" rid="B33-biomolecules-13-01678" class=" bibr popnode">33</a>], and pkCSM (<a href="https://biosig.lab.uq.edu.au/pkcsm/">https://biosig.lab.uq.edu.au/pkcsm/</a> accessed on 23&#x02013;28 July 2023) web servers were used for the drug-likeness analysis [<a href="#B34-biomolecules-13-01678" rid="B34-biomolecules-13-01678" class=" bibr popnode">34</a>], and the Protox-II web server (<a href="https://tox-new.charite.de/protox_II/">https://tox-new.charite.de/protox_II/</a> accessed on 23&#x02013;28 July 2023) was used for the analysis of the predicted ADMET properties [<a href="#B35-biomolecules-13-01678" rid="B35-biomolecules-13-01678" class=" bibr popnode">35</a>]. To predict ADMET properties (absorption, distribution, metabolism, excretion, and toxicity), the PubChem Database&#x02019;s canonical SMILES of the metabolites were consulted. Finally, we rejected the protein targets connected to Lipinski&#x02019;s rule of five (LO5) metabolites that broke more than two of its rules [<a href="#B36-biomolecules-13-01678" rid="B36-biomolecules-13-01678" class=" bibr popnode">36</a>]. Investigations were conducted using the remaining targets and metabolites.</p></div><div id="sec2dot6-biomolecules-13-01678" class="sec"><h3 id="sec2dot6-biomolecules-13-01678title">2.6. Validation of the Expression of the Hub Targets</h3><p class="p p-first-last">Information on the expression and distribution of various human proteins in diverse tissues is made available via the Human Protein Atlas database (HPA) v23.0 (<a href="https://www.proteinatlas.org/">https://www.proteinatlas.org/</a> accessed on 2 August 2023) [<a href="#B37-biomolecules-13-01678" rid="B37-biomolecules-13-01678" class=" bibr popnode">37</a>]. Using data from the Human Protein Atlas, we investigated the expression levels of hub targets in the kidneys and urinary bladder.</p></div><div id="sec2dot7-biomolecules-13-01678" class="sec"><h3 id="sec2dot7-biomolecules-13-01678title">2.7. Protein and Ligand Preparation</h3><p class="p p-first-last">From the RCSB protein data bank (<a href="http://www.rcsb.org">www.rcsb.org</a>, accessed on 1 August 2023), the crystal structures of MTOR (PDB:2FAP), PIK3CA (PDB:5DXT), IL6 (PDB:4ZST), PTGS2 (PDB:5IKQ), and IGF1R (PDB:5FXQ) were retrieved [<a href="#B38-biomolecules-13-01678" rid="B38-biomolecules-13-01678" class=" bibr popnode">38</a>]. All of the retrieved protein structures belonged to the human database. For the selection of protein structures, we considered mainly the X-ray diffraction experimental method, and also the refinement resolution range between approximately 1.5 and 2.5. The proteins were prepared by removing the cofactors, water molecules, and metal ions from the complex structure. After the non-polar hydrogen atoms were combined and polar hydrogen atoms were added, Gasteiger charges for the protein were computed [<a href="#B39-biomolecules-13-01678" rid="B39-biomolecules-13-01678" class=" bibr popnode">39</a>]. The aromatic carbons were located, the non-polar hydrogens were combined, and the molecule &#x02018;torsion tree&#x02019; was set up using AutoDock v4 Tools. For additional screening, the PDBQT file format of these results was used. Additionally, the PubChem Database (<a href="https://pubchem.ncbi.nlm.nih.gov/">https://pubchem.ncbi.nlm.nih.gov/</a> accessed on 14&#x02013;20 August 2023) was utilized to acquire the 3D structures of the active components [<a href="#B40-biomolecules-13-01678" rid="B40-biomolecules-13-01678" class=" bibr popnode">40</a>]. Finally, SDF files were used as the download format.</p></div><div id="sec2dot8-biomolecules-13-01678" class="sec"><h3 id="sec2dot8-biomolecules-13-01678title">2.8. Binding Site Identification and Grid Box Generation </h3><p class="p p-first-last">Binding sites were found by comparing pockets from known protein&#x02212;ligand interactions. PDB and CASTp (<a href="http://sts.bioe.uic.edu/castp/">http://sts.bioe.uic.edu/castp/</a> accessed on 14&#x02013;20 August 2023) [<a href="#B41-biomolecules-13-01678" rid="B41-biomolecules-13-01678" class=" bibr popnode">41</a>] were used to extract the known and unknown active sites of the protein structures, respectively, and BIOVIA Discovery Studio Visualizer v19.1 (BIOVIA) was used to examine the binding site of the proteins [<a href="#B42-biomolecules-13-01678" rid="B42-biomolecules-13-01678" class=" bibr popnode">42</a>]. The receptor grid was built using molecular docking and the binding sites were obtained from the complex structure using the PyRx&#x02014;Python Prescription 0.8 virtual screening tool [<a href="#B43-biomolecules-13-01678" rid="B43-biomolecules-13-01678" class=" bibr popnode">43</a>].</p></div><div id="sec2dot9-biomolecules-13-01678" class="sec sec-last"><h3 id="sec2dot9-biomolecules-13-01678title">2.9. Molecular Docking Simulation</h3><p class="p p-first-last">A molecular docking simulation was performed using the PyRx v0.8 tool to identify the candidates that were most compatible with the target proteins [<a href="#B30-biomolecules-13-01678" rid="B30-biomolecules-13-01678" class=" bibr popnode">30</a>]. AutoDock Vina and AutoDock v4 are included in PyRx, a free computational screening application that can assess a big dataset against a specific biologically targeted macromolecule. The default setting in PyRx v0.8 is the AutoDock Vina Wizard v4 [<a href="#B44-biomolecules-13-01678" rid="B44-biomolecules-13-01678" class=" bibr popnode">44</a>], which simulates molecular docking. In comparison with other compounds, the top compounds had the highest binding affinity (kcal/mol) to the target protein. Finally, using the default arrangement, receptor grids were created.</p></div></div><div id="sec3-biomolecules-13-01678" class="tsec sec"><h2 class="head no_bottom_margin" id="sec3-biomolecules-13-01678title">3. Results</h2><div id="sec3dot1-biomolecules-13-01678" class="sec sec-first"><h3 id="sec3dot1-biomolecules-13-01678title">3.1. Retrieve Metabolites and Potential Target Proteins Linked to Kidney Cancer and Kidney Inflammation</h3><p class="p p-first-last">We obtained 177 SCFA-producing microbes and 114 metabolites from the gutMgene microbiome database. A total of 1890 metabolite-related gene targets were predicted from the gutMgene, PubChem, and Human Metabolome (HMDB) databases, using the similarity ensemble approach (SEA), Swiss target prediction (STP), and Chemical Entities of Biological Interest (ChEBI) (<a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary S1</a>). A total of 13,104 target genes for kidney cancer and kidney inflammation-related diseases were retrieved from DisgeNET, genecard, and OMIM (<a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary S2</a>). The revealed targets and compounds were regarded as important factors for analyzing the treatment outcomes of the gut microbiota. Subsequently, 1436 overlapping targets related to kidney cancer and inflammation were identified through a Venn diagram (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f001/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f001" rid-ob="ob-biomolecules-13-01678-f001" co-legend-rid="lgnd_biomolecules-13-01678-f001"><span>Figure 1</span></a>A). Therefore, the 1436 selected targets were used for further analysis in GeneMANIA to find co-expressed genes that shared the same protein domain and related gene regulation pathways. Finally, we found 38 gene targets that were used for further analysis (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f001/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f001" rid-ob="ob-biomolecules-13-01678-f001" co-legend-rid="lgnd_biomolecules-13-01678-f001"><span>Figure 1</span></a>B, <a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary S3</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="biomolecules-13-01678-f001" co-legend-rid="lgnd_biomolecules-13-01678-f001"><a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f001/" target="figure" rid-figpopup="biomolecules-13-01678-f001" rid-ob="ob-biomolecules-13-01678-f001"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140445109513536"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10669250_biomolecules-13-01678-g001.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is biomolecules-13-01678-g001.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10669250/bin/biomolecules-13-01678-g001.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140445109513536"><a target="object" rel="noopener" href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f001/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_biomolecules-13-01678-f001"><div><a class="figpopup" href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f001/" target="figure" rid-figpopup="biomolecules-13-01678-f001" rid-ob="ob-biomolecules-13-01678-f001">Figure 1</a></div><!--caption a7--><div class="caption"><p>(<strong>A</strong>) Venn diagram analysis between metabolites and disease-related targets. (<strong>B</strong>) GeneMANIA analysis to predict co-expressed, same protein domain, and pathway targets.</p></div></div></div></div><div id="sec3dot2-biomolecules-13-01678" class="sec"><h3 id="sec3dot2-biomolecules-13-01678title">3.2. Distribution and Location of Genes</h3><p class="p p-first-last">Identifying the exact cellular and molecular locations of the genes is required to identify a protein at the transcription level. We used 38 final targets in the Metascape online server for this analysis. Most of the target genes of kidney cancer and kidney inflammation were expressed in the placenta (expressed value around 6), while others were also expressed in smooth muscle (above 3.5) and in the lungs (above 3) (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f002/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f002" rid-ob="ob-biomolecules-13-01678-f002" co-legend-rid="lgnd_biomolecules-13-01678-f002"><span>Figure 2</span></a>C). Additionally, in the analysis of kidney cancer and inflammation genes, the majority of the common target genes (six) were present in chromosome 3, and four genes were present in chromosomes 1, 7, and 11. Except for the 4, 5, 6, 8, 14, 16, 18, 21, 22, X, and Y chromosomes, the rest were evenly distributed throughout the genome (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f001/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f001" rid-ob="ob-biomolecules-13-01678-f001" co-legend-rid="lgnd_biomolecules-13-01678-f001"><span>Figure 1</span></a>A). As a cellular level, most of the genes were expressed (above 8) in lake adult kidney C8 descending thin limb, and were also highly expressed (around six) in the lake adult kidney C9 thin ascending limb, as well as the travaglini lung basophil mast 1 cell (near to six). In the other kidney and lung cells, the genes were thoroughly distributed (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f002/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f002" rid-ob="ob-biomolecules-13-01678-f002" co-legend-rid="lgnd_biomolecules-13-01678-f002"><span>Figure 2</span></a>B). Finally, we also analyzed and observed that most of the genes were expressed in different renal cell carcinomas and kidney carcinomas (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f002/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f002" rid-ob="ob-biomolecules-13-01678-f002" co-legend-rid="lgnd_biomolecules-13-01678-f002"><span>Figure 2</span></a>D).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="biomolecules-13-01678-f002" co-legend-rid="lgnd_biomolecules-13-01678-f002"><a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f002/" target="figure" rid-figpopup="biomolecules-13-01678-f002" rid-ob="ob-biomolecules-13-01678-f002"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140445105752448"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10669250_biomolecules-13-01678-g002.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is biomolecules-13-01678-g002.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10669250/bin/biomolecules-13-01678-g002.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140445105752448"><a target="object" rel="noopener" href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f002/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_biomolecules-13-01678-f002"><div><a class="figpopup" href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f002/" target="figure" rid-figpopup="biomolecules-13-01678-f002" rid-ob="ob-biomolecules-13-01678-f002">Figure 2</a></div><!--caption a7--><div class="caption"><p>Distribution of targeted genea. (<strong>A</strong>) Chromosomal distribution. (<strong>B</strong>) Cellular distribution. (<strong>C</strong>) Tissue-specific distribution. (<strong>D</strong>) Distribution in different cancer types.</p></div></div></div></div><div id="sec3dot3-biomolecules-13-01678" class="sec"><h3 id="sec3dot3-biomolecules-13-01678title">3.3. Gene Ontology and Pathway Analysis of Gene Targets</h3><p class="p p-first-last">To further investigate the potential mechanism of the genes for kidney cancer and inflammation, GO and KEGG enrichment analyses were conducted based on 38 targets. We found these genes enriched in 119 biological process (BP) terms, 46 molecular functions (MF) terms, and 33 cellular component (CC) terms for kidney cancer and inflammation. The top 12 entries from BP, 7 entries from MF, and 2 entries from CC terms are shown in <a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f003" rid-ob="ob-biomolecules-13-01678-f003" co-legend-rid="lgnd_biomolecules-13-01678-f003"><span>Figure 3</span></a> (<a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary S5</a>). In the case of kidney cancer and inflammation, BP analysis showed that associated targets were primarily centered on biological regulation and cellular and metabolic processes (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f003" rid-ob="ob-biomolecules-13-01678-f003" co-legend-rid="lgnd_biomolecules-13-01678-f003"><span>Figure 3</span></a>A). According to the MF analysis, potential kidney cancer and inflammation targets were determined mainly based on the binding affinity, catalytic activity, and molecular transducer activity. CC analysis indicated that related targets were primarily mainly centered on the cellular anatomical entity and protein-containing complex (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f003" rid-ob="ob-biomolecules-13-01678-f003" co-legend-rid="lgnd_biomolecules-13-01678-f003"><span>Figure 3</span></a>C). KEGG pathway enrichment analysis was also performed to investigate the pathways associated with the key targets. The results showed 65 significantly enriched signaling pathways for kidney cancer and inflammation. The top 32 significantly enriched pathways displayed in <a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f003" rid-ob="ob-biomolecules-13-01678-f003" co-legend-rid="lgnd_biomolecules-13-01678-f003"><span>Figure 3</span></a>D were closely correlated to kidney cancer and inflammation (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f003" rid-ob="ob-biomolecules-13-01678-f003" co-legend-rid="lgnd_biomolecules-13-01678-f003"><span>Figure 3</span></a>D). So, the pathway enrichment analysis indicated that the gonadotropin-releasing hormone receptor pathway, inflammation-mediated chemokine and cytokine signaling pathway, angiogenesis and apoptosis signaling pathway, endothelin signaling pathway, interleukin signaling pathway, and nicotinic acetylcholine receptor signaling pathway were most closely related to kidney cancer and inflammation (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f003/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f003" rid-ob="ob-biomolecules-13-01678-f003" co-legend-rid="lgnd_biomolecules-13-01678-f003"><span>Figure 3</span></a>D) (<a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary S4</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="biomolecules-13-01678-f003" co-legend-rid="lgnd_biomolecules-13-01678-f003"><a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f003/" target="figure" rid-figpopup="biomolecules-13-01678-f003" rid-ob="ob-biomolecules-13-01678-f003"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140445105598528"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10669250_biomolecules-13-01678-g003.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is biomolecules-13-01678-g003.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10669250/bin/biomolecules-13-01678-g003.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140445105598528"><a target="object" rel="noopener" href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f003/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_biomolecules-13-01678-f003"><div><a class="figpopup" href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f003/" target="figure" rid-figpopup="biomolecules-13-01678-f003" rid-ob="ob-biomolecules-13-01678-f003">Figure 3</a></div><!--caption a7--><div class="caption"><p>Gene ontology (GO) and KEGG pathway analysis of the target genes. (<strong>A</strong>) Biological process. (<strong>B</strong>) Molecular functions. (<strong>C</strong>) Cellular components. (<strong>D</strong>) KEGG pathway analysis.</p></div></div></div></div><div id="sec3dot4-biomolecules-13-01678" class="sec"><h3 id="sec3dot4-biomolecules-13-01678title">3.4. Screening of Hub Targets and PPI Network Construction</h3><p class="p p-first-last">In the PPI network analysis, we used 38 targets. Among these targets, the kidney cancer and inflammation targets FAAH2 and KCNMA1 did not interact with other targets that consisted of 38 nodes and 150 edges (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f004/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f004" rid-ob="ob-biomolecules-13-01678-f004" co-legend-rid="lgnd_biomolecules-13-01678-f004"><span>Figure 4</span></a>A). For further screening, we considered 78 nodes and 640 edges and selected the top 20% of targets based on the degree of centrality (DC). We also checked betweenness centrality (BC) and closeness centrality (CC) (<a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary S6</a>). Subsequently, we found the top 20 targets that were related to kidney cancer and inflammation (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f004/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f004" rid-ob="ob-biomolecules-13-01678-f004" co-legend-rid="lgnd_biomolecules-13-01678-f004"><span>Figure 4</span></a>B&#x02013;D) (<a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t001/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t001" rid-ob="ob-biomolecules-13-01678-t001" co-legend-rid=""><span>Table 1</span></a>). Furthermore, we selected the common targets among the 38 common targets of SCFA-producing microbes and kidney cancer and inflammation, and the top 20 PPI hub targets of kidney cancer and inflammation. Finally, we found nine hub targets (TP53, CTNNB1, MTOR, PIK3CA, IL6, ERBB2, PTGS2, IGF1R, and RELA) related to kidney cancer and inflammation. For additional conformation, we checked the RNA expression of the nine final target genes in the organs of the urinary bladder and whole kidneys. In this analysis, we found a positive expression in both the urinary bladder and whole kidneys (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f004/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f004" rid-ob="ob-biomolecules-13-01678-f004" co-legend-rid="lgnd_biomolecules-13-01678-f004"><span>Figure 4</span></a>E). Further analysis proceeded with the nine final targets, which were related to kidney cancer and inflammation.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="biomolecules-13-01678-f004" co-legend-rid="lgnd_biomolecules-13-01678-f004"><a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f004/" target="figure" rid-figpopup="biomolecules-13-01678-f004" rid-ob="ob-biomolecules-13-01678-f004"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140445106391904"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10669250_biomolecules-13-01678-g004.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is biomolecules-13-01678-g004.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10669250/bin/biomolecules-13-01678-g004.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140445106391904"><a target="object" rel="noopener" href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f004/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_biomolecules-13-01678-f004"><div><a class="figpopup" href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f004/" target="figure" rid-figpopup="biomolecules-13-01678-f004" rid-ob="ob-biomolecules-13-01678-f004">Figure 4</a></div><!--caption a7--><div class="caption"><p>Hub targets and PPI network construction. (<strong>A</strong>) Identification of kidney cancer and inflammation hub proteins. (<strong>B</strong>) PPI network analysis. (<strong>C</strong>,<strong>D</strong>) PPI network of kidney cancer and inflammation-related hub protein, and screening of the top 20 targets based on the degree of centrality (the nodes represent proteins, and the edges represent protein&#x02013;protein interactions). (<strong>E</strong>) RNA tissue specificity expression profile in the urinary bladder and whole kidneys.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="biomolecules-13-01678-t001"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Degree centrality (DC), betweenness centrality (BC), and closeness centrality (CC) of kidney cancer and kidney inflammation genes.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Name</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Betweenness Centrality</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Closeness Centrality</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Degree</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Number of Directed Edges</th></tr></thead><tbody><tr><td align="center" valign="middle" rowspan="1" colspan="1">TP53</td><td align="center" valign="middle" rowspan="1" colspan="1">0.319071552</td><td align="center" valign="middle" rowspan="1" colspan="1">0.872340426</td><td align="center" valign="middle" rowspan="1" colspan="1">35</td><td align="center" valign="middle" rowspan="1" colspan="1">2</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">CTNNB1</td><td align="center" valign="middle" rowspan="1" colspan="1">0.108958047</td><td align="center" valign="middle" rowspan="1" colspan="1">0.732142857</td><td align="center" valign="middle" rowspan="1" colspan="1">26</td><td align="center" valign="middle" rowspan="1" colspan="1">20</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">IL6</td><td align="center" valign="middle" rowspan="1" colspan="1">0.156313791</td><td align="center" valign="middle" rowspan="1" colspan="1">0.683333333</td><td align="center" valign="middle" rowspan="1" colspan="1">23</td><td align="center" valign="middle" rowspan="1" colspan="1">23</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MTOR</td><td align="center" valign="middle" rowspan="1" colspan="1">0.054536656</td><td align="center" valign="middle" rowspan="1" colspan="1">0.672131148</td><td align="center" valign="middle" rowspan="1" colspan="1">22</td><td align="center" valign="middle" rowspan="1" colspan="1">5</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">PTGS2</td><td align="center" valign="middle" rowspan="1" colspan="1">0.064988518</td><td align="center" valign="middle" rowspan="1" colspan="1">0.650793651</td><td align="center" valign="middle" rowspan="1" colspan="1">20</td><td align="center" valign="middle" rowspan="1" colspan="1">35</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">PIK3CA</td><td align="center" valign="middle" rowspan="1" colspan="1">0.03684383</td><td align="center" valign="middle" rowspan="1" colspan="1">0.630769231</td><td align="center" valign="middle" rowspan="1" colspan="1">18</td><td align="center" valign="middle" rowspan="1" colspan="1">17</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">ERBB2</td><td align="center" valign="middle" rowspan="1" colspan="1">0.0341558</td><td align="center" valign="middle" rowspan="1" colspan="1">0.630769231</td><td align="center" valign="middle" rowspan="1" colspan="1">17</td><td align="center" valign="middle" rowspan="1" colspan="1">10</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">NFKBIA</td><td align="center" valign="middle" rowspan="1" colspan="1">0.010607996</td><td align="center" valign="middle" rowspan="1" colspan="1">0.594202899</td><td align="center" valign="middle" rowspan="1" colspan="1">14</td><td align="center" valign="middle" rowspan="1" colspan="1">26</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">KEAP1</td><td align="center" valign="middle" rowspan="1" colspan="1">0.012401937</td><td align="center" valign="middle" rowspan="1" colspan="1">0.585714286</td><td align="center" valign="middle" rowspan="1" colspan="1">13</td><td align="center" valign="middle" rowspan="1" colspan="1">7</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">SIRT1</td><td align="center" valign="middle" rowspan="1" colspan="1">0.020993867</td><td align="center" valign="middle" rowspan="1" colspan="1">0.585714286</td><td align="center" valign="middle" rowspan="1" colspan="1">13</td><td align="center" valign="middle" rowspan="1" colspan="1">6</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">RELA</td><td align="center" valign="middle" rowspan="1" colspan="1">0.003634516</td><td align="center" valign="middle" rowspan="1" colspan="1">0.569444444</td><td align="center" valign="middle" rowspan="1" colspan="1">11</td><td align="center" valign="middle" rowspan="1" colspan="1">18</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">GSTP1</td><td align="center" valign="middle" rowspan="1" colspan="1">0.01601626</td><td align="center" valign="middle" rowspan="1" colspan="1">0.554054054</td><td align="center" valign="middle" rowspan="1" colspan="1">11</td><td align="center" valign="middle" rowspan="1" colspan="1">22</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">MET</td><td align="center" valign="middle" rowspan="1" colspan="1">0.005245046</td><td align="center" valign="middle" rowspan="1" colspan="1">0.561643836</td><td align="center" valign="middle" rowspan="1" colspan="1">10</td><td align="center" valign="middle" rowspan="1" colspan="1">4</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">SLC2A1</td><td align="center" valign="middle" rowspan="1" colspan="1">0.007293932</td><td align="center" valign="middle" rowspan="1" colspan="1">0.561643836</td><td align="center" valign="middle" rowspan="1" colspan="1">10</td><td align="center" valign="middle" rowspan="1" colspan="1">3</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">IGF1R</td><td align="center" valign="middle" rowspan="1" colspan="1">0.000513</td><td align="center" valign="middle" rowspan="1" colspan="1">0.554054054</td><td align="center" valign="middle" rowspan="1" colspan="1">9</td><td align="center" valign="middle" rowspan="1" colspan="1">6</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">VHL</td><td align="center" valign="middle" rowspan="1" colspan="1">0.003383222</td><td align="center" valign="middle" rowspan="1" colspan="1">0.525641026</td><td align="center" valign="middle" rowspan="1" colspan="1">8</td><td align="center" valign="middle" rowspan="1" colspan="1">7</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">LRRK2</td><td align="center" valign="middle" rowspan="1" colspan="1">0.006013884</td><td align="center" valign="middle" rowspan="1" colspan="1">0.532467532</td><td align="center" valign="middle" rowspan="1" colspan="1">8</td><td align="center" valign="middle" rowspan="1" colspan="1">14</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">CTSD</td><td align="center" valign="middle" rowspan="1" colspan="1">0.006634727</td><td align="center" valign="middle" rowspan="1" colspan="1">0.518987342</td><td align="center" valign="middle" rowspan="1" colspan="1">8</td><td align="center" valign="middle" rowspan="1" colspan="1">8</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">CYP1A1</td><td align="center" valign="middle" rowspan="1" colspan="1">0.005259476</td><td align="center" valign="middle" rowspan="1" colspan="1">0.539473684</td><td align="center" valign="middle" rowspan="1" colspan="1">7</td><td align="center" valign="middle" rowspan="1" colspan="1">1</td></tr><tr><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">FLT1</td><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.001178862</td><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">0.5125</td><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">7</td><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">1</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140445096234944"><a target="object" rel="noopener" href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t001/?report=objectonly">Open in a separate window</a></div></div></div><div id="sec3dot5-biomolecules-13-01678" class="sec"><h3 id="sec3dot5-biomolecules-13-01678title">3.5. Physiochemical and ADMET Property Analysis of Lead Compounds That Control the Hub Targets Expression</h3><p class="p p-first">After the PPI network analysis, we recovered nine hub targets for kidney disease and inflammation that could be regulated by gut SCFA-producing microbial metabolites (<a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t002/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t002" rid-ob="ob-biomolecules-13-01678-t002" co-legend-rid=""><span>Table 2</span></a>) (<a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary S7</a>). The physiochemical and ADMET (absorption, distribution, metabolism, excretion, and toxicity) features of these target-regulated metabolites (75 metabolites) were examined. After analyzing the ADME properties, we screened and rejected some metabolites that did not meet the drug properties (<a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary S7</a>). Finally, we found metabolites that regulated the nine hub targets (<a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t002/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t002" rid-ob="ob-biomolecules-13-01678-t002" co-legend-rid=""><span>Table 2</span></a>), that mainly determined by its pharmacodynamics (PD) and pharmacokinetics (PK) properties.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="biomolecules-13-01678-t002"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>Microbial metabolites and hub targets after ADME property analysis.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Hub Target Genes</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Gene Targeted Compounds and ID</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Target Microbes [<a href="#B19-biomolecules-13-01678" rid="B19-biomolecules-13-01678" class=" bibr popnode">19</a>]</th></tr></thead><tbody><tr><td align="center" valign="middle" rowspan="1" colspan="1">TP53</td><td align="left" valign="middle" rowspan="1" colspan="1">Bile acid (439520, CHEBI:22868)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacteroides distasonis</em>, <em>Clostridium scindens</em>, <em>Faecalibacterium prausnitzii</em>, <em>Haemophilus parainfluenzae.</em></td></tr><tr><td align="center" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">CTNNB1</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">2-Hydroxy-3-(5-hydroxy-1H-indol-3-yl)propanoic acid (192215)</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">
<em>Clostridium sporogenes.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-Hydroxy-4-methoxybenzenepropanoic acid (2752054, HMDB0131138)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Clostridium orbiscindens</em>, <em>Eubacterium ramulus</em>, </td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Dihydrocaffeic acid (348154, HMDB0000423, CHEBI:48400)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bifidobacterium</em>, <em>Bifidobacterium longum</em>, <em>Clostridium orbiscindens</em>, <em>Clostridium sporogenes</em>, <em>Eubacterium ramulus</em>, <em>Faecalibacterium prausnitzii</em>, <em>Lactobacillus mucosae</em>, <em>Lactobacillus zeae.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Indole-3-lactic acid (92904, CHEBI:24813)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Clostridium sporogenes.</em>
</td></tr><tr><td align="center" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">MTOR</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">11-Methoxycurvularin (10381440)</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1"><em>Bacillus</em> sp.</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Dihydrodaidzein (176907, HMDB0005760, CHEBI:75842)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Blautia producta</em>, <em>Bacillus</em> sp., <em>Clostridium</em> sp., <em>Lactobacillus mucosae</em>, <em>Lactococcus</em> sp., </td></tr><tr><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Glycocholic acid (10140, HMDB0000138, CHEBI:17687)</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><em>Bacteroides fragilis</em>, <em>Butyricicoccus pullicaecorum</em>, <em>Ruminococcus flavefaciens.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">PIK3CA</td><td align="left" valign="middle" rowspan="1" colspan="1">11-Methoxycurvularin (10381440)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacillus</em> sp.</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-Hydroxyphenethyl alcohol (83404)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Bifidobacterium.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Caffeic acid (689043, HMDB0001964, CHEBI:16433)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bifidobacterium</em>, <em>Bifidobacterium animalis.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Dihydrodaidzein (176907, HMDB0005760, CHEBI:75842)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Blautia producta</em>, <em>Bacillus</em> sp., <em>Clostridium</em> sp., <em>Lactobacillus mucosae</em>, <em>Lactococcus</em> sp., </td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Dihydroglycitein (101101166, CHEBI:174736)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Eubacterium limosum.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Glycocholic acid (10140, HMDB0000138, CHEBI:17687)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacteroides fragilis</em>, <em>Butyricicoccus pullicaecorum</em>, <em>Ruminococcus flavefaciens.</em></td></tr><tr><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Isoquercitrin (5280804, HMDB0037362, CHEBI:68352)</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><em>Bacillus</em> sp., <em>Bacteroides</em> sp., <em>Eubacterium ramulus.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">IL6</td><td align="left" valign="middle" rowspan="1" colspan="1">Acetate (175, CHEBI:30089)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacteroides thetaiotaomicron</em>, <em>Bacteroidetes</em>, <em>Bifidobacterium dentium</em>, <em>Bifidobacterium longum</em>, <em>Blautia faecis</em>, <em>Clostridium asparagiforme</em>, <em>Clostridium pasteurianum</em>, <em>Clostridium scindens</em>, <em>Clostridium sp. L2-50</em>, <em>Eubacterium limosum</em>, <em>Eubacterium ramulus</em>, <em>Eubacterium rectale</em>, <em>Lawsonibacter asaccharolyticus</em>, <em>Ruminococcus champanellensis</em>, <em>Succinivibrio dextrinosolvens</em>, </td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Butyrate (104775, CHEBI:17968)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Butyricimonas synergistica</em>, <em>Butyricimonas virosa</em>, <em>Clostridium</em>, <em>Clostridium butyricum</em>, <em>Clostridium pasteurianum</em>, <em>Clostridium tyrobutyricum</em>, <em>Eubacterium hallii</em>, <em>Eubacterium limosum</em>, <em>Eubacterium ramulus</em>, <em>Eubacterium rectale</em>, <em>Faecalibacterium prausnitzii</em>, <em>Firmicutes</em>, <em>Fusobacteriia</em>, <em>Lawsonibacter asaccharolyticus</em>, <em>Prevotella copri</em>, <em>Roseburia inulinivorans.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Isoquercitrin (5280804, HMDB0037362, CHEBI:68352)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacillus</em> sp., <em>Bacteroides</em> sp., <em>Eubacterium ramulus.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Propionate (104745, CHEBI:17272)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacteroides</em>, <em>Bacteroides thetaiotaomicron</em>, <em>Eubacterium limosum</em>, <em>Haemophilus parainfluenzae</em>, <em>Parasutterella excrementihominis</em>, <em>Phascolarctobacterium succinatutens</em>, <em>Propionibacterium avidum</em>, <em>Roseburia inulinivorans</em>, <em>Ruminococcus bromii</em>, <em>Veillonella</em>, <em>Veillonella ratti.</em></td></tr><tr><td align="center" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">ERBB2</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">2,3-Dihydroxypropyl (<em>E</em>)-3-(3,4-dihydroxyphenyl)prop-2-enoate (5315606)</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">
<em>Bifidobacterium.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">2-Hydroxy-3-(5-hydroxy-1H-indol-3-yl)propanoic acid (192215)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Clostridium sporogenes.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-Hydroxy-4-methoxybenzenepropanoic acid (2752054, HMDB0131138)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Clostridium orbiscindens</em>, <em>Eubacterium ramulus.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">4-Hydroxy-(3&#x02032;,4&#x02032;-dihydroxyphenyl)-valeric acid (52920332, HMDB0041679, CHEBI:137478)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Lactobacillus plantarum.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">5-(3,4-Dihydroxyphenyl)-valerolactone (45093073)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Lactobacillus plantarum.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Caffeic acid (689043, HMDB0001964, CHEBI:16433)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bifidobacterium</em>, <em>Bifidobacterium animalis.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Ethyl phenyllactate, (-)- (9877619, HMDB0032618)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacteroides caccae</em>, <em>Clostridium</em> sp.</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Hydroquinone (785, HMDB0002434, CHEBI:17594)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacteroides</em>, <em>Bifidobacterium</em>, <em>Bifidobacterium longum</em>, <em>Eubacterium.</em></td></tr><tr><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Indole-3-lactic acid (92904, CHEBI:24813)</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
<em>Clostridium sporogenes.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">PTGS2</td><td align="left" valign="middle" rowspan="1" colspan="1">(<em>R</em>)-3-(4-Hydroxyphenyl)lactate (9548632, CHEBI:10980)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacteroides caccae</em>, <em>Clostridium</em> sp.</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">2-(4-Hydroxyphenyl)propionic acid, (2<em>S</em>)- (6971268)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Eubacterium ramulus.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">2,3-Dihydroxypropyl (<em>E</em>)-3-(3,4-dihydroxyphenyl)prop-2-enoate (5315606)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bifidobacterium</em>, </td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">2-Hydroxy-3-(4-hydroxyphenyl)propanoic acid (9378, HMDB0000755, CHEBI:17385)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Clostridium sporogenes.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">2-Hydroxy-3-(5-hydroxy-1H-indol-3-yl)propanoic acid (192215)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Clostridium sporogenes.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-(3,4-Dihydroxyphenyl)-2-hydroxypropanoic acid (439435, HMDB0003503, CHEBI:17807)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Clostridium sporogenes.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-(3-Hydroxyphenyl)propanoic acid (91, HMDB0000375, CHEBI:1427)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Bifidobacterium.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-(4-Hydroxyphenyl)propionic acid (10394, HMDB0002199, CHEBI:32980)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Clostridium orbiscindens</em>, <em>Eubacterium ramulus.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3,4-Dihydroxybenzoic acid (72, CHEBI:36062)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacteroides</em> sp.</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3,4-Dihydroxyphenylacetic acid (547, HMDB0001336, CHEBI:41941)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Clostridium orbiscindens</em>, <em>Eubacterium ramulus</em>, </td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-Hydroxy-4-methoxybenzenepropanoic acid (2752054, HMDB0131138)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Clostridium orbiscindens</em>, <em>Eubacterium ramulus</em>, </td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-Hydroxybenzoic acid (7420, HMDB0002466, CHEBI:30764)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Eubacterium.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-Hydroxyphenethyl alcohol (83404)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Bifidobacterium.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-Phenylpropionic acid (107, CHEBI:28631)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Clostridium sporogenes</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">4-Hydroxy-(3&#x02032;,4&#x02032;-dihydroxyphenyl)-valeric acid (52920332, HMDB0041679, CHEBI:137478)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Lactobacillus plantarum.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">4-Hydroxybenzoic acid (135, HMDB0000500, CHEBI:30763)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Eubacterium.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">4-Hydroxyphenylacetic acid (127, HMDB0000020, CHEBI:18101)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Eubacterium ramulus</em>, </td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Caffeic acid (689043, HMDB0001964, CHEBI:16433)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bifidobacterium</em>, <em>Bifidobacterium animalis.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Dihydrocaffeic acid (348154, HMDB0000423, CHEBI:48400)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bifidobacterium</em>, <em>Bifidobacterium longum</em>, <em>Clostridium orbiscindens</em>, <em>Clostridium sporogenes</em>, <em>Eubacterium ramulus</em>, <em>Faecalibacterium prausnitzii</em>, <em>Lactobacillus mucosae</em>, <em>Lactobacillus zeae.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">D-Lactic acid (61503, HMDB0001311, CHEBI:42111)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Faecalibacterium prausnitzii</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Ethyl phenyllactate, (-)- (9877619, HMDB0032618)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacteroides caccae</em>, <em>Clostridium</em> sp.</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Isobutyric acid (6590, HMDB0001873, CHEBI:16135)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Butyricimonas synergistica</em>, <em>Butyricimonas virosa</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Isoquercitrin (5280804, HMDB0037362, CHEBI:68352)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacillus</em> sp., <em>Bacteroides</em> sp., <em>Eubacterium ramulus.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Leucine (6106, HMDB0000687, CHEBI:15603)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Blautia</em>, <em>Faecalibacterium prausnitzii</em>, <em>Ruminococcus</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Phenolic acid (CHEBI:166890)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Eubacterium ramulus.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Phenylacetic acid (999, HMDB0000209, CHEBI:30745)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Bifidobacterium.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Pipecolic acid (849, HMDB0000070, CHEBI:17964)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Lactobacillus casei.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Proline (145742, HMDB0000162, CHEBI:17203)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Blautia</em>, <em>Ruminococcus.</em></td></tr><tr><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Quinic acid (6508, HMDB0003072, CHEBI:17521)</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
<em>Bifidobacterium animalis.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">IGF1R</td><td align="left" valign="middle" rowspan="1" colspan="1">2-Amino-1-methyl-6-phenylimidazo[4,5-b] pyridine (1530, CHEBI:76290)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Blautia obeum</em>, <em>Faecalibacterium prausnitzii</em>, <em>Lactobacillus reuteri.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-(3,4-Dihydroxyphenyl)-2-hydroxypropanoic acid (439435, HMDB0003503, CHEBI:17807)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Clostridium sporogenes</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-(3-Hydroxyphenyl)propanoic acid (91, HMDB0000375, CHEBI:1427)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Bifidobacterium.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-(4-Hydroxyphenyl)propionic acid (10394, HMDB0002199, CHEBI:32980)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Clostridium orbiscindens</em>, <em>Eubacterium ramulus.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3,4-Dihydroxybenzoic acid (72, CHEBI:36062)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacteroides</em> sp.</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3,4-Dihydroxyphenylacetic acid (547, HMDB0001336, CHEBI:41941)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Clostridium orbiscindens</em>, <em>Eubacterium ramulus</em>, </td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-Hydroxy-4-methoxybenzenepropanoic acid (2752054, HMDB0131138)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Clostridium orbiscindens</em>, <em>Eubacterium ramulus</em>, </td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">3-Hydroxybenzoic acid (7420, HMDB0002466, CHEBI:30764)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Eubacterium.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">4-Hydroxy-(3&#x02032;,4&#x02032;-dihydroxyphenyl)-valeric acid (52920332, HMDB0041679, CHEBI:137478)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Lactobacillus plantarum.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">4-Hydroxybenzoic acid (135, HMDB0000500, CHEBI:30763)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Eubacterium.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Dihydrocaffeic acid (348154, HMDB0000423, CHEBI:48400)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bifidobacterium</em>, <em>Bifidobacterium longum</em>, <em>Clostridium orbiscindens</em>, <em>Clostridium sporogenes</em>, <em>Eubacterium ramulus</em>, <em>Faecalibacterium prausnitzii</em>, <em>Lactobacillus mucosae</em>, <em>Lactobacillus zeae.</em></td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Dihydrodaidzein (176907, HMDB0005760, CHEBI:75842)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Blautia producta</em>, <em>Bacillus</em> sp., <em>Clostridium</em> sp., <em>Lactobacillus mucosae</em>, <em>Lactococcus</em> sp., </td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Ethyl phenyllactate, (-)- (9877619, HMDB0032618)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacteroides caccae</em>, <em>Clostridium</em> sp.</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Glutathione (124886, HMDB0000125, CHEBI:16856)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Bacteroides thetaiotaomicron.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">
</td><td align="left" valign="middle" rowspan="1" colspan="1">Isoquercitrin (5280804, HMDB0037362, CHEBI:68352)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacillus</em> sp., <em>Bacteroides</em> sp., <em>Eubacterium ramulus.</em></td></tr><tr><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Phenolic acid (CHEBI:166890)</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
<em>Eubacterium ramulus.</em>
</td></tr><tr><td align="center" valign="middle" rowspan="1" colspan="1">RELA</td><td align="left" valign="middle" rowspan="1" colspan="1">2,3-Dihydroxypropyl (E)-3-(3,4-dihydroxyphenyl)prop-2-enoate (5315606)</td><td align="left" valign="middle" rowspan="1" colspan="1">
<em>Bifidobacterium.</em>
</td></tr><tr><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">
</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Caffeic acid (689043, HMDB0001964, CHEBI:16433)</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><em>Bifidobacterium</em>, <em>Bifidobacterium animalis.</em></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140445108663520"><a target="object" rel="noopener" href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t002/?report=objectonly">Open in a separate window</a></div></div><p class="p p-last">As per Lipinski&#x02019;s rule of five, an orally administrated drug must have HBA &#x02264; 10, log P &#x02264; 5, HBD &#x02264; 5, and molecular weight (MW) &#x0003c; 500 Daltons [<a href="#B45-biomolecules-13-01678" rid="B45-biomolecules-13-01678" class=" bibr popnode">45</a>]. Due to the violation of Lipinski&#x02019;s rules, we rejected (20S)-protopanaxadiol, aglycone, beta-D-Gal-(1-&#x0003e;4)-beta-D-GlcNAc-(1-&#x0003e;3)-beta-D-Gal-(1-&#x0003e;4)-D-Glc, and ginsenoside Rh2, as well as because of the violation of ADME properties, such as the plasma-protein binding property. The level of plasma-protein binding impacts the drug&#x02019;s effectiveness, clearance, and possible interactions. Only the unbound fraction of the drug is prone to clearance from the liver and is available for binding to the molecular target [<a href="#B46-biomolecules-13-01678" rid="B46-biomolecules-13-01678" class=" bibr popnode">46</a>]. As such, we rejected molecular candidates that did not have plasma-protein binding values below &#x0003c;90%. Candidates rejected based on the criterion included 10-keto-12Z-octadecenoic acid, 2,3-bis(3,4-dihydroxybenzyl) butyrolactone, 6,7,4&#x02032;,rihydroxyisoflavone, 6&#x02032;-hydroxy-O-desmethylangolensin, 8-prenylnaringenin, apigenin, arctigenin, chrysin, daidzein, deoxycholic acid, dihydrodaidzein, dihydroglycitein, equol, folic acid, genistein, glycitein, hesperetin dihydrochalcone, kaempferol, naringenin, norathyriol, O-desmethylangolensin, palmitic acid, phloretin, protopanaxadiol, quercetin, and secoisolariciresinol. Finally, we select 9 targets with 41 compounds for further analysis (<a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t002/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t002" rid-ob="ob-biomolecules-13-01678-t002" co-legend-rid=""><span>Table 2</span></a>, <a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary S7</a>).</p></div><div id="sec3dot6-biomolecules-13-01678" class="sec sec-last"><h3 id="sec3dot6-biomolecules-13-01678title">3.6. Molecular Docking of a Bioactive Compound with Its Target</h3><p class="p p-first">Molecular docking was used to examine the molecular interactions between the kidney cancer and inflammation-related targets described above, as well as the SCFAs that produce microbial metabolites. As a control, belzutifan (for kidney cancer) [<a href="#B47-biomolecules-13-01678" rid="B47-biomolecules-13-01678" class=" bibr popnode">47</a>] and levofloxacin (for kidney inflammation) [<a href="#B48-biomolecules-13-01678" rid="B48-biomolecules-13-01678" class=" bibr popnode">48</a>] were used, and the critical active site residues were flexibly maintained. The findings of the interaction were confirmed by the formation of hydrogen bonds and the binding energy to the necessary active residues and ligands. In molecular docking, we addressed the top nine targets for kidney cancer and inflammation. <a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary S4</a> relays information on the selected targets with ligands and the binding affinity. Among all of the molecular docking results, the maximum binding energy was observed to be &#x02212;2.9 kcal/mol for kidney cancer and inflammation, whereas the minimum binding energy was observed to be &#x02013;9.5 kcal/mol. After analyzing the obtained binding affinities, we obtained five targets (IGF1R, IL6, MTOR, PIK3CA, and PTGS2) for kidney cancer and inflammation (<a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t003/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t003" rid-ob="ob-biomolecules-13-01678-t003" co-legend-rid=""><span>Table 3</span></a>). After evaluating the docking results, we predict that 2-amino-1-methyl-6-phenylimidazo (4,5-b) pyridine and isoquercitrin with IGF1R target; isoquercitrin with IL6 target; 11-methoxycurvularin and glycocholic acid with MTOR target; 11-methoxycurvularin, glycocholic acid, and isoquercitrin with PIK3CA target; and isoquercitrin with PTGS2 target showed a lower binding affinity and better stability than the control ligands. </p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="biomolecules-13-01678-t003"><h3>Table 3</h3><!--caption a7--><div class="caption"><p>Molecular docking simulation between five targets and their associated compounds.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Targets</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Compound</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Binging Energy</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Hydrogen Bond</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Other Bonds</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Grid Box Center</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Dimension</th></tr></thead><tbody><tr><td rowspan="4" align="center" valign="middle" colspan="1">IGF1R</td><td align="left" valign="middle" rowspan="1" colspan="1">2-Amino-1-methyl-6-phenylimidazo(4,5-b) pyridine_CID_1530</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;7.4</td><td align="left" valign="middle" rowspan="1" colspan="1">MET</td><td align="left" valign="middle" rowspan="1" colspan="1">LEU, VAL, LYS, MET</td><td rowspan="4" align="left" valign="middle" colspan="1">x= 20.89, y = 3.76, z= 45.48</td><td rowspan="4" align="left" valign="middle" colspan="1">x = 74.703, y = 55.85, z = 62.013</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Isoquercetin_COMPOUND_CID_5280804</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;7.9</td><td align="left" valign="middle" rowspan="1" colspan="1">MET, THR</td><td align="left" valign="middle" rowspan="1" colspan="1">GLY, ARG, LEU, SER</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Control_Belzutifan_Cancer_CID_117947097</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;7.3</td><td align="left" valign="middle" rowspan="1" colspan="1">SER</td><td align="left" valign="middle" rowspan="1" colspan="1">LEU, ASP, MET, VAL</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Control_levofloxacin__inflammation_CID_149096</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;7.6</td><td align="left" valign="middle" rowspan="1" colspan="1">LEU</td><td align="left" valign="middle" rowspan="1" colspan="1">MET, VAL, ILE</td></tr><tr><td rowspan="3" align="center" valign="middle" style="border-top:solid thin" colspan="1">IL6</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">Isoquercetin_CID_5280804</td><td align="center" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">&#x02212;7.9</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">ALA, GLN, GLY, THR</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">VAL, GLU, PRO</td><td rowspan="3" align="left" valign="middle" style="border-top:solid thin" colspan="1">x = 16.06, y = 14.52, z = 22.37 </td><td rowspan="3" align="left" valign="middle" style="border-top:solid thin" colspan="1">x = 64.69, y = 83.398, z = 56.699</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Belzutifan_Cancer_CID_117947097</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;7.5</td><td align="left" valign="middle" rowspan="1" colspan="1">ARG, LEU, SER, TRP</td><td align="left" valign="middle" rowspan="1" colspan="1">ALA, LEU, ARG, VAL</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">levofloxacin__inflammation_CID_149096</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;7.5</td><td align="left" valign="middle" rowspan="1" colspan="1">HIS, LEU</td><td align="left" valign="middle" rowspan="1" colspan="1">SER, GLY, THR </td></tr><tr><td rowspan="3" align="center" valign="middle" style="border-top:solid thin" colspan="1">MTOR</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">2fap_Glycocholic acid_COMPOUND_CID_10140</td><td align="center" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">&#x02212;6.5</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">TYR</td><td align="center" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">
</td><td rowspan="3" align="left" valign="middle" style="border-top:solid thin" colspan="1">x = &#x02212;6.45, y = 21.63, z = 43.68</td><td rowspan="3" align="left" valign="middle" style="border-top:solid thin" colspan="1">x = 35.723, y = 45.28, z = 33.53</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">2fap_Belzutifan_Cancer_COMPOUND_CID_117947097</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;7.4</td><td align="left" valign="middle" rowspan="1" colspan="1">GLU</td><td align="left" valign="middle" rowspan="1" colspan="1">PHE</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">2fap_levofloxacin__inflammation_COMPOUND_CID_149096</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;6.3</td><td align="left" valign="middle" rowspan="1" colspan="1">GLU</td><td align="left" valign="middle" rowspan="1" colspan="1">TYR, ILE, VAL</td></tr><tr><td rowspan="5" align="center" valign="middle" style="border-top:solid thin" colspan="1">PIK3CA</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">5dxt.p_11-Methoxycurvularin_COMPOUND_CID_10381440</td><td align="center" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">&#x02212;8.2</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">GLN, SER</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">VAL, GLU, PRO</td><td rowspan="5" align="left" valign="middle" style="border-top:solid thin" colspan="1">x = &#x02212;1.83, y = 5.71, z = 17.04</td><td rowspan="5" align="left" valign="middle" style="border-top:solid thin" colspan="1">x = 79.48, y = 96.48, z = 89.84</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">5dxt.p_Glycocholic acid_COMPOUND_CID_10140</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;9.3</td><td align="left" valign="middle" rowspan="1" colspan="1">MET, GLU, ASP</td><td align="center" valign="middle" rowspan="1" colspan="1">
</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">5dxt.p_Isoquercetin_CID_5280804</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;8.4</td><td align="left" valign="middle" rowspan="1" colspan="1">GLN, LYN, SER, ASN</td><td align="left" valign="middle" rowspan="1" colspan="1">ASP</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">5dxt.p_Belzutifan__CID_117947097</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;7.3</td><td align="left" valign="middle" rowspan="1" colspan="1">ARG, HIS</td><td align="left" valign="middle" rowspan="1" colspan="1">SER, PHE, LEU</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">5dxt.p_levofloxacin__inflammation_COMPOUND_CID_149096</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;8.1</td><td align="left" valign="middle" rowspan="1" colspan="1">ARG, ASN</td><td align="left" valign="middle" rowspan="1" colspan="1">GLU, LEU, GLN</td></tr><tr><td rowspan="3" align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" colspan="1">PTGS2</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">5ikq.pp_Isoquercetin_COMPOUND_CID_5280804</td><td align="center" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">&#x02212;9.5</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">TYR, ASN</td><td align="left" valign="middle" style="border-top:solid thin" rowspan="1" colspan="1">PRO, CYS, ASP</td><td rowspan="3" align="left" valign="middle" style="border-top:solid thin;border-bottom:solid thin" colspan="1">x = 27.15, y = 38.38, z= 41.79</td><td rowspan="3" align="left" valign="middle" style="border-top:solid thin;border-bottom:solid thin" colspan="1">x = 82.98, y = 83.28, z = 104.06</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">5ikq.pp_Belzutifan_Cancer_COMPOUND_CID_117947097</td><td align="center" valign="middle" rowspan="1" colspan="1">&#x02212;9.1</td><td align="left" valign="middle" rowspan="1" colspan="1">ARG, GLN, ASN</td><td align="left" valign="middle" rowspan="1" colspan="1">LEU, GLY, PHE</td></tr><tr><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">5ikq.pp_levofloxacin__inflammation_COMPOUND_CID_149096</td><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">&#x02212;9</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">CYS</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">PRO, TYR, GLN</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140445094798800"><a target="object" rel="noopener" href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t003/?report=objectonly">Open in a separate window</a></div></div><p>Then, the results were visualized using Discovery Studio (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f005/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f005" rid-ob="ob-biomolecules-13-01678-f005" co-legend-rid="lgnd_biomolecules-13-01678-f005"><span>Figure 5</span></a>). We observed the 3D interaction modes of SCFA microbial metabolites with the kidney cancer and inflammation proteins targets.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="biomolecules-13-01678-f005" co-legend-rid="lgnd_biomolecules-13-01678-f005"><a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f005/" target="figure" rid-figpopup="biomolecules-13-01678-f005" rid-ob="ob-biomolecules-13-01678-f005"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140445089003488"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10669250_biomolecules-13-01678-g005a.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is biomolecules-13-01678-g005a.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10669250/bin/biomolecules-13-01678-g005a.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140445089003488"><a target="object" rel="noopener" href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f005/?report=objectonly">Open in a separate window</a></div><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140445089002528"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=10669250_biomolecules-13-01678-g005b.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="An external file that holds a picture, illustration, etc.&#10;Object name is biomolecules-13-01678-g005b.jpg" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC10669250/bin/biomolecules-13-01678-g005b.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm140445089002528"><a target="object" rel="noopener" href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f005/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_biomolecules-13-01678-f005"><div><a class="figpopup" href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f005/" target="figure" rid-figpopup="biomolecules-13-01678-f005" rid-ob="ob-biomolecules-13-01678-f005">Figure 5</a></div><!--caption a7--><div class="caption"><p>The 3D interactions of kidney cancer and inflammation-related targets with their related SCFA-producing microbial metabolites and control (Belzutifan_ Cancer and Levofloxacin_ inflammation). (<strong>A</strong>) (<strong>a</strong>) IGF1R with 2-amino-1-methyl-6-phenylimidazo(4,5-b) pyridine (CID 1530); (<strong>b</strong>) isoquercitrin (CID 5280804); (<strong>c</strong>) belzutifan (CID 117947097); (<strong>d</strong>) levofloxacin (CID 149096). (<strong>B</strong>) (<strong>a</strong>) IL6 with isoquercitrin (CID 5280804); (<strong>b</strong>) belzutifan (CID 117947097); (<strong>c</strong>) levofloxacin (CID 149096). (<strong>C</strong>) MTOR with glycocholic acid (CID 10140); belzutifan (CID 117947097); levofloxacin (CID 149096). (<strong>D</strong>) PIK3CA with 11-methoxycurvularin (<strong>a</strong>) (CID 10381440); (<strong>b</strong>) glycocholic acid (CID 10140); (<strong>c</strong>) isoquercitrin (CID 5280804); (<strong>d</strong>) belzutifan (CID 117947097); (<strong>e</strong>) levofloxacin (CID 149096). (<strong>E</strong>) PTGS2 with (<strong>a</strong>) isoquercitrin (CID 5280804), (<strong>b</strong>) belzutifan (CID 117947097), and (<strong>c</strong>) levofloxacin (CID 149096).</p></div></div></div><p>We predict that the IGF1R target interacted with 2-amino-1-methyl-6-phenylimidazo(4,5-b) pyridine via MET1082 (hydrogen bond), as well as LEU1005, VAL1013, LYS1033, and MET1156 (other bonds), and showing a binding affinity of &#x02212;7.4 kcal/mol. Another compound, isoquercitrin interacted via MET1082, MET1156, and THR1083 (hydrogen bond), as well as GLY1085, ARG1084, LEU1005, and SER1089 (other bonds), and with a binding affinity of &#x02212;7.9 kcal/mol. Meanwhile, belzutifan (control drug for kidney cancer) interacted via SER1089 (hydrogen bond), as well as LEU1005, ASP1086, MET1142, MET1156, and VAL1013 (other bonds), with a binding affinity of &#x02212;7.3 kcal/mol. Levofloxacin (control drug for kidney inflammation) interacted via LEU1005 (hydrogen bond), as well as MET1156, VAL1013, and ILE1160 (other bonds), and also with a binding affinity of &#x02212;7.6 kcal/mol (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f005/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f005" rid-ob="ob-biomolecules-13-01678-f005" co-legend-rid="lgnd_biomolecules-13-01678-f005"><span>Figure 5</span></a>A, <a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t003/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t003" rid-ob="ob-biomolecules-13-01678-t003" co-legend-rid=""><span>Table 3</span></a>, <a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary Figure S1A</a>).</p><p>Then, the IL6 target interacted with isoquercitrin via ALA177, GLN114, GLY43, and THR44 (hydrogen bond), as well as VAL94, GLU157, and PRO158 (other bonds), and with a binding affinity of &#x02212;7.9 kcal/mol, whereas both controls drugs showed the same binding affinity. Such as, belzutifan (control drug for kidney cancer) interacted via ARG103, LEU4, SER64, and TRP49 (hydrogen bond), as well as ALA99, LEU47, ARG103, and VAL100 (other bonds), and with a binding affinity of &#x02212;7.5 kcal/mol, and levofloxacin (control drug of kidney inflammation) interacted via HIS173 and LEU139 (hydrogen bond), as well as SER141, GLY171, and PHE175 (other bonds), and with a binding affinity of &#x02212;7.5 kcal/mol (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f005/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f005" rid-ob="ob-biomolecules-13-01678-f005" co-legend-rid="lgnd_biomolecules-13-01678-f005"><span>Figure 5</span></a>B, <a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t003/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t003" rid-ob="ob-biomolecules-13-01678-t003" co-legend-rid=""><span>Table 3</span></a>, <a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary Figure S1B</a>).</p><p>Moreover, MTOR target interacted with glycocholic acid via TYR82 (hydrogen bod) with a binding affinity of &#x02212;6.5 kcal/mol, whereas, both controls drugs showed different binding affinity. Such as, belzutifan (control drug for kidney cancer) interacted via GLU54 (hydrogen bond), as well as PHE46 (other bonds), with a binding affinity of &#x02212;7.4 kcal/mol, and levofloxacin (control drug of kidney inflammation) interacted via GLU54 (hydrogen bond), as well as TYR82, ILE56, and VAL55 (other bonds), with a binding affinity of &#x02212;6.3 kcal/mol (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f005/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f005" rid-ob="ob-biomolecules-13-01678-f005" co-legend-rid="lgnd_biomolecules-13-01678-f005"><span>Figure 5</span></a>C, <a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t003/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t003" rid-ob="ob-biomolecules-13-01678-t003" co-legend-rid=""><span>Table 3</span></a>, <a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary Figure S1C</a>).</p><p>In addition, PIK3CA target interacted with 11-methoxycurvularin via GLN682 and SER464 (hydrogen bond), as well as VAL680, GLU135, and PRO466 (other bonds), with a binding affinity of &#x02212;8.2 kcal/mol. Then, glycocholic acid interacted with MET811, GLU259, and ASP258 (hydrogen bod) with a binding affinity of &#x02212;9.3 kcal/mol. Finally, isoquercitrin interacted via GLN682, LYN678, SER464, ASN428 (hydrogen bond), and ASP133 (other bonds) with a binding affinity of &#x02212;8.4 kcal/mol, whereas belzutifan (control drug for kidney cancer) interacted via ARG818 and HIS670 (hydrogen bond), as well as SER629, PHE666, and LEU755 (other bonds) with a binding affinity of &#x02212;7.3 kcal/mol, and levofloxacin (control drug of kidney inflammation) interacted via ARG683 and ASN428 (hydrogen bond), as well as GLU135, LEU645, and GLN643 (other bonds), with a binding affinity of &#x02212;8.1 kcal/mol (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f005/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f005" rid-ob="ob-biomolecules-13-01678-f005" co-legend-rid="lgnd_biomolecules-13-01678-f005"><span>Figure 5</span></a>D, <a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t003/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t003" rid-ob="ob-biomolecules-13-01678-t003" co-legend-rid=""><span>Table 3</span></a>, <a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary Figure S1D</a>).</p><p class="p p-last">Finally, PTGS2 target interacted with isoquercitrin via TYR136 and ASN34 (hydrogen bond), as well as PRO156, CYS36, PRO154, and ASP157 (other bonds), with a binding affinity of &#x02212;9.5 kcal/mol, whereas belzutifan (control drug for kidney cancer) interacted via ARG376, GLN375, and ASN376 (hydrogen bond), as well as LEU224, GLY225, and PHE142 (other bonds), with a binding affinity of &#x02212;9.1 kcal/mol. Levofloxacin (control drug of kidney inflammation) interacted via CYS39 and CYS47 (hydrogen bond), as well as PRO156, TYR136, PRO154, and GLN328 (other bonds), with a binding affinity of &#x02212;9 kcal/mol (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f005/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f005" rid-ob="ob-biomolecules-13-01678-f005" co-legend-rid="lgnd_biomolecules-13-01678-f005"><span>Figure 5</span></a>E, <a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t003/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t003" rid-ob="ob-biomolecules-13-01678-t003" co-legend-rid=""><span>Table 3</span></a>, <a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary Figure S1E</a>).</p></div></div><div id="sec4-biomolecules-13-01678" class="tsec sec"><h2 class="head no_bottom_margin" id="sec4-biomolecules-13-01678title">4. Discussion</h2><p class="p p-first">According to epidemiological data for kidney cancer (KC) as a whole, renal cell carcinoma (RCC) represents the vast majority (90%) of KC cases, with clear cell RCC (ccRCC; 70%), papillary RCC (pRCC; 10&#x02013;15%), and chromophobe RCC (5%) being the most common types [<a href="#B49-biomolecules-13-01678" rid="B49-biomolecules-13-01678" class=" bibr popnode">49</a>]. Recently, research topics like kidney cancer are a major challenge for scientists. To address challenges like the lack of efficacy and the development of resistance to single-targeted drugs, drug discovery frequently necessitates a system-level pharmacology approach. Network pharmacology techniques are created and used more frequently to identify new therapeutic possibilities and repurpose current medications [<a href="#B50-biomolecules-13-01678" rid="B50-biomolecules-13-01678" class=" bibr popnode">50</a>]. To assess the pharmacological significance of the primary target discovered using microbiome analysis, we conducted a network pharmacology inquiry. Using data-driven analysis, we investigated the interaction of kidney cancer and inflammation with gut microbiome metabolites. Malnutrition, hypertension or hypotension, microinflammation, immune system dysbiosis, and numerous oxidative stresses are frequently associated with kidney illness and can all be treated with SCFAs [<a href="#B14-biomolecules-13-01678" rid="B14-biomolecules-13-01678" class=" bibr popnode">14</a>]. SCFA-producing microbes also release different types of metabolites. However, currently, there is a lack of information on the role and interaction of gene targets with SCFA-producing microbial metabolites in regulating kidney cancer and inflammation. To investigate such a novel hypothesis, we performed several network pharmacology-based analyses. In this study, we analyzed SCFA-producing microbial metabolites and disease-related targets to investigate SCFA-producing microbial metabolites and disease networks.</p><p class="p p-last">During data collection, we selected microbes that produce SCFAs (177 different species, 114 metabolites, and 1890 gene targets from gutMgene [<a href="#B19-biomolecules-13-01678" rid="B19-biomolecules-13-01678" class=" bibr popnode">19</a>]) and 13,104 disease targets from the DisgeNET [<a href="#B20-biomolecules-13-01678" rid="B20-biomolecules-13-01678" class=" bibr popnode">20</a>], Genecard [<a href="#B22-biomolecules-13-01678" rid="B22-biomolecules-13-01678" class=" bibr popnode">22</a>], and OMIM [<a href="#B21-biomolecules-13-01678" rid="B21-biomolecules-13-01678" class=" bibr popnode">21</a>] database platforms. For validation of the data, we collected gene targets from three different platforms. Furthermore, we performed PPI network analysis; the physical connections between proteins in a cell were mathematically modeled by PPI networks. These unique interactions between specified binding sites in the proteins have a particular biological significance (i.e., they perform a particular function) [<a href="#B51-biomolecules-13-01678" rid="B51-biomolecules-13-01678" class=" bibr popnode">51</a>]; therefore, we found functionally correlated genes. For further validation, we observed the RNA expression in different organs, and found a positive expression in the urinary bladder and kidney (<a href="/pmc/articles/PMC10669250/figure/biomolecules-13-01678-f004/" target="figure" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-f004" rid-ob="ob-biomolecules-13-01678-f004" co-legend-rid="lgnd_biomolecules-13-01678-f004"><span>Figure 4</span></a>E). Absorption, distribution, metabolism, and excretion are together referred to as ADME. This group of characteristics is essential for a drug molecule to be an acceptable drug candidate within the human body. The ADME profile is affected by a number of variables, including physicochemical qualities, protein binding, solubility, permeability, and inhibitory screening results. A drug&#x02019;s probability of success will also be significantly influenced by its ADME profile [<a href="#B52-biomolecules-13-01678" rid="B52-biomolecules-13-01678" class=" bibr popnode">52</a>]. As a result, ADME features were used to select potentially significant substances. Finally, we performed a molecular docking simulation to predict possible targets. Due to its capacity to predict the binding conformation of small molecule ligands to the proper target binding site, molecular docking is one of the most widely utilized techniques in structure-based drug design. The rational design of medications and understanding of the basic biological processes both benefit greatly from the characterization of the binding behavior [<a href="#B53-biomolecules-13-01678" rid="B53-biomolecules-13-01678" class=" bibr popnode">53</a>,<a href="#B54-biomolecules-13-01678" rid="B54-biomolecules-13-01678" class=" bibr popnode">54</a>]. Therefore, after PPI network analysis, RNA expression validation, ADMET properties screening, and molecular docking simulation, we selected five (5) different targets (IGF1R, IL6, MTOR, PIK3CA, and PTGS2) and four (4) different metabolites including different targets (isoquercitrin, glycocholic acid, 11-methoxycurvularin, and 2-amino-1-methyl-6-phenylimidazo (4,5-b) pyridine). Among our selected metabolites, isoquercitrin prevents the spread of bladder, pancreatic, and liver cancer. The main mechanisms are that isoquercitrin activates caspase-3, -8, and -9; reduces the phosphorylation of ERK; and promotes the phosphorylation of c-Jun N-terminal kinase (JNK). Additionally, isoquercitrin blocks the cell cycle in the G1 phase to promote the death of cancer cells through apoptosis [<a href="#B55-biomolecules-13-01678" rid="B55-biomolecules-13-01678" class=" bibr popnode">55</a>,<a href="#B56-biomolecules-13-01678" rid="B56-biomolecules-13-01678" class=" bibr popnode">56</a>,<a href="#B57-biomolecules-13-01678" rid="B57-biomolecules-13-01678" class=" bibr popnode">57</a>]. Further, isoquercitrin dramatically reduces the mRNA expression of proinflammatory factors such as tumor necrosis factor-, interleukin (IL)-1, IL-6, monocyte chemoattractant protein-1, and prostaglandin E synthase 2 (PTGES2). As a result, isoquercitrin serves as a possible pharmaceutical substitute for the treatment of diseases caused by inflammation [<a href="#B58-biomolecules-13-01678" rid="B58-biomolecules-13-01678" class=" bibr popnode">58</a>]. Through our network pharmacology study, we found that the isoquercitrin compound can be isolated from <em>Bacillus sp.</em>, <em>Bacteroides sp.</em>, and <em>Eubacterium ramulus</em> [<a href="#B19-biomolecules-13-01678" rid="B19-biomolecules-13-01678" class=" bibr popnode">19</a>] SCFA-producing microbes, and can be used to treat kidney cancer and inflammation-related diseases by regulating PIK3CA, IL6, PTGS2, and IGF1R gene targets (<a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t004/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t004" rid-ob="ob-biomolecules-13-01678-t004" co-legend-rid=""><span>Table 4</span></a>). Another compound, glycocholic acid, was discovered to be a highly effective and secure anti-inflammatory medication. It is a lead compound that can be utilized to treat an overactive immune system [<a href="#B59-biomolecules-13-01678" rid="B59-biomolecules-13-01678" class=" bibr popnode">59</a>]. We also found that glycocholic acid can be isolated from <em>Bacteroides fragilis</em>, <em>Butyricicoccus pullicaecorum</em>, and <em>Ruminococcus flavefaciens</em> [<a href="#B19-biomolecules-13-01678" rid="B19-biomolecules-13-01678" class=" bibr popnode">19</a>] SCFA-producing microbes, and can be used to treat kidney cancer and inflammation-related diseases through regulating MTOR and PIK3CA targets (<a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t004/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t004" rid-ob="ob-biomolecules-13-01678-t004" co-legend-rid=""><span>Table 4</span></a>). Furthermore, 11-methoxycurvularin derived from a fungal strain <em>Penicillium</em> sp. and acts as an anti-inflammatory compound that exhibits strong inhibitory effects on nitric oxide (NO) and prostaglandin E<sub>2</sub> (PGE<sub>2</sub>), with IC<sub>50</sub> values ranging from 1.9 to 18.1&#x003bc;m, and also on IC<sub>50</sub> values from 2.8 to 18.7 &#x003bc;M, respectively, in RAW264.7 cells induced by LPS [<a href="#B60-biomolecules-13-01678" rid="B60-biomolecules-13-01678" class=" bibr popnode">60</a>]. In our study, we found that the 11-methoxycurvularin compound could be isolated from <em>Bacillus sp.</em> SCFAs produce microbes [<a href="#B19-biomolecules-13-01678" rid="B19-biomolecules-13-01678" class=" bibr popnode">19</a>] and can be used to treat kidney cancer and inflammation-related diseases through regulating MTOR and PIK3CA targets (<a href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t004/" target="table" class="fig-table-link figpopup" rid-figpopup="biomolecules-13-01678-t004" rid-ob="ob-biomolecules-13-01678-t004" co-legend-rid=""><span>Table 4</span></a>). Finally, pathway analysis and gene&#x02212;gene interactions show that 2-amino-1-methyl-6-phenylimidazo[4,5-b] pyridine (PhIP) regulates STAT3-regulated genes and starts leptin signaling through the JAK/STAT and MAPK pathway cascades. PhIP can be isolated from <em>Blautia obeum</em>, <em>Faecalibacterium prausnitzii</em>, and <em>Lactobacillus reuteri</em> SCFA microbes [<a href="#B19-biomolecules-13-01678" rid="B19-biomolecules-13-01678" class=" bibr popnode">19</a>]. As a result of the many limitations and toxicity of this compound, it cannot be used as a postbiotic compound [<a href="#B61-biomolecules-13-01678" rid="B61-biomolecules-13-01678" class=" bibr popnode">61</a>]. However, further experiments need to be conducted to confirm the role and relation of these compounds with the IGF1R target of kidney inflammation and kidney cancer.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="biomolecules-13-01678-t004"><h3>Table 4</h3><!--caption a7--><div class="caption"><p>Final targets, compounds, and microbial sources for kidney cancer and inflammation-related diseases.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Final Target Gene</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Gene Targeted Compounds and ID</th><th align="center" valign="middle" style="border-top:solid thin;border-bottom:solid thin" rowspan="1" colspan="1">Target Microbes [<a href="#B19-biomolecules-13-01678" rid="B19-biomolecules-13-01678" class=" bibr popnode">19</a>]</th></tr></thead><tbody><tr><td rowspan="2" align="center" valign="middle" style="border-bottom:solid thin" colspan="1">MTOR</td><td align="left" valign="middle" rowspan="1" colspan="1">11-Methoxycurvularin (10381440)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacillus</em> sp.</td></tr><tr><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Glycocholic acid (10140, HMDB0000138, CHEBI:17687)</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><em>Bacteroides fragilis</em>, <em>Butyricicoccus pullicaecorum</em>, <em>Ruminococcus flavefaciens.</em></td></tr><tr><td rowspan="3" align="center" valign="middle" style="border-bottom:solid thin" colspan="1">PIK3CA</td><td align="left" valign="middle" rowspan="1" colspan="1">11-Methoxycurvularin (10381440)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacillus</em> sp.</td></tr><tr><td align="left" valign="middle" rowspan="1" colspan="1">Glycocholic acid (10140, HMDB0000138, CHEBI:17687)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Bacteroides fragilis</em>, <em>Butyricicoccus pullicaecorum</em>, <em>Ruminococcus flavefaciens.</em></td></tr><tr><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Isoquercitrin (5280804, HMDB0037362, CHEBI:68352)</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><em>Bacillus</em> sp., <em>Bacteroides</em> sp., <em>Eubacterium ramulus.</em></td></tr><tr><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">IL6</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Isoquercitrin (5280804, HMDB0037362, CHEBI:68352)</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><em>Bacillus</em> sp., <em>Bacteroides</em> sp., <em>Eubacterium ramulus.</em></td></tr><tr><td align="center" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">PTGS2</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Isoquercitrin (5280804, HMDB0037362, CHEBI:68352)</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><em>Bacillus</em> sp., <em>Bacteroides</em> sp., <em>Eubacterium ramulus.</em></td></tr><tr><td rowspan="2" align="center" valign="middle" style="border-bottom:solid thin" colspan="1">IGF1R</td><td align="left" valign="middle" rowspan="1" colspan="1">2-Amino-1-methyl-6-phenylimidazo[4,5-b] pyridine (1530, CHEBI:76290)</td><td align="left" valign="middle" rowspan="1" colspan="1"><em>Blautia obeum</em>, <em>Faecalibacterium prausnitzii</em>, <em>Lactobacillus reuteri.</em></td></tr><tr><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1">Isoquercitrin (5280804, HMDB0037362, CHEBI:68352)</td><td align="left" valign="middle" style="border-bottom:solid thin" rowspan="1" colspan="1"><em>Bacillus</em> sp., <em>Bacteroides</em> sp., <em>Eubacterium ramulus.</em></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140445104347440"><a target="object" rel="noopener" href="/pmc/articles/PMC10669250/table/biomolecules-13-01678-t004/?report=objectonly">Open in a separate window</a></div></div></div><div id="sec5-biomolecules-13-01678" class="tsec sec"><h2 class="head no_bottom_margin" id="sec5-biomolecules-13-01678title">5. Conclusions</h2><p class="p p-first">Our findings imply that a variety of microorganisms can offer crucial metabolites against kidney cancer and disorders linked to inflammation. We found four typical microbial metabolites along with five gene targets related to kidney cancer and inflammatory diseases. SCFA-producing microbial metabolites, such as isoquercitrin, 11-methoxycurvularin, and glycocholic acid, are involved in regulating kidney cancer and inflammation-related diseases, and could be useful as postbiotics. Moreover, isoquercitrin with PIK3CA, IL6, PTGS2, and IGF1R gene targets; 11-methoxycurvularin; and glycocholic acid with MTOR and PIK3CA gene targets are all involved in controlling kidney cancer and diseases related to kidney inflammation.</p><p class="p p-last">However, there are certain limitations on the amount of information that has been assembled on the microbiome. Because of the limitations of bioinformatics and chem-informatics, we propose conducting additional preclinical or clinical experiments to confirm these findings.</p></div><div id="ack-a.t.a" class="tsec sec"><h2 class="head no_bottom_margin" id="ack-a.t.atitle">Acknowledgments</h2><div class="sec"><p>We appreciate the work of the teams of Hanbangbio Inc., Yong-si 17104, Gyeonggi-do, Republic of Korea. We also thanked ABIOME, 6, Jeonmin-ro 30 beon-gil, Yuseong-gu 34052, Daejeon, Republic of Korea for supporting our study.</p></div></div><div id="app1-biomolecules-13-01678" class="tsec sec"><h2 class="head no_bottom_margin" id="app1-biomolecules-13-01678title">Supplementary Materials</h2><!--/article/back/app-group/app/--><p>The following supporting information can be downloaded at: <a href="https://www.mdpi.com/article/10.3390/biom13111678/s1">https://www.mdpi.com/article/10.3390/biom13111678/s1</a>. Supplementary S1. SCFAs microbe&#x02019;s database (compounds and targets). Supplementary S2: Kidney cancer and inflammation database (diseases targets). Supplementary S3: GeneMANIA analysis result (suitable target genes). Supplementary S4: Molecular docking simulation results (between all selected compounds and targets). Supplementary S5: GO and KEGG pathway analysis. Supplementary S6: PPI network analysis. Top 20 target selection. Supplementary S7: ADME properties of selected compounds. Supplementary Figure S1: Molecular docking analysis of selected compounds and targets showing different bond attractions between the compounds and targets.</p><div class="sec suppmat" id="biomolecules-13-01678-s001"><div class="sup-box half_rhythm" id="media-a.t.c.a.c.a"><a href="/pmc/articles/PMC10669250/bin/biomolecules-13-01678-s001.zip" data-ga-action="click_feat_suppl">Click here for additional data file.</a><sup>(4.9M, zip)</sup></div></div></div><div id="funding-group-a.r.b.x" class="tsec sec"><h2 class="head no_bottom_margin" id="funding-group-a.r.b.xtitle">Funding Statement</h2><p>This research received no external funding.</p></div><div id="notes-a.t.d" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.t.dtitle">Author Contributions</h2><p>M.R.K.: data analysis, analysis design, and writing the original manuscript; M.N.M.: data analysis and writing the original manuscript; S.I.: editing and review; S.M.: review; R.M.: editing; D.C.Y.: review and editing; Y.J.K.: grammar check, review, and editing, J.H.S.: review, supervision, and editing; D.U.Y.: review and supervision. All authors have read and agreed to the published version of the manuscript.</p></div><div id="notes-a.t.e" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.t.etitle">Institutional Review Board Statement</h2><p>Not applicable.</p></div><div id="notes-a.t.f" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.t.ftitle">Informed Consent Statement</h2><p>Not applicable.</p></div><div id="notes-a.t.g" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.t.gtitle">Data Availability Statement</h2><p>All data generated or analyzed during this study are included in this published article (and its <a href="#app1-biomolecules-13-01678" rid="app1-biomolecules-13-01678" class=" app">Supplementary Information files</a>).</p></div><div id="notes-a.t.h" class="tsec sec"><h2 class="head no_bottom_margin" id="notes-a.t.htitle">Conflicts of Interest</h2><p>We appreciate the work of the teams of Hanbangbio Inc., Yong-si 17104, Gyeonggi-do, Republic of Korea.</p></div><div id="fn-group-a.t.b" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.t.btitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="fn-a.t.b.a"><p class="p p-first-last"><strong>Disclaimer/Publisher&#x02019;s Note:</strong> The statements, opinions and data contained in all publications are solely those of the individual author(s) and contributor(s) and not of MDPI and/or the editor(s). MDPI and/or the editor(s) disclaim responsibility for any injury to people or property resulting from any ideas, methods, instructions or products referred to in the content.</p></p></div></div><div id="ref-list-a.t.i" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.t.ititle">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="B1-biomolecules-13-01678">1. <span class="element-citation">Shi J., Wang K., Xiong Z., Yuan C., Wang C., Cao Q., Yu H., Meng X., Xie K., Cheng Z., et al. Impact of inflammation and immunotherapy in renal cell carcinoma. <span><span class="ref-journal">Oncol. Lett. </span>2020;<span class="ref-vol">20</span>:272. doi: 10.3892/ol.2020.12135.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7520756/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33014151" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3892%2Fol.2020.12135" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.+Lett.&amp;title=Impact+of+inflammation+and+immunotherapy+in+renal+cell+carcinoma&amp;author=J.+Shi&amp;author=K.+Wang&amp;author=Z.+Xiong&amp;author=C.+Yuan&amp;author=C.+Wang&amp;volume=20&amp;publication_year=2020&amp;pages=272&amp;pmid=33014151&amp;doi=10.3892/ol.2020.12135&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B2-biomolecules-13-01678">2. <span class="element-citation">Sabarwal A., Kumar K., Singh R.P. Hazardous effects of chemical pesticides on human health-Cancer and other associated disorders. <span><span class="ref-journal">Environ. Toxicol. Pharmacol. </span>2018;<span class="ref-vol">63</span>:103–114. doi: 10.1016/j.etap.2018.08.018.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30199797" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.etap.2018.08.018" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Environ.+Toxicol.+Pharmacol.&amp;title=Hazardous+effects+of+chemical+pesticides+on+human+health-Cancer+and+other+associated+disorders&amp;author=A.+Sabarwal&amp;author=K.+Kumar&amp;author=R.P.+Singh&amp;volume=63&amp;publication_year=2018&amp;pages=103-114&amp;pmid=30199797&amp;doi=10.1016/j.etap.2018.08.018&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B3-biomolecules-13-01678">3. <span class="element-citation">Chhikara B.S., Parang K. Global Cancer Statistics 2022: The trends projection analysis. <span><span class="ref-journal">Chem. Biol. Lett. </span>2022;<span class="ref-vol">10</span>:451 </span> <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Chem.+Biol.+Lett.&amp;title=Global+Cancer+Statistics+2022:+The+trends+projection+analysis&amp;author=B.S.+Chhikara&amp;author=K.+Parang&amp;volume=10&amp;publication_year=2022&amp;pages=451&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B4-biomolecules-13-01678">4. <span class="element-citation">Galdiero M.R., Marone G., Mantovani A. Cancer Inflammation and Cytokines. <span><span class="ref-journal">Cold Spring Harb. Perspect Biol. </span>2018;<span class="ref-vol">10</span>:a028662.  doi: 10.1101/cshperspect.a028662.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6071493/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28778871" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fcshperspect.a028662" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cold+Spring+Harb.+Perspect+Biol.&amp;title=Cancer+Inflammation+and+Cytokines&amp;author=M.R.+Galdiero&amp;author=G.+Marone&amp;author=A.+Mantovani&amp;volume=10&amp;publication_year=2018&amp;pages=a028662&amp;pmid=28778871&amp;doi=10.1101/cshperspect.a028662&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B5-biomolecules-13-01678">5. <span class="element-citation">Kay J., Thadhani E., Samson L., Engelward B. Inflammation-induced DNA damage, mutations and cancer. <span><span class="ref-journal">DNA Repair. </span>2019;<span class="ref-vol">83</span>:102673. doi: 10.1016/j.dnarep.2019.102673.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6801086/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31387777" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.dnarep.2019.102673" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=DNA+Repair&amp;title=Inflammation-induced+DNA+damage,+mutations+and+cancer&amp;author=J.+Kay&amp;author=E.+Thadhani&amp;author=L.+Samson&amp;author=B.+Engelward&amp;volume=83&amp;publication_year=2019&amp;pages=102673&amp;pmid=31387777&amp;doi=10.1016/j.dnarep.2019.102673&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B6-biomolecules-13-01678">6. <span class="element-citation">Ha H., Debnath B., Neamati N. Role of the CXCL8-CXCR1/2 Axis in Cancer and Inflammatory Diseases. <span><span class="ref-journal">Theranostics. </span>2017;<span class="ref-vol">7</span>:1543–1588. doi: 10.7150/thno.15625.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5436513/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28529637" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7150%2Fthno.15625" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Theranostics&amp;title=Role+of+the+CXCL8-CXCR1/2+Axis+in+Cancer+and+Inflammatory+Diseases&amp;author=H.+Ha&amp;author=B.+Debnath&amp;author=N.+Neamati&amp;volume=7&amp;publication_year=2017&amp;pages=1543-1588&amp;pmid=28529637&amp;doi=10.7150/thno.15625&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B7-biomolecules-13-01678">7. <span class="element-citation">Brighi N., Farolfi A., Conteduca V., Gurioli G., Gargiulo S., Gall&#x000e0; V., Schepisi G., Lolli C., Casadei C., De Giorgi U. The Interplay between Inflammation, Anti-Angiogenic Agents, and Immune Checkpoint Inhibitors: Perspectives for Renal Cell Cancer Treatment. <span><span class="ref-journal">Cancers. </span>2019;<span class="ref-vol">11</span>:1935.  doi: 10.3390/cancers11121935.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6966461/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31817109" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fcancers11121935" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancers&amp;title=The+Interplay+between+Inflammation,+Anti-Angiogenic+Agents,+and+Immune+Checkpoint+Inhibitors:+Perspectives+for+Renal+Cell+Cancer+Treatment&amp;author=N.+Brighi&amp;author=A.+Farolfi&amp;author=V.+Conteduca&amp;author=G.+Gurioli&amp;author=S.+Gargiulo&amp;volume=11&amp;publication_year=2019&amp;pages=1935&amp;pmid=31817109&amp;doi=10.3390/cancers11121935&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B8-biomolecules-13-01678">8. <span class="element-citation">Nakamura K., Smyth M.J. Targeting cancer-related inflammation in the era of immunotherapy. <span><span class="ref-journal">Immunol. Cell Biol. </span>2017;<span class="ref-vol">95</span>:325–332. doi: 10.1038/icb.2016.126.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27999432" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Ficb.2016.126" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Immunol.+Cell+Biol.&amp;title=Targeting+cancer-related+inflammation+in+the+era+of+immunotherapy&amp;author=K.+Nakamura&amp;author=M.J.+Smyth&amp;volume=95&amp;publication_year=2017&amp;pages=325-332&amp;pmid=27999432&amp;doi=10.1038/icb.2016.126&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B9-biomolecules-13-01678">9. <span class="element-citation">Hammers H.J., Plimack E.R., Infante J.R., Rini B.I., McDermott D.F., Lewis L.D., Voss M.H., Sharma P., Pal S.K., Razak A.R.A., et al. Safety and Efficacy of Nivolumab in Combination with Ipilimumab in Metastatic Renal Cell Carcinoma: The CheckMate 016 Study. <span><span class="ref-journal">J. Clin. Oncol. </span>2017;<span class="ref-vol">35</span>:3851–3858. doi: 10.1200/JCO.2016.72.1985.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7587408/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28678668" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1200%2FJCO.2016.72.1985" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Clin.+Oncol.&amp;title=Safety+and+Efficacy+of+Nivolumab+in+Combination+with+Ipilimumab+in+Metastatic+Renal+Cell+Carcinoma:+The+CheckMate+016+Study&amp;author=H.J.+Hammers&amp;author=E.R.+Plimack&amp;author=J.R.+Infante&amp;author=B.I.+Rini&amp;author=D.F.+McDermott&amp;volume=35&amp;publication_year=2017&amp;pages=3851-3858&amp;pmid=28678668&amp;doi=10.1200/JCO.2016.72.1985&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B10-biomolecules-13-01678">10. <span class="element-citation">Hsu C.N., Tain Y.L. Chronic Kidney Disease and Gut Microbiota: What Is Their Connection in Early Life? <span><span class="ref-journal">Int. J. Mol. Sci. </span>2022;<span class="ref-vol">23</span>:3954.  doi: 10.3390/ijms23073954.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9000069/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35409313" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fijms23073954" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Mol.+Sci.&amp;title=Chronic+Kidney+Disease+and+Gut+Microbiota:+What+Is+Their+Connection+in+Early+Life?&amp;author=C.N.+Hsu&amp;author=Y.L.+Tain&amp;volume=23&amp;publication_year=2022&amp;pages=3954&amp;pmid=35409313&amp;doi=10.3390/ijms23073954&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B11-biomolecules-13-01678">11. <span class="element-citation">Yang T., Richards E.M., Pepine C.J., Raizada M.K. The gut microbiota and the brain-gut-kidney axis in hypertension and chronic kidney disease. <span><span class="ref-journal">Nat. Rev. Nephrol. </span>2018;<span class="ref-vol">14</span>:442–456. doi: 10.1038/s41581-018-0018-2.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6385605/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29760448" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41581-018-0018-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Nephrol.&amp;title=The+gut+microbiota+and+the+brain-gut-kidney+axis+in+hypertension+and+chronic+kidney+disease&amp;author=T.+Yang&amp;author=E.M.+Richards&amp;author=C.J.+Pepine&amp;author=M.K.+Raizada&amp;volume=14&amp;publication_year=2018&amp;pages=442-456&amp;pmid=29760448&amp;doi=10.1038/s41581-018-0018-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B12-biomolecules-13-01678">12. <span class="element-citation">Hobby G.P., Karaduta O., Dusio G.F., Singh M., Zybailov B.L., Arthur J.M. Chronic kidney disease and the gut microbiome. <span><span class="ref-journal">Am. J. Physiol. Renal Physiol. </span>2019;<span class="ref-vol">316</span>:F1211–F1217. doi: 10.1152/ajprenal.00298.2018.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6620595/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30864840" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1152%2Fajprenal.00298.2018" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Physiol.+Renal+Physiol.&amp;title=Chronic+kidney+disease+and+the+gut+microbiome&amp;author=G.P.+Hobby&amp;author=O.+Karaduta&amp;author=G.F.+Dusio&amp;author=M.+Singh&amp;author=B.L.+Zybailov&amp;volume=316&amp;publication_year=2019&amp;pages=F1211-F1217&amp;pmid=30864840&amp;doi=10.1152/ajprenal.00298.2018&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B13-biomolecules-13-01678">13. <span class="element-citation">Al Khodor S., Shatat I.F. Gut microbiome and kidney disease: A bidirectional relationship. <span><span class="ref-journal">Pediatr. Nephrol. </span>2017;<span class="ref-vol">32</span>:921–931. doi: 10.1007/s00467-016-3392-7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5399049/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27129691" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00467-016-3392-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr.+Nephrol.&amp;title=Gut+microbiome+and+kidney+disease:+A+bidirectional+relationship&amp;author=S.+Al+Khodor&amp;author=I.F.+Shatat&amp;volume=32&amp;publication_year=2017&amp;pages=921-931&amp;pmid=27129691&amp;doi=10.1007/s00467-016-3392-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B14-biomolecules-13-01678">14. <span class="element-citation">Li L., Ma L., Fu P. Gut microbiota-derived short-chain fatty acids and kidney diseases. <span><span class="ref-journal">Drug Des. Devel. Ther. </span>2017;<span class="ref-vol">11</span>:3531–3542. doi: 10.2147/DDDT.S150825.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5729884/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29270002" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2147%2FDDDT.S150825" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Drug+Des.+Devel.+Ther.&amp;title=Gut+microbiota-derived+short-chain+fatty+acids+and+kidney+diseases&amp;author=L.+Li&amp;author=L.+Ma&amp;author=P.+Fu&amp;volume=11&amp;publication_year=2017&amp;pages=3531-3542&amp;doi=10.2147/DDDT.S150825&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B15-biomolecules-13-01678">15. <span class="element-citation">Diaz Heijtz R., Wang S., Anuar F., Qian Y., Bj&#x000f6;rkholm B., Samuelsson A., Hibberd M.L., Forssberg H., Pettersson S. Normal gut microbiota modulates brain development and behavior. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2011;<span class="ref-vol">108</span>:3047–3052. doi: 10.1073/pnas.1010529108.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3041077/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21282636" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1010529108" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Normal+gut+microbiota+modulates+brain+development+and+behavior&amp;author=R.+Diaz+Heijtz&amp;author=S.+Wang&amp;author=F.+Anuar&amp;author=Y.+Qian&amp;author=B.+Bj&#x000f6;rkholm&amp;volume=108&amp;publication_year=2011&amp;pages=3047-3052&amp;pmid=21282636&amp;doi=10.1073/pnas.1010529108&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B16-biomolecules-13-01678">16. <span class="element-citation">Vijay-Kumar M., Aitken J.D., Carvalho F.A., Cullender T.C., Mwangi S., Srinivasan S., Sitaraman S.V., Knight R., Ley R.E., Gewirtz A.T. Metabolic syndrome and altered gut microbiota in mice lacking Toll-like receptor 5. <span><span class="ref-journal">Science. </span>2010;<span class="ref-vol">328</span>:228–231. doi: 10.1126/science.1179721.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4714868/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20203013" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.1179721" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=Metabolic+syndrome+and+altered+gut+microbiota+in+mice+lacking+Toll-like+receptor+5&amp;author=M.+Vijay-Kumar&amp;author=J.D.+Aitken&amp;author=F.A.+Carvalho&amp;author=T.C.+Cullender&amp;author=S.+Mwangi&amp;volume=328&amp;publication_year=2010&amp;pages=228-231&amp;pmid=20203013&amp;doi=10.1126/science.1179721&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B17-biomolecules-13-01678">17. <span class="element-citation">Uronis J.M., M&#x000fc;hlbauer M., Herfarth H.H., Rubinas T.C., Jones G.S., Jobin C. Modulation of the intestinal microbiota alters colitis-associated colorectal cancer susceptibility. <span><span class="ref-journal">PLoS ONE. </span>2009;<span class="ref-vol">4</span>:e6026.  doi: 10.1371/journal.pone.0006026.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2696084/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19551144" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0006026" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Modulation+of+the+intestinal+microbiota+alters+colitis-associated+colorectal+cancer+susceptibility&amp;author=J.M.+Uronis&amp;author=M.+M&#x000fc;hlbauer&amp;author=H.H.+Herfarth&amp;author=T.C.+Rubinas&amp;author=G.S.+Jones&amp;volume=4&amp;publication_year=2009&amp;pages=e6026&amp;pmid=19551144&amp;doi=10.1371/journal.pone.0006026&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B18-biomolecules-13-01678">18. <span class="element-citation">De Filippo C., Cavalieri D., Di Paola M., Ramazzotti M., Poullet J.B., Massart S., Collini S., Pieraccini G., Lionetti P. Impact of diet in shaping gut microbiota revealed by a comparative study in children from Europe and rural Africa. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2010;<span class="ref-vol">107</span>:14691–14696. doi: 10.1073/pnas.1005963107.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2930426/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20679230" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1005963107" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Impact+of+diet+in+shaping+gut+microbiota+revealed+by+a+comparative+study+in+children+from+Europe+and+rural+Africa&amp;author=C.+De+Filippo&amp;author=D.+Cavalieri&amp;author=M.+Di+Paola&amp;author=M.+Ramazzotti&amp;author=J.B.+Poullet&amp;volume=107&amp;publication_year=2010&amp;pages=14691-14696&amp;pmid=20679230&amp;doi=10.1073/pnas.1005963107&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B19-biomolecules-13-01678">19. <span class="element-citation">Cheng L., Qi C., Yang H., Lu M., Cai Y., Fu T., Ren J., Jin Q., Zhang X. gutMGene: A comprehensive database for target genes of gut microbes and microbial metabolites. <span><span class="ref-journal">Nucleic Acids Res. </span>2022;<span class="ref-vol">50</span>:D795–D800. doi: 10.1093/nar/gkab786.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8728193/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34500458" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkab786" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=gutMGene:+A+comprehensive+database+for+target+genes+of+gut+microbes+and+microbial+metabolites&amp;author=L.+Cheng&amp;author=C.+Qi&amp;author=H.+Yang&amp;author=M.+Lu&amp;author=Y.+Cai&amp;volume=50&amp;publication_year=2022&amp;pages=D795-D800&amp;pmid=34500458&amp;doi=10.1093/nar/gkab786&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B20-biomolecules-13-01678">20. <span class="element-citation">Pi&#x000f1;ero J., Ram&#x000ed;rez-Anguita J.M., Sa&#x000fc;ch-Pitarch J., Ronzano F., Centeno E., Sanz F., Furlong L.I. The DisGeNET knowledge platform for disease genomics: 2019 update. <span><span class="ref-journal">Nucleic Acids Res. </span>2020;<span class="ref-vol">48</span>:D845–D855. doi: 10.1093/nar/gkz1021.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7145631/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31680165" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkz1021" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+DisGeNET+knowledge+platform+for+disease+genomics:+2019+update&amp;author=J.+Pi&#x000f1;ero&amp;author=J.M.+Ram&#x000ed;rez-Anguita&amp;author=J.+Sa&#x000fc;ch-Pitarch&amp;author=F.+Ronzano&amp;author=E.+Centeno&amp;volume=48&amp;publication_year=2020&amp;pages=D845-D855&amp;pmid=31680165&amp;doi=10.1093/nar/gkz1021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B21-biomolecules-13-01678">21. <span class="element-citation">Hamosh A., Scott A.F., Amberger J.S., Bocchini C.A., McKusick V.A. Online Mendelian Inheritance in Man (OMIM), a knowledgebase of human genes and genetic disorders. <span><span class="ref-journal">Nucleic Acids Res. </span>2005;<span class="ref-vol">33</span>:D514–D517. doi: 10.1093/nar/gki033.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC539987/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15608251" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgki033" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Online+Mendelian+Inheritance+in+Man+(OMIM),+a+knowledgebase+of+human+genes+and+genetic+disorders&amp;author=A.+Hamosh&amp;author=A.F.+Scott&amp;author=J.S.+Amberger&amp;author=C.A.+Bocchini&amp;author=V.A.+McKusick&amp;volume=33&amp;publication_year=2005&amp;pages=D514-D517&amp;pmid=15608251&amp;doi=10.1093/nar/gki033&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B22-biomolecules-13-01678">22. <span class="element-citation">Stelzer G., Rosen N., Plaschkes I., Zimmerman S., Twik M., Fishilevich S., Stein T.I., Nudel R., Lieder I., Mazor Y., et al. The GeneCards Suite: From Gene Data Mining to Disease Genome Sequence Analyses. <span><span class="ref-journal">Curr. Protoc. Bioinform. </span>2016;<span class="ref-vol">54</span>:1.30.31–31.30.33. doi: 10.1002/cpbi.5.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27322403" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fcpbi.5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Protoc.+Bioinform.&amp;title=The+GeneCards+Suite:+From+Gene+Data+Mining+to+Disease+Genome+Sequence+Analyses&amp;author=G.+Stelzer&amp;author=N.+Rosen&amp;author=I.+Plaschkes&amp;author=S.+Zimmerman&amp;author=M.+Twik&amp;volume=54&amp;publication_year=2016&amp;pages=1.30.31-31.30.33&amp;doi=10.1002/cpbi.5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B23-biomolecules-13-01678">23. <span class="element-citation">Warde-Farley D., Donaldson S.L., Comes O., Zuberi K., Badrawi R., Chao P., Franz M., Grouios C., Kazi F., Lopes C.T., et al. The GeneMANIA prediction server: Biological network integration for gene prioritization and predicting gene function. <span><span class="ref-journal">Nucleic Acids Res. </span>2010;<span class="ref-vol">38</span>:W214–W220. doi: 10.1093/nar/gkq537.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2896186/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20576703" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkq537" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+GeneMANIA+prediction+server:+Biological+network+integration+for+gene+prioritization+and+predicting+gene+function&amp;author=D.+Warde-Farley&amp;author=S.L.+Donaldson&amp;author=O.+Comes&amp;author=K.+Zuberi&amp;author=R.+Badrawi&amp;volume=38&amp;publication_year=2010&amp;pages=W214-W220&amp;pmid=20576703&amp;doi=10.1093/nar/gkq537&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B24-biomolecules-13-01678">24. <span class="element-citation">Ge S.X., Jung D., Yao R. ShinyGO: A graphical gene-set enrichment tool for animals and plants. <span><span class="ref-journal">Bioinformatics. </span>2020;<span class="ref-vol">36</span>:2628–2629. doi: 10.1093/bioinformatics/btz931.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7178415/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31882993" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtz931" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=ShinyGO:+A+graphical+gene-set+enrichment+tool+for+animals+and+plants&amp;author=S.X.+Ge&amp;author=D.+Jung&amp;author=R.+Yao&amp;volume=36&amp;publication_year=2020&amp;pages=2628-2629&amp;pmid=31882993&amp;doi=10.1093/bioinformatics/btz931&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B25-biomolecules-13-01678">25. <span class="element-citation">Zhou Y., Zhou B., Pache L., Chang M., Khodabakhshi A.H., Tanaseichuk O., Benner C., Chanda S.K. Metascape provides a biologist-oriented resource for the analysis of systems-level datasets. <span><span class="ref-journal">Nat. Commun. </span>2019;<span class="ref-vol">10</span>:1523. doi: 10.1038/s41467-019-09234-6.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6447622/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30944313" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-019-09234-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Metascape+provides+a+biologist-oriented+resource+for+the+analysis+of+systems-level+datasets&amp;author=Y.+Zhou&amp;author=B.+Zhou&amp;author=L.+Pache&amp;author=M.+Chang&amp;author=A.H.+Khodabakhshi&amp;volume=10&amp;publication_year=2019&amp;pages=1523&amp;pmid=30944313&amp;doi=10.1038/s41467-019-09234-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B26-biomolecules-13-01678">26. <span class="element-citation">Shi L., Yu L., Zou F., Hu H., Liu K., Lin Z. Gene expression profiling and functional analysis reveals that p53 pathway-related gene expression is highly activated in cancer cells treated by cold atmospheric plasma-activated medium. <span><span class="ref-journal">PeerJ. </span>2017;<span class="ref-vol">5</span>:e3751. doi: 10.7717/peerj.3751.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5572956/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28852598" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7717%2Fpeerj.3751" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Gene+expression+profiling+and+functional+analysis+reveals+that+p53+pathway-related+gene+expression+is+highly+activated+in+cancer+cells+treated+by+cold+atmospheric+plasma-activated+medium&amp;author=L.+Shi&amp;author=L.+Yu&amp;author=F.+Zou&amp;author=H.+Hu&amp;author=K.+Liu&amp;volume=5&amp;publication_year=2017&amp;pages=e3751&amp;pmid=28852598&amp;doi=10.7717/peerj.3751&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B27-biomolecules-13-01678">27. <span class="element-citation">Mi H., Muruganujan A., Huang X., Ebert D., Mills C., Guo X., Thomas P.D. Protocol Update for large-scale genome and gene function analysis with the PANTHER classification system (v.14.0) <span><span class="ref-journal">Nat. Protoc. </span>2019;<span class="ref-vol">14</span>:703–721. doi: 10.1038/s41596-019-0128-8.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6519457/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30804569" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41596-019-0128-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Protoc.&amp;title=Protocol+Update+for+large-scale+genome+and+gene+function+analysis+with+the+PANTHER+classification+system+(v.14.0)&amp;author=H.+Mi&amp;author=A.+Muruganujan&amp;author=X.+Huang&amp;author=D.+Ebert&amp;author=C.+Mills&amp;volume=14&amp;publication_year=2019&amp;pages=703-721&amp;pmid=30804569&amp;doi=10.1038/s41596-019-0128-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B28-biomolecules-13-01678">28. <span class="element-citation">Siki&#x00107; M., Tomi&#x00107; S., Vlahovicek K. Prediction of protein-protein interaction sites in sequences and 3D structures by random forests. <span><span class="ref-journal">PLoS Comput. Biol. </span>2009;<span class="ref-vol">5</span>:e1000278.  doi: 10.1371/journal.pcbi.1000278.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2621338/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19180183" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pcbi.1000278" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+Comput.+Biol.&amp;title=Prediction+of+protein-protein+interaction+sites+in+sequences+and+3D+structures+by+random+forests&amp;author=M.+Siki&#x00107;&amp;author=S.+Tomi&#x00107;&amp;author=K.+Vlahovicek&amp;volume=5&amp;publication_year=2009&amp;pages=e1000278&amp;pmid=19180183&amp;doi=10.1371/journal.pcbi.1000278&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B29-biomolecules-13-01678">29. <span class="element-citation">Rao V.S., Srinivas K., Sujini G.N., Kumar G.N. Protein-protein interaction detection: Methods and analysis. <span><span class="ref-journal">Int. J. Proteom. </span>2014;<span class="ref-vol">2014</span>:147648. doi: 10.1155/2014/147648.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3947875/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24693427" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1155%2F2014%2F147648" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Proteom.&amp;title=Protein-protein+interaction+detection:+Methods+and+analysis&amp;author=V.S.+Rao&amp;author=K.+Srinivas&amp;author=G.N.+Sujini&amp;author=G.N.+Kumar&amp;volume=2014&amp;publication_year=2014&amp;pages=147648&amp;doi=10.1155/2014/147648&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B30-biomolecules-13-01678">30. <span class="element-citation">Zhou G., Soufan O., Ewald J., Hancock R.E.W., Basu N., Xia J. NetworkAnalyst 3.0: A visual analytics platform for comprehensive gene expression profiling and meta-analysis. <span><span class="ref-journal">Nucleic Acids Res. </span>2019;<span class="ref-vol">47</span>:W234–W241. doi: 10.1093/nar/gkz240.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6602507/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30931480" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkz240" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=NetworkAnalyst+3.0:+A+visual+analytics+platform+for+comprehensive+gene+expression+profiling+and+meta-analysis&amp;author=G.+Zhou&amp;author=O.+Soufan&amp;author=J.+Ewald&amp;author=R.E.W.+Hancock&amp;author=N.+Basu&amp;volume=47&amp;publication_year=2019&amp;pages=W234-W241&amp;pmid=30931480&amp;doi=10.1093/nar/gkz240&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B31-biomolecules-13-01678">31. <span class="element-citation">Tang Y., Li M., Wang J., Pan Y., Wu F.X. CytoNCA: A cytoscape plugin for centrality analysis and evaluation of protein interaction networks. <span><span class="ref-journal">Biosystems. </span>2015;<span class="ref-vol">127</span>:67–72. doi: 10.1016/j.biosystems.2014.11.005.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25451770" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.biosystems.2014.11.005" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biosystems&amp;title=CytoNCA:+A+cytoscape+plugin+for+centrality+analysis+and+evaluation+of+protein+interaction+networks&amp;author=Y.+Tang&amp;author=M.+Li&amp;author=J.+Wang&amp;author=Y.+Pan&amp;author=F.X.+Wu&amp;volume=127&amp;publication_year=2015&amp;pages=67-72&amp;pmid=25451770&amp;doi=10.1016/j.biosystems.2014.11.005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B32-biomolecules-13-01678">32. <span class="element-citation">Daina A., Michielin O., Zoete V. SwissADME: A free web tool to evaluate pharmacokinetics, drug-likeness and medicinal chemistry friendliness of small molecules. <span><span class="ref-journal">Sci. Rep. </span>2017;<span class="ref-vol">7</span>:42717. doi: 10.1038/srep42717.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5335600/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28256516" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fsrep42717" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=SwissADME:+A+free+web+tool+to+evaluate+pharmacokinetics,+drug-likeness+and+medicinal+chemistry+friendliness+of+small+molecules&amp;author=A.+Daina&amp;author=O.+Michielin&amp;author=V.+Zoete&amp;volume=7&amp;publication_year=2017&amp;pages=42717&amp;pmid=28256516&amp;doi=10.1038/srep42717&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B33-biomolecules-13-01678">33. <span class="element-citation">Xiong G., Wu Z., Yi J., Fu L., Yang Z., Hsieh C., Yin M., Zeng X., Wu C., Lu A., et al. ADMETlab 2.0: An integrated online platform for accurate and comprehensive predictions of ADMET properties. <span><span class="ref-journal">Nucleic Acids Res. </span>2021;<span class="ref-vol">49</span>:W5–W14. doi: 10.1093/nar/gkab255.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8262709/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33893803" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkab255" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=ADMETlab+2.0:+An+integrated+online+platform+for+accurate+and+comprehensive+predictions+of+ADMET+properties&amp;author=G.+Xiong&amp;author=Z.+Wu&amp;author=J.+Yi&amp;author=L.+Fu&amp;author=Z.+Yang&amp;volume=49&amp;publication_year=2021&amp;pages=W5-W14&amp;pmid=33893803&amp;doi=10.1093/nar/gkab255&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B34-biomolecules-13-01678">34. <span class="element-citation">Pires D.E.V., Blundell T.L., Ascher D.B. pkCSM: Predicting Small-Molecule Pharmacokinetic and Toxicity Properties Using Graph-Based Signatures. <span><span class="ref-journal">J. Med. Chem. </span>2015;<span class="ref-vol">58</span>:4066–4072. doi: 10.1021/acs.jmedchem.5b00104.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4434528/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25860834" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.jmedchem.5b00104" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=pkCSM:+Predicting+Small-Molecule+Pharmacokinetic+and+Toxicity+Properties+Using+Graph-Based+Signatures&amp;author=D.E.V.+Pires&amp;author=T.L.+Blundell&amp;author=D.B.+Ascher&amp;volume=58&amp;publication_year=2015&amp;pages=4066-4072&amp;pmid=25860834&amp;doi=10.1021/acs.jmedchem.5b00104&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B35-biomolecules-13-01678">35. <span class="element-citation">Banerjee P., Eckert A.O., Schrey A.K., Preissner R. ProTox-II: A webserver for the prediction of toxicity of chemicals. <span><span class="ref-journal">Nucleic Acids Res. </span>2018;<span class="ref-vol">46</span>:W257–W263. doi: 10.1093/nar/gky318.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6031011/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29718510" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgky318" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=ProTox-II:+A+webserver+for+the+prediction+of+toxicity+of+chemicals&amp;author=P.+Banerjee&amp;author=A.O.+Eckert&amp;author=A.K.+Schrey&amp;author=R.+Preissner&amp;volume=46&amp;publication_year=2018&amp;pages=W257-W263&amp;pmid=29718510&amp;doi=10.1093/nar/gky318&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B36-biomolecules-13-01678">36. <span class="element-citation">Benet L.Z., Hosey C.M., Ursu O., Oprea T.I. BDDCS, the Rule of 5 and drugability. <span><span class="ref-journal">Adv. Drug Deliv. Rev. </span>2016;<span class="ref-vol">101</span>:89–98. doi: 10.1016/j.addr.2016.05.007.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4910824/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27182629" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.addr.2016.05.007" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv.+Drug+Deliv.+Rev.&amp;title=BDDCS,+the+Rule+of+5+and+drugability&amp;author=L.Z.+Benet&amp;author=C.M.+Hosey&amp;author=O.+Ursu&amp;author=T.I.+Oprea&amp;volume=101&amp;publication_year=2016&amp;pages=89-98&amp;pmid=27182629&amp;doi=10.1016/j.addr.2016.05.007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B37-biomolecules-13-01678">37. <span class="element-citation">Thul P.J., Lindskog C. The human protein atlas: A spatial map of the human proteome. <span><span class="ref-journal">Protein Sci. </span>2018;<span class="ref-vol">27</span>:233–244. doi: 10.1002/pro.3307.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5734309/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28940711" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fpro.3307" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Protein+Sci.&amp;title=The+human+protein+atlas:+A+spatial+map+of+the+human+proteome&amp;author=P.J.+Thul&amp;author=C.+Lindskog&amp;volume=27&amp;publication_year=2018&amp;pages=233-244&amp;pmid=28940711&amp;doi=10.1002/pro.3307&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B38-biomolecules-13-01678">38. <span class="element-citation">Pallesen J., Wang N., Corbett K.S., Wrapp D., Kirchdoerfer R.N., Turner H.L., Cottrell C.A., Becker M.M., Wang L., Shi W. Immunogenicity and structures of a rationally designed prefusion MERS-CoV spike antigen. <span><span class="ref-journal">Proc. Natl. Acad. Sci. USA. </span>2017;<span class="ref-vol">114</span>:E7348–E7357. doi: 10.1073/pnas.1707304114.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5584442/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28807998" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1707304114" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Immunogenicity+and+structures+of+a+rationally+designed+prefusion+MERS-CoV+spike+antigen&amp;author=J.+Pallesen&amp;author=N.+Wang&amp;author=K.S.+Corbett&amp;author=D.+Wrapp&amp;author=R.N.+Kirchdoerfer&amp;volume=114&amp;publication_year=2017&amp;pages=E7348-E7357&amp;pmid=28807998&amp;doi=10.1073/pnas.1707304114&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B39-biomolecules-13-01678">39. <span class="element-citation">Opo F.A., Rahman M.M., Ahammad F., Ahmed I., Bhuiyan M.A., Asiri A.M. Structure based pharmacophore modeling, virtual screening, molecular docking and ADMET approaches for identification of natural anti-cancer agents targeting XIAP protein. <span><span class="ref-journal">Sci. Rep. </span>2021;<span class="ref-vol">11</span>:4049. doi: 10.1038/s41598-021-83626-x.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7892887/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33603068" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-021-83626-x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=Structure+based+pharmacophore+modeling,+virtual+screening,+molecular+docking+and+ADMET+approaches+for+identification+of+natural+anti-cancer+agents+targeting+XIAP+protein&amp;author=F.A.+Opo&amp;author=M.M.+Rahman&amp;author=F.+Ahammad&amp;author=I.+Ahmed&amp;author=M.A.+Bhuiyan&amp;volume=11&amp;publication_year=2021&amp;pages=4049&amp;pmid=33603068&amp;doi=10.1038/s41598-021-83626-x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B40-biomolecules-13-01678">40. <span class="element-citation">Zhang J., Liu X., Zhou W., Cheng G., Wu J., Guo S., Jia S., Liu Y., Li B., Zhang X., et al. A bioinformatics investigation into molecular mechanism of Yinzhihuang granules for treating hepatitis B by network pharmacology and molecular docking verification. <span><span class="ref-journal">Sci. Rep. </span>2020;<span class="ref-vol">10</span>:11448. doi: 10.1038/s41598-020-68224-7.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7351787/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32651427" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-020-68224-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep.&amp;title=A+bioinformatics+investigation+into+molecular+mechanism+of+Yinzhihuang+granules+for+treating+hepatitis+B+by+network+pharmacology+and+molecular+docking+verification&amp;author=J.+Zhang&amp;author=X.+Liu&amp;author=W.+Zhou&amp;author=G.+Cheng&amp;author=J.+Wu&amp;volume=10&amp;publication_year=2020&amp;pages=11448&amp;pmid=32651427&amp;doi=10.1038/s41598-020-68224-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B41-biomolecules-13-01678">41. <span class="element-citation">Tian W., Chen C., Lei X., Zhao J., Liang J. CASTp 3.0: Computed atlas of surface topography of proteins. <span><span class="ref-journal">Nucleic Acids Res. </span>2018;<span class="ref-vol">46</span>:W363–W367. doi: 10.1093/nar/gky473.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6031066/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29860391" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgky473" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=CASTp+3.0:+Computed+atlas+of+surface+topography+of+proteins&amp;author=W.+Tian&amp;author=C.+Chen&amp;author=X.+Lei&amp;author=J.+Zhao&amp;author=J.+Liang&amp;volume=46&amp;publication_year=2018&amp;pages=W363-W367&amp;pmid=29860391&amp;doi=10.1093/nar/gky473&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B42-biomolecules-13-01678">42. <span class="element-citation">Kemmish H., Fasnacht M., Yan L. Fully automated antibody structure prediction using BIOVIA tools: Validation study. <span><span class="ref-journal">PLoS ONE. </span>2017;<span class="ref-vol">12</span>:e0177923.  doi: 10.1371/journal.pone.0177923.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5436848/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28542300" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0177923" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+ONE&amp;title=Fully+automated+antibody+structure+prediction+using+BIOVIA+tools:+Validation+study&amp;author=H.+Kemmish&amp;author=M.+Fasnacht&amp;author=L.+Yan&amp;volume=12&amp;publication_year=2017&amp;pages=e0177923&amp;pmid=28542300&amp;doi=10.1371/journal.pone.0177923&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B43-biomolecules-13-01678">43. <span class="element-citation">Dallakyan S., Olson A.J. Small-molecule library screening by docking with PyRx. <span><span class="ref-journal">Methods Mol. Biol. </span>2015;<span class="ref-vol">1263</span>:243–250. doi: 10.1007/978-1-4939-2269-7_19.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25618350" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2F978-1-4939-2269-7_19" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Methods+Mol.+Biol.&amp;title=Small-molecule+library+screening+by+docking+with+PyRx&amp;author=S.+Dallakyan&amp;author=A.J.+Olson&amp;volume=1263&amp;publication_year=2015&amp;pages=243-250&amp;pmid=25618350&amp;doi=10.1007/978-1-4939-2269-7_19&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B44-biomolecules-13-01678">44. <span class="element-citation">Trott O., Olson A.J. AutoDock Vina: Improving the speed and accuracy of docking with a new scoring function, efficient optimization, and multithreading. <span><span class="ref-journal">J. Comput. Chem. </span>2010;<span class="ref-vol">31</span>:455–461. doi: 10.1002/jcc.21334.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3041641/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19499576" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fjcc.21334" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comput.+Chem.&amp;title=AutoDock+Vina:+Improving+the+speed+and+accuracy+of+docking+with+a+new+scoring+function,+efficient+optimization,+and+multithreading&amp;author=O.+Trott&amp;author=A.J.+Olson&amp;volume=31&amp;publication_year=2010&amp;pages=455-461&amp;pmid=19499576&amp;doi=10.1002/jcc.21334&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B45-biomolecules-13-01678">45. <span class="element-citation">Lipinski C.A., Lombardo F., Dominy B.W., Feeney P.J. Experimental and computational approaches to estimate solubility and permeability in drug discovery and development settings. <span><span class="ref-journal">Adv. Drug Deliv. Rev. </span>2001;<span class="ref-vol">46</span>:3–26. doi: 10.1016/S0169-409X(00)00129-0.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/11259830" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2FS0169-409X(00)00129-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv.+Drug+Deliv.+Rev.&amp;title=Experimental+and+computational+approaches+to+estimate+solubility+and+permeability+in+drug+discovery+and+development+settings&amp;author=C.A.+Lipinski&amp;author=F.+Lombardo&amp;author=B.W.+Dominy&amp;author=P.J.+Feeney&amp;volume=46&amp;publication_year=2001&amp;pages=3-26&amp;pmid=11259830&amp;doi=10.1016/S0169-409X(00)00129-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B46-biomolecules-13-01678">46. <span class="element-citation">Toutain P.L., Bousquet-Melou A. Free drug fraction vs. free drug concentration: A matter of frequent confusion. <span><span class="ref-journal">J. Vet. Pharmacol. Ther. </span>2002;<span class="ref-vol">25</span>:460–463. doi: 10.1046/j.1365-2885.2002.00442.x.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12485352" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1046%2Fj.1365-2885.2002.00442.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Vet.+Pharmacol.+Ther.&amp;title=Free+drug+fraction+vs.+free+drug+concentration:+A+matter+of+frequent+confusion&amp;author=P.L.+Toutain&amp;author=A.+Bousquet-Melou&amp;volume=25&amp;publication_year=2002&amp;pages=460-463&amp;pmid=12485352&amp;doi=10.1046/j.1365-2885.2002.00442.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B47-biomolecules-13-01678">47. <span class="element-citation">Zhou J., Gong K. Belzutifan: A novel therapy for von Hippel&#x02013;Lindau disease. <span><span class="ref-journal">Nat. Rev. Nephrol. </span>2022;<span class="ref-vol">18</span>:205–206. doi: 10.1038/s41581-022-00544-5.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35132216" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41581-022-00544-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Nephrol.&amp;title=Belzutifan:+A+novel+therapy+for+von+Hippel&#x02013;Lindau+disease&amp;author=J.+Zhou&amp;author=K.+Gong&amp;volume=18&amp;publication_year=2022&amp;pages=205-206&amp;pmid=35132216&amp;doi=10.1038/s41581-022-00544-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B48-biomolecules-13-01678">48. <span class="element-citation">McGregor J.C., Allen G.P., Bearden D.T. Levofloxacin in the treatment of complicated urinary tract infections and acute pyelonephritis. <span><span class="ref-journal">Ther. Clin. Risk Manag. </span>2008;<span class="ref-vol">4</span>:843–853. doi: 10.2147/TCRM.S3426.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2621400/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19209267" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2147%2FTCRM.S3426" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ther.+Clin.+Risk+Manag.&amp;title=Levofloxacin+in+the+treatment+of+complicated+urinary+tract+infections+and+acute+pyelonephritis&amp;author=J.C.+McGregor&amp;author=G.P.+Allen&amp;author=D.T.+Bearden&amp;volume=4&amp;publication_year=2008&amp;pages=843-853&amp;pmid=19209267&amp;doi=10.2147/TCRM.S3426&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B49-biomolecules-13-01678">49. <span class="element-citation">Bukavina L., Bensalah K., Bray F., Carlo M., Challacombe B., Karam J.A., Kassouf W., Mitchell T., Montironi R., O&#x02019;Brien T., et al. Epidemiology of Renal Cell Carcinoma: 2022 Update. <span><span class="ref-journal">Eur. Urol. </span>2022;<span class="ref-vol">82</span>:529–542. doi: 10.1016/j.eururo.2022.08.019.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36100483" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.eururo.2022.08.019" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Eur.+Urol.&amp;title=Epidemiology+of+Renal+Cell+Carcinoma:+2022+Update&amp;author=L.+Bukavina&amp;author=K.+Bensalah&amp;author=F.+Bray&amp;author=M.+Carlo&amp;author=B.+Challacombe&amp;volume=82&amp;publication_year=2022&amp;pages=529-542&amp;pmid=36100483&amp;doi=10.1016/j.eururo.2022.08.019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B50-biomolecules-13-01678">50. <span class="element-citation">Kibble M., Saarinen N., Tang J., Wennerberg K., M&#x000e4;kel&#x000e4; S., Aittokallio T. Network pharmacology applications to map the unexplored target space and therapeutic potential of natural products. <span><span class="ref-journal">Nat. Prod. Rep. </span>2015;<span class="ref-vol">32</span>:1249–1266. doi: 10.1039/C5NP00005J.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26030402" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1039%2FC5NP00005J" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Prod.+Rep.&amp;title=Network+pharmacology+applications+to+map+the+unexplored+target+space+and+therapeutic+potential+of+natural+products&amp;author=M.+Kibble&amp;author=N.+Saarinen&amp;author=J.+Tang&amp;author=K.+Wennerberg&amp;author=S.+M&#x000e4;kel&#x000e4;&amp;volume=32&amp;publication_year=2015&amp;pages=1249-1266&amp;pmid=26030402&amp;doi=10.1039/C5NP00005J&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B51-biomolecules-13-01678">51. <span class="element-citation">Athanasios A., Charalampos V., Vasileios T., Ashraf G.M. Protein-Protein Interaction (PPI) Network: Recent Advances in Drug Discovery. <span><span class="ref-journal">Curr. Drug Metab. </span>2017;<span class="ref-vol">18</span>:5–10. doi: 10.2174/138920021801170119204832.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28889796" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2174%2F138920021801170119204832" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Drug+Metab.&amp;title=Protein-Protein+Interaction+(PPI)+Network:+Recent+Advances+in+Drug+Discovery&amp;author=A.+Athanasios&amp;author=V.+Charalampos&amp;author=T.+Vasileios&amp;author=G.M.+Ashraf&amp;volume=18&amp;publication_year=2017&amp;pages=5-10&amp;pmid=28889796&amp;doi=10.2174/138920021801170119204832&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B52-biomolecules-13-01678">52. <span class="element-citation">Balani S.K., Miwa G.T., Gan L.S., Wu J.T., Lee F.W. Strategy of utilizing in vitro and in vivo ADME tools for lead optimization and drug candidate selection. <span><span class="ref-journal">Curr. Top. Med. Chem. </span>2005;<span class="ref-vol">5</span>:1033–1038. doi: 10.2174/156802605774297038.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16181128" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2174%2F156802605774297038" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Top.+Med.+Chem.&amp;title=Strategy+of+utilizing+in+vitro+and+in+vivo+ADME+tools+for+lead+optimization+and+drug+candidate+selection&amp;author=S.K.+Balani&amp;author=G.T.+Miwa&amp;author=L.S.+Gan&amp;author=J.T.+Wu&amp;author=F.W.+Lee&amp;volume=5&amp;publication_year=2005&amp;pages=1033-1038&amp;pmid=16181128&amp;doi=10.2174/156802605774297038&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B53-biomolecules-13-01678">53. <span class="element-citation">Kitchen D.B., Decornez H., Furr J.R., Bajorath J. Docking and scoring in virtual screening for drug discovery: Methods and applications. <span><span class="ref-journal">Nat. Rev. Drug Discov. </span>2004;<span class="ref-vol">3</span>:935–949. doi: 10.1038/nrd1549.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15520816" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrd1549" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Drug+Discov.&amp;title=Docking+and+scoring+in+virtual+screening+for+drug+discovery:+Methods+and+applications&amp;author=D.B.+Kitchen&amp;author=H.+Decornez&amp;author=J.R.+Furr&amp;author=J.+Bajorath&amp;volume=3&amp;publication_year=2004&amp;pages=935-949&amp;pmid=15520816&amp;doi=10.1038/nrd1549&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B54-biomolecules-13-01678">54. <span class="element-citation">Mostashari-Rad T., Arian R., Sadri H., Mehridehnavi A., Mokhtari M., Ghasemi F., Fassihi A. Study of CXCR4 chemokine receptor inhibitors using QSPR and molecular docking methodologies. <span><span class="ref-journal">J. Theor. Comput. Chem. </span>2019;<span class="ref-vol">18</span>:1950018. doi: 10.1142/S0219633619500184.</span> [<a href="//doi.org/10.1142%2FS0219633619500184" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Theor.+Comput.+Chem.&amp;title=Study+of+CXCR4+chemokine+receptor+inhibitors+using+QSPR+and+molecular+docking+methodologies&amp;author=T.+Mostashari-Rad&amp;author=R.+Arian&amp;author=H.+Sadri&amp;author=A.+Mehridehnavi&amp;author=M.+Mokhtari&amp;volume=18&amp;publication_year=2019&amp;pages=1950018&amp;doi=10.1142/S0219633619500184&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B55-biomolecules-13-01678">55. <span class="element-citation">Chen F., Chen X., Yang D., Che X., Wang J., Li X., Zhang Z., Wang Q., Zheng W., Wang L., et al. Isoquercitrin inhibits bladder cancer progression in vivo and in vitro by regulating the PI3K/Akt and PKC signaling pathways. <span><span class="ref-journal">Oncol. Rep. </span>2016;<span class="ref-vol">36</span>:165–172. doi: 10.3892/or.2016.4794.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27177091" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3892%2For.2016.4794" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.+Rep.&amp;title=Isoquercitrin+inhibits+bladder+cancer+progression+in+vivo+and+in+vitro+by+regulating+the+PI3K/Akt+and+PKC+signaling+pathways&amp;author=F.+Chen&amp;author=X.+Chen&amp;author=D.+Yang&amp;author=X.+Che&amp;author=J.+Wang&amp;volume=36&amp;publication_year=2016&amp;pages=165-172&amp;pmid=27177091&amp;doi=10.3892/or.2016.4794&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B56-biomolecules-13-01678">56. <span class="element-citation">Chen Q., Li P., Li P., Xu Y., Li Y., Tang B. Isoquercitrin inhibits the progression of pancreatic cancer in vivo and in vitro by regulating opioid receptors and the mitogen-activated protein kinase signalling pathway. <span><span class="ref-journal">Oncol. Rep. </span>2015;<span class="ref-vol">33</span>:840–848. doi: 10.3892/or.2014.3626.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25434366" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3892%2For.2014.3626" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.+Rep.&amp;title=Isoquercitrin+inhibits+the+progression+of+pancreatic+cancer+in+vivo+and+in+vitro+by+regulating+opioid+receptors+and+the+mitogen-activated+protein+kinase+signalling+pathway&amp;author=Q.+Chen&amp;author=P.+Li&amp;author=P.+Li&amp;author=Y.+Xu&amp;author=Y.+Li&amp;volume=33&amp;publication_year=2015&amp;pages=840-848&amp;pmid=25434366&amp;doi=10.3892/or.2014.3626&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B57-biomolecules-13-01678">57. <span class="element-citation">Huang G., Tang B., Tang K., Dong X., Deng J., Liao L., Liao Z., Yang H., He S. Isoquercitrin inhibits the progression of liver cancer in vivo and in vitro via the MAPK signalling pathway. <span><span class="ref-journal">Oncol. Rep. </span>2014;<span class="ref-vol">31</span>:2377–2384. doi: 10.3892/or.2014.3099.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24676882" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3892%2For.2014.3099" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.+Rep.&amp;title=Isoquercitrin+inhibits+the+progression+of+liver+cancer+in+vivo+and+in+vitro+via+the+MAPK+signalling+pathway&amp;author=G.+Huang&amp;author=B.+Tang&amp;author=K.+Tang&amp;author=X.+Dong&amp;author=J.+Deng&amp;volume=31&amp;publication_year=2014&amp;pages=2377-2384&amp;pmid=24676882&amp;doi=10.3892/or.2014.3099&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B58-biomolecules-13-01678">58. <span class="element-citation">Lee E.-H., Park H.-J., Jung H.-Y., Kang I.-K., Kim B.-O., Cho Y.-J. Isoquercitrin isolated from newly bred Green ball aapple peel in lipopolysaccharide-stimulated macrophage regulates NF-&#x003ba;B inflammatory pathways and cytokines. <span><span class="ref-journal">3 Biotech. </span>2022;<span class="ref-vol">12</span>:100.  doi: 10.1007/s13205-022-03118-1.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8938568/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35463046" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs13205-022-03118-1" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=3+Biotech&amp;title=Isoquercitrin+isolated+from+newly+bred+Green+ball+aapple+peel+in+lipopolysaccharide-stimulated+macrophage+regulates+NF-&#x003ba;B+inflammatory+pathways+and+cytokines&amp;author=E.-H.+Lee&amp;author=H.-J.+Park&amp;author=H.-Y.+Jung&amp;author=I.-K.+Kang&amp;author=B.-O.+Kim&amp;volume=12&amp;publication_year=2022&amp;pages=100&amp;doi=10.1007/s13205-022-03118-1&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B59-biomolecules-13-01678">59. <span class="element-citation">Ge X., Huang S., Ren C., Zhao L. Taurocholic Acid and Glycocholic Acid Inhibit Inflammation and Activate Farnesoid X Receptor Expression in LPS-Stimulated Zebrafish and Macrophages. <span><span class="ref-journal">Molecules. </span>2023;<span class="ref-vol">28</span>:2005.  doi: 10.3390/molecules28052005.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10003765/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36903252" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fmolecules28052005" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecules&amp;title=Taurocholic+Acid+and+Glycocholic+Acid+Inhibit+Inflammation+and+Activate+Farnesoid+X+Receptor+Expression+in+LPS-Stimulated+Zebrafish+and+Macrophages&amp;author=X.+Ge&amp;author=S.+Huang&amp;author=C.+Ren&amp;author=L.+Zhao&amp;volume=28&amp;publication_year=2023&amp;pages=2005&amp;pmid=36903252&amp;doi=10.3390/molecules28052005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B60-biomolecules-13-01678">60. <span class="element-citation">Ha T.M., Ko W., Lee S.J., Kim Y.-C., Son J.-Y., Sohn J.H., Yim J.H., Oh H. Anti-Inflammatory Effects of Curvularin-Type Metabolites from a Marine-Derived Fungal Strain Penicillium sp. SF-5859 in Lipopolysaccharide-Induced RAW264.7 Macrophages. <span><span class="ref-journal">Mar. Drugs. </span>2017;<span class="ref-vol">15</span>:282.  doi: 10.3390/md15090282.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5618421/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28869509" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3390%2Fmd15090282" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mar.+Drugs&amp;title=Anti-Inflammatory+Effects+of+Curvularin-Type+Metabolites+from+a+Marine-Derived+Fungal+Strain+Penicillium+sp.+SF-5859+in+Lipopolysaccharide-Induced+RAW264.7+Macrophages&amp;author=T.M.+Ha&amp;author=W.+Ko&amp;author=S.J.+Lee&amp;author=Y.-C.+Kim&amp;author=J.-Y.+Son&amp;volume=15&amp;publication_year=2017&amp;pages=282&amp;pmid=28869509&amp;doi=10.3390/md15090282&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="B61-biomolecules-13-01678">61. <span class="element-citation">Rogers L.J., Basnakian A.G., Orloff M.S., Ning B., Yao-Borengasser A., Raj V., Kadlubar S. 2-amino-1-methyl-6-phenylimidazo(4,5-b)pyridine (PhIP) induces gene expression changes in JAK/STAT and MAPK pathways related to inflammation, diabetes and cancer. <span><span class="ref-journal">Nutr. Metab. </span>2016;<span class="ref-vol">13</span>:54. doi: 10.1186/s12986-016-0111-0.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4992261/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27547236" ref="reftype=pubmed&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs12986-016-0111-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nutr.+Metab.&amp;title=2-amino-1-methyl-6-phenylimidazo(4,5-b)pyridine+(PhIP)+induces+gene+expression+changes+in+JAK/STAT+and+MAPK+pathways+related+to+inflammation,+diabetes+and+cancer&amp;author=L.J.+Rogers&amp;author=A.G.+Basnakian&amp;author=M.S.+Orloff&amp;author=B.+Ning&amp;author=A.+Yao-Borengasser&amp;volume=13&amp;publication_year=2016&amp;pages=54&amp;pmid=27547236&amp;doi=10.1186/s12986-016-0111-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=10669250&amp;issue-id=449700&amp;journal-id=2397&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Biomolecules</span> are provided here courtesy of <strong>Multidisciplinary Digital Publishing Institute  (MDPI)</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC10669250/pdf/biomolecules-13-01678.pdf" class="int-view">PDF (3.4M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/10669250/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/10669250/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="10669250" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10669250%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="VXU0SHIBVOQbNlodCGewOPYoX8t6lb5ydxyA4H9n69Te4Ju9RYawKZ1Rqpsz0KHW">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10669250%2F&amp;text=A%20Network%20Pharmacology%20and%20Molecular-Docking-Based%20Approach%20to%20Identify%20the%20Probable%20Targets%20of%20Short-Chain%20Fatty-Acid-Producing%20..." alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC10669250%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC10669250/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC10669250/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38002360/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC10669250/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38002360/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC10669250/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/10669250/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC10669250/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
