
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-journal.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Exploration of prognostic and treatment markers in hepatocellular carcinoma via GPCR-related genes analysis - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="322C300B24F3920500004085AD7F74F4.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="heliyon" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11058304/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11058304/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Heliyon" /><meta name="citation_title" content="Exploration of prognostic and treatment markers in hepatocellular carcinoma via GPCR-related genes analysis" /><meta name="citation_author" content="Yuxin Wang" /><meta name="citation_author" content="Bao Jin" /><meta name="citation_author" content="Xiangan Wu" /><meta name="citation_author" content="Jiali Xing" /><meta name="citation_author" content="Baoluhe Zhang" /><meta name="citation_author" content="Xiaokun Chen" /><meta name="citation_author" content="Xiao Liu" /><meta name="citation_author" content="Xueshuai Wan" /><meta name="citation_author" content="Shunda Du" /><meta name="citation_publication_date" content="2024/05/05" /><meta name="citation_issue" content="9" /><meta name="citation_volume" content="10" /><meta name="citation_doi" content="10.1016/j.heliyon.2024.e29659" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11058304/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11058304/" /><meta name="citation_pmid" content="38694033" /><meta name="DC.Title" content="Exploration of prognostic and treatment markers in hepatocellular carcinoma via GPCR-related genes analysis" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="Elsevier" /><meta name="DC.Contributor" content="Yuxin Wang" /><meta name="DC.Contributor" content="Bao Jin" /><meta name="DC.Contributor" content="Xiangan Wu" /><meta name="DC.Contributor" content="Jiali Xing" /><meta name="DC.Contributor" content="Baoluhe Zhang" /><meta name="DC.Contributor" content="Xiaokun Chen" /><meta name="DC.Contributor" content="Xiao Liu" /><meta name="DC.Contributor" content="Xueshuai Wan" /><meta name="DC.Contributor" content="Shunda Du" /><meta name="DC.Date" content="2024 May 15" /><meta name="DC.Identifier" content="10.1016/j.heliyon.2024.e29659" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Exploration of prognostic and treatment markers in hepatocellular carcinoma via GPCR-related genes analysis" /><meta property="og:type" content="article" /><meta property="og:description" content="G protein-coupled receptors (GPCRs), the biggest family of signaling receptors, account for 34&#x000a0;% of all the drug targets approved by the Food and Drug Administration (FDA). It has been gradually recognized that GPCRs are of significance for tumorigenesis, ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11058304/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11058304/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2992/">Heliyon</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/461321/">v.10(9); 2024 May 15</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11058304
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11058304/pdf/main.pdf" class="int-view">PDF (14M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11058304/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11058304/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11058304" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11058304%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="8z10ESaZOnYQJFJNEUZVWEPiV0tv0gvDTNQfPUq078o5Yv9cN836qezJrB1ZNbgZ">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11058304%2F&amp;text=Exploration%20of%20prognostic%20and%20treatment%20markers%20in%20hepatocellular%20carcinoma%20via%20GPCR-related%20genes%20analysis" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11058304%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11058304/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11058304/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38694033/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11058304/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38694033/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11058304/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11058304/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/journals/2992/">Heliyon</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a class="navlink" href="/pmc/issues/461321/">v.10(9); 2024 May 15</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11058304
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-heliyon.png" alt="Logo of heliyon" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="http://www.journals.elsevier.com/heliyon" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div class="citation-default"><div class="part1"><span id="pmcmata">Heliyon.</span> 2024 May 15; 10(9): e29659. </div><div class="part2"><span class="fm-vol-iss-date">Published online 2024 Apr 15. </span>  <span class="doi"><span>doi: </span><a href="//doi.org/10.1016%2Fj.heliyon.2024.e29659" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.heliyon.2024.e29659</a></span></div></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11058304</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38694033">38694033</a></div></div></div><h1 class="content-title">Exploration of prognostic and treatment markers in hepatocellular carcinoma via GPCR-related genes analysis</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wang%20Y%5BAuthor%5D" id="au1">Yuxin Wang</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Jin%20B%5BAuthor%5D" id="au2">Bao Jin</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wu%20X%5BAuthor%5D" id="au3">Xiangan Wu</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Xing%20J%5BAuthor%5D" id="au4">Jiali Xing</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhang%20B%5BAuthor%5D" id="au5">Baoluhe Zhang</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chen%20X%5BAuthor%5D" id="au6">Xiaokun Chen</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Liu%20X%5BAuthor%5D" id="au7">Xiao Liu</a>, <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wan%20X%5BAuthor%5D" id="au8">Xueshuai Wan</a>, and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Du%20S%5BAuthor%5D" id="au9">Shunda Du</a><sup>&#x0204e;</sup></div><div style="display:none" class="contrib-group aff-tip"></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm139883509316368_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm139883509316368_an">Article notes</a> <a href="#" class="pmctoggle" rid="idm139883509316368_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm139883509316368_ai" style="display:none"><div class="fm-affl" id="aff1">Department of Liver Surgery, Peking Union Medical College Hospital, Chinese Academy of Medical Sciences and Peking Union Medical College (CAMS &#x00026; PUMC), Beijing, China</div><div><span class="contrib-email" id="au9">Shunda Du: <a href="mailto:dev@null" data-email="nc.hcmup@dhsud" class="oemail">nc.hcmup@dhsud</a></span> </div><div id="cor1"><sup>&#x0204e;</sup>Corresponding author. <a href="mailto:dev@null" data-email="nc.hcmup@dhsud" class="oemail">nc.hcmup@dhsud</a></div></div><div class="fm-article-notes hide half_rhythm" id="idm139883509316368_an" style="display:none"><div class="fm-pubdate half_rhythm">Received 2023 Dec 28; Revised 2024 Apr 10; Accepted 2024 Apr 12.</div></div><div class="permissions half_rhythm hide" id="idm139883509316368_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">Copyright</a> &#x000a9; 2024 The Authors</div><div class="license half_rhythm">This is an open access article under the CC BY-NC license (http://creativecommons.org/licenses/by-nc/4.0/).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm hidden" role="complementary" aria-label="Related or updated information about this article."></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-length="122" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><p class="p p-first-last">The names of the public repositories and their relevant accession links are detailed in the Materials and Methods section.</p></dd></dl></div><div id="abs0010" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="abs0010title">Abstract</h2><!--article-meta--><div><div id="sec-a.q.b.p.a" class="sec sec-first"><h3 id="sec-a.q.b.p.atitle">Background</h3><p class="p p-first-last">G protein-coupled receptors (GPCRs), the biggest family of signaling receptors, account for 34&#x000a0;% of all the drug targets approved by the Food and Drug Administration (FDA). It has been gradually recognized that GPCRs are of significance for tumorigenesis, but in-depth studies are still required to explore specific mechanisms. In this study, the role of GPCRs in hepatocellular carcinoma (HCC) was elucidated, and GPCR-related genes were employed for building a risk-score model for the prognosis and treatment efficacy prediction of HCC patients.</p></div><div id="sec-a.q.b.p.b" class="sec"><h3 id="sec-a.q.b.p.btitle">Methods</h3><p class="p p-first-last">Patients&#x02019; data on HCC were sourced from the Liver Hepatocellular Carcinoma-Japan (LIRI-JP) and The Cancer Genome Atlas (TCGA) databases, while GPCR-related genes were obtained from the Molecular Signatures Database (MSigDB). Univariant and multivariant Cox regression analyses, as well as least absolute shrinkage and selection operator (LASSO) were performed with the aim of identifying differentially expressed GPCR-related genes and grouping patients. Differential expression and functional enrichment analyses were performed; protein-protein interaction (PPI) mechanisms were explored; hub genes and micro ribonucleic acid (miRNA)-target gene regulatory networks were constructed. The tumor immune dysfunction and exclusion (TIDE) algorithm was utilized to evaluate immune infiltration levels and genetic variations. Sensitivity to immunotherapy and common antitumor drugs was predicted via the database Genomics of Drug Sensitivity in Cancer (GDSC).</p></div><div id="sec-a.q.b.p.c" class="sec"><h3 id="sec-a.q.b.p.ctitle">Results</h3><p class="p p-first-last">A GPCR-related risk score containing eight GPCR-related genes (atypical chemokine receptor 3 (ACKR3), C&#x02013;C chemokine receptor type 3 (CCR3), CCR7, frizzled homolog 5 (FZD5), metabotropic glutamate receptor 8 (GRM8), hydroxycarboxylic acid receptor 1 (HCAR1), 5-hydroxytryptamine receptor 5A (HTR5A) and nucleotide-binding oligomerization domain-like receptor family pyrin domain containing 6 (NLRP6)) was set up. In addition, patients were classified into groups with high and low risks. Patients in the high-risk group exhibited a worse prognosis but demonstrated a more favorable immunotherapy response rate compared with those in the low-risk group. Distinct sensitivity to chemotherapeutic drugs was observed. A clinical prediction model on the basis of GPCR-related risk scores was constructed. Areas under the curves (AUC) corresponding to one-, three- and five-year survival were 0.731, 0.765 and 0.731, respectively.</p></div><div id="sec-a.q.b.p.d" class="sec sec-last"><h3 id="sec-a.q.b.p.dtitle">Conclusions</h3><p class="p p-first-last">In this study, an efficient HCC prognostic prediction model was constructed by only GPCR-related genes, which are all potential targets for HCC treatment.</p></div></div><div class="sec"><strong class="kwd-title">Keywords: </strong><span class="kwd-text">HCC, GPCR, Prognosis, Systemic treatment</span></div></div><div id="sec1" class="tsec sec"><h2 class="head no_bottom_margin" id="sec1title">1. Introduction</h2><p id="p0010" class="p p-first">As the most prevalent and malignant cancer, hepatocellular carcinoma (HCC) accounts for 75%&#x02013;85&#x000a0;% of primary liver cancer and is considered to be the third major cause of cancer-associated mortality across the globe [<a href="#bib1" rid="bib1" class=" bibr popnode">1</a>]. Current standard therapeutic strategies encompass maximal surgical resection combined with transarterial chemoembolization, immunotherapy, chemotherapy, molecular targeted therapy, as well as radiotherapy if available. Owning to the inherent heterogeneity of HCC, patients show great differences in their response to immunotherapy and transarterial chemotherapy [<a href="#bib2" rid="bib2" class=" bibr popnode">[2]</a>, <a href="#bib3" rid="bib3" class=" bibr popnode">[3]</a>, <a href="#bib4" rid="bib4" class=" bibr popnode">[4]</a>, <a href="#bib5" rid="bib5" class=" bibr popnode">[5]</a>]. The prediction and assessment of immunotherapy and chemotherapy sensitivity in HCC patients has long been a prominent area of research interest.</p><p id="p0015">G protein-coupled receptors (GPCRs), which feature a seven-transmembrane receptor with an intracellular C-terminus and an extracellular N-terminus, are the largest superfamily of cell-surface receptors [<a href="#bib6" rid="bib6" class=" bibr popnode">6</a>]. They serve as essential pivots to communicate between internal and external cellular environments and transduce extracellular signals into intracellular pathways and intricate downstream effectors. Thus, multiple cellular processes are mediated, including sensory, neural, endocrine and immune systems, etc [<a href="#bib6" rid="bib6" class=" bibr popnode">6</a>]. Given the abundance of druggable sites and their involvement in a wide array of physiological and pathophysiological processes, GPCRs have emerged as highly valuable drug targets and have been the focus of research for decades. Of the current drugs approved by the Food and Drug Administration (FDA), 481 drugs target GPCRs, and take up 34&#x000a0;% of all approved drugs. Currently, approximately 320 GPCR-targeted agents are being clinically tested [<a href="#bib7" rid="bib7" class=" bibr popnode">7</a>]. Still and all, only less than 15&#x000a0;% of the discovered human GPCRs are targeted and more than half of the non-olfactory GPCRs encoded by genomes remain unexploited in terms of treatment [<a href="#bib7" rid="bib7" class=" bibr popnode">7</a>]. With breakthroughs in structural biology, the expanded knowledge about the activation of receptors and the application of new methodologies [<a href="#bib8" rid="bib8" class=" bibr popnode">8</a>], the pace of GPCR research and discovery of GPCR-targeted drugs has accelerated. These concepts lead to new opportunities for the identification of novel GPCR and GPCR-targeted drugs.</p><p id="p0020">About 800 GPCR members have been discovered in humans, and nearly 20&#x000a0;% of human tumors harbor mutations in GPCR-related genes [<a href="#bib9" rid="bib9" class=" bibr popnode">9</a>,<a href="#bib10" rid="bib10" class=" bibr popnode">10</a>]. Those mutations are identical in specific tumor types and vary widely among different tumors, which provides the possibility for the application of GPCR-targeted drugs in cancer treatments [<a href="#bib9" rid="bib9" class=" bibr popnode">9</a>]. Mounting studies have demonstrated that aberrant GPCR expression has a direct relationship with the proliferation, apoptosis, invasion, migration and metastasis of tumor cells, tumor angiogenesis and the formation of the tumor microenvironment [<a href="#bib10" rid="bib10" class=" bibr popnode">[10]</a>, <a href="#bib11" rid="bib11" class=" bibr popnode">[11]</a>, <a href="#bib12" rid="bib12" class=" bibr popnode">[12]</a>, <a href="#bib13" rid="bib13" class=" bibr popnode">[13]</a>]. Around 70 GPCR somatic mutants were reported in HCC [<a href="#bib14" rid="bib14" class=" bibr popnode">14</a>] and part of the role played by GPCRs in HCC was also investigated. It was found that some GPCRs exhibited elevated expression levels in HCC compared with normal liver tissues, such as C&#x02013;C chemokine receptor type 9 (CCR9), chemokine (C-X-C motif) receptor 3 (CXCR3), CXCR6 and G protein-coupled receptor 37 (GPR37). They were linked to tumor number, pathological differentiation and vascular invasion, and could serve as prognostic markers for HCC [<a href="#bib15" rid="bib15" class=" bibr popnode">[15]</a>, <a href="#bib16" rid="bib16" class=" bibr popnode">[16]</a>, <a href="#bib17" rid="bib17" class=" bibr popnode">[17]</a>, <a href="#bib18" rid="bib18" class=" bibr popnode">[18]</a>]. Some GPCRs were observed to be strongly associated with angiogenesis, including CCR6, CCR7, CXCR2, CXCR7, angiotensin II type 1 receptor (AT1R), AT2R, etc [<a href="#bib19" rid="bib19" class=" bibr popnode">[19]</a>, <a href="#bib20" rid="bib20" class=" bibr popnode">[20]</a>, <a href="#bib21" rid="bib21" class=" bibr popnode">[21]</a>, <a href="#bib22" rid="bib22" class=" bibr popnode">[22]</a>]. Some were demonstrated to induce epithelial&#x02013;mesenchymal transition (EMT) and enhance HCC cell migration and invasion, such as CXCR2, CXCR4 and CXCR7 [<a href="#bib19" rid="bib19" class=" bibr popnode">19</a>,<a href="#bib20" rid="bib20" class=" bibr popnode">20</a>]. Regarding treatment and drug resistance, leucine-rich repeat-containing G protein-coupled receptor 5 (Lgr5) could lead to doxorubicin resistance in HCC by regulating the programmed cell death protein 5 (PDCD5)/tumor protein 53 (p53) signaling axis [<a href="#bib23" rid="bib23" class=" bibr popnode">23</a>]. GPCRs were also found to assist in shaping the protumor microenvironment. CXCR6 was demonstrated to up-regulate interleukin-6 (IL-6) and IL-8 and increase the accumulation of neutrophils, which created an inflammatory environment and promoted HCC initiation and invasion [<a href="#bib16" rid="bib16" class=" bibr popnode">16</a>]. G protein-coupled estrogen receptor (GPER) deficiency could enhance macrophage infiltration and liver fibrosis [<a href="#bib24" rid="bib24" class=" bibr popnode">24</a>]. CCR4 could mediate regulatory T cells into the tumor microenvironment and promote immune escape in hepatitis B virus (HBV)<sup>+</sup>HCC [<a href="#bib25" rid="bib25" class=" bibr popnode">25</a>].</p><p id="p0025" class="p p-last">As mentioned above, GPCRs are perfect drug targets and have been demonstrated to play an essential role in HCC development and metastasis, as well as the regulation of the HCC microenvironment. Therefore, we innovatively adopted bioinformatics analysis methods, using large samples from public databases to individually analyze GPCR-related genes in HCC, further exploring the role of GPCR-related genes in predicting the prognosis and treatment sensitivity of HCC. This study can provide a comprehensive and holistic perspective on potential GPCR-related targets and the use of GPCR-related drugs in HCC treatment. With the help of the extensive development of GPCR-related drugs, this study may potentially offer new insights for the treatment of HCC.</p></div><div id="sec2" class="tsec sec"><h2 class="head no_bottom_margin" id="sec2title">2. Materials and methods</h2><div id="sec2.1" class="sec sec-first"><h3 id="sec2.1title">2.1. Data collection</h3><p id="p0030" class="p p-first">The HCC dataset The Cancer Genome Atlas (TCGA) was provided by University of California, Santa Cruz (UCSC) Xena (<a href="https://xena.ucsc.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://xena.ucsc.edu</a>). The gene expression sequencing data count and fragments per kilobase million (FPKM) values of patients (n&#x000a0;=&#x000a0;424) were downloaded and further normalized to transcripts per million (TPM) values. Meantime, the clinical data of patients were downloaded, such as age, gender, tumor-node-metastasis (TNM) stage, survival time and information on survival status, and patients without clinical information were removed. Finally, 363 samples with complete clinical information were retained. <a href="/pmc/articles/PMC11058304/table/tbl1/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl1" rid-ob="ob-tbl1" co-legend-rid=""><span>Table 1</span></a> shows detailed clinical information. Meanwhile, the mutation data of patients (n&#x000a0;=&#x000a0;415) were downloaded from Genomic Data Commons (GDC), selected as &#x0201c;Masked Somatic Mutation&#x0201d; and visualized by use of the maftools R package. In addition, the HCC dataset Liver Hepatocellular Carcinoma-Japan (LIRI-JP) was retrieved and downloaded from the LIRI-JP database, where LIRI-JP contained transcriptomic data for 232 samples and survival data for 260 samples. After the two pieces of information were matched, 232 samples containing survival data were obtained for analysis.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl1"><h3>Table 1</h3><!--caption a7--><div class="caption"><p>Baseline information of TCGA-LIHC patients in the TCGA database.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">Characteristic</th><th rowspan="1" colspan="1">levels</th><th rowspan="1" colspan="1">Overall</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1"><strong>n</strong></td><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">363</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>Age, n (%)</strong></td><td align="left" rowspan="1" colspan="1">&#x0003c;60</td><td align="left" rowspan="1" colspan="1">165 (45.5&#x000a0;%)</td></tr><tr><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">&#x02265;60</td><td align="left" rowspan="1" colspan="1">198 (54.5&#x000a0;%)</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>Gender, n (%)</strong></td><td align="left" rowspan="1" colspan="1">female</td><td align="left" rowspan="1" colspan="1">118 (32.5&#x000a0;%)</td></tr><tr><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">male</td><td align="left" rowspan="1" colspan="1">245 (67.5&#x000a0;%)</td></tr><tr><td align="left" rowspan="1" colspan="1"><strong>Stage, n (%)</strong></td><td align="left" rowspan="1" colspan="1">not reported</td><td align="left" rowspan="1" colspan="1">24 (6.7&#x000a0;%)</td></tr><tr><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">stage I</td><td align="left" rowspan="1" colspan="1">170 (46.8&#x000a0;%)</td></tr><tr><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">stage II</td><td align="left" rowspan="1" colspan="1">84 (23.1&#x000a0;%)</td></tr><tr><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">stage III</td><td align="left" rowspan="1" colspan="1">81 (22.3&#x000a0;%)</td></tr><tr><td rowspan="1" colspan="1"></td><td align="left" rowspan="1" colspan="1">stage IV</td><td align="left" rowspan="1" colspan="1">4 (1.1&#x000a0;%)</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139883497508144"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/table/tbl1/?report=objectonly">Open in a separate window</a></div></div><p id="p0035" class="p p-last">GPCRs bind chemicals in the extracellular environment and activate multiple intracellular signaling pathways, which finally causes cellular changes. A total of 870 GPCR genes were obtained on the basis of the Molecular Signatures Database (MSigDB) (<a href="https://www.gsea-msigdb.org/gsea/msigdb/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.gsea-msigdb.org/gsea/msigdb/</a>), of which 770 were matched with TCGA and LIRI-JP datasets.</p></div><div id="sec2.2" class="sec"><h3 id="sec2.2title">2.2. Subtype analysis of patients with HCC based on GPCR genes</h3><p id="p0040" class="p p-first-last">According to TCGA-LIHC expression data and GPCR genes, R package &#x02018;ConsensusClusterPlus&#x02019; [<a href="#bib26" rid="bib26" class=" bibr popnode">26</a>] was adopted to perform unsupervised clustering analysis and identify GPCR subtypes in HCC patients. This consistent clustering algorithm was utilized for identifying cluster numbers, and the analysis included 1000 iterations to make sure the classification was stable. Principal component analysis (PCA) was performed on patients with grouped subtypes, to see sample differences, and the associations of different subtypes with prognosis were determined using survival analysis.</p></div><div id="sec2.3" class="sec"><h3 id="sec2.3title">2.3. GPCR-related prognostic gene screening and risk score model construction</h3><p id="p0045" class="p p-first">To assess the prognostic value of GPCR genes in HCC patients, a univariate Cox analysis was first used to screen GPCR genes that have a threshold value of P&#x000a0;&#x0003c;&#x000a0;0.05. To further identify independent prognostic factors, least absolute shrinkage and selection operator (LASSO) regression [<a href="#bib27" rid="bib27" class=" bibr popnode">27</a>] and multivariate Cox proportional-hazards models were explored for the screening of independent prognostic factors. A prognostic GPCR-related model was established after 10-fold cross-validation. Below is the computational formula for the risk score.</p><div class="disp-formula" id="ufd1"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M1" altimg="si1.svg"><mrow><mtext>risk</mtext><mi>S</mi><mi>c</mi><mi>o</mi><mi>r</mi><mi>e</mi><mo linebreak="badbreak">=</mo><mrow><munder><mo>&#x02211;</mo><mi>i</mi></munder><mrow><mi>C</mi><mi>o</mi><mi>e</mi><mi>f</mi><mi>f</mi><mi>i</mi><mi>c</mi><mi>i</mi><mi>e</mi><mi>n</mi><mi>t</mi><mspace width="0.25em"></mspace><mrow><mo stretchy="true">(</mo><msub><mrow><mi>g</mi><mi>e</mi><mi>n</mi><mi>e</mi></mrow><mi>i</mi></msub><mo stretchy="true">)</mo></mrow><mo>*</mo><mi>m</mi><mi>R</mi><mi>N</mi><mi>A</mi><mspace width="0.25em"></mspace><mi>E</mi><mi>x</mi><mi>p</mi><mi>r</mi><mi>e</mi><mi>s</mi><mi>s</mi><mi>i</mi><mi>o</mi><mi>n</mi><mspace width="0.25em"></mspace><mrow><mo stretchy="true">(</mo><msub><mrow><mi>g</mi><mi>e</mi><mi>n</mi><mi>e</mi></mrow><mi>i</mi></msub><mo stretchy="true">)</mo></mrow></mrow></mrow></mrow></math></div></div><p></p><p id="p0050" class="p p-last">Patients were categorized into groups with high and low risks in accordance with the median of GPCR risk scores. Receiver operating characteristic (ROC) curves were plotted for one, three and five years by time-dependent ROC curves for the purpose of determining accuracy. Additionally, LIRI-JP, an external test set, was used to validate the model. Based on the gene regression coefficients in the model, model accuracy was evaluated by plotting time-dependent ROC curves.</p></div><div id="sec2.4" class="sec"><h3 id="sec2.4title">2.4. Analysis of DEGs and functional enrichment in the GPCR-related risk score model</h3><p id="p0055" class="p p-first">DESeq2 package in R [<a href="#bib28" rid="bib28" class=" bibr popnode">28</a>] was utilized for identifying DEGs in both groups of HCC patients in TCGA. differentially expressed genes (DEGs) were defined to be genes with adj. <em>P</em>-value &#x0003c;0.05 and an absolute value of LogFC &#x0003e;1. Heat and volcano maps were used to visually display the results.</p><p id="p0060">Gene Ontology (GO) analysis [<a href="#bib29" rid="bib29" class=" bibr popnode">29</a>] is a frequently used approach to performing large-scale research on functional enrichment, including biological processes (BPs), molecular functions (MFs) and cellular components (CCs). Kyoto Encyclopedia of Genes and Genomes (KEGG) [<a href="#bib30" rid="bib30" class=" bibr popnode">30</a>] is a database widely used for storing information about genomes, drugs, diseases and biological pathways. ClusterProfiler package in R [<a href="#bib31" rid="bib31" class=" bibr popnode">31</a>] was adopted to conduct GO and KEGG pathway enrichment analyses. It was considered that a critical value of &#x0003c;0.05 for the false discovery rate (FDR) showed statistical significance.</p><p id="p0065" class="p p-last">For the purpose of investigating the differences in BPs between all subgroups, GSEA [<a href="#bib32" rid="bib32" class=" bibr popnode">32</a>] was used to perform a gene set enrichment analysis based on the HCC patients&#x02019; gene expression profile dataset. As a computational method of analyzing the statistical difference between two biological states in a particular gene set, GSEA is frequently used for estimating the pathway and changes in BP activity. The effect of groups with high and low risks on tumor KEGG was assessed by downloading the &#x0201c;c2.cp.kegg.symbols.gmt&#x0201d; gene set from the MSigDB database for GSEA. FDR &#x0003c;0.25 was taken into account in the results.</p></div><div id="sec2.5" class="sec"><h3 id="sec2.5title">2.5. Protein-protein interaction (PPI) and regulatory networks</h3><p id="p0070" class="p p-first-last">The STRING database [<a href="#bib33" rid="bib33" class=" bibr popnode">33</a>] was used to analyze the interactions between DEG-encoded proteins, and the results were exported to further screen hub genes through the plugin CytoHubba in Cytoscape. Based on the prediction of the mirTarbse database (<a href="https://mirtarbase.cuhk.edu.cn/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mirtarbase.cuhk.edu.cn/</a>), possible regulatory micro ribonucleic acids (miRNAs) were predicted based on the evidence of &#x0201c;luciferase reporter gene assay&#x0201d;, and related long non-coding RNAs (lncRNAs) were further predicted. The Cytoscape software was used to visualize the results of competitive endogenous RNA (ceRNA) analysis.</p></div><div id="sec2.6" class="sec"><h3 id="sec2.6title">2.6. Identification and correlation analysis of tumor immune infiltrating cells</h3><p id="p0075" class="p p-first">The estimation of stromal and immune cells in malignant tumor tissues using the expression data (ESTIMATE) package for R [<a href="#bib34" rid="bib34" class=" bibr popnode">34</a>] was used to assess the immunoreactivity of the tumor microenvironment. ESTIMATE analysis quantified immunoreactivity (immune infiltration level) in tumor samples based on gene expression profiles, and an immune score was obtained for each tumor sample. A comparison was made between the differences in the immune infiltration characteristics of patients with HCC in both groups.</p><p id="p0080" class="p p-last">To clarify the specific infiltration level of immune cells, the CIBERSORT algorithm [<a href="#bib35" rid="bib35" class=" bibr popnode">35</a>] was used for deconvoluting the HCC transcriptome expression matrix, which thus estimated immune cell composition and abundance in mixed cells. Data of the gene expression matrix were uploaded to CIBERSORT, and the output of samples with p&#x000a0;&#x0003c;&#x000a0;0.05 was filtered to obtain the matrix of immune cell infiltration. The ggplot2 package in R was used to draw histograms showing how 22 immune cell infiltrates were distributed in each sample.</p></div><div id="sec2.7" class="sec"><h3 id="sec2.7title">2.7. Analysis of single nucleotide polymorphisms and copy number variation</h3><p id="p0085" class="p p-first-last">Maftools package was utilized for analyzing the high-frequency mutated genes of patients in groups with high and low risks to analyze single nucleotide polymorphisms (SNPs). Then, the Masked Copy Number Segment data were downloaded through GDC. The analysis of the downloaded copy number variation (CNV) segments was completed by GenePattern with Genomic Identification of Significant Targets in Cancer (GISTIC) 2.0. GISTIC 2.0 analysis was performed using default parameters.</p></div><div id="sec2.8" class="sec"><h3 id="sec2.8title">2.8. TMB, MSI and predictive analysis of tumor immunotherapy response</h3><p id="p0090" class="p p-first-last">The maftools R package was used to calculate the data on the TMB of HCC patients. MSI-Sensor data were gathered from the cBioportal database (<a href="https://www.cbioportal.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.cbioportal.org</a>). The underlying tumor treatment response of immune checkpoint blockade (ICB) was predicted by the tumor immune dysfunction and exclusion (TIDE) score [<a href="#bib36" rid="bib36" class=" bibr popnode">36</a>], a computational algorithm based on gene expression profiles (<a href="http://tide.dfci.harvard.edu" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://tide.dfci.harvard.edu</a>). Differences between both groups were compared with the TIDE analysis results.</p></div><div id="sec2.9" class="sec"><h3 id="sec2.9title">2.9. Analysis of drug sensitivity</h3><p id="p0095" class="p p-first-last">The database GDSC (<a href="http://www.cancerrxgene.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">www.cancerrxgene.org/</a>) [<a href="#bib37" rid="bib37" class=" bibr popnode">37</a>] was utilized for finding data on oncology drug response and sensitive genome markers. The pRRophetic algorithm [<a href="#bib38" rid="bib38" class=" bibr popnode">38</a>] was used for the construction of a ridge regression model on the basis of gene expression profiles to forecast the sensitivity of high- and low-risk groups to commonly used anticancer drugs by half maximal inhibitory concentration (IC50) values. Moreover, common oncological chemotherapeutic agents were extracted, including cisplatin, etoposide, paclitaxel, cyclophosphamide, etc. Differences in chemotherapeutic drug sensitivity between different subgroups were clarified.</p></div><div id="sec2.10" class="sec"><h3 id="sec2.10title">2.10. Clinical prognosis prediction model construction based on GPCR risk scores</h3><p id="p0100" class="p p-first-last">The individualized evaluation of GPCR risk scores in combination with the clinicopathological characteristics of patient prognosis was demonstrated by performing a multivariant Cox analysis and constructing a nomogram. The predicted values of the model were compared with the actual observed survival to generate calibration curves and assess performance. Based on the rms R package (<a href="https://cran.r-project.org/web/packages/rms/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://cran.r-project.org/web/packages/rms/</a>), the model was resampled 1000 times using the bootstrap method to ensure that the nomogram was accurate.</p></div><div id="sec2.11" class="sec sec-last"><h3 id="sec2.11title">2.11. Statistical analysis</h3><p id="p0105" class="p p-first-last">R software (version 4.1.3) was used to perform all data processing and analysis. To compare the continuous variables of both groups, independent Student t-tests were used to estimate whether normally distributed variables were statistically significant. Additionally, the Mann-Whitney <em>U</em> test (i.e. Wilcoxon rank sum test) was utilized to analyze the differences between non-normally distributed variables. A chi-square or Fisher exact test was utilized for comparing and analyzing whether the categorical variables of both groups were statistically significant. Survival analysis was performed using the survivor package of R. Survival differences were shown using Kaplan-Meier (KM) survival curves. A log-rank test was utilized to assess whether both groups showed significant differences in survival time. Then, pROC package in R was applied to plot time-dependent ROC curves, and the calculation of areas under the curve (AUCs) [<a href="#bib39" rid="bib39" class=" bibr popnode">39</a>] was conducted to evaluate the accuracy of risk scores and thus estimate prognosis. Each statistical P value was two-sided and showed statistical significance at P&#x000a0;&#x0003c;&#x000a0;0.05.</p></div></div><div id="sec3" class="tsec sec"><h2 class="head no_bottom_margin" id="sec3title">3. Results</h2><div id="sec3.1" class="sec sec-first"><h3 id="sec3.1title">3.1. Expression and mutational alterations of GPCR-associated genes in HCC patients</h3><p id="p0110" class="p p-first-last">The research workflow is shown in <a href="/pmc/articles/PMC11058304/figure/fig1/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig1" rid-ob="ob-fig1" co-legend-rid="lgnd_fig1"><span>Fig. 1</span></a>. First, 770 GPCR-related genes were extracted from the RNA-sequencing (RNA-seq) data of TCGA-LIHC, and a comparison was made between the expression differences of HCC and normal liver tissues. After a univariant Cox regression analysis, 38 of these 770 GPCR-related genes were observed to be differentially expressed, and almost all of them were overexpressed in HCC (<a href="/pmc/articles/PMC11058304/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>A), which suggested that these genes may play an essential role in HCC tumorigenesis. Then, the mutation information of those 38 GPCR-related genes was extracted by maftools package, and their SNP mutation frequencies were found to be generally low (<a href="/pmc/articles/PMC11058304/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>B). The above results revealed that these GPCR-related genes may play a major role at the transcriptional or post-transcriptional level.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig1" co-legend-rid="lgnd_fig1"><a href="/pmc/articles/PMC11058304/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139883504037232"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11058304_gr1.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 1" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11058304/bin/gr1.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139883504037232"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/figure/fig1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig1"><div><a class="figpopup" href="/pmc/articles/PMC11058304/figure/fig1/" target="figure" rid-figpopup="fig1" rid-ob="ob-fig1">Fig. 1</a></div><!--caption a7--><div class="caption"><p>Flow chart of the whole research work.</p></div></div></div><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig2" co-legend-rid="lgnd_fig2"><a href="/pmc/articles/PMC11058304/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139883505262496"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11058304_gr2.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 2" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11058304/bin/gr2.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139883505262496"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/figure/fig2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig2"><div><a class="figpopup" href="/pmc/articles/PMC11058304/figure/fig2/" target="figure" rid-figpopup="fig2" rid-ob="ob-fig2">Fig. 2</a></div><!--caption a7--><div class="caption"><p>Differential expression of GPCR genes and the identification of subtypes in TCGA-LIHC patients. (A) Differential expression of GPCR genes in tumor and tumor groups. (B) Alternation in the somatic mutation of GPCR genes in the group with LIHC patients. (C) Heatmap of K&#x000a0;=&#x000a0;2 sample clustering by use of consistent clustering. (D) The plot of sample tracking when K&#x000a0;=&#x000a0;2&#x02013;9, where different colors stand for the distribution of various cluster samples, and vertical and horizontal coordinates represent K values and samples, respectively. (E) PCA plot for subgroups 1 and 2. (F) KM method for comparing the difference between subgroups 1 and 2 in survival; both subtypes of patients were not statistically and significantly different. (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.)</p></div></div></div></div><div id="sec3.2" class="sec"><h3 id="sec3.2title">3.2. Identification of HCC GPCR subtypes</h3><p id="p0115" class="p p-first-last">On the basis of the above DEGs related to GPCRs, consistent clustering was used for the clustering of HCC samples. When k&#x000a0;=&#x000a0;2, the best clustering distribution was obtained, and subgroups 1 and 2 were identified (<a href="/pmc/articles/PMC11058304/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>C and D). HCC transcriptome data were downscaled by PCA, and principal components PC1 and PC2 were obtained. The PCA results of the two groups were plotted, and it was found that both groups were clearly distinguished (<a href="/pmc/articles/PMC11058304/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>E). After that, the prognostic characteristics of the two subgroups were analyzed by KM curves, but the results revealed no significant prognostic differences (<a href="/pmc/articles/PMC11058304/figure/fig2/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig2" rid-ob="ob-fig2" co-legend-rid="lgnd_fig2"><span>Fig. 2</span></a>F). The above results suggested that differentially expressed GPCR-related genes could well guide the subtyping of HCC patients. Nevertheless, subtyping could not yet assess prognostic differences.</p></div><div id="sec3.3" class="sec"><h3 id="sec3.3title">3.3. Construction and evaluation of GPCR-related risk scores</h3><p id="p0120" class="p p-first">LASSO and multivariant Cox regression analyses were performed among 38 genes (<a href="/pmc/articles/PMC11058304/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>A and B). Eight genes with the optimal prognostic value were obtained, namely atypical chemokine receptor 3 (ACKR3), CCR3, CCR7, frizzled homolog 5 (FZD5), metabotropic glutamate receptor 8 (GRM8), hydroxycarboxylic acid receptor 1 (HCAR1), 5-hydroxytryptamine receptor 5A (HTR5A) and nucleotide-binding oligomerization domain-like receptor family pyrin domain containing 6 (NLRP6) (<a href="/pmc/articles/PMC11058304/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>C). Meanwhile, the coefficients of key characteristic genes computed by the multivariate Cox analysis were obtained. Gene expression was multiplied with corresponding coefficients and summed to set up a GPCR-related risk score. The final risk scores of all samples were calculated.</p><div class="disp-formula" id="ufd2"><div class="f"><math xmlns:mml="http://www.w3.org/1998/Math/MathML" display="block" id="M2" altimg="si2.svg"><mrow><mtext>risk</mtext><mi>S</mi><mi>c</mi><mi>o</mi><mi>r</mi><mi>e</mi><mo linebreak="badbreak">=</mo><mi>A</mi><mi>C</mi><mi>K</mi><mi>R</mi><mn>3</mn><mo linebreak="goodbreak">&#x000d7;</mo><mn>0.213</mn><mo linebreak="goodbreak">+</mo><mi>C</mi><mi>C</mi><mi>R</mi><mn>3</mn><mo linebreak="goodbreak">&#x000d7;</mo><mn>0.386</mn><mo linebreak="goodbreak">+</mo><mi>C</mi><mi>C</mi><mi>R</mi><mn>7</mn><mo linebreak="goodbreak">&#x000d7;</mo><mn>0.409</mn><mo linebreak="goodbreak">+</mo><mi>F</mi><mi>Z</mi><mi>D</mi><mn>5</mn><mo linebreak="goodbreak">&#x000d7;</mo><mn>0.201</mn><mo linebreak="goodbreak">+</mo><mi>G</mi><mi>R</mi><mi>M</mi><mn>8</mn><mo linebreak="goodbreak">&#x000d7;</mo><mn>0.295</mn><mo linebreak="goodbreak">+</mo><mi>H</mi><mi>C</mi><mi>A</mi><mi>R</mi><mn>1</mn><mo linebreak="goodbreak">&#x000d7;</mo><mn>0.689</mn><mo linebreak="goodbreak">+</mo><mi>H</mi><mi>T</mi><mi>R</mi><mn>5</mn><mi>A</mi><mo linebreak="goodbreak">&#x000d7;</mo><mn>0.866</mn><mo>&#x02212;</mo><mi>N</mi><mi>L</mi><mi>R</mi><mi>P</mi><mn>6</mn><mo linebreak="goodbreak">&#x000d7;</mo><mn>0.268</mn></mrow></math></div></div><p></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig3" co-legend-rid="lgnd_fig3"><a href="/pmc/articles/PMC11058304/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139883513287104"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11058304_gr3.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 3" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11058304/bin/gr3.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139883513287104"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/figure/fig3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig3"><div><a class="figpopup" href="/pmc/articles/PMC11058304/figure/fig3/" target="figure" rid-figpopup="fig3" rid-ob="ob-fig3">Fig. 3</a></div><!--caption a7--><div class="caption"><p>GPCR-related risk score model construction. (A&#x02013;B) LASSO regression analyses with the best lamda value corresponding to the number of seven variables. (C) Results of multifactorial Cox stepwise regression (direction&#x000a0;=&#x000a0;&#x0201c;both&#x0201d;) analysis; eight genes were finally identified as independent prognostic factors. (D) Risk score distribution, patient survival and heatmap of characteristic gene expression in LIHC patients. (E) The left side is the KM curve of patients in both groups in the training dataset, which demonstrates a greatly worse prognosis in the high-risk group of patients; the right side shows the time-dependent ROC curve of TCGA-LIHC patients. (F) The left side is the KM curve of patients in both groups in the validation dataset LIRI-JP, which indicates a greatly worse prognosis in high-risk group patients; the right side shows the time-dependent ROC curve analysis of LIRI-JP.</p></div></div></div><p id="p0125" class="p p-last">Patients were divided into groups with high and low risks on the basis of the median expression values of risk scores in the TCGA dataset, and a heatmap was plotted (<a href="/pmc/articles/PMC11058304/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>D). KM curves demonstrated a significantly worse prognosis in the high-risk group of patients and a great difference in survival between high- and low-risk groups of patients (p&#x000a0;&#x0003c;&#x000a0;0.001) (<a href="/pmc/articles/PMC11058304/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>E). Additionally, time-dependent ROC curve analysis demonstrated that the risk score had excellent predictive power for the overall survival (OS) of HCC patients, where AUCs were 0.804, 0.787 and 0.750 for one-, three- and five-year OS, respectively (<a href="/pmc/articles/PMC11058304/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>E). In the external validation dataset LIRI-JP, the high-risk group of patients were also found to have a significantly worse prognosis (p&#x000a0;&#x0003c;&#x000a0;0.01), and ROC curves had an AUC of 0.764, 0.704 and 0.746 for one-, three- and five-year OS, respectively. This indicated that the model performed well in the external dataset as well (<a href="/pmc/articles/PMC11058304/figure/fig3/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig3" rid-ob="ob-fig3" co-legend-rid="lgnd_fig3"><span>Fig. 3</span></a>F).</p></div><div id="sec3.4" class="sec"><h3 id="sec3.4title">3.4. Analysis of DEGs in patients with GPCR-related risk scores in high- and low-risk groups</h3><p id="p0130" class="p p-first">To analyze the association between GPCR-related risk scores and HCC development, PCA was explored and plots demonstrated an obvious distinction between both groups of patients (<a href="/pmc/articles/PMC11058304/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>A). Later, a differential expression analysis was conducted on both groups of patients. The results showed that 1976 significant DEGs were obtained, including 1371 up-regulated DEGs and 605 down-regulated DEGs (<a href="/pmc/articles/PMC11058304/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>B and C).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig4" co-legend-rid="lgnd_fig4"><a href="/pmc/articles/PMC11058304/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139883511716096"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11058304_gr4.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 4" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11058304/bin/gr4.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139883511716096"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/figure/fig4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig4"><div><a class="figpopup" href="/pmc/articles/PMC11058304/figure/fig4/" target="figure" rid-figpopup="fig4" rid-ob="ob-fig4">Fig. 4</a></div><!--caption a7--><div class="caption"><p>DEG and functional enrichment analyses on the basis of the GPCR-related risk score model. (A) PCA plots of the overall differences between the high and low-risk groups of TCGA. (B&#x02013;C) Volcano and heatmap demonstrating the expression of DEGs between LIHC patients with high and low risks in the TCGA dataset. (D) GO analysis results between TCGA-LIHC patients in both groups. (E) KEGG analysis results between TCGA-LIHC patients in both groups.</p></div></div></div><p id="p0135">Next, a functional enrichment analysis was performed on these 1976 DEGs. The GO analysis suggested that DEGs were associated with nuclear division, mitotic nuclear division and mitotic sister chromatid segregation in BPs. In addition, they were related to synaptic and basolateral plasma membranes, kinesin complex and condensed nuclear chromosome kinetochore in CCs, and substrate-specific channel, channel, passive transmembrane transporter and ion channel activities in MFs (<a href="/pmc/articles/PMC11058304/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>D). The KEGG functional analysis results indicated that DEGs principally affect bile secretion and neuroactive ligand- and extracellular matrix (ECM)-receptor interactions (<a href="/pmc/articles/PMC11058304/figure/fig4/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig4" rid-ob="ob-fig4" co-legend-rid="lgnd_fig4"><span>Fig. 4</span></a>E). <a href="/pmc/articles/PMC11058304/table/tbl2/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl2" rid-ob="ob-tbl2" co-legend-rid=""><span>Table 2</span></a>, <a href="/pmc/articles/PMC11058304/table/tbl3/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl3" rid-ob="ob-tbl3" co-legend-rid=""><span>Table 3</span></a> show detailed GO and KEGG results.</p><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl2"><h3>Table 2</h3><!--caption a7--><div class="caption"><p>Top 20 significant genes in GO analysis.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">ONTOLOGY</th><th rowspan="1" colspan="1">ID</th><th rowspan="1" colspan="1">Description</th><th rowspan="1" colspan="1">GeneRatio</th><th rowspan="1" colspan="1">pvalue</th><th rowspan="1" colspan="1">qvalue</th><th rowspan="1" colspan="1">Count</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0140014</td><td align="left" rowspan="1" colspan="1">mitotic nuclear division</td><td align="left" rowspan="1" colspan="1">51/1334</td><td align="left" rowspan="1" colspan="1">5.5297E-11</td><td align="left" rowspan="1" colspan="1">2.6409E-07</td><td align="left" rowspan="1" colspan="1">51</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0000280</td><td align="left" rowspan="1" colspan="1">nuclear division</td><td align="left" rowspan="1" colspan="1">66/1334</td><td align="left" rowspan="1" colspan="1">2.6889E-10</td><td align="left" rowspan="1" colspan="1">6.4208E-07</td><td align="left" rowspan="1" colspan="1">66</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0000070</td><td align="left" rowspan="1" colspan="1">mitotic sister chromatid segregation</td><td align="left" rowspan="1" colspan="1">34/1334</td><td align="left" rowspan="1" colspan="1">1.4812E-09</td><td align="left" rowspan="1" colspan="1">2.3579E-06</td><td align="left" rowspan="1" colspan="1">34</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0048285</td><td align="left" rowspan="1" colspan="1">organelle fission</td><td align="left" rowspan="1" colspan="1">67/1334</td><td align="left" rowspan="1" colspan="1">6.7362E-09</td><td align="left" rowspan="1" colspan="1">8.0427E-06</td><td align="left" rowspan="1" colspan="1">67</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:1902850</td><td align="left" rowspan="1" colspan="1">microtubule cytoskeleton organization involved in mitosis</td><td align="left" rowspan="1" colspan="1">30/1334</td><td align="left" rowspan="1" colspan="1">8.8873E-09</td><td align="left" rowspan="1" colspan="1">8.4888E-06</td><td align="left" rowspan="1" colspan="1">30</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0000819</td><td align="left" rowspan="1" colspan="1">sister chromatid segregation</td><td align="left" rowspan="1" colspan="1">37/1334</td><td align="left" rowspan="1" colspan="1">1.6462E-08</td><td align="left" rowspan="1" colspan="1">1.301E-05</td><td align="left" rowspan="1" colspan="1">37</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0007052</td><td align="left" rowspan="1" colspan="1">mitotic spindle organization</td><td align="left" rowspan="1" colspan="1">26/1334</td><td align="left" rowspan="1" colspan="1">1.9069E-08</td><td align="left" rowspan="1" colspan="1">1.301E-05</td><td align="left" rowspan="1" colspan="1">26</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0030198</td><td align="left" rowspan="1" colspan="1">extracellular matrix organization</td><td align="left" rowspan="1" colspan="1">57/1334</td><td align="left" rowspan="1" colspan="1">2.4111E-08</td><td align="left" rowspan="1" colspan="1">1.4394E-05</td><td align="left" rowspan="1" colspan="1">57</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0007059</td><td align="left" rowspan="1" colspan="1">chromosome segregation</td><td align="left" rowspan="1" colspan="1">50/1334</td><td align="left" rowspan="1" colspan="1">1.4519E-07</td><td align="left" rowspan="1" colspan="1">7.7043E-05</td><td align="left" rowspan="1" colspan="1">50</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0043062</td><td align="left" rowspan="1" colspan="1">extracellular structure organization</td><td align="left" rowspan="1" colspan="1">60/1334</td><td align="left" rowspan="1" colspan="1">2.3461E-07</td><td align="left" rowspan="1" colspan="1">0.00010762</td><td align="left" rowspan="1" colspan="1">60</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0098813</td><td align="left" rowspan="1" colspan="1">nuclear chromosome segregation</td><td align="left" rowspan="1" colspan="1">43/1334</td><td align="left" rowspan="1" colspan="1">2.4788E-07</td><td align="left" rowspan="1" colspan="1">0.00010762</td><td align="left" rowspan="1" colspan="1">43</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0051783</td><td align="left" rowspan="1" colspan="1">regulation of nuclear division</td><td align="left" rowspan="1" colspan="1">34/1334</td><td align="left" rowspan="1" colspan="1">4.4798E-07</td><td align="left" rowspan="1" colspan="1">0.00017829</td><td align="left" rowspan="1" colspan="1">34</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0048562</td><td align="left" rowspan="1" colspan="1">embryonic organ morphogenesis</td><td align="left" rowspan="1" colspan="1">45/1334</td><td align="left" rowspan="1" colspan="1">5.5201E-07</td><td align="left" rowspan="1" colspan="1">0.00020279</td><td align="left" rowspan="1" colspan="1">45</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0099177</td><td align="left" rowspan="1" colspan="1">regulation of <em>trans</em>-synaptic signaling</td><td align="left" rowspan="1" colspan="1">60/1334</td><td align="left" rowspan="1" colspan="1">7.7998E-07</td><td align="left" rowspan="1" colspan="1">0.00024928</td><td align="left" rowspan="1" colspan="1">60</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0016266</td><td align="left" rowspan="1" colspan="1">O-glycan processing</td><td align="left" rowspan="1" colspan="1">17/1334</td><td align="left" rowspan="1" colspan="1">7.8294E-07</td><td align="left" rowspan="1" colspan="1">0.00024928</td><td align="left" rowspan="1" colspan="1">17</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0048568</td><td align="left" rowspan="1" colspan="1">embryonic organ development</td><td align="left" rowspan="1" colspan="1">59/1334</td><td align="left" rowspan="1" colspan="1">8.4259E-07</td><td align="left" rowspan="1" colspan="1">0.0002515</td><td align="left" rowspan="1" colspan="1">59</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0042445</td><td align="left" rowspan="1" colspan="1">hormone metabolic process</td><td align="left" rowspan="1" colspan="1">38/1334</td><td align="left" rowspan="1" colspan="1">1.2818E-06</td><td align="left" rowspan="1" colspan="1">0.00036009</td><td align="left" rowspan="1" colspan="1">38</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0050804</td><td align="left" rowspan="1" colspan="1">modulation of chemical synaptic transmission</td><td align="left" rowspan="1" colspan="1">59/1334</td><td align="left" rowspan="1" colspan="1">1.5535E-06</td><td align="left" rowspan="1" colspan="1">0.00039072</td><td align="left" rowspan="1" colspan="1">59</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0051480</td><td align="left" rowspan="1" colspan="1">regulation of cytosolic calcium ion concentration</td><td align="left" rowspan="1" colspan="1">51/1334</td><td align="left" rowspan="1" colspan="1">1.6155E-06</td><td align="left" rowspan="1" colspan="1">0.00039072</td><td align="left" rowspan="1" colspan="1">51</td></tr><tr><td align="left" rowspan="1" colspan="1">BP</td><td align="left" rowspan="1" colspan="1">GO:0007088</td><td align="left" rowspan="1" colspan="1">regulation of mitotic nuclear division</td><td align="left" rowspan="1" colspan="1">30/1334</td><td align="left" rowspan="1" colspan="1">1.6362E-06</td><td align="left" rowspan="1" colspan="1">0.00039072</td><td align="left" rowspan="1" colspan="1">30</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0016323</td><td align="left" rowspan="1" colspan="1">basolateral plasma membrane</td><td align="left" rowspan="1" colspan="1">41/1391</td><td align="left" rowspan="1" colspan="1">5.8276E-09</td><td align="left" rowspan="1" colspan="1">2.6868E-06</td><td align="left" rowspan="1" colspan="1">41</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0097060</td><td align="left" rowspan="1" colspan="1">synaptic membrane</td><td align="left" rowspan="1" colspan="1">63/1391</td><td align="left" rowspan="1" colspan="1">2.9299E-08</td><td align="left" rowspan="1" colspan="1">6.7541E-06</td><td align="left" rowspan="1" colspan="1">63</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0005871</td><td align="left" rowspan="1" colspan="1">kinesin complex</td><td align="left" rowspan="1" colspan="1">17/1391</td><td align="left" rowspan="1" colspan="1">1.2539E-07</td><td align="left" rowspan="1" colspan="1">1.6497E-05</td><td align="left" rowspan="1" colspan="1">17</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0000778</td><td align="left" rowspan="1" colspan="1">condensed nuclear chromosome kinetochore</td><td align="left" rowspan="1" colspan="1">9/1391</td><td align="left" rowspan="1" colspan="1">1.4313E-07</td><td align="left" rowspan="1" colspan="1">1.6497E-05</td><td align="left" rowspan="1" colspan="1">9</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0000779</td><td align="left" rowspan="1" colspan="1">condensed chromosome, centromeric region</td><td align="left" rowspan="1" colspan="1">25/1391</td><td align="left" rowspan="1" colspan="1">5.8004E-07</td><td align="left" rowspan="1" colspan="1">4.3583E-05</td><td align="left" rowspan="1" colspan="1">25</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0000775</td><td align="left" rowspan="1" colspan="1">chromosome, centromeric region</td><td align="left" rowspan="1" colspan="1">34/1391</td><td align="left" rowspan="1" colspan="1">6.3612E-07</td><td align="left" rowspan="1" colspan="1">4.3583E-05</td><td align="left" rowspan="1" colspan="1">34</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0098793</td><td align="left" rowspan="1" colspan="1">presynapse</td><td align="left" rowspan="1" colspan="1">65/1391</td><td align="left" rowspan="1" colspan="1">6.617E-07</td><td align="left" rowspan="1" colspan="1">4.3583E-05</td><td align="left" rowspan="1" colspan="1">65</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0000793</td><td align="left" rowspan="1" colspan="1">condensed chromosome</td><td align="left" rowspan="1" colspan="1">37/1391</td><td align="left" rowspan="1" colspan="1">9.553E-07</td><td align="left" rowspan="1" colspan="1">5.5056E-05</td><td align="left" rowspan="1" colspan="1">37</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0000940</td><td align="left" rowspan="1" colspan="1">condensed chromosome outer kinetochore</td><td align="left" rowspan="1" colspan="1">8/1391</td><td align="left" rowspan="1" colspan="1">1.2298E-06</td><td align="left" rowspan="1" colspan="1">6.2999E-05</td><td align="left" rowspan="1" colspan="1">8</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0000776</td><td align="left" rowspan="1" colspan="1">kinetochore</td><td align="left" rowspan="1" colspan="1">26/1391</td><td align="left" rowspan="1" colspan="1">2.36E-06</td><td align="left" rowspan="1" colspan="1">0.00010881</td><td align="left" rowspan="1" colspan="1">26</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0099572</td><td align="left" rowspan="1" colspan="1">postsynaptic specialization</td><td align="left" rowspan="1" colspan="1">49/1391</td><td align="left" rowspan="1" colspan="1">2.77E-06</td><td align="left" rowspan="1" colspan="1">0.00011553</td><td align="left" rowspan="1" colspan="1">49</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0098978</td><td align="left" rowspan="1" colspan="1">glutamatergic synapse</td><td align="left" rowspan="1" colspan="1">49/1391</td><td align="left" rowspan="1" colspan="1">3.0069E-06</td><td align="left" rowspan="1" colspan="1">0.00011553</td><td align="left" rowspan="1" colspan="1">49</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0000777</td><td align="left" rowspan="1" colspan="1">condensed chromosome kinetochore</td><td align="left" rowspan="1" colspan="1">22/1391</td><td align="left" rowspan="1" colspan="1">3.2962E-06</td><td align="left" rowspan="1" colspan="1">0.0001169</td><td align="left" rowspan="1" colspan="1">22</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:1902495</td><td align="left" rowspan="1" colspan="1">transmembrane transporter complex</td><td align="left" rowspan="1" colspan="1">46/1391</td><td align="left" rowspan="1" colspan="1">4.4261E-06</td><td align="left" rowspan="1" colspan="1">0.00014576</td><td align="left" rowspan="1" colspan="1">46</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0000780</td><td align="left" rowspan="1" colspan="1">condensed nuclear chromosome, centromeric region</td><td align="left" rowspan="1" colspan="1">10/1391</td><td align="left" rowspan="1" colspan="1">5.515E-06</td><td align="left" rowspan="1" colspan="1">0.00016951</td><td align="left" rowspan="1" colspan="1">10</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0016324</td><td align="left" rowspan="1" colspan="1">apical plasma membrane</td><td align="left" rowspan="1" colspan="1">45/1391</td><td align="left" rowspan="1" colspan="1">6.1116E-06</td><td align="left" rowspan="1" colspan="1">0.00017611</td><td align="left" rowspan="1" colspan="1">45</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0034702</td><td align="left" rowspan="1" colspan="1">ion channel complex</td><td align="left" rowspan="1" colspan="1">43/1391</td><td align="left" rowspan="1" colspan="1">7.6656E-06</td><td align="left" rowspan="1" colspan="1">0.00019618</td><td align="left" rowspan="1" colspan="1">43</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0042734</td><td align="left" rowspan="1" colspan="1">presynaptic membrane</td><td align="left" rowspan="1" colspan="1">28/1391</td><td align="left" rowspan="1" colspan="1">7.7805E-06</td><td align="left" rowspan="1" colspan="1">0.00019618</td><td align="left" rowspan="1" colspan="1">28</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:0005604</td><td align="left" rowspan="1" colspan="1">basement membrane</td><td align="left" rowspan="1" colspan="1">20/1391</td><td align="left" rowspan="1" colspan="1">8.4264E-06</td><td align="left" rowspan="1" colspan="1">0.00019618</td><td align="left" rowspan="1" colspan="1">20</td></tr><tr><td align="left" rowspan="1" colspan="1">CC</td><td align="left" rowspan="1" colspan="1">GO:1990351</td><td align="left" rowspan="1" colspan="1">transporter complex</td><td align="left" rowspan="1" colspan="1">46/1391</td><td align="left" rowspan="1" colspan="1">8.51E-06</td><td align="left" rowspan="1" colspan="1">0.00019618</td><td align="left" rowspan="1" colspan="1">46</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0022838</td><td align="left" rowspan="1" colspan="1">substrate-specific channel activity</td><td align="left" rowspan="1" colspan="1">66/1314</td><td align="left" rowspan="1" colspan="1">1.0166E-08</td><td align="left" rowspan="1" colspan="1">3.1517E-06</td><td align="left" rowspan="1" colspan="1">66</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0015267</td><td align="left" rowspan="1" colspan="1">channel activity</td><td align="left" rowspan="1" colspan="1">69/1314</td><td align="left" rowspan="1" colspan="1">1.0332E-08</td><td align="left" rowspan="1" colspan="1">3.1517E-06</td><td align="left" rowspan="1" colspan="1">69</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0022803</td><td align="left" rowspan="1" colspan="1">passive transmembrane transporter activity</td><td align="left" rowspan="1" colspan="1">69/1314</td><td align="left" rowspan="1" colspan="1">1.1313E-08</td><td align="left" rowspan="1" colspan="1">3.1517E-06</td><td align="left" rowspan="1" colspan="1">69</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0005216</td><td align="left" rowspan="1" colspan="1">ion channel activity</td><td align="left" rowspan="1" colspan="1">63/1314</td><td align="left" rowspan="1" colspan="1">4.3919E-08</td><td align="left" rowspan="1" colspan="1">9.1768E-06</td><td align="left" rowspan="1" colspan="1">63</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0008509</td><td align="left" rowspan="1" colspan="1">anion transmembrane transporter activity</td><td align="left" rowspan="1" colspan="1">52/1314</td><td align="left" rowspan="1" colspan="1">1.3809E-07</td><td align="left" rowspan="1" colspan="1">2.3083E-05</td><td align="left" rowspan="1" colspan="1">52</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0022836</td><td align="left" rowspan="1" colspan="1">gated channel activity</td><td align="left" rowspan="1" colspan="1">52/1314</td><td align="left" rowspan="1" colspan="1">6.3595E-07</td><td align="left" rowspan="1" colspan="1">7.6378E-05</td><td align="left" rowspan="1" colspan="1">52</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0046873</td><td align="left" rowspan="1" colspan="1">metal ion transmembrane transporter activity</td><td align="left" rowspan="1" colspan="1">62/1314</td><td align="left" rowspan="1" colspan="1">6.3969E-07</td><td align="left" rowspan="1" colspan="1">7.6378E-05</td><td align="left" rowspan="1" colspan="1">62</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0022839</td><td align="left" rowspan="1" colspan="1">ion gated channel activity</td><td align="left" rowspan="1" colspan="1">49/1314</td><td align="left" rowspan="1" colspan="1">3.4496E-06</td><td align="left" rowspan="1" colspan="1">0.00036039</td><td align="left" rowspan="1" colspan="1">49</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0005261</td><td align="left" rowspan="1" colspan="1">cation channel activity</td><td align="left" rowspan="1" colspan="1">46/1314</td><td align="left" rowspan="1" colspan="1">1.0766E-05</td><td align="left" rowspan="1" colspan="1">0.00099982</td><td align="left" rowspan="1" colspan="1">46</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0008017</td><td align="left" rowspan="1" colspan="1">microtubule binding</td><td align="left" rowspan="1" colspan="1">38/1314</td><td align="left" rowspan="1" colspan="1">1.2906E-05</td><td align="left" rowspan="1" colspan="1">0.00107865</td><td align="left" rowspan="1" colspan="1">38</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0008514</td><td align="left" rowspan="1" colspan="1">organic anion transmembrane transporter activity</td><td align="left" rowspan="1" colspan="1">34/1314</td><td align="left" rowspan="1" colspan="1">1.4699E-05</td><td align="left" rowspan="1" colspan="1">0.00111687</td><td align="left" rowspan="1" colspan="1">34</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:1901618</td><td align="left" rowspan="1" colspan="1">organic hydroxy compound transmembrane transporter activity</td><td align="left" rowspan="1" colspan="1">12/1314</td><td align="left" rowspan="1" colspan="1">6.0364E-05</td><td align="left" rowspan="1" colspan="1">0.00420433</td><td align="left" rowspan="1" colspan="1">12</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0008146</td><td align="left" rowspan="1" colspan="1">sulfotransferase activity</td><td align="left" rowspan="1" colspan="1">13/1314</td><td align="left" rowspan="1" colspan="1">8.0923E-05</td><td align="left" rowspan="1" colspan="1">0.00499379</td><td align="left" rowspan="1" colspan="1">13</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0015631</td><td align="left" rowspan="1" colspan="1">tubulin binding</td><td align="left" rowspan="1" colspan="1">45/1314</td><td align="left" rowspan="1" colspan="1">8.3649E-05</td><td align="left" rowspan="1" colspan="1">0.00499379</td><td align="left" rowspan="1" colspan="1">45</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0003777</td><td align="left" rowspan="1" colspan="1">microtubule motor activity</td><td align="left" rowspan="1" colspan="1">17/1314</td><td align="left" rowspan="1" colspan="1">0.00012412</td><td align="left" rowspan="1" colspan="1">0.0069158</td><td align="left" rowspan="1" colspan="1">17</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0005201</td><td align="left" rowspan="1" colspan="1">extracellular matrix structural constituent</td><td align="left" rowspan="1" colspan="1">26/1314</td><td align="left" rowspan="1" colspan="1">0.00016952</td><td align="left" rowspan="1" colspan="1">0.00885503</td><td align="left" rowspan="1" colspan="1">26</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0004222</td><td align="left" rowspan="1" colspan="1">metalloendopeptidase activity</td><td align="left" rowspan="1" colspan="1">19/1314</td><td align="left" rowspan="1" colspan="1">0.00018593</td><td align="left" rowspan="1" colspan="1">0.00903072</td><td align="left" rowspan="1" colspan="1">19</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0005249</td><td align="left" rowspan="1" colspan="1">voltage-gated potassium channel activity</td><td align="left" rowspan="1" colspan="1">17/1314</td><td align="left" rowspan="1" colspan="1">0.00019449</td><td align="left" rowspan="1" colspan="1">0.00903072</td><td align="left" rowspan="1" colspan="1">17</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0005267</td><td align="left" rowspan="1" colspan="1">potassium channel activity</td><td align="left" rowspan="1" colspan="1">21/1314</td><td align="left" rowspan="1" colspan="1">0.00026951</td><td align="left" rowspan="1" colspan="1">0.01185541</td><td align="left" rowspan="1" colspan="1">21</td></tr><tr><td align="left" rowspan="1" colspan="1">MF</td><td align="left" rowspan="1" colspan="1">GO:0022843</td><td align="left" rowspan="1" colspan="1">voltage-gated cation channel activity</td><td align="left" rowspan="1" colspan="1">23/1314</td><td align="left" rowspan="1" colspan="1">0.00031315</td><td align="left" rowspan="1" colspan="1">0.01308632</td><td align="left" rowspan="1" colspan="1">23</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139883495490352"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/table/tbl2/?report=objectonly">Open in a separate window</a></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl3"><h3>Table 3</h3><!--caption a7--><div class="caption"><p>KEGG analysis results.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">ONTOLOGY</th><th rowspan="1" colspan="1">ID</th><th rowspan="1" colspan="1">Description</th><th rowspan="1" colspan="1">GeneRatio</th><th rowspan="1" colspan="1">BgRatio</th><th rowspan="1" colspan="1">pvalue</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa04080</td><td align="left" rowspan="1" colspan="1">Neuroactive ligand-receptor interaction</td><td align="left" rowspan="1" colspan="1">47/572</td><td align="left" rowspan="1" colspan="1">341/8076</td><td align="left" rowspan="1" colspan="1">6.3794E-06</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa04512</td><td align="left" rowspan="1" colspan="1">ECM-receptor interaction</td><td align="left" rowspan="1" colspan="1">19/572</td><td align="left" rowspan="1" colspan="1">88/8076</td><td align="left" rowspan="1" colspan="1">9.2739E-06</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa04976</td><td align="left" rowspan="1" colspan="1">Bile secretion</td><td align="left" rowspan="1" colspan="1">19/572</td><td align="left" rowspan="1" colspan="1">90/8076</td><td align="left" rowspan="1" colspan="1">1.3074E-05</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa00830</td><td align="left" rowspan="1" colspan="1">Retinol metabolism</td><td align="left" rowspan="1" colspan="1">14/572</td><td align="left" rowspan="1" colspan="1">68/8076</td><td align="left" rowspan="1" colspan="1">0.00023607</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa04974</td><td align="left" rowspan="1" colspan="1">Protein digestion and absorption</td><td align="left" rowspan="1" colspan="1">18/572</td><td align="left" rowspan="1" colspan="1">103/8076</td><td align="left" rowspan="1" colspan="1">0.00028894</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa04911</td><td align="left" rowspan="1" colspan="1">Insulin secretion</td><td align="left" rowspan="1" colspan="1">16/572</td><td align="left" rowspan="1" colspan="1">86/8076</td><td align="left" rowspan="1" colspan="1">0.00029839</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa04110</td><td align="left" rowspan="1" colspan="1">Cell cycle</td><td align="left" rowspan="1" colspan="1">20/572</td><td align="left" rowspan="1" colspan="1">124/8076</td><td align="left" rowspan="1" colspan="1">0.00040995</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa05030</td><td align="left" rowspan="1" colspan="1">Cocaine addiction</td><td align="left" rowspan="1" colspan="1">11/572</td><td align="left" rowspan="1" colspan="1">49/8076</td><td align="left" rowspan="1" colspan="1">0.00050003</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa04060</td><td align="left" rowspan="1" colspan="1">Cytokine-cytokine receptor interaction</td><td align="left" rowspan="1" colspan="1">35/572</td><td align="left" rowspan="1" colspan="1">295/8076</td><td align="left" rowspan="1" colspan="1">0.00167833</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa04918</td><td align="left" rowspan="1" colspan="1">Thyroid hormone synthesis</td><td align="left" rowspan="1" colspan="1">13/572</td><td align="left" rowspan="1" colspan="1">75/8076</td><td align="left" rowspan="1" colspan="1">0.00211463</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa04724</td><td align="left" rowspan="1" colspan="1">Glutamatergic synapse</td><td align="left" rowspan="1" colspan="1">17/572</td><td align="left" rowspan="1" colspan="1">114/8076</td><td align="left" rowspan="1" colspan="1">0.002612</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa04024</td><td align="left" rowspan="1" colspan="1">cAMP signaling pathway</td><td align="left" rowspan="1" colspan="1">27/572</td><td align="left" rowspan="1" colspan="1">216/8076</td><td align="left" rowspan="1" colspan="1">0.00264907</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG</td><td align="left" rowspan="1" colspan="1">hsa00591</td><td align="left" rowspan="1" colspan="1">Linoleic acid metabolism</td><td align="left" rowspan="1" colspan="1">7/572</td><td align="left" rowspan="1" colspan="1">29/8076</td><td align="left" rowspan="1" colspan="1">0.00340721</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139883511262448"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/table/tbl3/?report=objectonly">Open in a separate window</a></div></div><p id="p0140" class="p p-last">GSEA was further explored, and the results of the pathway database were summarized on the basis of the KEGG results. The results showed the significant enrichment of fatty acid, glycine, serine, threonine and retinol metabolism, primary bile acid biosynthesis and other hallmarks in the group with high risk (<a href="/pmc/articles/PMC11058304/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>A). Additionally, the results indicated the significant enrichment of ECM- and neuroactive ligand-receptor interactions, o-glycan biosynthesis and cell cycle in the group with low risk (<a href="/pmc/articles/PMC11058304/figure/fig5/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig5" rid-ob="ob-fig5" co-legend-rid="lgnd_fig5"><span>Fig. 5</span></a>B). <a href="/pmc/articles/PMC11058304/table/tbl4/" target="table" class="fig-table-link figpopup" rid-figpopup="tbl4" rid-ob="ob-tbl4" co-legend-rid=""><span>Table 4</span></a> presents the detailed GSEA results of metabolism-associated pathways.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig5" co-legend-rid="lgnd_fig5"><a href="/pmc/articles/PMC11058304/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139883509593936"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11058304_gr5.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 5" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11058304/bin/gr5.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139883509593936"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/figure/fig5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig5"><div><a class="figpopup" href="/pmc/articles/PMC11058304/figure/fig5/" target="figure" rid-figpopup="fig5" rid-ob="ob-fig5">Fig. 5</a></div><!--caption a7--><div class="caption"><p>GSEA analysis. (A) Significant enrichment of primary bile acid biosynthesis, fatty acid, glycine, serine, threonine and retinol metabolism, etc. in the high-risk group. (B) Significant enrichment of ECM- and neuroactive ligand-receptor interactions, o-glycan biosynthesis, cell cycle, etc. in the low-risk group.</p></div></div></div><!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="tbl4"><h3>Table 4</h3><!--caption a7--><div class="caption"><p>GSEA analysis results.</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th rowspan="1" colspan="1">ID</th><th rowspan="1" colspan="1">setSize</th><th rowspan="1" colspan="1">NES</th><th rowspan="1" colspan="1">p.adjust</th><th rowspan="1" colspan="1">qvalue</th></tr></thead><tbody><tr><td align="left" rowspan="1" colspan="1">KEGG_FATTY_ACID_METABOLISM</td><td align="left" rowspan="1" colspan="1">39</td><td align="left" rowspan="1" colspan="1">4.362864</td><td align="left" rowspan="1" colspan="1">8.75E-09</td><td align="left" rowspan="1" colspan="1">7.00E-09</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_PRIMARY_BILE_ACID_BIOSYNTHESIS</td><td align="left" rowspan="1" colspan="1">16</td><td align="left" rowspan="1" colspan="1">3.427042</td><td align="left" rowspan="1" colspan="1">8.75E-09</td><td align="left" rowspan="1" colspan="1">7.00E-09</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_RETINOL_METABOLISM</td><td align="left" rowspan="1" colspan="1">49</td><td align="left" rowspan="1" colspan="1">3.066979</td><td align="left" rowspan="1" colspan="1">1.42E-07</td><td align="left" rowspan="1" colspan="1">1.14E-07</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_GLYCINE_SERINE_AND_THREONINE_METABOLISM</td><td align="left" rowspan="1" colspan="1">29</td><td align="left" rowspan="1" colspan="1">3.084273</td><td align="left" rowspan="1" colspan="1">2.24E-06</td><td align="left" rowspan="1" colspan="1">1.79E-06</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_PRIMARY_IMMUNODEFICIENCY</td><td align="left" rowspan="1" colspan="1">25</td><td align="left" rowspan="1" colspan="1">3.061342</td><td align="left" rowspan="1" colspan="1">4.80E-06</td><td align="left" rowspan="1" colspan="1">3.84E-06</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_NEUROACTIVE_LIGAND_RECEPTOR_INTERACTION</td><td align="left" rowspan="1" colspan="1">143</td><td align="left" rowspan="1" colspan="1">&#x02212;1.66461</td><td align="left" rowspan="1" colspan="1">3.12E-05</td><td align="left" rowspan="1" colspan="1">2.50E-05</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_VALINE_LEUCINE_AND_ISOLEUCINE_DEGRADATION</td><td align="left" rowspan="1" colspan="1">39</td><td align="left" rowspan="1" colspan="1">2.542273</td><td align="left" rowspan="1" colspan="1">8.39E-05</td><td align="left" rowspan="1" colspan="1">6.71E-05</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_ECM_RECEPTOR_INTERACTION</td><td align="left" rowspan="1" colspan="1">66</td><td align="left" rowspan="1" colspan="1">&#x02212;1.8386</td><td align="left" rowspan="1" colspan="1">8.58E-05</td><td align="left" rowspan="1" colspan="1">6.86E-05</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_LINOLEIC_ACID_METABOLISM</td><td align="left" rowspan="1" colspan="1">22</td><td align="left" rowspan="1" colspan="1">2.678666</td><td align="left" rowspan="1" colspan="1">0.000336</td><td align="left" rowspan="1" colspan="1">0.000269</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_TRYPTOPHAN_METABOLISM</td><td align="left" rowspan="1" colspan="1">32</td><td align="left" rowspan="1" colspan="1">2.503453</td><td align="left" rowspan="1" colspan="1">0.000414</td><td align="left" rowspan="1" colspan="1">0.000331</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_PROPANOATE_METABOLISM</td><td align="left" rowspan="1" colspan="1">27</td><td align="left" rowspan="1" colspan="1">2.385105</td><td align="left" rowspan="1" colspan="1">0.001341</td><td align="left" rowspan="1" colspan="1">0.001073</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_BUTANOATE_METABOLISM</td><td align="left" rowspan="1" colspan="1">28</td><td align="left" rowspan="1" colspan="1">2.401353</td><td align="left" rowspan="1" colspan="1">0.001459</td><td align="left" rowspan="1" colspan="1">0.001167</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_TYROSINE_METABOLISM</td><td align="left" rowspan="1" colspan="1">30</td><td align="left" rowspan="1" colspan="1">2.27098</td><td align="left" rowspan="1" colspan="1">0.002981</td><td align="left" rowspan="1" colspan="1">0.002385</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_RENIN_ANGIOTENSIN_SYSTEM</td><td align="left" rowspan="1" colspan="1">14</td><td align="left" rowspan="1" colspan="1">2.110067</td><td align="left" rowspan="1" colspan="1">0.008246</td><td align="left" rowspan="1" colspan="1">0.006597</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_STEROID_HORMONE_BIOSYNTHESIS</td><td align="left" rowspan="1" colspan="1">43</td><td align="left" rowspan="1" colspan="1">1.985456</td><td align="left" rowspan="1" colspan="1">0.012971</td><td align="left" rowspan="1" colspan="1">0.010377</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_ASTHMA</td><td align="left" rowspan="1" colspan="1">15</td><td align="left" rowspan="1" colspan="1">1.965837</td><td align="left" rowspan="1" colspan="1">0.015989</td><td align="left" rowspan="1" colspan="1">0.012791</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_O_GLYCAN_BIOSYNTHESIS</td><td align="left" rowspan="1" colspan="1">23</td><td align="left" rowspan="1" colspan="1">&#x02212;1.77524</td><td align="left" rowspan="1" colspan="1">0.015989</td><td align="left" rowspan="1" colspan="1">0.012791</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_BETA_ALANINE_METABOLISM</td><td align="left" rowspan="1" colspan="1">20</td><td align="left" rowspan="1" colspan="1">2.002323</td><td align="left" rowspan="1" colspan="1">0.021878</td><td align="left" rowspan="1" colspan="1">0.017502</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_HISTIDINE_METABOLISM</td><td align="left" rowspan="1" colspan="1">25</td><td align="left" rowspan="1" colspan="1">1.977649</td><td align="left" rowspan="1" colspan="1">0.026026</td><td align="left" rowspan="1" colspan="1">0.02082</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_PROTEASOME</td><td align="left" rowspan="1" colspan="1">37</td><td align="left" rowspan="1" colspan="1">1.8651</td><td align="left" rowspan="1" colspan="1">0.026026</td><td align="left" rowspan="1" colspan="1">0.02082</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_DRUG_METABOLISM_OTHER_ENZYMES</td><td align="left" rowspan="1" colspan="1">47</td><td align="left" rowspan="1" colspan="1">1.840722</td><td align="left" rowspan="1" colspan="1">0.026026</td><td align="left" rowspan="1" colspan="1">0.02082</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_CELL_CYCLE</td><td align="left" rowspan="1" colspan="1">121</td><td align="left" rowspan="1" colspan="1">&#x02212;1.40341</td><td align="left" rowspan="1" colspan="1">0.029283</td><td align="left" rowspan="1" colspan="1">0.023427</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_SYSTEMIC_LUPUS_ERYTHEMATOSUS</td><td align="left" rowspan="1" colspan="1">42</td><td align="left" rowspan="1" colspan="1">1.811271</td><td align="left" rowspan="1" colspan="1">0.045285</td><td align="left" rowspan="1" colspan="1">0.036228</td></tr><tr><td align="left" rowspan="1" colspan="1">KEGG_ARACHIDONIC_ACID_METABOLISM</td><td align="left" rowspan="1" colspan="1">45</td><td align="left" rowspan="1" colspan="1">1.765372</td><td align="left" rowspan="1" colspan="1">0.045285</td><td align="left" rowspan="1" colspan="1">0.036228</td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm139883509590752"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/table/tbl4/?report=objectonly">Open in a separate window</a></div></div></div><div id="sec3.5" class="sec"><h3 id="sec3.5title">3.5. Analysis of PPI and regulation networks</h3><p id="p0145" class="p p-first">Hub genes playing a key role were further clarified, and an attempt was made to reveal their potential molecular interaction mechanisms. The STRING database was used for analyzing the PPI mechanism. It can be seen from <a href="/pmc/articles/PMC11058304/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6</span></a>A that the number of PPI nodes (proteins) was 38 after confidence&#x000a0;=&#x000a0;0.400 screening; the number of linkages (edges) was 48; the average connectivity per node was 2.53; the whole PPI had an enrichment statistic P value of below 1.0e-16.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig6" co-legend-rid="lgnd_fig6"><a href="/pmc/articles/PMC11058304/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139883501241504"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11058304_gr6.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 6" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11058304/bin/gr6.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139883501241504"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/figure/fig6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig6"><div><a class="figpopup" href="/pmc/articles/PMC11058304/figure/fig6/" target="figure" rid-figpopup="fig6" rid-ob="ob-fig6">Fig. 6</a></div><!--caption a7--><div class="caption"><p>Analysis of PPI regulatory network. (A) PPI regulatory network with specific information on its nodes, connecting lines and all sub-networks. (B) Calculation of hub gene regulatory network on the basis of CytoHubba. (C) Prediction of miRNA-hub gene regulatory network using the mirTarbase database.</p></div></div></div><p id="p0150" class="p p-last">Afterwards, the plugin CytoHubba in Cytoscape was utilized for further clarifying the proteins in which interactions occur to serve as hub genes. After calculation, it was found that GPR119, cannabinoid receptor 1 (CNR1), arginine vasopressin receptor 1B (AVPR1B), 5-hydroxytryptamine receptor 1A (HTR1A) and thyrotropin-releasing hormone receptor (TRHR) were the highest scoring interacting proteins (<a href="/pmc/articles/PMC11058304/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6</span></a>B). After that, miRNA molecules potentially regulating these hub genes were analyzed using the mirTarbase database, and Cytoscape was employed for constructing miRNA-target gene regulatory networks (<a href="/pmc/articles/PMC11058304/figure/fig6/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig6" rid-ob="ob-fig6" co-legend-rid="lgnd_fig6"><span>Fig. 6</span></a>C).</p></div><div id="sec3.6" class="sec"><h3 id="sec3.6title">3.6. Immune infiltration analysis of groups with high and low risks</h3><p id="p0155" class="p p-first-last">Next, the associations of GPCR-related risk scores with the level of different immune cell infiltrates and overall immune characteristics in HCC patients were evaluated. When a comparison was made between the immune cell infiltration in groups with high and low risks, it was found that only naive B cells were significantly different in both groups, but any other immune cells were not statistically and significantly different (<a href="/pmc/articles/PMC11058304/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>A). The analysis of correlation between immune cells showed that macrophages, monocytes and B cells accountable for antigen presentation were not significantly correlated with CD4 and CD8 T cells responsible for killing (<a href="/pmc/articles/PMC11058304/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>B). According to the ESTIMATE results, groups with high and low risks were statistically and significantly different in the cellular infiltration level of the tumor microenvironment and immune correlation scores; the immune and stromal cell infiltration of the high-risk group was significantly lower than that of the low-risk group; the high-risk group exhibited higher tumor purity than the low-risk one (<a href="/pmc/articles/PMC11058304/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>C). Further, the correlation of patients&#x02019; risk scores with immune cells was compared. It was discovered that risk scores presented a positive trend with cells like M0 macrophages and memory B cells and a negative trend with M1 macrophages, CD8 T and naive B cells, etc. Nonetheless, the degree of these correlations was low and not statistically significant (<a href="/pmc/articles/PMC11058304/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>D). Subsequently, a comparison was made between the expression of immune checkpoint-related genes in both groups. It was noted that a number of important immune checkpoint genes, including a cluster of differentiation 8a (CD8A), hepatitis A virus cellular receptor 2 (HAVCR2), chemokine (C-X-C motif) ligand 10 (CXCL10), CXCL9, Perforin 1 (PRF1), etc., were differentially expressed in both groups (<a href="/pmc/articles/PMC11058304/figure/fig7/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig7" rid-ob="ob-fig7" co-legend-rid="lgnd_fig7"><span>Fig. 7</span></a>E). The above results suggested that GPCR risk scores were not significantly associated with the tumor microenvironment, which highly indicated that the expression of GPCR-related DEGs mainly affects the changes in the tumor itself and has less effect on immune or stromal cells.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig7" co-legend-rid="lgnd_fig7"><a href="/pmc/articles/PMC11058304/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139883505365360"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11058304_gr7.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 7" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11058304/bin/gr7.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139883505365360"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/figure/fig7/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig7"><div><a class="figpopup" href="/pmc/articles/PMC11058304/figure/fig7/" target="figure" rid-figpopup="fig7" rid-ob="ob-fig7">Fig. 7</a></div><!--caption a7--><div class="caption"><p>Relationships between GPCR-related risk scores with different infiltration levels of immune cells. (A) Analysis of differences in the levels of 22 different immune cell infiltrates between both groups. (B) Panoramic analysis of immune cell infiltration and immune cell correlation analysis in the LIHC dataset. (C) Differences in stromal cells and ESTIMATE and immune scores between both groups. (D) Bar chart of the correlation analysis of GPCR-related risk scores with immune cell infiltration. (E) Differential expression analysis of different immune checkpoint genes between both subgroups (*P&#x000a0;&#x0003c;&#x000a0;0.05, **P&#x000a0;&#x0003c;&#x000a0;0.01 and ***P&#x000a0;&#x0003c;&#x000a0;0.001).</p></div></div></div></div><div id="sec3.7" class="sec"><h3 id="sec3.7title">3.7. Effect of GPCR-related risk scores on genomic changes in HCC patients</h3><p id="p0160" class="p p-first-last">Later, the effect of GPCR-related risk scores on changes in the level of genetic variation was further evaluated, like SNPs and CNVs. The SNP mutation analysis results of driver genes in common tumorigenesis revealed that both groups of patients demonstrated similar mutation levels (<a href="/pmc/articles/PMC11058304/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span>Fig. 8</span></a>A-B). In the comparison of TMB and MSI, TMB showed no significant difference (<a href="/pmc/articles/PMC11058304/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span>Fig. 8</span></a>C), while MSI exhibited an elevation in the high-risk group (<a href="/pmc/articles/PMC11058304/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span>Fig. 8</span></a>D). The study of CNV alteration frequency showed widespread CNV in both groups of patients but was not significantly different (<a href="/pmc/articles/PMC11058304/figure/fig8/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig8" rid-ob="ob-fig8" co-legend-rid="lgnd_fig8"><span>Fig. 8</span></a>E-F).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig8" co-legend-rid="lgnd_fig8"><a href="/pmc/articles/PMC11058304/figure/fig8/" target="figure" rid-figpopup="fig8" rid-ob="ob-fig8"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139883509322944"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11058304_gr8.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 8" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11058304/bin/gr8.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139883509322944"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/figure/fig8/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig8"><div><a class="figpopup" href="/pmc/articles/PMC11058304/figure/fig8/" target="figure" rid-figpopup="fig8" rid-ob="ob-fig8">Fig. 8</a></div><!--caption a7--><div class="caption"><p>Impact of GPCR-related risk score groupings on immunotherapy and genetic variation in patients with HCC. (A&#x02013;B) Common tumorigenic driver gene mutation profiles of patients in both groups. The information on the mutation of genes in samples is illustrated in the waterfall plot, in which a variety of colors indicate different types of mutation; the mutational load is denoted by the vignettes above the legend. (C&#x02013;D) Differences in TMB and MSI levels between patients in both groups; the high-risk group has significantly higher TMB and MSI levels than the low-risk one. (E&#x02013;F) Copy number levels of genes in patients in both groups; red represents genes with a significant increase in copy number and blue stands for genes with a significant decrease in copy number; more copy number amplification and missing fragments are seen in the low-risk group (*P&#x000a0;&#x0003c;&#x000a0;0.05, **P&#x000a0;&#x0003c;&#x000a0;0.01 and ***P&#x000a0;&#x0003c;&#x000a0;0.001). (For interpretation of the references to color in this figure legend, the reader is referred to the Web version of this article.)</p></div></div></div></div><div id="sec3.8" class="sec"><h3 id="sec3.8title">3.8. Differences in drug sensitivity between both groups of patients</h3><p id="p0165" class="p p-first">Then, the sensitivity phenotypes of commonly used antitumor drugs between groups with high and low risks were evaluated via the GDSC database. Firstly, GDSC data were taken as a training set, and the pRRophetic package was used to construct a ridge regression model. After that, the data of the TCGA-LIHC dataset were input for testing. As revealed by the test results, 51 out of 138 input drugs were observed to show statistical differences between both groups. Then, boxplots were used to plot the six most statistically different drugs and small molecule compounds. Among these drugs and small molecule compounds, BIRB.0796 and all-trans retinoic acid (ATRA) showed higher sensitivity to the high-risk group of patients, while AZD8055, EHT.1864, PF.4708671 and GW.441756 showed higher sensitivity to the low-risk group of patients (<a href="/pmc/articles/PMC11058304/figure/fig9/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig9" rid-ob="ob-fig9" co-legend-rid="lgnd_fig9"><span>Fig. 9</span></a>A&#x02013;F).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig9" co-legend-rid="lgnd_fig9"><a href="/pmc/articles/PMC11058304/figure/fig9/" target="figure" rid-figpopup="fig9" rid-ob="ob-fig9"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139883513137840"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11058304_gr9.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 9" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11058304/bin/gr9.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139883513137840"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/figure/fig9/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig9"><div><a class="figpopup" href="/pmc/articles/PMC11058304/figure/fig9/" target="figure" rid-figpopup="fig9" rid-ob="ob-fig9">Fig. 9</a></div><!--caption a7--><div class="caption"><p>Drug sensitivity analysis of patients in both groups in the GPCR-associated risk score model. (A&#x02013;F) Top six drugs in terms of the P value of IC50 based on the Wilcox test between both groups; they are all less sensitive in the high-risk group. (G&#x02013;I) Differences in TIDE, dysfunction and exclusive scores between both groups; this suggests the possibility that the high-risk group shows better responsiveness to immunotherapy (*P&#x000a0;&#x0003c;&#x000a0;0.05, **P&#x000a0;&#x0003c;&#x000a0;0.01 and ***P&#x000a0;&#x0003c;&#x000a0;0.001).</p></div></div></div><p id="p0170" class="p p-last">Considering that immunotherapy plays a vital part in HCC treatment currently, the TIDE algorithm was adopted to evaluate the sensitivity of patients in both groups to immunotherapy. As shown in <a href="/pmc/articles/PMC11058304/figure/fig9/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig9" rid-ob="ob-fig9" co-legend-rid="lgnd_fig9"><span>Fig. 9</span></a>G&#x02013;I, the high-risk group exhibited lower TIDE scores than the low-risk one, which indicated that patients in the high-risk group might have better responsiveness to immunotherapy than those in the low-risk group. Dysfunction and exclusion scores were combined, which demonstrated immune dysfunction and possibly stronger immune escape in the group with high risk. Through the combination of previous analyses that the high-risk group exhibited an MSI score, it was inferred that the responsiveness of patients in the group with high risk to immunotherapy might be better.</p></div><div id="sec3.9" class="sec sec-last"><h3 id="sec3.9title">3.9. Clinical prediction model construction based on GPCR-associated risk scores</h3><p id="p0175" class="p p-first">To further examine the potential clinical significance of GPCR-associated risk scores, the composition of gender, age and TNM stage among both groups of patients in the TCGA-LIHC dataset was analyzed first. It was noticed that the age and gender composition of patients in both groups were not significantly different, but the results of the staging comparison demonstrated that the low-risk group had a higher proportion of stage I patients than the high-risk one (<a href="/pmc/articles/PMC11058304/figure/fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig10" rid-ob="ob-fig10" co-legend-rid="lgnd_fig10"><span>Fig. 10</span></a>A&#x02013;C).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="fig10" co-legend-rid="lgnd_fig10"><a href="/pmc/articles/PMC11058304/figure/fig10/" target="figure" rid-figpopup="fig10" rid-ob="ob-fig10"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm139883500163248"><a class="inline_block ts_canvas" href="/core/lw/2.0/html/tileshop_pmc/tileshop_pmc_inline.html?title=Click%20on%20image%20to%20zoom&amp;p=PMC3&amp;id=11058304_gr10.jpg" target="tileshopwindow" rel="noopener"><div class="ts_bar small" title="Click on image to zoom"></div><img loading="lazy" alt="Fig. 10" title="Click on image to zoom" class="tileshop" src="/pmc/articles/PMC11058304/bin/gr10.jpg" /></a></div><div class="largeobj-link align_right" id="largeobj_idm139883500163248"><a target="object" rel="noopener" href="/pmc/articles/PMC11058304/figure/fig10/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_fig10"><div><a class="figpopup" href="/pmc/articles/PMC11058304/figure/fig10/" target="figure" rid-figpopup="fig10" rid-ob="ob-fig10">Fig. 10</a></div><!--caption a7--><div class="caption"><p>Predictive power analysis of GPCR-related risk scores on the prognosis of patients with HCC. (A&#x02013;C) Superimposed histograms showing the proportion of gender, age and stage among patients in both groups. Both groups are similar in the proportion of age and gender, and the low-risk group has significantly more early-stage patients than the high-risk one. (D) Nomogram of a clinical prediction model on the basis of GPCR-related risk scores in combination with clinicopathological characteristics. (E) Calibration curve of the nomogram; horizontal and vertical coordinates are the nomogram-predicted and actual observed survival statuses, respectively. Curves show that the nomogram model has good predictive value at one, three and five years.</p></div></div></div><p id="p0180" class="p p-last">Subsequently, a prognostic model was built based on the GPCR-associated risk scores and clinicopathological features (gender, age and TNM stage) of HCC patients and visualized in the nomogram (<a href="/pmc/articles/PMC11058304/figure/fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig10" rid-ob="ob-fig10" co-legend-rid="lgnd_fig10"><span>Fig. 10</span></a>D). Calibration curves were used for assessing model accuracy, and the modeled one-, three- and three-year OS estimates were found to well agree with the actual observations of patients (<a href="/pmc/articles/PMC11058304/figure/fig10/" target="figure" class="fig-table-link figpopup" rid-figpopup="fig10" rid-ob="ob-fig10" co-legend-rid="lgnd_fig10"><span>Fig. 10</span></a>E).</p></div></div><div id="sec4" class="tsec sec"><h2 class="head no_bottom_margin" id="sec4title">4. Discussion</h2><p id="p0185" class="p p-first">Systemic therapy benefits a small proportion of patients despite having made significant advancements and playing an increasingly crucial role in treating HCC. The lack of direct targets may be one of the reasons for low response rates, which thus makes it critical to identify potential target drugs. GPCRs, the most prominent drug targets among FDA-approved drugs, have played a significant role in HCC development and progression. Nevertheless, specific mechanisms have not been extensively studied, which hence offers significant potential and research value.</p><p id="p0190">In the present study, the focus was placed on the association of GPCRs with HCC. The clinical and expression data of 595 HCC patients were obtained from TCGA and LIRI-JP databases, and the focus was put on GPCR-related gene expression with the assistance of GPCR gene sets downloaded from MsigDB. LASSO and multivariant Cox regression analyses were followed by the identification of eight GPCR-related genes and the establishment of a GPCR-related risk score. Among these eight genes, NLRP6 was the only gene whose expression level was negatively related to the risk score. HCC patients were categorized into groups with high and low risks. Patients in the high-risk group showed a significantly worse prognosis. Differential expression analyses were performed, and significant differences in cell proliferation and differentiation, cell membrane channel activity, and metabolism were found. This conclusion further confirmed the effectiveness of the risk score model. PPI mechanisms were further explored. Hub genes and miRNA-target gene regulatory networks were constructed to see protein interactions. Five miRNAs including miR-155-5p, miR-135a-5p, miR-494-3p, miR-34a-5p and miR-410-3p were predicted as potentially regulating miRNAs, of which four [<a href="#bib40" rid="bib40" class=" bibr popnode">40</a>,<a href="#bib41" rid="bib41" class=" bibr popnode">41</a>] (miR-155-5p, miR-135a-5p, miR-494-3p and miR-34a-5p) had already proved to act as regulators in HCC. In addition, miR-410-3p was demonstrated to promote or suppress tumor progression in prostate [<a href="#bib42" rid="bib42" class=" bibr popnode">42</a>] and breast [<a href="#bib43" rid="bib43" class=" bibr popnode">43</a>] as well. This further validated the accuracy of the risk model indirectly. Immune infiltration level was evaluated, and the number of immune and stromal cells was found to be lower in the high-risk group. The high-risk group exhibited lower TIDE scores as well. Together, these suggested that patients in the high-risk group tended to respond to immunotherapy. The sensitivity of common HCC treatment drugs was evaluated. BIRB.0796 (p38 mitogen-activated protein kinase (MAPK) inhibitor) and ATRA showed higher sensitivity to the high-risk group of patients. However, AZD8055 (mammalian target of rapamycin (mTOR) inhibitor), EHT.1864 (Ras-related C3 botulinum toxin substrate (Rac1) inhibitor), PF.4708671 (ribosomal protein S6 kinase beta-1 (S6K1) inhibitor), and GW.441756 (tyrosine kinase-A (TrkA) inhibitor) demonstrated higher sensitivity to the low-risk group of patients. Finally, a clinical prediction model on the basis of GPCR-related risk scores was established, with excellent predictive power for the OS of patients with HCC.</p><p id="p0195">Subsequently, in-depth research was conducted on eight genes in the predictive model. It was confirmed that six of the eight genes played a part in the occurrence, development, immune regulation or prognosis prediction of HCC. Also called CXCR, ACKR3 mainly serves as a scavenger receptor for CXCL12 [<a href="#bib44" rid="bib44" class=" bibr popnode">44</a>]. Research has shown that CXCR7 overexpression was linked to the differentiation [<a href="#bib45" rid="bib45" class=" bibr popnode">45</a>] and proliferation of HCC, and regulated the expression of vascular endothelial growth factor A (VEGFA) and galectin-3 to result in tumor angiogenesis, cell invasiveness, as well as metastasis [<a href="#bib46" rid="bib46" class=" bibr popnode">46</a>,<a href="#bib47" rid="bib47" class=" bibr popnode">47</a>]. Additionally, ACKR3 controlled the secretion of various inflammatory factors and induced immune escape by regulating M2 macrophage migration in HCC [<a href="#bib47" rid="bib47" class=" bibr popnode">47</a>]. Primarily expressed on the surface of eosinophils, CCR3 binds and responds to CCL11, CCL24 and CCL26, respectively [<a href="#bib48" rid="bib48" class=" bibr popnode">48</a>]. It was observed that CCL26 showed a strong relationship with HCC proliferation, migration, invasion and angiogenesis [<a href="#bib49" rid="bib49" class=" bibr popnode">49</a>], and CCL24 caused metastasis and neovascularization through the RhoB-VEGFA-VEGFR2 pathway and demonstrated a poor prognosis in HCC [<a href="#bib50" rid="bib50" class=" bibr popnode">50</a>]. CCR7 mainly binds to CCL19 and CCL21 [<a href="#bib51" rid="bib51" class=" bibr popnode">51</a>], whose axis often recruits regulatory T cells and tumor-infiltrating lymphocytes into the tumor niche and contributes to the improvement of prognosis [<a href="#bib51" rid="bib51" class=" bibr popnode">51</a>]. The research results of CCR7 in HCC signified that CCR7 facilitated the progression of EMT, enhanced the invasion and proliferation of tumor cells, and demonstrated a relationship with lymphatic and intrahepatic dissemination [<a href="#bib52" rid="bib52" class=" bibr popnode">52</a>,<a href="#bib53" rid="bib53" class=" bibr popnode">53</a>]. It also increased levels of IL-10 and transforming growth factor-&#x003b2;1 (TGF-&#x003b2;1), decreased levels of IL-12 and interferon-gamma (IFN-&#x003b3;), and played a role in shaping the tumor microenvironment [<a href="#bib53" rid="bib53" class=" bibr popnode">53</a>]. FZD5 (Frizzled-5) belongs to one of the Frizzled receptor subtypes, namely wingless (Wnt) receptors, mediates the Wnt/<strong><em>&#x003b2;</em></strong>-catenin signaling pathway, activates liver cancer stem cells, and facilitates the progression [<a href="#bib54" rid="bib54" class=" bibr popnode">54</a>] and metastasis of HCC [<a href="#bib55" rid="bib55" class=" bibr popnode">55</a>]. HCAR1 is also called HCA1 or GPR81 and is viewed as a lactate receptor. It has been found that HCAR1 is overexpressed in HCC, and regulates angiogenesis and growth induced by tumor lactate [<a href="#bib56" rid="bib56" class=" bibr popnode">56</a>,<a href="#bib57" rid="bib57" class=" bibr popnode">57</a>]. NLRP6, a novel member of the nucleotide binding domain and leucine-rich repeat-containing (NLR) family, shows abundant expression in the liver and intestine and plays an essential role in regulating inflammation and shaping intestine microbiota [<a href="#bib58" rid="bib58" class=" bibr popnode">58</a>]. It is demonstrated that NLRP6 drives the progression of liver diseases by causing dysbiosis of the intestine, and its expression level is strongly related to the prognosis of HCC by regulating the progress of pyroptosis [<a href="#bib59" rid="bib59" class=" bibr popnode">59</a>,<a href="#bib60" rid="bib60" class=" bibr popnode">60</a>]. GRM8 modulates glutamatergic and gamma-aminobutyric acid (GABA) neurotransmission, and HTR5A belongs to the 5-HT receptor family and is deeply involved in regulating the nervous system [<a href="#bib61" rid="bib61" class=" bibr popnode">61</a>,<a href="#bib62" rid="bib62" class=" bibr popnode">62</a>].</p><p id="p0200">Most of the GPCRs included in the risk score model used in this study are either FDA-approved drug targets or drug targets in the phase of clinical trials. Small molecule inhibitors of ACKR3 were manufactured, including CCX771, CCX754 and CCX733 [<a href="#bib63" rid="bib63" class=" bibr popnode">63</a>], among which the former two had already shown the capability of reducing nearly half of colorectal carcinoma lung metastases in vivo [<a href="#bib64" rid="bib64" class=" bibr popnode">64</a>]. Maraviroc, including CCR5 and CCR3 antagonists, has been approved for the treatment of human immunodeficiency virus (HIV) infection [<a href="#bib65" rid="bib65" class=" bibr popnode">65</a>]. CCR3 antagonist <a id="__tag_1147621403" class="tag_hotlink" href="/nuccore/GW766994" ref="/nuccore/GW766994"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"GW766994","term_id":"292553463","term_text":"GW766994"}}</span>GW766994</a> showed safety and efficacy in asthma and eosinophilic bronchitis [<a href="#bib66" rid="bib66" class=" bibr popnode">66</a>]. HTR5A antagonist SB-699551 has been proven to reduce the survival of breast tumor-initiating cells [<a href="#bib67" rid="bib67" class=" bibr popnode">67</a>]. UCSF678, an arrestin-biased partial agonist, was developed with a deeper understanding of the HTR5A structure. MSOP, microtubule-associated protein 4 (MAP4), (RS)-&#x003b1;-Cyclopropyl-4-phosphonophenylglycine (CPPG), UBP111 and <a id="__tag_1147621404" class="tag_hotlink" href="/nuccore/LY341495" ref="/nuccore/LY341495"><span class="tag-json" style="display:none">{"type":"entrez-nucleotide","attrs":{"text":"LY341495","term_id":"1257705759","term_text":"LY341495"}}</span>LY341495</a> were manufactured as GRM8 antagonists recently. The crystal structure of CCR7 was elucidated. Thiazole-dioxide, thiadiazole-dioxide or cyclobutene-dione motifs are crucial for interaction with a conservative transmembrane helix 7-helix 8 (TM7-H8) protein patch in the binding site of Gi protein in chemokine receptors like CCR7 [<a href="#bib68" rid="bib68" class=" bibr popnode">68</a>].</p><p id="p0205">Some limitations exist in this study. Firstly, validation and training datasets had no large sample sizes, which may limit the statistical power of data analysis. Secondly, it is necessary to conduct extra experimental validations to expound the mechanisms of DEGs identified. Further studies in diverse populations or settings would be beneficial to validating and extending the scope of the research results.</p><p id="p0210" class="p p-last">It must be admitted that GPCRs in oncology are a relatively new topic and many underlying mechanisms remain unclear and need much further exploration compared with GPCRs in metabolic and physical disorders, as well as infectious diseases. Currently, some drugs for upstream or downstream targets have already been from GPCRs although not many GPCR-targeted drugs are directly used for antitumor therapy [<a href="#bib69" rid="bib69" class=" bibr popnode">69</a>]. New avenues for the discovery of GPCR drugs have appeared thanks to recent advancements in receptor pharmacology, biotechnological innovations and breakthroughs in structural biology. Therefore, investigating the relationship between GPCRs and HCC further provides insights into both GPCR and GPCR downstream targets, which thus expands the repertoire for future HCC treatments.</p></div><div id="sec5" class="tsec sec"><h2 class="head no_bottom_margin" id="sec5title">Ethics statement</h2><p id="p0215" class="p p-first-last">All data used in the study was obtained from public databases, hence, ethics approval and informed consent were not required.</p></div><div id="sec6" class="tsec sec"><h2 class="head no_bottom_margin" id="sec6title">Consent to participate</h2><p id="p0220" class="p p-first-last">Not applicable.</p></div><div id="sec7" class="tsec sec"><h2 class="head no_bottom_margin" id="sec7title">Consent for publication</h2><p id="p0225" class="p p-first-last">The authors declare that they agree to submit the article for publication.</p></div><div id="sec8" class="tsec sec"><h2 class="head no_bottom_margin" id="sec8title">Funding</h2><p id="p0230" class="p p-first-last">This work was supported by National High Level Hospital Clinical Research Funding (No. 2022-PUMCH-C-047).</p></div><div id="sec9" class="tsec sec"><h2 class="head no_bottom_margin" id="sec9title">Data availability statement</h2><p id="p0235" class="p p-first-last">The names of the public repositories and their relevant accession links are detailed in the Materials and Methods section.</p></div><div id="sec10" class="tsec sec"><h2 class="head no_bottom_margin" id="sec10title">CRediT authorship contribution statement</h2><p id="p0240" class="p p-first-last"><strong>Yuxin Wang:</strong> Writing &#x02013; original draft, Methodology, Investigation, Formal analysis, Data curation, Conceptualization. <strong>Bao Jin:</strong> Supervision, Methodology, Investigation. <strong>Xiangan Wu:</strong> Visualization, Methodology, Formal analysis. <strong>Jiali Xing:</strong> Visualization, Validation, Methodology. <strong>Baoluhe Zhang:</strong> Validation, Supervision, Investigation. <strong>Xiaokun Chen:</strong> Writing &#x02013; review &#x00026; editing, Validation, Formal analysis. <strong>Xiao Liu:</strong> Writing &#x02013; review &#x00026; editing, Validation, Supervision. <strong>Xueshuai Wan:</strong> Writing &#x02013; review &#x00026; editing, Visualization, Validation. <strong>Shunda Du:</strong> Writing &#x02013; review &#x00026; editing, Validation, Funding acquisition, Conceptualization.</p></div><div id="sec-a.r.k" class="tsec sec"><h2 class="head no_bottom_margin" id="sec-a.r.ktitle">Declaration of competing interest</h2><p id="p0245" class="p p-first-last">The authors declare that they have no known competing financial interests or personal relationships that could have appeared to influence the work reported in this paper.</p></div><div id="appsec1" class="tsec bk-sec"><h2 class="head no_bottom_margin" id="appsec1title">Abbreviations</h2><!--/article/back/sec/--><p id="p0250" class="p p-first-last">
</p><div class="def-list"><dl class="labeled-list"><dt id="d0010">AUC</dt><dd>Area Under the Curves</dd><dt id="d0015">ACKR3</dt><dd>Atypical Chemokine Receptor 3</dd><dt id="d0020">BP</dt><dd>Biological Process</dd><dt id="d0025">CCR3</dt><dd>C&#x02013;C Chemokine Receptor Type 3</dd><dt id="d0030">CCR7</dt><dd>C&#x02013;C Chemokine Receptor Type 7</dd><dt id="d0035">CNV</dt><dd>Copy Number Alteration</dd><dt id="d0040">DEG</dt><dd>Differentially Expressed Gene</dd><dt id="d0045">EMT</dt><dd>Epithelial&#x02013;Mesenchymal Transition</dd><dt id="d0050">ESTIMATE</dt><dd>Estimation of Stromal and Immune Cells in Malignant Tumor Tissues Using Expression Data</dd><dt id="d0055">FDA</dt><dd>Food and Drug Administration</dd><dt id="d0060">FDR</dt><dd>False Discovery Rate</dd><dt id="d0065">FPKM</dt><dd>Fragments Per Kilobase Million</dd><dt id="d0070">FZD5</dt><dd>Frizzled Class Receptor 5</dd><dt id="d0075">GDSC</dt><dd>Genomics of Drug Sensitivity in Cancer</dd><dt id="d0080">GO</dt><dd>Gene Ontology</dd><dt id="d0085">GPCR</dt><dd>G Protein-Coupled Receptor</dd><dt id="d0090">GSEA</dt><dd>Gene Set Enrichment Analysis</dd><dt id="d0095">GISTIC</dt><dd>Genomic Identification of Significant Targets in Cancer</dd><dt id="d0100">HAR1</dt><dd>Hydroxycarboxylic Acid Receptor 1</dd><dt id="d0105">HCC</dt><dd>Hepatocellular Carcinoma</dd><dt id="d0110">HTR5A</dt><dd>5-Hydroxytryptamine Receptor 5A</dd><dt id="d0115">ICB</dt><dd>Immune Checkpoint Blockade</dd><dt id="d0120">IL-6</dt><dd>Interleukin-6</dd><dt id="d0125">KEGG</dt><dd>Kyoto Encyclopedia of Genes and Genomes</dd><dt id="d0130">LIRI-JP</dt><dd>Liver Hepatocellular Carcinoma-Japan</dd><dt id="d0135">MF</dt><dd>Molecular Function</dd><dt id="d0140">MSI</dt><dd>Microsatellite Instability</dd><dt id="d0145">MSigDB</dt><dd>Molecular Signatures Database</dd><dt id="d0150">PPI</dt><dd>Protein-Protein Interaction</dd><dt id="d0155">ROC</dt><dd>Receiver Operating Characteristic</dd><dt id="d0160">SNP</dt><dd>Single Nucleotide Polymorphism</dd><dt id="d0165">TCGA</dt><dd>The Cancer Genome Atlas</dd><dt id="d0170">TIDE</dt><dd>Tumor Immune Dysfunction and Exclusion</dd><dt id="d0175">TMB</dt><dd>Tumor Mutational Burden</dd><dt id="d0180">TNM</dt><dd>Tumor Node Metastasis</dd><dt id="d0185">TPM</dt><dd>Transcripts Per Million</dd></dl></div><p>
</p></div><div id="cebib0010" class="tsec sec"><h2 class="head no_bottom_margin" id="cebib0010title">References</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="bib1">1. <span class="element-citation" id="sref1">Sung H., Ferlay J., Siegel R.L., et al. Global cancer statistics 2020: GLOBOCAN estimates of incidence and mortality worldwide for 36 cancers in 185 Countries. <span><span class="ref-journal">CA Cancer J Clin. </span>2021;<span class="ref-vol">71</span>(3):209–249.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33538338" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=CA+Cancer+J+Clin&amp;title=Global+cancer+statistics+2020:+GLOBOCAN+estimates+of+incidence+and+mortality+worldwide+for+36+cancers+in+185+Countries&amp;author=H.+Sung&amp;author=J.+Ferlay&amp;author=R.L.+Siegel&amp;volume=71&amp;issue=3&amp;publication_year=2021&amp;pages=209-249&amp;pmid=33538338&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib2">2. <span class="element-citation" id="sref2">EL-Khoueiry A.B., Sangro B., Yau T., et al. Nivolumab in patients with advanced hepatocellular carcinoma (CheckMate 040): an open-label, non-comparative, phase 1/2 dose escalation and expansion trial. <span><span class="ref-journal">Lancet. </span>2017;<span class="ref-vol">389</span>(10088):2492–2502.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7539326/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28434648" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet&amp;title=Nivolumab+in+patients+with+advanced+hepatocellular+carcinoma+(CheckMate+040):+an+open-label,+non-comparative,+phase+1/2+dose+escalation+and+expansion+trial&amp;author=A.B.+EL-Khoueiry&amp;author=B.+Sangro&amp;author=T.+Yau&amp;volume=389&amp;issue=10088&amp;publication_year=2017&amp;pages=2492-2502&amp;pmid=28434648&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib3">3. <span class="element-citation" id="sref3">Yau T., Hsu C., Kim T.Y., et al. Nivolumab in advanced hepatocellular carcinoma: sorafenib-experienced Asian cohort analysis. <span><span class="ref-journal">J. Hepatol. </span>2019;<span class="ref-vol">71</span>(3):543–552.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31176752" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hepatol.&amp;title=Nivolumab+in+advanced+hepatocellular+carcinoma:+sorafenib-experienced+Asian+cohort+analysis&amp;author=T.+Yau&amp;author=C.+Hsu&amp;author=T.Y.+Kim&amp;volume=71&amp;issue=3&amp;publication_year=2019&amp;pages=543-552&amp;pmid=31176752&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib4">4. <span class="element-citation" id="sref4">Finkelmeier F., Czauderna C., Perkhofer L., et al. Feasibility and safety of nivolumab in advanced hepatocellular carcinoma: real-life experience from three German centers. <span><span class="ref-journal">J. Cancer Res. Clin. Oncol. </span>2019;<span class="ref-vol">145</span>(1):253–259.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30374657" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cancer+Res.+Clin.+Oncol.&amp;title=Feasibility+and+safety+of+nivolumab+in+advanced+hepatocellular+carcinoma:+real-life+experience+from+three+German+centers&amp;author=F.+Finkelmeier&amp;author=C.+Czauderna&amp;author=L.+Perkhofer&amp;volume=145&amp;issue=1&amp;publication_year=2019&amp;pages=253-259&amp;pmid=30374657&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib5">5. <span class="element-citation" id="sref5">Yeh C.T., Liang K.H., Lin C.C., et al. A single nucleotide polymorphism on the GALNT14 gene as an effective predictor of response to chemotherapy in advanced hepatocellular carcinoma. <span><span class="ref-journal">Int. J. Cancer. </span>2014;<span class="ref-vol">134</span>(5):1214–1224.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23959947" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Cancer&amp;title=A+single+nucleotide+polymorphism+on+the+GALNT14+gene+as+an+effective+predictor+of+response+to+chemotherapy+in+advanced+hepatocellular+carcinoma&amp;author=C.T.+Yeh&amp;author=K.H.+Liang&amp;author=C.C.+Lin&amp;volume=134&amp;issue=5&amp;publication_year=2014&amp;pages=1214-1224&amp;pmid=23959947&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib6">6. <span class="element-citation" id="sref6">Rosenbaum D.M., Rasmussen S.G., Kobilka B.K. The structure and function of G-protein-coupled receptors. <span><span class="ref-journal">Nature. </span>2009;<span class="ref-vol">459</span>(7245):356–363.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3967846/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19458711" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=The+structure+and+function+of+G-protein-coupled+receptors&amp;author=D.M.+Rosenbaum&amp;author=S.G.+Rasmussen&amp;author=B.K.+Kobilka&amp;volume=459&amp;issue=7245&amp;publication_year=2009&amp;pages=356-363&amp;pmid=19458711&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib7">7. <span class="element-citation" id="sref7">Hauser A.S., Attwood M.M., Rask-Andersen M., et al. Trends in GPCR drug discovery: new agents, targets and indications. <span><span class="ref-journal">Nat. Rev. Drug Discov. </span>2017;<span class="ref-vol">16</span>(12):829–842.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6882681/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29075003" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Drug+Discov.&amp;title=Trends+in+GPCR+drug+discovery:+new+agents,+targets+and+indications&amp;author=A.S.+Hauser&amp;author=M.M.+Attwood&amp;author=M.+Rask-Andersen&amp;volume=16&amp;issue=12&amp;publication_year=2017&amp;pages=829-842&amp;pmid=29075003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib8">8. <span class="element-citation" id="sref8">Insel P.A., Sriram K., Gorr M.W., et al. GPCRomics: an approach to discover GPCR drug targets. <span><span class="ref-journal">Trends Pharmacol. Sci. </span>2019;<span class="ref-vol">40</span>(6):378–387.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6604616/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31078319" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Pharmacol.+Sci.&amp;title=GPCRomics:+an+approach+to+discover+GPCR+drug+targets&amp;author=P.A.+Insel&amp;author=K.+Sriram&amp;author=M.W.+Gorr&amp;volume=40&amp;issue=6&amp;publication_year=2019&amp;pages=378-387&amp;pmid=31078319&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib9">9. <span class="element-citation" id="sref9">O'Hayre M., Vazquez-Prado J., Kufareva I., et al. The emerging mutational landscape of G proteins and G-protein-coupled receptors in cancer. <span><span class="ref-journal">Nat. Rev. Cancer. </span>2013;<span class="ref-vol">13</span>(6):412–424.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4068741/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23640210" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Cancer&amp;title=The+emerging+mutational+landscape+of+G+proteins+and+G-protein-coupled+receptors+in+cancer&amp;author=M.+O'Hayre&amp;author=J.+Vazquez-Prado&amp;author=I.+Kufareva&amp;volume=13&amp;issue=6&amp;publication_year=2013&amp;pages=412-424&amp;pmid=23640210&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib10">10. <span class="element-citation" id="sref10">Kan Z., Jaiswal B.S., Stinson J., et al. Diverse somatic mutation patterns and pathway alterations in human cancers. <span><span class="ref-journal">Nature. </span>2010;<span class="ref-vol">466</span>(7308):869–873.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20668451" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Diverse+somatic+mutation+patterns+and+pathway+alterations+in+human+cancers&amp;author=Z.+Kan&amp;author=B.S.+Jaiswal&amp;author=J.+Stinson&amp;volume=466&amp;issue=7308&amp;publication_year=2010&amp;pages=869-873&amp;pmid=20668451&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib11">11. <span class="element-citation" id="sref11">O'Hayre M., Degese M.S., Gutkind J.S. Novel insights into G protein and G protein-coupled receptor signaling in cancer. <span><span class="ref-journal">Curr. Opin. Cell Biol. </span>2014;<span class="ref-vol">27</span>:126–135.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4021379/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24508914" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Opin.+Cell+Biol.&amp;title=Novel+insights+into+G+protein+and+G+protein-coupled+receptor+signaling+in+cancer&amp;author=M.+O'Hayre&amp;author=M.S.+Degese&amp;author=J.S.+Gutkind&amp;volume=27&amp;publication_year=2014&amp;pages=126-135&amp;pmid=24508914&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib12">12. <span class="element-citation" id="sref12">Lappano R., Maggiolini M. G protein-coupled receptors: novel targets for drug discovery in cancer. <span><span class="ref-journal">Nat. Rev. Drug Discov. </span>2011;<span class="ref-vol">10</span>(1):47–60.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21193867" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Drug+Discov.&amp;title=G+protein-coupled+receptors:+novel+targets+for+drug+discovery+in+cancer&amp;author=R.+Lappano&amp;author=M.+Maggiolini&amp;volume=10&amp;issue=1&amp;publication_year=2011&amp;pages=47-60&amp;pmid=21193867&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib13">13. <span class="element-citation" id="sref13">Li S., Huang S., Peng S.B. Overexpression of G protein-coupled receptors in cancer cells: involvement in tumor progression. <span><span class="ref-journal">Int. J. Oncol. </span>2005;<span class="ref-vol">27</span>(5):1329–1339.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16211229" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Oncol.&amp;title=Overexpression+of+G+protein-coupled+receptors+in+cancer+cells:+involvement+in+tumor+progression&amp;author=S.+Li&amp;author=S.+Huang&amp;author=S.B.+Peng&amp;volume=27&amp;issue=5&amp;publication_year=2005&amp;pages=1329-1339&amp;pmid=16211229&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib14">14. <span class="element-citation" id="sref14">Forbes S.A., Bindal N., Bamford S., et al. COSMIC: mining complete cancer genomes in the Catalogue of Somatic Mutations in Cancer. <span><span class="ref-journal">Nucleic Acids Res. </span>2011;<span class="ref-vol">39</span>(Database issue):D945–D950.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3013785/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20952405" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=COSMIC:+mining+complete+cancer+genomes+in+the+Catalogue+of+Somatic+Mutations+in+Cancer&amp;author=S.A.+Forbes&amp;author=N.+Bindal&amp;author=S.+Bamford&amp;volume=39&amp;issue=Database+issue&amp;publication_year=2011&amp;pages=D945-D950&amp;pmid=20952405&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib15">15. <span class="element-citation" id="sref15">Zhang Z., Qin C., Wu Y., et al. CCR9 as a prognostic marker and therapeutic target in hepatocellular carcinoma. <span><span class="ref-journal">Oncol. Rep. </span>2014;<span class="ref-vol">31</span>(4):1629–1636.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24481516" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.+Rep.&amp;title=CCR9+as+a+prognostic+marker+and+therapeutic+target+in+hepatocellular+carcinoma&amp;author=Z.+Zhang&amp;author=C.+Qin&amp;author=Y.+Wu&amp;volume=31&amp;issue=4&amp;publication_year=2014&amp;pages=1629-1636&amp;pmid=24481516&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib16">16. <span class="element-citation" id="sref16">Gao Q., Zhao Y.J., Wang X.Y., et al. CXCR6 upregulation contributes to a proinflammatory tumor microenvironment that drives metastasis and poor patient outcomes in hepatocellular carcinoma. <span><span class="ref-journal">Cancer Res. </span>2012;<span class="ref-vol">72</span>(14):3546–3556.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22710437" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=CXCR6+upregulation+contributes+to+a+proinflammatory+tumor+microenvironment+that+drives+metastasis+and+poor+patient+outcomes+in+hepatocellular+carcinoma&amp;author=Q.+Gao&amp;author=Y.J.+Zhao&amp;author=X.Y.+Wang&amp;volume=72&amp;issue=14&amp;publication_year=2012&amp;pages=3546-3556&amp;pmid=22710437&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib17">17. <span class="element-citation" id="sref17">Liu F., Zhu C., Huang X., et al. A low level of GPR37 is associated with human hepatocellular carcinoma progression and poor patient survival. <span><span class="ref-journal">Pathol. Res. Pract. </span>2014;<span class="ref-vol">210</span>(12):885–892.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25169131" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pathol.+Res.+Pract.&amp;title=A+low+level+of+GPR37+is+associated+with+human+hepatocellular+carcinoma+progression+and+poor+patient+survival&amp;author=F.+Liu&amp;author=C.+Zhu&amp;author=X.+Huang&amp;volume=210&amp;issue=12&amp;publication_year=2014&amp;pages=885-892&amp;pmid=25169131&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib18">18. <span class="element-citation" id="sref18">Ding Q., Xia Y., Ding S., et al. An alternatively spliced variant of CXCR3 mediates the metastasis of CD133+ liver cancer cells induced by CXCL9. <span><span class="ref-journal">Oncotarget. </span>2016;<span class="ref-vol">7</span>(12):14405–14414.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4924724/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26883105" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=An+alternatively+spliced+variant+of+CXCR3+mediates+the+metastasis+of+CD133++liver+cancer+cells+induced+by+CXCL9&amp;author=Q.+Ding&amp;author=Y.+Xia&amp;author=S.+Ding&amp;volume=7&amp;issue=12&amp;publication_year=2016&amp;pages=14405-14414&amp;pmid=26883105&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib19">19. <span class="element-citation" id="sref19">Liu Z., Yang L., Xu J., et al. Enhanced expression and clinical significance of chemokine receptor CXCR2 in hepatocellular carcinoma. <span><span class="ref-journal">J. Surg. Res. </span>2011;<span class="ref-vol">166</span>(2):241–246.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20018298" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Surg.+Res.&amp;title=Enhanced+expression+and+clinical+significance+of+chemokine+receptor+CXCR2+in+hepatocellular+carcinoma&amp;author=Z.+Liu&amp;author=L.+Yang&amp;author=J.+Xu&amp;volume=166&amp;issue=2&amp;publication_year=2011&amp;pages=241-246&amp;pmid=20018298&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib20">20. <span class="element-citation" id="sref20">Zheng K., Li H.Y., Su X.L., et al. Chemokine receptor CXCR7 regulates the invasion, angiogenesis and tumor growth of human hepatocellular carcinoma cells. <span><span class="ref-journal">J. Exp. Clin. Cancer Res. </span>2010;<span class="ref-vol">29</span>(1):31.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2859378/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20380740" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Clin.+Cancer+Res.&amp;title=Chemokine+receptor+CXCR7+regulates+the+invasion,+angiogenesis+and+tumor+growth+of+human+hepatocellular+carcinoma+cells&amp;author=K.+Zheng&amp;author=H.Y.+Li&amp;author=X.L.+Su&amp;volume=29&amp;issue=1&amp;publication_year=2010&amp;pages=31&amp;pmid=20380740&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib21">21. <span class="element-citation" id="sref21">Kuang D.M., Zhao Q., Wu Y., et al. Peritumoral neutrophils link inflammatory response to disease progression by fostering angiogenesis in hepatocellular carcinoma. <span><span class="ref-journal">J. Hepatol. </span>2011;<span class="ref-vol">54</span>(5):948–955.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21145847" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hepatol.&amp;title=Peritumoral+neutrophils+link+inflammatory+response+to+disease+progression+by+fostering+angiogenesis+in+hepatocellular+carcinoma&amp;author=D.M.+Kuang&amp;author=Q.+Zhao&amp;author=Y.+Wu&amp;volume=54&amp;issue=5&amp;publication_year=2011&amp;pages=948-955&amp;pmid=21145847&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib22">22. <span class="element-citation" id="sref22">Deshayes F., Nahmias C. Angiotensin receptors: a new role in cancer? <span><span class="ref-journal">Trends Endocrinol Metab. </span>2005;<span class="ref-vol">16</span>(7):293–299.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16061390" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Endocrinol+Metab&amp;title=Angiotensin+receptors:+a+new+role+in+cancer?&amp;author=F.+Deshayes&amp;author=C.+Nahmias&amp;volume=16&amp;issue=7&amp;publication_year=2005&amp;pages=293-299&amp;pmid=16061390&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib23">23. <span class="element-citation" id="sref23">Ma Z., Guo D., Wang Q., et al. Lgr5-mediated p53 Repression through PDCD5 leads to doxorubicin resistance in Hepatocellular Carcinoma. <span><span class="ref-journal">Theranostics. </span>2019;<span class="ref-vol">9</span>(10):2967–2983.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6568175/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31244936" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Theranostics&amp;title=Lgr5-mediated+p53+Repression+through+PDCD5+leads+to+doxorubicin+resistance+in+Hepatocellular+Carcinoma&amp;author=Z.+Ma&amp;author=D.+Guo&amp;author=Q.+Wang&amp;volume=9&amp;issue=10&amp;publication_year=2019&amp;pages=2967-2983&amp;pmid=31244936&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib24">24. <span class="element-citation" id="sref24">Wei T., Chen W., Wen L., et al. G protein-coupled estrogen receptor deficiency accelerates liver tumorigenesis by enhancing inflammation and fibrosis. <span><span class="ref-journal">Cancer Lett. </span>2016;<span class="ref-vol">382</span>(2):195–202.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27594673" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Lett.&amp;title=G+protein-coupled+estrogen+receptor+deficiency+accelerates+liver+tumorigenesis+by+enhancing+inflammation+and+fibrosis&amp;author=T.+Wei&amp;author=W.+Chen&amp;author=L.+Wen&amp;volume=382&amp;issue=2&amp;publication_year=2016&amp;pages=195-202&amp;pmid=27594673&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib25">25. <span class="element-citation" id="sref25">Gao Y., You M., Fu J., et al. Intratumoral stem-like CCR4+ regulatory T cells orchestrate the immunosuppressive microenvironment in HCC associated with hepatitis B. <span><span class="ref-journal">J. Hepatol. </span>2022;<span class="ref-vol">76</span>(1):148–159.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34689996" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Hepatol.&amp;title=Intratumoral+stem-like+CCR4++regulatory+T+cells+orchestrate+the+immunosuppressive+microenvironment+in+HCC+associated+with+hepatitis+B&amp;author=Y.+Gao&amp;author=M.+You&amp;author=J.+Fu&amp;volume=76&amp;issue=1&amp;publication_year=2022&amp;pages=148-159&amp;pmid=34689996&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib26">26. <span class="element-citation" id="sref26">Wilkerson M.D., Hayes D.N. ConsensusClusterPlus: a class discovery tool with confidence assessments and item tracking. <span><span class="ref-journal">Bioinformatics. </span>2010;<span class="ref-vol">26</span>(12):1572–1573.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2881355/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20427518" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinformatics&amp;title=ConsensusClusterPlus:+a+class+discovery+tool+with+confidence+assessments+and+item+tracking&amp;author=M.D.+Wilkerson&amp;author=D.N.+Hayes&amp;volume=26&amp;issue=12&amp;publication_year=2010&amp;pages=1572-1573&amp;pmid=20427518&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib27">27. <span class="element-citation" id="sref999">Cai W., Van Der Laan M. Nonparametric bootstrap inference for the targeted highly adaptive least absolute shrinkage and selection operator (LASSO) estimator. <span><span class="ref-journal">Int. J. Biostat. </span>2020;<span class="ref-vol">16</span>(2):1–34.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32772002" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Biostat.&amp;title=Nonparametric+bootstrap+inference+for+the+targeted+highly+adaptive+least+absolute+shrinkage+and+selection+operator+(LASSO)+estimator&amp;author=W.+Cai&amp;author=M.+Van+Der+Laan&amp;volume=16&amp;issue=2&amp;publication_year=2020&amp;pages=1-34&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib28">28. <span class="element-citation" id="sref28">Love M.I., Huber W., Anders S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <span><span class="ref-journal">Genome Biol. </span>2014;<span class="ref-vol">15</span>(12):550.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4302049/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol.&amp;title=Moderated+estimation+of+fold+change+and+dispersion+for+RNA-seq+data+with+DESeq2&amp;author=M.I.+Love&amp;author=W.+Huber&amp;author=S.+Anders&amp;volume=15&amp;issue=12&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib29">29. <span class="element-citation" id="sref29">Ashburner M., Ball C.A., Blake J.A., et al. Gene ontology: tool for the unification of biology. The Gene Ontology Consortium. <span><span class="ref-journal">Nat. Genet. </span>2000;<span class="ref-vol">25</span>(1):25–29.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3037419/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/10802651" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Genet.&amp;title=Gene+ontology:+tool+for+the+unification+of+biology.+The+Gene+Ontology+Consortium&amp;author=M.+Ashburner&amp;author=C.A.+Ball&amp;author=J.A.+Blake&amp;volume=25&amp;issue=1&amp;publication_year=2000&amp;pages=25-29&amp;pmid=10802651&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib30">30. <span class="element-citation" id="sref30">Ogata H., Goto S., Sato K., et al. KEGG: kyoto Encyclopedia of genes and genomes. <span><span class="ref-journal">Nucleic Acids Res. </span>1999;<span class="ref-vol">27</span>(1):29–34.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC148090/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/9847135" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=KEGG:+kyoto+Encyclopedia+of+genes+and+genomes&amp;author=H.+Ogata&amp;author=S.+Goto&amp;author=K.+Sato&amp;volume=27&amp;issue=1&amp;publication_year=1999&amp;pages=29-34&amp;pmid=9847135&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib31">31. <span class="element-citation" id="sref31">Yu G., Wang L.G., Han Y., et al. clusterProfiler: an R package for comparing biological themes among gene clusters. <span><span class="ref-journal">OMICS. </span>2012;<span class="ref-vol">16</span>(5):284–287.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3339379/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22455463" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=OMICS&amp;title=clusterProfiler:+an+R+package+for+comparing+biological+themes+among+gene+clusters&amp;author=G.+Yu&amp;author=L.G.+Wang&amp;author=Y.+Han&amp;volume=16&amp;issue=5&amp;publication_year=2012&amp;pages=284-287&amp;pmid=22455463&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib32">32. <span class="element-citation" id="sref32">Subramanian A., Tamayo P., Mootha V.K., et al. Gene set enrichment analysis: a knowledge-based approach for interpreting genome-wide expression profiles. <span><span class="ref-journal">Proc Natl Acad Sci U S A. </span>2005;<span class="ref-vol">102</span>(43):15545–15550.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1239896/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16199517" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci+U+S+A&amp;title=Gene+set+enrichment+analysis:+a+knowledge-based+approach+for+interpreting+genome-wide+expression+profiles&amp;author=A.+Subramanian&amp;author=P.+Tamayo&amp;author=V.K.+Mootha&amp;volume=102&amp;issue=43&amp;publication_year=2005&amp;pages=15545-15550&amp;pmid=16199517&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib33">33. <span class="element-citation" id="sref33">Szklarczyk D., Gable A.L., Nastou K.C., et al. The STRING database in 2021: customizable protein-protein networks, and functional characterization of user-uploaded gene/measurement sets. <span><span class="ref-journal">Nucleic Acids Res. </span>2021;<span class="ref-vol">49</span>(D1):D605–D612.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7779004/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33237311" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=The+STRING+database+in+2021:+customizable+protein-protein+networks,+and+functional+characterization+of+user-uploaded+gene/measurement+sets&amp;author=D.+Szklarczyk&amp;author=A.L.+Gable&amp;author=K.C.+Nastou&amp;volume=49&amp;issue=D1&amp;publication_year=2021&amp;pages=D605-D612&amp;pmid=33237311&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib34">34. <span class="element-citation" id="sref34">Yoshihara K., Shahmoradgoli M., Martinez E., et al. Inferring tumour purity and stromal and immune cell admixture from expression data. <span><span class="ref-journal">Nat. Commun. </span>2013;<span class="ref-vol">4</span>:2612.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3826632/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24113773" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun.&amp;title=Inferring+tumour+purity+and+stromal+and+immune+cell+admixture+from+expression+data&amp;author=K.+Yoshihara&amp;author=M.+Shahmoradgoli&amp;author=E.+Martinez&amp;volume=4&amp;publication_year=2013&amp;pages=2612&amp;pmid=24113773&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib35">35. <span class="element-citation" id="sref35">Newman A.M., Liu C.L., Green M.R., et al. Robust enumeration of cell subsets from tissue expression profiles. <span><span class="ref-journal">Nat. Methods. </span>2015;<span class="ref-vol">12</span>(5):453–457.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4739640/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25822800" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Robust+enumeration+of+cell+subsets+from+tissue+expression+profiles&amp;author=A.M.+Newman&amp;author=C.L.+Liu&amp;author=M.R.+Green&amp;volume=12&amp;issue=5&amp;publication_year=2015&amp;pages=453-457&amp;pmid=25822800&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib36">36. <span class="element-citation" id="sref36">Fu J., Li K., Zhang W., et al. Large-scale public data reuse to model immunotherapy response and resistance. <span><span class="ref-journal">Genome Med. </span>2020;<span class="ref-vol">12</span>(1):21.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7045518/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32102694" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Med.&amp;title=Large-scale+public+data+reuse+to+model+immunotherapy+response+and+resistance&amp;author=J.+Fu&amp;author=K.+Li&amp;author=W.+Zhang&amp;volume=12&amp;issue=1&amp;publication_year=2020&amp;pages=21&amp;pmid=32102694&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib37">37. <span class="element-citation" id="sref37">Yang W., Soares J., Greninger P., et al. Genomics of Drug Sensitivity in Cancer (GDSC): a resource for therapeutic biomarker discovery in cancer cells. <span><span class="ref-journal">Nucleic Acids Res. </span>2013;<span class="ref-vol">41</span>(Database issue):D955–D961.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3531057/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23180760" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res.&amp;title=Genomics+of+Drug+Sensitivity+in+Cancer+(GDSC):+a+resource+for+therapeutic+biomarker+discovery+in+cancer+cells&amp;author=W.+Yang&amp;author=J.+Soares&amp;author=P.+Greninger&amp;volume=41&amp;issue=Database+issue&amp;publication_year=2013&amp;pages=D955-D961&amp;pmid=23180760&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib38">38. <span class="element-citation" id="sref38">Geeleher P., Cox N., Huang R.S. pRRophetic: an R package for prediction of clinical chemotherapeutic response from tumor gene expression levels. <span><span class="ref-journal">PLoS One. </span>2014;<span class="ref-vol">9</span>(9)</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4167990/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25229481" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=pRRophetic:+an+R+package+for+prediction+of+clinical+chemotherapeutic+response+from+tumor+gene+expression+levels&amp;author=P.+Geeleher&amp;author=N.+Cox&amp;author=R.S.+Huang&amp;volume=9&amp;issue=9&amp;publication_year=2014&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib39">39. <span class="element-citation" id="sref39">Blanche P., Dartigues J.F., Jacqmin-Gadda H. Estimating and comparing time-dependent areas under receiver operating characteristic curves for censored event times with competing risks. <span><span class="ref-journal">Stat. Med. </span>2013;<span class="ref-vol">32</span>(30):5381–5397.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24027076" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Stat.+Med.&amp;title=Estimating+and+comparing+time-dependent+areas+under+receiver+operating+characteristic+curves+for+censored+event+times+with+competing+risks&amp;author=P.+Blanche&amp;author=J.F.+Dartigues&amp;author=H.+Jacqmin-Gadda&amp;volume=32&amp;issue=30&amp;publication_year=2013&amp;pages=5381-5397&amp;pmid=24027076&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib40">40. <span class="element-citation" id="sref40">Li Y., Zhang Y., Zhang S., et al. circRNA circARNT2 suppressed the sensitivity of hepatocellular carcinoma cells to cisplatin by targeting the miR-155-5p/PDK1 Axis. <span><span class="ref-journal">Mol. Ther. Nucleic Acids. </span>2021;<span class="ref-vol">23</span>:244–254.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7772525/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33425483" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Ther.+Nucleic+Acids&amp;title=circRNA+circARNT2+suppressed+the+sensitivity+of+hepatocellular+carcinoma+cells+to+cisplatin+by+targeting+the+miR-155-5p/PDK1+Axis&amp;author=Y.+Li&amp;author=Y.+Zhang&amp;author=S.+Zhang&amp;volume=23&amp;publication_year=2021&amp;pages=244-254&amp;pmid=33425483&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib41">41. <span class="element-citation" id="sref41">Van Renne N., Roca Suarez A.A., Duong F.H.T., et al. miR-135a-5p-mediated downregulation of protein tyrosine phosphatase receptor delta is a candidate driver of HCV-associated hepatocarcinogenesis. <span><span class="ref-journal">Gut. </span>2018;<span class="ref-vol">67</span>(5):953–962.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5540823/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28159835" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=miR-135a-5p-mediated+downregulation+of+protein+tyrosine+phosphatase+receptor+delta+is+a+candidate+driver+of+HCV-associated+hepatocarcinogenesis&amp;author=N.+Van+Renne&amp;author=A.A.+Roca+Suarez&amp;author=F.H.T.+Duong&amp;volume=67&amp;issue=5&amp;publication_year=2018&amp;pages=953-962&amp;pmid=28159835&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib42">42. <span class="element-citation" id="sref42">Zhang Y., Zhang D., Lv J., et al. miR-410-3p promotes prostate cancer progression via regulating PTEN/AKT/mTOR signaling pathway. <span><span class="ref-journal">Biochem. Biophys. Res. Commun. </span>2018;<span class="ref-vol">503</span>(4):2459–2465.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29969630" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+Biophys.+Res.+Commun.&amp;title=miR-410-3p+promotes+prostate+cancer+progression+via+regulating+PTEN/AKT/mTOR+signaling+pathway&amp;author=Y.+Zhang&amp;author=D.+Zhang&amp;author=J.+Lv&amp;volume=503&amp;issue=4&amp;publication_year=2018&amp;pages=2459-2465&amp;pmid=29969630&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib43">43. <span class="element-citation" id="sref43">Zhang Y.F., Yu Y., Song W.Z., et al. miR-410-3p suppresses breast cancer progression by targeting Snail. <span><span class="ref-journal">Oncol. Rep. </span>2016;<span class="ref-vol">36</span>(1):480–486.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27221455" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.+Rep.&amp;title=miR-410-3p+suppresses+breast+cancer+progression+by+targeting+Snail&amp;author=Y.F.+Zhang&amp;author=Y.+Yu&amp;author=W.Z.+Song&amp;volume=36&amp;issue=1&amp;publication_year=2016&amp;pages=480-486&amp;pmid=27221455&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib44">44. <span class="element-citation" id="sref44">Burns J.M., Summers B.C., Wang Y., et al. A novel chemokine receptor for SDF-1 and I-TAC involved in cell survival, cell adhesion, and tumor development. <span><span class="ref-journal">J. Exp. Med. </span>2006;<span class="ref-vol">203</span>(9):2201–2213.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2118398/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16940167" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Exp.+Med.&amp;title=A+novel+chemokine+receptor+for+SDF-1+and+I-TAC+involved+in+cell+survival,+cell+adhesion,+and+tumor+development&amp;author=J.M.+Burns&amp;author=B.C.+Summers&amp;author=Y.+Wang&amp;volume=203&amp;issue=9&amp;publication_year=2006&amp;pages=2201-2213&amp;pmid=16940167&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib45">45. <span class="element-citation" id="sref45">Xue T.C., Jia Q.A., Bu Y., et al. CXCR7 correlates with the differentiation of hepatocellular carcinoma and suppresses HNF4alpha expression through the ERK pathway. <span><span class="ref-journal">Oncol. Rep. </span>2014;<span class="ref-vol">32</span>(6):2387–2396.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25242412" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.+Rep.&amp;title=CXCR7+correlates+with+the+differentiation+of+hepatocellular+carcinoma+and+suppresses+HNF4alpha+expression+through+the+ERK+pathway&amp;author=T.C.+Xue&amp;author=Q.A.+Jia&amp;author=Y.+Bu&amp;volume=32&amp;issue=6&amp;publication_year=2014&amp;pages=2387-2396&amp;pmid=25242412&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib46">46. <span class="element-citation" id="sref46">Lin L., Han M.M., Wang F., et al. CXCR7 stimulates MAPK signaling to regulate hepatocellular carcinoma progression. <span><span class="ref-journal">Cell Death Dis. </span>2014;<span class="ref-vol">5</span>(10)</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4649507/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25341042" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Death+Dis.&amp;title=CXCR7+stimulates+MAPK+signaling+to+regulate+hepatocellular+carcinoma+progression&amp;author=L.+Lin&amp;author=M.M.+Han&amp;author=F.+Wang&amp;volume=5&amp;issue=10&amp;publication_year=2014&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib47">47. <span class="element-citation" id="sref47">Chen Y., Teng F., Wang G., et al. Overexpression of CXCR7 induces angiogenic capacity of human hepatocellular carcinoma cells via the AKT signaling pathway. <span><span class="ref-journal">Oncol. Rep. </span>2016;<span class="ref-vol">36</span>(4):2275–2281.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27572688" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.+Rep.&amp;title=Overexpression+of+CXCR7+induces+angiogenic+capacity+of+human+hepatocellular+carcinoma+cells+via+the+AKT+signaling+pathway&amp;author=Y.+Chen&amp;author=F.+Teng&amp;author=G.+Wang&amp;volume=36&amp;issue=4&amp;publication_year=2016&amp;pages=2275-2281&amp;pmid=27572688&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib48">48. <span class="element-citation" id="sref48">Millard C.J., Ludeman J.P., Canals M., et al. Structural basis of receptor sulfotyrosine recognition by a CC chemokine: the N-terminal region of CCR3 bound to CCL11/eotaxin-1. <span><span class="ref-journal">Structure. </span>2014;<span class="ref-vol">22</span>(11):1571–1581.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25450766" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Structure&amp;title=Structural+basis+of+receptor+sulfotyrosine+recognition+by+a+CC+chemokine:+the+N-terminal+region+of+CCR3+bound+to+CCL11/eotaxin-1&amp;author=C.J.+Millard&amp;author=J.P.+Ludeman&amp;author=M.+Canals&amp;volume=22&amp;issue=11&amp;publication_year=2014&amp;pages=1571-1581&amp;pmid=25450766&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib49">49. <span class="element-citation" id="sref49">Lin Z.Y., Chuang Y.H., Chuang W.L. Cancer-associated fibroblasts up-regulate CCL2, CCL26, IL6 and LOXL2 genes related to promotion of cancer progression in hepatocellular carcinoma cells. <span><span class="ref-journal">Biomed. Pharmacother. </span>2012;<span class="ref-vol">66</span>(7):525–529.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22739041" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biomed.+Pharmacother.&amp;title=Cancer-associated+fibroblasts+up-regulate+CCL2,+CCL26,+IL6+and+LOXL2+genes+related+to+promotion+of+cancer+progression+in+hepatocellular+carcinoma+cells&amp;author=Z.Y.+Lin&amp;author=Y.H.+Chuang&amp;author=W.L.+Chuang&amp;volume=66&amp;issue=7&amp;publication_year=2012&amp;pages=525-529&amp;pmid=22739041&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib50">50. <span class="element-citation" id="sref50">Jin L., Liu W.R., Tian M.X., et al. CCL24 contributes to HCC malignancy via RhoB- VEGFA-VEGFR2 angiogenesis pathway and indicates poor prognosis. <span><span class="ref-journal">Oncotarget. </span>2017;<span class="ref-vol">8</span>(3):5135–5148.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5354897/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28042950" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncotarget&amp;title=CCL24+contributes+to+HCC+malignancy+via+RhoB-+VEGFA-VEGFR2+angiogenesis+pathway+and+indicates+poor+prognosis&amp;author=L.+Jin&amp;author=W.R.+Liu&amp;author=M.X.+Tian&amp;volume=8&amp;issue=3&amp;publication_year=2017&amp;pages=5135-5148&amp;pmid=28042950&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib51">51. <span class="element-citation" id="sref51">Correale P., Rotundo M.S., Botta C., et al. Tumor infiltration by T lymphocytes expressing chemokine receptor 7 (CCR7) is predictive of favorable outcome in patients with advanced colorectal carcinoma. <span><span class="ref-journal">Clin. Cancer Res. </span>2012;<span class="ref-vol">18</span>(3):850–857.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22142823" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Cancer+Res.&amp;title=Tumor+infiltration+by+T+lymphocytes+expressing+chemokine+receptor+7+(CCR7)+is+predictive+of+favorable+outcome+in+patients+with+advanced+colorectal+carcinoma&amp;author=P.+Correale&amp;author=M.S.+Rotundo&amp;author=C.+Botta&amp;volume=18&amp;issue=3&amp;publication_year=2012&amp;pages=850-857&amp;pmid=22142823&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib52">52. <span class="element-citation" id="sref52">Yang L., Chang Y., Cao P. CCR7 preservation via histone deacetylase inhibition promotes epithelial-mesenchymal transition of hepatocellular carcinoma cells. <span><span class="ref-journal">Exp. Cell Res. </span>2018;<span class="ref-vol">371</span>(1):231–237.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30107147" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Cell+Res.&amp;title=CCR7+preservation+via+histone+deacetylase+inhibition+promotes+epithelial-mesenchymal+transition+of+hepatocellular+carcinoma+cells&amp;author=L.+Yang&amp;author=Y.+Chang&amp;author=P.+Cao&amp;volume=371&amp;issue=1&amp;publication_year=2018&amp;pages=231-237&amp;pmid=30107147&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib53">53. <span class="element-citation" id="sref53">Schimanski C.C., Bahre R., Gockel I., et al. Chemokine receptor CCR7 enhances intrahepatic and lymphatic dissemination of human hepatocellular cancer. <span><span class="ref-journal">Oncol. Rep. </span>2006;<span class="ref-vol">16</span>(1):109–113.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16786131" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Oncol.+Rep.&amp;title=Chemokine+receptor+CCR7+enhances+intrahepatic+and+lymphatic+dissemination+of+human+hepatocellular+cancer&amp;author=C.C.+Schimanski&amp;author=R.+Bahre&amp;author=I.+Gockel&amp;volume=16&amp;issue=1&amp;publication_year=2006&amp;pages=109-113&amp;pmid=16786131&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib54">54. <span class="element-citation" id="sref54">Liu X., Qin J., Gao T., et al. YTHDF1 facilitates the progression of hepatocellular carcinoma by promoting FZD5 mRNA translation in an m6A-dependent manner. <span><span class="ref-journal">Mol. Ther. Nucleic Acids. </span>2020;<span class="ref-vol">22</span>:750–765.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7595883/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33230473" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Ther.+Nucleic+Acids&amp;title=YTHDF1+facilitates+the+progression+of+hepatocellular+carcinoma+by+promoting+FZD5+mRNA+translation+in+an+m6A-dependent+manner&amp;author=X.+Liu&amp;author=J.+Qin&amp;author=T.+Gao&amp;volume=22&amp;publication_year=2020&amp;pages=750-765&amp;pmid=33230473&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span><link class="retraction-forward" pmcaccid="PMC9096254" /> <a class="red" href="/pmc/articles/PMC9096254/">Retracted</a></span></div><div class="ref-cit-blk half_rhythm" id="bib55">55. <span class="element-citation" id="sref55">Zhu Q., Lu G., Luo Z., et al. CircRNA circ_0067934 promotes tumor growth and metastasis in hepatocellular carcinoma through regulation of miR-1324/FZD5/Wnt/beta-catenin axis. <span><span class="ref-journal">Biochem. Biophys. Res. Commun. </span>2018;<span class="ref-vol">497</span>(2):626–632.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29458020" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biochem.+Biophys.+Res.+Commun.&amp;title=CircRNA+circ_0067934+promotes+tumor+growth+and+metastasis+in+hepatocellular+carcinoma+through+regulation+of+miR-1324/FZD5/Wnt/beta-catenin+axis&amp;author=Q.+Zhu&amp;author=G.+Lu&amp;author=Z.+Luo&amp;volume=497&amp;issue=2&amp;publication_year=2018&amp;pages=626-632&amp;pmid=29458020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib56">56. <span class="element-citation" id="sref56">Payen V.L., Mina E., Van Hee V.F., et al. Monocarboxylate transporters in cancer. <span><span class="ref-journal">Mol Metab. </span>2020;<span class="ref-vol">33</span>:48–66.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7056923/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31395464" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol+Metab&amp;title=Monocarboxylate+transporters+in+cancer&amp;author=V.L.+Payen&amp;author=E.+Mina&amp;author=V.F.+Van+Hee&amp;volume=33&amp;publication_year=2020&amp;pages=48-66&amp;pmid=31395464&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib57">57. <span class="element-citation" id="sref57">Roland C.L., Arumugam T., Deng D., et al. Cell surface lactate receptor GPR81 is crucial for cancer cell survival. <span><span class="ref-journal">Cancer Res. </span>2014;<span class="ref-vol">74</span>(18):5301–5310.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4167222/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24928781" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Res.&amp;title=Cell+surface+lactate+receptor+GPR81+is+crucial+for+cancer+cell+survival&amp;author=C.L.+Roland&amp;author=T.+Arumugam&amp;author=D.+Deng&amp;volume=74&amp;issue=18&amp;publication_year=2014&amp;pages=5301-5310&amp;pmid=24928781&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib58">58. <span class="element-citation" id="sref58">Hara H., Seregin S.S., Yang D., et al. The NLRP6 inflammasome recognizes lipoteichoic acid and regulates gram-positive pathogen infection. <span><span class="ref-journal">Cell. </span>2018;<span class="ref-vol">175</span>(6):1651–1664 e14.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6294477/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30392956" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+NLRP6+inflammasome+recognizes+lipoteichoic+acid+and+regulates+gram-positive+pathogen+infection&amp;author=H.+Hara&amp;author=S.S.+Seregin&amp;author=D.+Yang&amp;volume=175&amp;issue=6&amp;publication_year=2018&amp;pages=1651-1664+e14&amp;pmid=30392956&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib59">59. <span class="element-citation" id="sref59">Wang H., Zhang B., Shang Y., et al. A novel risk score model based on pyroptosis-related genes for predicting survival and immunogenic landscape in hepatocellular carcinoma. <span><span class="ref-journal">Aging (Albany NY) </span>2023;<span class="ref-vol">15</span>(5):1412–1444.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10042690/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36920176" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Aging+(Albany+NY)&amp;title=A+novel+risk+score+model+based+on+pyroptosis-related+genes+for+predicting+survival+and+immunogenic+landscape+in+hepatocellular+carcinoma&amp;author=H.+Wang&amp;author=B.+Zhang&amp;author=Y.+Shang&amp;volume=15&amp;issue=5&amp;publication_year=2023&amp;pages=1412-1444&amp;pmid=36920176&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib60">60. <span class="element-citation" id="sref60">Gao X., Wang W.X., Zhang X.L. A novel pyroptosis risk model composed of NLRP6 effectively predicts the prognosis of hepatocellular carcinoma patients. <span><span class="ref-journal">Cancer Med. </span>2023;<span class="ref-vol">12</span>(1):808–823.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9844607/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35651286" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cancer+Med.&amp;title=A+novel+pyroptosis+risk+model+composed+of+NLRP6+effectively+predicts+the+prognosis+of+hepatocellular+carcinoma+patients&amp;author=X.+Gao&amp;author=W.X.+Wang&amp;author=X.L.+Zhang&amp;volume=12&amp;issue=1&amp;publication_year=2023&amp;pages=808-823&amp;pmid=35651286&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib61">61. <span class="element-citation" id="sref61">Vaidya A., Jain S., Jain A.K., et al. Metabotropic glutamate receptors: a review on prospectives and therapeutic aspects. <span><span class="ref-journal">Mini Rev. Med. Chem. </span>2013;<span class="ref-vol">13</span>(13):1967–1981.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22530579" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mini+Rev.+Med.+Chem.&amp;title=Metabotropic+glutamate+receptors:+a+review+on+prospectives+and+therapeutic+aspects&amp;author=A.+Vaidya&amp;author=S.+Jain&amp;author=A.K.+Jain&amp;volume=13&amp;issue=13&amp;publication_year=2013&amp;pages=1967-1981&amp;pmid=22530579&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib62">62. <span class="element-citation" id="sref62">Raber J., Duvoisin R.M. Novel metabotropic glutamate receptor 4 and glutamate receptor 8 therapeutics for the treatment of anxiety. <span><span class="ref-journal">Expert Opin Investig Drugs. </span>2015;<span class="ref-vol">24</span>(4):519–528.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25518990" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Expert+Opin+Investig+Drugs&amp;title=Novel+metabotropic+glutamate+receptor+4+and+glutamate+receptor+8+therapeutics+for+the+treatment+of+anxiety&amp;author=J.+Raber&amp;author=R.M.+Duvoisin&amp;volume=24&amp;issue=4&amp;publication_year=2015&amp;pages=519-528&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib63">63. <span class="element-citation" id="sref63">Yang Y., Li J., Lei W., et al. CXCL12-CXCR4/CXCR7 Axis in cancer: from mechanisms to clinical applications. <span><span class="ref-journal">Int. J. Biol. Sci. </span>2023;<span class="ref-vol">19</span>(11):3341–3359.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10367567/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37497001" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Biol.+Sci.&amp;title=CXCL12-CXCR4/CXCR7+Axis+in+cancer:+from+mechanisms+to+clinical+applications&amp;author=Y.+Yang&amp;author=J.+Li&amp;author=W.+Lei&amp;volume=19&amp;issue=11&amp;publication_year=2023&amp;pages=3341-3359&amp;pmid=37497001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib64">64. <span class="element-citation" id="sref64">Guillemot E., Karimdjee-Soilihi B., Pradelli E., et al. CXCR7 receptors facilitate the progression of colon carcinoma within lung not within liver. <span><span class="ref-journal">Br. J. Cancer. </span>2012;<span class="ref-vol">107</span>(12):1944–1949.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3516689/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23169289" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Br.+J.+Cancer&amp;title=CXCR7+receptors+facilitate+the+progression+of+colon+carcinoma+within+lung+not+within+liver&amp;author=E.+Guillemot&amp;author=B.+Karimdjee-Soilihi&amp;author=E.+Pradelli&amp;volume=107&amp;issue=12&amp;publication_year=2012&amp;pages=1944-1949&amp;pmid=23169289&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib65">65. <span class="element-citation" id="sref65">Horuk R. Chemokine receptor antagonists: overcoming developmental hurdles. <span><span class="ref-journal">Nat. Rev. Drug Discov. </span>2009;<span class="ref-vol">8</span>(1):23–33.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19079127" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Drug+Discov.&amp;title=Chemokine+receptor+antagonists:+overcoming+developmental+hurdles&amp;author=R.+Horuk&amp;volume=8&amp;issue=1&amp;publication_year=2009&amp;pages=23-33&amp;pmid=19079127&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib66">66. <span class="element-citation" id="sref66">Neighbour H., Boulet L.P., Lemiere C., et al. Safety and efficacy of an oral CCR3 antagonist in patients with asthma and eosinophilic bronchitis: a randomized, placebo-controlled clinical trial. <span><span class="ref-journal">Clin. Exp. Allergy. </span>2014;<span class="ref-vol">44</span>(4):508–516.</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24286456" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Clin.+Exp.+Allergy&amp;title=Safety+and+efficacy+of+an+oral+CCR3+antagonist+in+patients+with+asthma+and+eosinophilic+bronchitis:+a+randomized,+placebo-controlled+clinical+trial&amp;author=H.+Neighbour&amp;author=L.P.+Boulet&amp;author=C.+Lemiere&amp;volume=44&amp;issue=4&amp;publication_year=2014&amp;pages=508-516&amp;pmid=24286456&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib67">67. <span class="element-citation" id="sref67">Levit Kaplan A., Strachan R.T., Braz J.M., et al. Structure-based design of a chemical probe set for the 5-HT(5A) serotonin receptor. <span><span class="ref-journal">J. Med. Chem. </span>2022;<span class="ref-vol">65</span>(5):4201–4217.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9116900/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35195401" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Med.+Chem.&amp;title=Structure-based+design+of+a+chemical+probe+set+for+the+5-HT(5A)+serotonin+receptor&amp;author=A.+Levit+Kaplan&amp;author=R.T.+Strachan&amp;author=J.M.+Braz&amp;volume=65&amp;issue=5&amp;publication_year=2022&amp;pages=4201-4217&amp;pmid=35195401&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib68">68. <span class="element-citation" id="sref68">Jaeger K., Bruenle S., Weinert T., et al. Structural basis for allosteric ligand recognition in the human CC chemokine receptor 7. <span><span class="ref-journal">Cell. </span>2019;<span class="ref-vol">178</span>(5):1222–1230 e10.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6709783/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31442409" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Structural+basis+for+allosteric+ligand+recognition+in+the+human+CC+chemokine+receptor+7&amp;author=K.+Jaeger&amp;author=S.+Bruenle&amp;author=T.+Weinert&amp;volume=178&amp;issue=5&amp;publication_year=2019&amp;pages=1222-1230+e10&amp;pmid=31442409&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="bib69">69. <span class="element-citation" id="sref69">Sriram K., Insel P.A. G protein-coupled receptors as targets for approved drugs: how many targets and how many drugs? <span><span class="ref-journal">Mol. Pharmacol. </span>2018;<span class="ref-vol">93</span>(4):251–258.</span> <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5820538/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29298813" ref="reftype=pubmed&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Pharmacol.&amp;title=G+protein-coupled+receptors+as+targets+for+approved+drugs:+how+many+targets+and+how+many+drugs?&amp;author=K.+Sriram&amp;author=P.A.+Insel&amp;volume=93&amp;issue=4&amp;publication_year=2018&amp;pages=251-258&amp;pmid=29298813&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11058304&amp;issue-id=461321&amp;journal-id=2992&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div class="courtesy-note whole_rhythm small"><hr /><div class="half_rhythm">Articles from <span class="acknowledgment-journal-title">Heliyon</span> are provided here courtesy of <strong>Elsevier</strong></div><hr /></div><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11058304/pdf/main.pdf" class="int-view">PDF (14M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11058304/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11058304/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11058304" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11058304%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="8z10ESaZOnYQJFJNEUZVWEPiV0tv0gvDTNQfPUq078o5Yv9cN836qezJrB1ZNbgZ">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11058304%2F&amp;text=Exploration%20of%20prognostic%20and%20treatment%20markers%20in%20hepatocellular%20carcinoma%20via%20GPCR-related%20genes%20analysis" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11058304%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11058304/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11058304/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38694033/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11058304/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38694033/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11058304/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11058304/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript" src="/core/mathjax/2.7.9/MathJax.js?config=/corehtml/pmc/js/mathjax-config-classic.3.4.js"></script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11058304/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
