
    
<!DOCTYPE html>




<html lang="en" >
<head >
    <meta charset="UTF-8">
    <meta http-equiv="X-UA-Compatible" content="IE=edge">

    <!-- Mobile properties -->
    <meta name="HandheldFriendly" content="True">
    <meta name="MobileOptimized" content="320">
    <meta name="viewport" content="width=device-width, initial-scale=1.0">

  
    <!-- Stylesheets -->
    <link rel="stylesheet" href="/pmc/static/CACHE/css/output.cc84172e9e16.css" type="text/css">
  
  <link rel="stylesheet" href="/pmc/static/CACHE/css/output.b634933a977d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.913c7dcab8ff.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.3766d7ad0d2d.css" type="text/css"><link rel="stylesheet" href="/pmc/static/CACHE/css/output.e3c3c2c84eb3.css" type="text/css">

  
    <link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.7.0/css/font-awesome.min.css"/>
  
<link type="text/css" href="/pmc/static/bundles/base/base.a2ef7ca69e4b20dff539.css" rel="stylesheet" />


    <link rel="stylesheet" href="/corehtml/pmc/css/fonts/stix/stixfonts.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/3.18/pmcrefs1.min.css" type="text/css" /><link rel="stylesheet" href="/corehtml/pmc/css/pmc2020_1.1/ncbi_web.min.css?_=a" type="text/css" /><style type="text/css">.pmc-wm {background:transparent repeat-y top left;background-image:url(/corehtml/pmc/pmcgifs/wm-manuscript.gif);background-size: auto, contain}</style><style type="text/css">.print-view{display:block}</style>

    <link type="text/css" href="/pmc/static/bundles/article/article.3c5d12ff2e4beea20136.css" rel="stylesheet" />
    <link type="text/css" href="/pmc/static/django_pmc_cite_box/lib/cite-box.css" rel="stylesheet" />


    <title>Peripheral neuronal activation shapes the microbiome and alters gut physiology - PMC</title>

  
  <!-- Favicons -->
  <link rel="shortcut icon" type="image/ico" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.ico" />
  <link rel="icon" type="image/png" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon.png" />

  <!-- 192x192, as recommended for Android
  http://updates.html5rocks.com/2014/11/Support-for-theme-color-in-Chrome-39-for-Android
  -->
  <link rel="icon" type="image/png" sizes="192x192" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-192.png" />

  <!-- 57x57 (precomposed) for iPhone 3GS, pre-2011 iPod Touch and older Android devices -->
  <link rel="apple-touch-icon-precomposed" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-57.png">
  <!-- 72x72 (precomposed) for 1st generation iPad, iPad 2 and iPad mini -->
  <link rel="apple-touch-icon-precomposed" sizes="72x72" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-72.png">
  <!-- 114x114 (precomposed) for iPhone 4, 4S, 5 and post-2011 iPod Touch -->
  <link rel="apple-touch-icon-precomposed" sizes="114x114" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-114.png">
  <!-- 144x144 (precomposed) for iPad 3rd and 4th generation -->
  <link rel="apple-touch-icon-precomposed" sizes="144x144" href="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/favicons/favicon-144.png">



    
        <!-- Logging params: Pinger defaults -->

<meta name="ncbi_app" content="pmc-frontend" />

<meta name="ncbi_db" content="pmc" />

<meta name="ncbi_phid" content="D0BD41E4AC192A450000318DDB0C2FF4.1.m_2" />


        <!-- Logging params: Pinger custom -->

<meta name="ncbi_pdid" content="article" />

<meta name="ncbi_op" content="retrieved" />

<meta name="ncbi_app_version" content="0.0" />

<meta name="ncbi_domain" content="nihpa" />

<meta name="ncbi_type" content="fulltext" />

<meta name="ncbi_pcid" content="/articles/PMC11132177/" />


    


    <script>
        
            var ncbiBaseUrl = "https://www.ncbi.nlm.nih.gov";
        
        var useOfficialGovtHeader = true;
    </script>


    <meta name="robots" content="INDEX,NOFOLLOW,NOARCHIVE" /><link rel="canonical" href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11132177/" /><link rel="schema.DC" href="http://purl.org/DC/elements/1.0/" /><meta name="citation_journal_title" content="Cell reports" /><meta name="citation_title" content="Peripheral neuronal activation shapes the microbiome and alters gut physiology" /><meta name="citation_author" content="Jessica A. Griffiths" /><meta name="citation_author_institution" content="Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA" /><meta name="citation_author_institution" content="Aligning Science Across Parkinson&#x02019;s (ASAP) Collaborative Research Network, Chevy Chase, MD 20815, USA" /><meta name="citation_author_institution" content="These authors contributed equally" /><meta name="citation_author" content="Bryan B. Yoo" /><meta name="citation_author_institution" content="Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA" /><meta name="citation_author_institution" content="These authors contributed equally" /><meta name="citation_author" content="Peter Thuy-Boun" /><meta name="citation_author_institution" content="Departments of Molecular Medicine and Integrative Structural and Computational Biology, The Scripps Research Institute, 10550 North Torrey Pines Road, La Jolla, CA 92037, USA" /><meta name="citation_author_institution" content="These authors contributed equally" /><meta name="citation_author" content="Victor J. Cantu" /><meta name="citation_author_institution" content="Department of Pediatrics, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author_institution" content="These authors contributed equally" /><meta name="citation_author" content="Kelly C. Weldon" /><meta name="citation_author_institution" content="Collaborative Mass Spectrometry Innovation Center, Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author_institution" content="UCSD Center for Microbiome Innovation, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author_institution" content="These authors contributed equally" /><meta name="citation_author" content="Collin Challis" /><meta name="citation_author_institution" content="Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA" /><meta name="citation_author" content="Michael J. Sweredoski" /><meta name="citation_author_institution" content="Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA" /><meta name="citation_author" content="Ken Y. Chan" /><meta name="citation_author_institution" content="Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA" /><meta name="citation_author_institution" content="Present address: Stanley Center for Psychiatric Research, Broad Institute, Massachusetts Institute of Technology, Cambridge, MA 02142, USA" /><meta name="citation_author" content="Taren M. Thron" /><meta name="citation_author_institution" content="Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA" /><meta name="citation_author" content="Gil Sharon" /><meta name="citation_author_institution" content="Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA" /><meta name="citation_author" content="Annie Moradian" /><meta name="citation_author_institution" content="Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA" /><meta name="citation_author_institution" content="Present address: Precision Biomarker Labs, Cedars-Sinai, 8700 Beverly Blvd., Davis 2904, Los Angeles, CA 90048, USA" /><meta name="citation_author" content="Gregory Humphrey" /><meta name="citation_author_institution" content="Department of Pediatrics, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author" content="Qiyun Zhu" /><meta name="citation_author_institution" content="Department of Pediatrics, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author_institution" content="Present address: School of Life Sciences, Arizona State University, Tempe, AZ 85281, USA" /><meta name="citation_author" content="Justin P. Shaffer" /><meta name="citation_author_institution" content="Department of Pediatrics, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author_institution" content="Present address: Department of Biology, College of Science and Mathematics, Fresno, CA 93740, USA" /><meta name="citation_author" content="Dennis W. Wolan" /><meta name="citation_author_institution" content="Departments of Molecular Medicine and Integrative Structural and Computational Biology, The Scripps Research Institute, 10550 North Torrey Pines Road, La Jolla, CA 92037, USA" /><meta name="citation_author" content="Pieter C. Dorrestein" /><meta name="citation_author_institution" content="Department of Pediatrics, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author_institution" content="Collaborative Mass Spectrometry Innovation Center, Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author_institution" content="UCSD Center for Microbiome Innovation, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author" content="Rob Knight" /><meta name="citation_author_institution" content="Department of Pediatrics, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author_institution" content="UCSD Center for Microbiome Innovation, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author_institution" content="Department of Computer Science and Engineering, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author_institution" content="Shu Chien-Gene Lay Department of Engineering, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author_institution" content="Hal&#x00131;c&#x00131;o&#x0011f;lu Data Science Institute, University of California, San Diego, San Diego, CA, USA" /><meta name="citation_author" content="Viviana Gradinaru" /><meta name="citation_author_institution" content="Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA" /><meta name="citation_author_institution" content="Aligning Science Across Parkinson&#x02019;s (ASAP) Collaborative Research Network, Chevy Chase, MD 20815, USA" /><meta name="citation_author" content="Sarkis K. Mazmanian" /><meta name="citation_author_institution" content="Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA" /><meta name="citation_author_institution" content="Aligning Science Across Parkinson&#x02019;s (ASAP) Collaborative Research Network, Chevy Chase, MD 20815, USA" /><meta name="citation_author_institution" content="Lead contact" /><meta name="citation_publication_date" content="2024/04/04" /><meta name="citation_issue" content="4" /><meta name="citation_volume" content="43" /><meta name="citation_firstpage" content="113953" /><meta name="citation_doi" content="10.1016/j.celrep.2024.113953" /><meta name="citation_abstract_html_url" content="/pmc/articles/PMC11132177/?report=abstract" /><meta name="citation_fulltext_html_url" content="/pmc/articles/PMC11132177/" /><meta name="citation_pmid" content="38517896" /><meta name="DC.Title" content="Peripheral neuronal activation shapes the microbiome and alters gut physiology" /><meta name="DC.Type" content="Text" /><meta name="DC.Publisher" content="NIH Public Access" /><meta name="DC.Contributor" content="Jessica A. Griffiths" /><meta name="DC.Contributor" content="Bryan B. Yoo" /><meta name="DC.Contributor" content="Peter Thuy-Boun" /><meta name="DC.Contributor" content="Victor J. Cantu" /><meta name="DC.Contributor" content="Kelly C. Weldon" /><meta name="DC.Contributor" content="Collin Challis" /><meta name="DC.Contributor" content="Michael J. Sweredoski" /><meta name="DC.Contributor" content="Ken Y. Chan" /><meta name="DC.Contributor" content="Taren M. Thron" /><meta name="DC.Contributor" content="Gil Sharon" /><meta name="DC.Contributor" content="Annie Moradian" /><meta name="DC.Contributor" content="Gregory Humphrey" /><meta name="DC.Contributor" content="Qiyun Zhu" /><meta name="DC.Contributor" content="Justin P. Shaffer" /><meta name="DC.Contributor" content="Dennis W. Wolan" /><meta name="DC.Contributor" content="Pieter C. Dorrestein" /><meta name="DC.Contributor" content="Rob Knight" /><meta name="DC.Contributor" content="Viviana Gradinaru" /><meta name="DC.Contributor" content="Sarkis K. Mazmanian" /><meta name="DC.Date" content="2024 Apr 23" /><meta name="DC.Identifier" content="10.1016/j.celrep.2024.113953" /><meta name="DC.Language" content="en" /><meta property="og:title" content="Peripheral neuronal activation shapes the microbiome and alters gut physiology" /><meta property="og:type" content="article" /><meta property="og:description" content="The gastrointestinal (GI) tract is innervated by intrinsic neurons of the enteric nervous system (ENS) and extrinsic neurons of the central nervous system and peripheral ganglia. The GI tract also harbors a diverse microbiome, but interactions between ..." /><meta property="og:url" content="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11132177/" /><meta property="og:site_name" content="PubMed Central (PMC)" /><meta property="og:image" content="https://www.ncbi.nlm.nih.gov/corehtml/pmc/pmcgifs/pmc-card-share.jpg?_=0" /><meta name="twitter:card" content="summary_large_image" /><meta name="twitter:site" content="@ncbi" /><meta name="ncbi_feature" content="associated_data" />


</head>
<body >

   
    
        <button
          class="back-to-top back-to-top--bottom"
          data-ga-category="pagination"
          data-ga-action="back_to_top">
          Back to Top
        </button>
    

    <a class="usa-skipnav" href="#main-content">Skip to main content</a>
    <!-- ========== BEGIN HEADER ========== -->
<section class="usa-banner" style="display: none;">
  <div class="usa-accordion">
    <header class="usa-banner-header">
      <div class="usa-grid usa-banner-inner">
        <img src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/favicons/favicon-57.png" alt="U.S. flag" />
        <p>An official website of the United States government</p>
        <button
          class="usa-accordion-button usa-banner-button"
          aria-expanded="false"
          aria-controls="gov-banner-top"
        >
          <span class="usa-banner-button-text">Here's how you know</span>
        </button>
      </div>
    </header>
    <div
      class="usa-banner-content usa-grid usa-accordion-content"
      id="gov-banner-top"
    >
      <div class="usa-banner-guidance-gov usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-dot-gov.svg"
          alt="Dot gov"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The .gov means it’s official.</strong>
            <br />
            Federal government websites often end in .gov or .mil. Before
            sharing sensitive information, make sure you’re on a federal
            government site.
          </p>
        </div>
      </div>
      <div class="usa-banner-guidance-ssl usa-width-one-half">
        <img
          class="usa-banner-icon usa-media_block-img"
          src="https://www.ncbi.nlm.nih.gov/coreutils/uswds/img/icon-https.svg"
          alt="Https"
        />
        <div class="usa-media_block-body">
          <p>
            <strong>The site is secure.</strong>
            <br />
            The <strong>https://</strong> ensures that you are connecting to the
            official website and that any information you provide is encrypted
            and transmitted securely.
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
<div class="usa-overlay"></div>
<header class="ncbi-header" role="banner" data-section="Header">

	<div class="usa-grid">
		<div class="usa-width-one-whole">

            <div class="ncbi-header__logo">
                <a href="https://www.ncbi.nlm.nih.gov/" class="logo" aria-label="NCBI Logo" data-ga-action="click_image" data-ga-label="NIH NLM Logo">
                  <img src="https://www.ncbi.nlm.nih.gov/coreutils/nwds/img/logos/AgencyLogo.svg" alt="NIH NLM Logo" />
                </a>
            </div>

			<div class="ncbi-header__account">
				<a id="account_login" href="https://account.ncbi.nlm.nih.gov" class="usa-button header-button" style="display:none" data-ga-action="open_menu" data-ga-label="account_menu">Log in</a>
				<button id="account_info" class="header-button" style="display:none"
						aria-controls="account_popup">
					<span class="fa fa-user" aria-hidden="true"></span>
					<span class="username desktop-only" aria-hidden="true" id="uname_short"></span>
					<span class="sr-only">Show account info</span>
				</button>
			</div>

			<div class="ncbi-popup-anchor">
				<div class="ncbi-popup account-popup" id="account_popup" aria-hidden="true">
					<div class="ncbi-popup-head">
						<button class="ncbi-close-button" data-ga-action="close_menu" data-ga-label="account_menu"><span class="fa fa-times"></span><span class="usa-sr-only">Close</span></button>
						<h4>Account</h4>
					</div>
					<div class="account-user-info">
						Logged in as:<br/>
						<b><span class="username" id="uname_long">username</span></b>
					</div>
					<div class="account-links">
						<ul class="usa-unstyled-list">
							<li><a id="account_myncbi" href="/myncbi/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_myncbi">Dashboard</a></li>
							<li><a id="account_pubs" href="/myncbi/collections/bibliography/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_pubs">Publications</a></li>
							<li><a id="account_settings" href="/account/settings/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_settings">Account settings</a></li>
							<li><a id="account_logout" href="/account/signout/" class="set-base-url" data-ga-action="click_menu_item" data-ga-label="account_logout">Log out</a></li>
						</ul>
					</div>
				</div>
			</div>

		</div>
	</div>
</header>
<div role="navigation" aria-label="access keys">
<a id="nws_header_accesskey_0" href="https://www.ncbi.nlm.nih.gov/guide/browsers/#ncbi_accesskeys" class="usa-sr-only" accesskey="0" tabindex="-1">Access keys</a>
<a id="nws_header_accesskey_1" href="https://www.ncbi.nlm.nih.gov" class="usa-sr-only" accesskey="1" tabindex="-1">NCBI Homepage</a>
<a id="nws_header_accesskey_2" href="/myncbi/" class="set-base-url usa-sr-only" accesskey="2" tabindex="-1">MyNCBI Homepage</a>
<a id="nws_header_accesskey_3" href="#maincontent" class="usa-sr-only" accesskey="3" tabindex="-1">Main Content</a>
<a id="nws_header_accesskey_4" href="#" class="usa-sr-only" accesskey="4" tabindex="-1">Main Navigation</a>
</div>
<section data-section="Alerts">
	<div class="ncbi-alerts-placeholder"></div>
</section>
<!-- ========== END HEADER ========== -->

    
    


    
        

<section class="pmc-alerts">
    
    <div role="alert" class="pmc-alert pmc-alert--info" role="region" aria-label="Alert" aria-hidden="false"  data-key="pmc-alert-welcome">
        <div class="pmc-alert__body pmc-alert--ncbi-icon">
            <div class="pmc-alert__content">
                <p>
                    <strong>
                        Preview improvements coming to the PMC website in October 2024.
                        <a href="https://ncbiinsights.ncbi.nlm.nih.gov/2024/03/14/preview-pmc-improvements/" data-ga-category="cloud_beta_banner" data-ga-label="learn_more" data-ga-action="on_pmc2020">Learn More</a> or
                        <a href="https://pmc.ncbi.nlm.nih.gov/articles/PMC11132177/" data-ga-category="cloud_beta_banner" data-ga-action="on_pmc2020" data-ga-label="go_to_cloud">Try it out now</a>.
                    </strong>
                </p>
            </div>
        </div>
    </div>
    
</section>
    

    
    
        
        <header class="pmc-header usa-header-extended" role="banner">
    <div class="pmc-header__bar">
        <div class="pmc-header__control usa-accordion">
            
                <button class="usa-menu-btn pmc-header--button pmc-header--left">
                    <i class="fa fa-ellipsis-v" aria-hidden="true"></i>
                </button>
            

            <div class="usa-logo pmc-header__logo pmc-header--stretch
                
               " id="extended-mega-logo">
                <div class="usa-logo-text">
                    <a href="/pmc/" title="Home" aria-label="Home" ></a>
                </div>
            </div>
            <button class="usa-accordion-button pmc-header--right pmc-header--button pmc-header__search--control pmc-header--right" aria-expanded="false" aria-controls="a1">
                <i class="fa fa-search" aria-hidden="true"></i>
                <i class="fa fa-times" aria-hidden="true"></i>
            </button>
        </div>
        <div class="pmc-header__search usa-accordion-content" id="a1">
            <div role="search"  class="pmc-header--stretch"  >
    <form class="usa-search " autocomplete="off">
        <div>
              <label class="usa-sr-only" for="pmc-search">Search PMC Full-Text Archive</label>
              <span class="clearable">
                <input
                  required="required"
                  autocomplete-url="/pmc/autocomplete/pmc/"
                  placeholder="Search PMC Full-Text Archive"
                  id="pmc-search" type="search" name="term"/>
                  <i class="clear-btn" ></i>
              </span>
              <button type="submit" formaction="/pmc/">
                <span class="usa-search-submit-text">Search in PMC</span>
              </button>
        </div>
    </form>
</div>


            <ul class="usa-unstyled-list usa-nav-secondary-links pmc-header--offset-two">
                    <li>
                        <a href="https://www.ncbi.nlm.nih.gov/pmc/advanced" data-ga-action="featured_link" data-ga-label="advanced_search">
                            Advanced Search
                        </a>
                    </li>
                    <li>
                        <a  href="/pmc/about/userguide/" data-ga-action="featured_link"
                        data-ga-label="user guide">
                            User Guide
                        </a>
                    </li>
            </ul>
        </div>
    </div>
     <nav role="navigation" class="usa-nav ">
        <button class="usa-nav-close">
            <i class="fa fa-times" aria-hidden="true"></i>
        </button>
        <div class="usa-breadcrumb usa-breadcrumb--wrap usa-breadcrumb--hack">
             
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11132177
                </li>
            
    </ul>
 

        </div>
        
        
            <div class="pmc-sidebar pmc-sidebar-hack">
                

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11132177/pdf/nihms-1988700.pdf" class="int-view">PDF (3.4M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11132177/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11132177/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11132177" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11132177%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="pqU7rL8SzYlKJ3KnG6tPtxSidQIUtuVmlFmqyr8eOUO6T30PmV1OWab9aIRSjHX7">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-header "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-header ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-header "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-header ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11132177%2F&amp;text=Peripheral%20neuronal%20activation%20shapes%20the%20microbiome%20and%20alters%20gut%20physiology" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11132177%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11132177/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11132177/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38517896/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-header" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11132177/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38517896/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-header"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11132177/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-header"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11132177/" class="usa-accordion-content" id="links-accordion-header" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>
            </div>
        

    </nav>

</header>

    
    

    <div class="usa-overlay"></div>
    
<main id="main-content" class="usa-grid usa-layout-docs pmc-main">
    <article class="usa-width-three-fourths usa-layout-docs-main_content pmc-article">
        <section class="usa-breadcrumb usa-breadcrumb--wrap">
         
    <ul class="usa-breadcrumb__list">
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/journals/" class="navlink">Journal List</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                    >
                    <a href="/pmc/?term=hhs%20author%20manuscript[filter]" class="navlink">HHS Author Manuscripts</a>
                </li>
            
                <li class="usa-breadcrumb__list-item"
                     aria-current="page" >
                    PMC11132177
                </li>
            
    </ul>
 

        </section>
        
  <div class="pmc-article__disclaimer" role="complementary" aria-label="Disclaimer note">
    As a library, NLM provides access to scientific literature. Inclusion in an NLM database does not imply endorsement of, or agreement with,
    the contents by NLM or the National Institutes of Health.<br/>
    Learn more:
    <a data-ga-category="Link click" data-ga-action="Disclaimer" data-ga-label="New disclaimer box" href="/pmc/about/disclaimer/">PMC Disclaimer</a>
    |
    <a data-ga-category="Link click" data-ga-action="PMC Copyright Notice" data-ga-label="New disclaimer box" href="/pmc/about/copyright/">
        PMC Copyright Notice
    </a>
</div>

        <section class="pmc-page-banner" role="banner">
            
                
                    <div><img src="/corehtml/pmc/pmcgifs/logo-nihpa.png" alt="Logo of nihpa" usemap="#logo-imagemap" /><map id="logo-imagemap" name="logo-imagemap"><area alt="Link to Publisher's site" title="Link to Publisher's site" shape="default" coords="0,0,499,74" href="https://www.hhs.gov/open/public-access-guiding-principles/index.html" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBanner&amp;TO=Publisher%7COther%7CN/A" /></map></div> 
                
            
        </section>
        <section  role="document">
            
                <div id="mc" class=" article lit-style content pmc-wm slang-all page-box"><!--main-content--><div class="jig-ncbiinpagenav" data-jigconfig="smoothScroll: false, allHeadingLevels: ['h2'], headingExclude: ':hidden,.nomenu'"><div class="fm-sec half_rhythm no_top_margin"><div class="fm-flexbox"><div class="fm-citation"><div><span id="pmcmata">Cell Rep.</span> Author manuscript; available in PMC 2024 May 28.</div><div></div><div class="final-form"><em>Published in final edited form as:</em></div><div class="final-citation"><div class="fm-vol-iss-date"><a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=38517896" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Cell Rep. 2024 Apr 23; 43(4): 113953. </a></div><span class="fm-vol-iss-date">Published online 2024 Mar 21. </span> <span class="doi"><span>doi: </span><a href="//doi.org/10.1016%2Fj.celrep.2024.113953" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CCrosslink%7CDOI">10.1016/j.celrep.2024.113953</a></span></div></div><div class="fm-ids"><div class="fm-citation-pmcid"><span class="fm-citation-ids-label">PMCID: </span><span>PMC11132177</span></div><div class="fm-citation-manuscriptid"><span class="fm-citation-ids-label">NIHMSID: </span><span>NIHMS1988700</span></div><div class="fm-citation-pmid">PMID: <a href="https://pubmed.ncbi.nlm.nih.gov/38517896">38517896</a></div></div></div><h1 class="content-title">Peripheral neuronal activation shapes the microbiome and alters gut physiology</h1><div class="half_rhythm"><div class="contrib-group fm-author"><a href="https://pubmed.ncbi.nlm.nih.gov/?term=Griffiths%20JA%5BAuthor%5D" class="affpopup" co-rid="_co_idm140265030573712" co-class="co-affbox">Jessica A. Griffiths</a>,<sup>1,</sup><sup>2,</sup><sup>10</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yoo%20BB%5BAuthor%5D" class="affpopup" co-rid="_co_idm140264992192672" co-class="co-affbox">Bryan B. Yoo</a>,<sup>1,</sup><sup>10</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Thuy-Boun%20P%5BAuthor%5D" class="affpopup" co-rid="_co_idm140265030570944" co-class="co-affbox">Peter Thuy-Boun</a>,<sup>3,</sup><sup>11</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Cantu%20VJ%5BAuthor%5D" class="affpopup" co-rid="_co_idm140265030568896" co-class="co-affbox">Victor J. Cantu</a>,<sup>4,</sup><sup>11</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Weldon%20KC%5BAuthor%5D" class="affpopup" co-rid="_co_idm140265030566848" co-class="co-affbox">Kelly C. Weldon</a>,<sup>5,</sup><sup>6,</sup><sup>11</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Challis%20C%5BAuthor%5D" class="affpopup" co-rid="_co_idm140264990962640" co-class="co-affbox">Collin Challis</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sweredoski%20MJ%5BAuthor%5D" class="affpopup" co-rid="_co_idm140264990960880" co-class="co-affbox">Michael J. Sweredoski</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chan%20KY%5BAuthor%5D" class="affpopup" co-rid="_co_idm140264990958896" co-class="co-affbox">Ken Y. Chan</a>,<sup>1,</sup><sup>12</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Thron%20TM%5BAuthor%5D" class="affpopup" co-rid="_co_idm140264990956176" co-class="co-affbox">Taren M. Thron</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sharon%20G%5BAuthor%5D" class="affpopup" co-rid="_co_idm140264990954192" co-class="co-affbox">Gil Sharon</a>,<sup>1</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Moradian%20A%5BAuthor%5D" class="affpopup" co-rid="_co_idm140264990952208" co-class="co-affbox">Annie Moradian</a>,<sup>1,</sup><sup>13</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Humphrey%20G%5BAuthor%5D" class="affpopup" co-rid="_co_idm140264990949488" co-class="co-affbox">Gregory Humphrey</a>,<sup>4</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhu%20Q%5BAuthor%5D" class="affpopup" co-rid="_co_idm140265033163456" co-class="co-affbox">Qiyun Zhu</a>,<sup>4,</sup><sup>14</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shaffer%20JP%5BAuthor%5D" class="affpopup" co-rid="_co_idm140265033160736" co-class="co-affbox">Justin P. Shaffer</a>,<sup>4,</sup><sup>15</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wolan%20DW%5BAuthor%5D" class="affpopup" co-rid="_co_idm140265033158016" co-class="co-affbox">Dennis W. Wolan</a>,<sup>3</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Dorrestein%20PC%5BAuthor%5D" class="affpopup" co-rid="_co_idm140265033156032" co-class="co-affbox">Pieter C. Dorrestein</a>,<sup>4,</sup><sup>5,</sup><sup>6</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Knight%20R%5BAuthor%5D" class="affpopup" co-rid="_co_idm140265033152576" co-class="co-affbox">Rob Knight</a>,<sup>4,</sup><sup>6,</sup><sup>7,</sup><sup>8,</sup><sup>9</sup> <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gradinaru%20V%5BAuthor%5D" class="affpopup" co-rid="_co_idm140264993403984" co-class="co-affbox">Viviana Gradinaru</a>,<sup>1,</sup><sup>2</sup> and  <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mazmanian%20SK%5BAuthor%5D" class="affpopup" co-rid="_co_idm140264993401264" co-class="co-affbox">Sarkis K. Mazmanian</a><sup>1,</sup><sup>2,</sup><sup>16,</sup><sup>*</sup></div><div style="display:none" class="contrib-group aff-tip"><div id="_co_idm140265030573712"><h3 class="no_margin">Jessica A. Griffiths</h3><p><sup>1</sup>Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</p><p><sup>2</sup>Aligning Science Across Parkinson&#x02019;s (ASAP) Collaborative Research Network, Chevy Chase, MD 20815, USA</p><p><sup>10</sup>These authors contributed equally</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Griffiths%20JA%5BAuthor%5D">Jessica A. Griffiths</a></div></div><div id="_co_idm140264992192672"><h3 class="no_margin">Bryan B. Yoo</h3><p><sup>1</sup>Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</p><p><sup>10</sup>These authors contributed equally</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Yoo%20BB%5BAuthor%5D">Bryan B. Yoo</a></div></div><div id="_co_idm140265030570944"><h3 class="no_margin">Peter Thuy-Boun</h3><p><sup>3</sup>Departments of Molecular Medicine and Integrative Structural and Computational Biology, The Scripps Research Institute, 10550 North Torrey Pines Road, La Jolla, CA 92037, USA</p><p><sup>11</sup>These authors contributed equally</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Thuy-Boun%20P%5BAuthor%5D">Peter Thuy-Boun</a></div></div><div id="_co_idm140265030568896"><h3 class="no_margin">Victor J. Cantu</h3><p><sup>4</sup>Department of Pediatrics, University of California, San Diego, San Diego, CA, USA</p><p><sup>11</sup>These authors contributed equally</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Cantu%20VJ%5BAuthor%5D">Victor J. Cantu</a></div></div><div id="_co_idm140265030566848"><h3 class="no_margin">Kelly C. Weldon</h3><p><sup>5</sup>Collaborative Mass Spectrometry Innovation Center, Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego, San Diego, CA, USA</p><p><sup>6</sup>UCSD Center for Microbiome Innovation, University of California, San Diego, San Diego, CA, USA</p><p><sup>11</sup>These authors contributed equally</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Weldon%20KC%5BAuthor%5D">Kelly C. Weldon</a></div></div><div id="_co_idm140264990962640"><h3 class="no_margin">Collin Challis</h3><p><sup>1</sup>Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Challis%20C%5BAuthor%5D">Collin Challis</a></div></div><div id="_co_idm140264990960880"><h3 class="no_margin">Michael J. Sweredoski</h3><p><sup>1</sup>Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sweredoski%20MJ%5BAuthor%5D">Michael J. Sweredoski</a></div></div><div id="_co_idm140264990958896"><h3 class="no_margin">Ken Y. Chan</h3><p><sup>1</sup>Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</p><p><sup>12</sup>Present address: Stanley Center for Psychiatric Research, Broad Institute, Massachusetts Institute of Technology, Cambridge, MA 02142, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Chan%20KY%5BAuthor%5D">Ken Y. Chan</a></div></div><div id="_co_idm140264990956176"><h3 class="no_margin">Taren M. Thron</h3><p><sup>1</sup>Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Thron%20TM%5BAuthor%5D">Taren M. Thron</a></div></div><div id="_co_idm140264990954192"><h3 class="no_margin">Gil Sharon</h3><p><sup>1</sup>Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Sharon%20G%5BAuthor%5D">Gil Sharon</a></div></div><div id="_co_idm140264990952208"><h3 class="no_margin">Annie Moradian</h3><p><sup>1</sup>Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</p><p><sup>13</sup>Present address: Precision Biomarker Labs, Cedars-Sinai, 8700 Beverly Blvd., Davis 2904, Los Angeles, CA 90048, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Moradian%20A%5BAuthor%5D">Annie Moradian</a></div></div><div id="_co_idm140264990949488"><h3 class="no_margin">Gregory Humphrey</h3><p><sup>4</sup>Department of Pediatrics, University of California, San Diego, San Diego, CA, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Humphrey%20G%5BAuthor%5D">Gregory Humphrey</a></div></div><div id="_co_idm140265033163456"><h3 class="no_margin">Qiyun Zhu</h3><p><sup>4</sup>Department of Pediatrics, University of California, San Diego, San Diego, CA, USA</p><p><sup>14</sup>Present address: School of Life Sciences, Arizona State University, Tempe, AZ 85281, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Zhu%20Q%5BAuthor%5D">Qiyun Zhu</a></div></div><div id="_co_idm140265033160736"><h3 class="no_margin">Justin P. Shaffer</h3><p><sup>4</sup>Department of Pediatrics, University of California, San Diego, San Diego, CA, USA</p><p><sup>15</sup>Present address: Department of Biology, College of Science and Mathematics, Fresno, CA 93740, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Shaffer%20JP%5BAuthor%5D">Justin P. Shaffer</a></div></div><div id="_co_idm140265033158016"><h3 class="no_margin">Dennis W. Wolan</h3><p><sup>3</sup>Departments of Molecular Medicine and Integrative Structural and Computational Biology, The Scripps Research Institute, 10550 North Torrey Pines Road, La Jolla, CA 92037, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Wolan%20DW%5BAuthor%5D">Dennis W. Wolan</a></div></div><div id="_co_idm140265033156032"><h3 class="no_margin">Pieter C. Dorrestein</h3><p><sup>4</sup>Department of Pediatrics, University of California, San Diego, San Diego, CA, USA</p><p><sup>5</sup>Collaborative Mass Spectrometry Innovation Center, Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego, San Diego, CA, USA</p><p><sup>6</sup>UCSD Center for Microbiome Innovation, University of California, San Diego, San Diego, CA, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Dorrestein%20PC%5BAuthor%5D">Pieter C. Dorrestein</a></div></div><div id="_co_idm140265033152576"><h3 class="no_margin">Rob Knight</h3><p><sup>4</sup>Department of Pediatrics, University of California, San Diego, San Diego, CA, USA</p><p><sup>6</sup>UCSD Center for Microbiome Innovation, University of California, San Diego, San Diego, CA, USA</p><p><sup>7</sup>Department of Computer Science and Engineering, University of California, San Diego, San Diego, CA, USA</p><p><sup>8</sup>Shu Chien-Gene Lay Department of Engineering, University of California, San Diego, San Diego, CA, USA</p><p><sup>9</sup>Hal&#x00131;c&#x00131;o&#x0011f;lu Data Science Institute, University of California, San Diego, San Diego, CA, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Knight%20R%5BAuthor%5D">Rob Knight</a></div></div><div id="_co_idm140264993403984"><h3 class="no_margin">Viviana Gradinaru</h3><p><sup>1</sup>Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</p><p><sup>2</sup>Aligning Science Across Parkinson&#x02019;s (ASAP) Collaborative Research Network, Chevy Chase, MD 20815, USA</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Gradinaru%20V%5BAuthor%5D">Viviana Gradinaru</a></div></div><div id="_co_idm140264993401264"><h3 class="no_margin">Sarkis K. Mazmanian</h3><p><sup>1</sup>Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</p><p><sup>2</sup>Aligning Science Across Parkinson&#x02019;s (ASAP) Collaborative Research Network, Chevy Chase, MD 20815, USA</p><p><sup>16</sup>Lead contact</p><div>Find articles by <a href="https://pubmed.ncbi.nlm.nih.gov/?term=Mazmanian%20SK%5BAuthor%5D">Sarkis K. Mazmanian</a></div></div></div></div><div class="half_rhythm"><div class="togglers fm-copyright-license"><a href="#" class="pmctoggle" rid="idm140264994022080_ai">Author information</a> <a href="#" class="pmctoggle" rid="idm140264994022080_cpl">Copyright and License information</a> <a href="/pmc/about/disclaimer/" style="margin-left: 1em">PMC Disclaimer</a></div><div class="fm-authors-info hide half_rhythm" id="idm140264994022080_ai" style="display:none"><div class="fm-affl" id="A1"><sup>1</sup>Division of Biology &#x00026; Biological Engineering, California Institute of Technology, Pasadena, CA 91125, USA</div><div class="fm-affl" id="A2"><sup>2</sup>Aligning Science Across Parkinson&#x02019;s (ASAP) Collaborative Research Network, Chevy Chase, MD 20815, USA</div><div class="fm-affl" id="A3"><sup>3</sup>Departments of Molecular Medicine and Integrative Structural and Computational Biology, The Scripps Research Institute, 10550 North Torrey Pines Road, La Jolla, CA 92037, USA</div><div class="fm-affl" id="A4"><sup>4</sup>Department of Pediatrics, University of California, San Diego, San Diego, CA, USA</div><div class="fm-affl" id="A5"><sup>5</sup>Collaborative Mass Spectrometry Innovation Center, Skaggs School of Pharmacy and Pharmaceutical Sciences, University of California, San Diego, San Diego, CA, USA</div><div class="fm-affl" id="A6"><sup>6</sup>UCSD Center for Microbiome Innovation, University of California, San Diego, San Diego, CA, USA</div><div class="fm-affl" id="A7"><sup>7</sup>Department of Computer Science and Engineering, University of California, San Diego, San Diego, CA, USA</div><div class="fm-affl" id="A8"><sup>8</sup>Shu Chien-Gene Lay Department of Engineering, University of California, San Diego, San Diego, CA, USA</div><div class="fm-affl" id="A9"><sup>9</sup>Hal&#x00131;c&#x00131;o&#x0011f;lu Data Science Institute, University of California, San Diego, San Diego, CA, USA</div><div class="fm-affl" id="A10"><sup>10</sup>These authors contributed equally</div><div class="fm-affl" id="A11"><sup>11</sup>These authors contributed equally</div><div class="fm-affl" id="A12"><sup>12</sup>Present address: Stanley Center for Psychiatric Research, Broad Institute, Massachusetts Institute of Technology, Cambridge, MA 02142, USA</div><div class="fm-affl" id="A13"><sup>13</sup>Present address: Precision Biomarker Labs, Cedars-Sinai, 8700 Beverly Blvd., Davis 2904, Los Angeles, CA 90048, USA</div><div class="fm-affl" id="A14"><sup>14</sup>Present address: School of Life Sciences, Arizona State University, Tempe, AZ 85281, USA</div><div class="fm-affl" id="A15"><sup>15</sup>Present address: Department of Biology, College of Science and Mathematics, Fresno, CA 93740, USA</div><div class="fm-affl" id="A16"><sup>16</sup>Lead contact</div><div><p id="P1" class="p p-first">AUTHOR CONTRIBUTIONS</p>J.A.G., B.B.Y., and S.K.M. designed experiments. J.A.G. and B.B.Y. performed experiments. K.Y.C., C.C. (supervised by V.G.), and J.A.G. helped with AAV-mediated ENS characterization. P.T.-B. (supervised by D.W.W.), M.J.S., and A.M. performed acquisition and analysis of proteomics data. V.C., G.H., G.S., Q.Z., and J.S. (supervised by R.K.) performed acquisition and analysis of metagenomics data. K.W. (supervised by P.C.D.) performed acquisition and analysis of metabolomics data. T.M.T. assisted with animal-related work. J.A.G., B.B.Y., and S.K.M wrote the manuscript with input from all authors.</div><div id="CR1"><sup>*</sup>Correspondence: <a href="mailto:dev@null" data-email="ude.hcetlac@sikras" class="oemail">ude.hcetlac@sikras</a></div></div><div class="fm-article-notes half_rhythm"></div><div class="permissions half_rhythm hide" id="idm140264994022080_cpl" style="display:none"><div class="fm-copyright half_rhythm"><a href="/pmc/about/copyright/">PMC Copyright notice</a> </div><div class="license half_rhythm">This is an open access article under the CC BY license (<a href="https://creativecommons.org/licenses/by/4.0/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=External%7CLink%7CURI" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>).</div></div></div><div id="pmclinksbox" class="links-box whole_rhythm" role="complementary" aria-label="Related or updated information about this article."><div>The publisher's final edited version of this article is available free at <a href="/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;retmode=ref&amp;cmd=prlinks&amp;id=38517896" target="_blank" rel="noopener noreferrer" ref="reftype=publisher&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CFront%20Matter&amp;TO=Content%20Provider%7CArticle%7CFree%20Access">Cell Rep</a></div></div></div><div class="sec"></div><div id="ass-data" class="tsec fm-sec whole_rhythm" data-section="Featured_PMC_Datacitation"><h2 class="nomenu">Associated Data</h2><dl data-count="6" class="box-data-suppmats whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-suppmats" data-ga-action="click_feat_toggler" data-ga-label="Supplementary Materials" class="pmctoggle">Supplementary Materials</a></dt><dd id="data-suppmats" style="display: none;"><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>1.</strong></div><div><a href="/pmc/articles/PMC11132177/bin/NIHMS1988700-supplement-1.pdf" data-ga-action="click_feat_suppl">NIHMS1988700-supplement-1.pdf</a><span style="color:gray"> (53M)</span></div><div class="small guid">GUID: 1165D156-4AD1-4E20-9183-69B4F8305FA8</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>2.</strong></div><div><a href="/pmc/articles/PMC11132177/bin/NIHMS1988700-supplement-2.xlsx" data-ga-action="click_feat_suppl">NIHMS1988700-supplement-2.xlsx</a><span style="color:gray"> (15K)</span></div><div class="small guid">GUID: 8CEB32EA-4A3A-465B-8F73-8775FF5ED6F0</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>3.</strong></div><div><a href="/pmc/articles/PMC11132177/bin/NIHMS1988700-supplement-3.xlsx" data-ga-action="click_feat_suppl">NIHMS1988700-supplement-3.xlsx</a><span style="color:gray"> (90K)</span></div><div class="small guid">GUID: BA46F66B-22ED-431F-83F7-267F8116E904</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>4.</strong></div><div><a href="/pmc/articles/PMC11132177/bin/NIHMS1988700-supplement-4.xlsx" data-ga-action="click_feat_suppl">NIHMS1988700-supplement-4.xlsx</a><span style="color:gray"> (14K)</span></div><div class="small guid">GUID: 2A413489-59AD-4042-B216-5301B5E405CD</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>5.</strong></div><div><a href="/pmc/articles/PMC11132177/bin/NIHMS1988700-supplement-5.xlsx" data-ga-action="click_feat_suppl">NIHMS1988700-supplement-5.xlsx</a><span style="color:gray"> (28K)</span></div><div class="small guid">GUID: 8CA6DE20-7092-403D-BF88-FF31960B694C</div></div><div class="half_rhythm"><div class="caption half_rhythm no_bottom_margin"><strong>6.</strong></div><div><a href="/pmc/articles/PMC11132177/bin/NIHMS1988700-supplement-6.mp4" data-ga-action="click_feat_suppl">NIHMS1988700-supplement-6.mp4</a><span style="color:gray"> (8.6M)</span></div><div class="small guid">GUID: 6D21D372-68AB-4891-9EA4-C2BAD437CC29</div></div></dd></dl><dl data-length="805" class="box-data-avail whole_rhythm no_bottom_margin"><dt><a href="#" rid="data-avl-stmnt" data-ga-action="click_feat_toggler" data-ga-label="Data Availability Statement" class="pmctoggle">Data Availability Statement</a></dt><dd id="data-avl-stmnt" style="display: none;"><ul class="unordered" style="list-style-type:disc"><li><div>Microbial sequencing data have been deposited at the European Bioinformatics Institute (ERP131523), Metabolomic data at UCSD MassIVE repository (MSV000084550), proteomic data at UCSD MassIVE repository (MSV000087917), and QuantSeq data at NCBI GEO repository (<a class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE180961" ref="/geo/query/acc.cgi?acc=GSE180961"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE180961","term_id":"180961"}}</span>GSE180961</a>) and are publicly available as of the date of publication. All other experimental data used to generate the figures reported in this paper can be found at (DOI:<a href="//doi.org/10.5281/zenodo.10525220" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">10.5281/zenodo.10525220</a>, <a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/</a>) and are publicly available as of the date of publication.</div></li><li><div>This paper does not report original code.</div></li><li><div>Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</div></li></ul></dd></dl></div><div id="ABS1" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="ABS1title">SUMMARY</h2><!--article-meta--><div><p id="P3" class="p p-first-last">The gastrointestinal (GI) tract is innervated by intrinsic neurons of the enteric nervous system (ENS) and extrinsic neurons of the central nervous system and peripheral ganglia. The GI tract also harbors a diverse microbiome, but interactions between the ENS and the microbiome remain poorly understood. Here, we activate choline acetyltransferase (ChAT)-expressing or tyrosine hydroxylase (TH)-expressing gut-associated neurons in mice to determine effects on intestinal microbial communities and their metabolites as well as on host physiology. The resulting multi-omics datasets support broad roles for discrete peripheral neuronal subtypes in shaping microbiome structure, including modulating bile acid profiles and fungal colonization. Physiologically, activation of either ChAT<sup>+</sup> or TH<sup>+</sup> neurons increases fecal output, while only ChAT<sup>+</sup> activation results in increased colonic contractility and diarrhea-like fluid secretion. These findings suggest that specific subsets of peripherally activated neurons differentially regulate the gut microbiome and GI physiology in mice without involvement of signals from the brain.</p></div></div><div id="ABS2" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="ABS2title">Graphical abstract</h2><!--article-meta--><div><p id="P4" class="p p-first-last">
</p><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140264990004320"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0001.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0001.jpg" src="/pmc/articles/PMC11132177/bin/nihms-1988700-f0001.jpg" /></div><p>
</p></div></div><div id="ABS3" lang="en" class="tsec sec"><h2 class="head no_bottom_margin" id="ABS3title">In brief</h2><!--article-meta--><div><p id="P5" class="p p-first-last">Griffiths et al. examine how activating ChAT<sup>+</sup> or TH<sup>+</sup> gut-associated neurons shapes the gut microbiome and physiology in mice. They uncover unique alterations in the metagenome, transcriptome, metabolome, and proteome, as well as GI motility, suggesting that subpopulations of peripherally activated neurons differentially regulate the gut microbiome and GI function with no involvement of signals from the brain.</p></div></div><div id="S1" class="tsec sec"><h2 class="head no_bottom_margin" id="S1title">INTRODUCTION</h2><p id="P6" class="p p-first">Diverse cell types in the gastrointestinal (GI) tract coordinate physiology within the gut<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup> and throughout the body.<sup><a href="#R2" rid="R2" class=" bibr popnode">2</a></sup> The mammalian gut receives and transmits neuronal signals through ~100,000 extrinsic nerve fibers originating from the sympathetic, parasym-pathetic, and sensory nervous systems.<sup><a href="#R3" rid="R3" class=" bibr popnode">3</a></sup> The GI tract is also innervated by an extensive network of over 100 million intrinsic neurons organized into two distinct compartments within the GI tract; namely, the myenteric plexus and submucosal plexus.<sup><a href="#R4" rid="R4" class=" bibr popnode">4</a></sup> The neurons of the GI tract, composing the enteric nervous system (ENS), have been implicated in processes including digestion,<sup><a href="#R5" rid="R5" class=" bibr popnode">5</a></sup> immunity,<sup><a href="#R6" rid="R6" class=" bibr popnode">6</a>,<a href="#R7" rid="R7" class=" bibr popnode">7</a></sup> and even complex behaviors,<sup><a href="#R8" rid="R8" class=" bibr popnode">8</a></sup> in mice. Interactions between neurons of the GI tract and other cell types highlight the diverse roles of the ENS. For example, neuronal pathways in the gut regulate nutrient sensation through intestinal enteroendocrine cells,<sup><a href="#R9" rid="R9" class=" bibr popnode">9</a></sup> modulate the epithelial barrier and mucosal immunity,<sup><a href="#R10" rid="R10" class=" bibr popnode">10</a>&#x02013;<a href="#R12" rid="R12" class=" bibr popnode">12</a></sup> and dynamically interface with the microbiome.<sup><a href="#R13" rid="R13" class=" bibr popnode">13</a>,<a href="#R14" rid="R14" class=" bibr popnode">14</a></sup> Exposure of the ENS to changing diet, microbiome, and xenobiotics creates inputs distinct from those in the central nervous system (CNS); i.e., the brain and spinal cord.</p><p id="P7">Choline acetyltransferase (ChAT) and tyrosine hydroxylase (TH) are the rate-limiting enzymes in acetylcholine and catecholamine biosynthesis, respectively, and are key chemical mediators of neurotransmission in the brain and the periphery. Acetylcholine is the primary excitatory neurotransmitter of the gut, and cholinergic neurons represent 60% of the ENS, mediating intestinal propulsion and secretion.<sup><a href="#R15" rid="R15" class=" bibr popnode">15</a>,<a href="#R16" rid="R16" class=" bibr popnode">16</a></sup> Several studies have established correlations between neuronal activity, abundance, and specific physiological outcomes.<sup><a href="#R17" rid="R17" class=" bibr popnode">17</a>&#x02013;<a href="#R19" rid="R19" class=" bibr popnode">19</a></sup> For example, age-associated reduction of ChAT<sup>+</sup> neurons in the ENS coincides with constipation and evacuation disorders,<sup><a href="#R15" rid="R15" class=" bibr popnode">15</a>,<a href="#R16" rid="R16" class=" bibr popnode">16</a></sup> and clinical studies have shown that anticholinergic drugs cause constipation and cholinergic agonists can cause diarrhea.<sup><a href="#R20" rid="R20" class=" bibr popnode">20</a>,<a href="#R21" rid="R21" class=" bibr popnode">21</a></sup> In a disease context, cholera toxin induces hypersecretion and sustained activation of submucosal ChAT<sup>+</sup> neurons in mice.<sup><a href="#R22" rid="R22" class=" bibr popnode">22</a>,<a href="#R23" rid="R23" class=" bibr popnode">23</a></sup> Although less characterized, TH<sup>+</sup> neurons and dopamine signaling pathways have also been shown to affect GI motility,<sup><a href="#R24" rid="R24" class=" bibr popnode">24</a></sup> and TH<sup>+</sup> neuronal damage in individuals with Parkinson&#x02019;s disease (PD) correlates with increased constipation.<sup><a href="#R25" rid="R25" class=" bibr popnode">25</a>,<a href="#R26" rid="R26" class=" bibr popnode">26</a></sup></p><p id="P8" class="p p-last">Though known to be important for motility and secretomotor function, ChAT<sup>+</sup> and TH<sup>+</sup> neurons have not yet been systematically characterized and interrogated for their roles in GI physiology.<sup><a href="#R17" rid="R17" class=" bibr popnode">17</a>,<a href="#R27" rid="R27" class=" bibr popnode">27</a></sup> One barrier to modulation of neuronal populations in the ENS is its size: 35&#x02013;40 cm in mice. To circumvent the need for direct delivery of effectors, we leveraged a systemically delivered engineered adeno-associated virus (AAV) with enhanced tropism for the ENS and other peripheral ganglia of mice.<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup> Importantly, this vector, AAV-PHP.S, does not transduce the CNS, allowing us to uncouple peripheral activation from brain-to-gut signaling. We find that activating gut-associated ChAT<sup>+</sup> and TH<sup>+</sup> neurons of mice with chemogenetic modulators<sup><a href="#R29" rid="R29" class=" bibr popnode">29</a></sup> alters the transcriptional and proteomic landscape of the intestines as well as the gut metagenome and metabolome. Multiomics analyses allow us to characterize detailed and complex host-microbial interactions and enable prediction of neuronal influence on a number of biological processes in the gut, including providing insights into secondary bile acid production and control of fungal populations, among other interesting associations. In addition, we show that activation of gut-associated neurons strikingly impacts GI function, including motility and fluid secretion. Together, this work reveals differential effects of non-brain activation of ChAT<sup>+</sup> and TH<sup>+</sup> neurons in shaping the gut environment and GI physiology and generates rich datasets as a resource for further exploration (<a href="//doi.org/10.5281/zenodo.10525220" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">https://doi.org/10.5281/zenodo.10525220</a>, <a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/</a>).</p></div><div id="S2" class="tsec sec"><h2 class="head no_bottom_margin" id="S2title">RESULTS</h2><div id="S3" class="sec sec-first"><h3 id="S3title">Distinct spatial localization of ChAT<sup>+</sup> and TH<sup>+</sup> neurons in the ENS</h3><p id="P9" class="p p-first">Broad ENS morphology has been characterized previously using immunohistochemistry (IHC).<sup><a href="#R16" rid="R16" class=" bibr popnode">16</a>,<a href="#R30" rid="R30" class=" bibr popnode">30</a>,<a href="#R31" rid="R31" class=" bibr popnode">31</a></sup> To map neurons in mice with higher resolution, we used recombinant AAVs to fluorescently label enteric neurons <em>in vivo</em> and tissue-clearing techniques to enhance visualization of intact GI tissue.<sup><a href="#R32" rid="R32" class=" bibr popnode">32</a>&#x02013;<a href="#R34" rid="R34" class=" bibr popnode">34</a></sup> Imaging whole tissue, without the need for sectioning, preserves neuronal architectures over large distances and across both longitudinal and cross-sectional axes. The AAV capsid variant AAV-PHP.S is optimized for systemic delivery in mice<sup><a href="#R35" rid="R35" class=" bibr popnode">35</a></sup> and displays increased tropism for the peripheral nervous system (PNS), including the ENS.<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup> To further optimize ENS expression, we replaced the CAG promoter used previously<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup> with the human synapsin 1 (hSYN1) promoter, which has been shown to restrict gene expression to neurons<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup> and minimize expression in peripheral targets, such as the dorsal root ganglia (DRGs).<sup><a href="#R36" rid="R36" class=" bibr popnode">36</a></sup> To assess off-target effects, we compared expression of AAV-PHP.S-delivered hSYN1-mNeonGreen to that of CAG-mNeonGreen in various non-ENS tissues known to affect GI function (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S1</a>). Expression from the hSYN1 construct only sparsely labeled the DRGs and jugular-nodose ganglia and did not label neuronal projections in the vagus nerve or dorsal root, unlike the previously used CAG construct (<a href="#SD1" rid="SD1" class=" supplementary-material">Figures S2A</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">S2B</a>).<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup> In the CNS, AAV-PHP.S-hSYN1 did not label neurons in the brain, brain stem, or spinal cord (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S3</a>).</p><p id="P10">We packaged genes encoding fluorescent proteins (tdTomato or mNeonGreen) under control of the hSYN1 promoter into AAV-PHP.S, delivered them systemically, and found that 90% (&#x000b1;2.6% SD) of ENS cells labeled with antibodies against protein gene product 9.5 (PGP9.5), a pan-neuronal protein, co-localized with virally labeled neurons in the small intestine (SI) and colon (<a href="/pmc/articles/PMC11132177/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1A</span></a>). A single systemic injection of AAV-PHP.S-hSYN1-mNeonGreen at a dose of 10<sup>12</sup> viral genomes (vg) was sufficient to label spatially diverse regions of the ENS, such as ganglia proximal and distal to the mesentery (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S4A</a>). Viral transduction was uniform throughout the SI and colon, aside from a small (~1.5-cm) section of the medial colon that, for unknown reasons, was consistently not well transduced and was therefore excluded from further analysis (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S4B</a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F1" co-legend-rid="lgnd_F1"><a href="/pmc/articles/PMC11132177/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140264990327392"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0002.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0002.jpg" src="/pmc/articles/PMC11132177/bin/nihms-1988700-f0002.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140264990327392"><a target="object" rel="noopener" href="/pmc/articles/PMC11132177/figure/F1/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F1"><div><a class="figpopup" href="/pmc/articles/PMC11132177/figure/F1/" target="figure" rid-figpopup="F1" rid-ob="ob-F1">Figure 1.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>ChAT<sup>+</sup> versus TH<sup>+</sup> neuronal distribution in the ENS</strong><p id="P82">(A) Representative images of SI and colon from mice infected with AAV-PHP.S-hSYN1-tdTomato and immunolabeled with the pan-neuronal antibody PGP9.5. Inset: quantification of the ratio of tdTomato<sup>+</sup> cells/PGP9.5<sup>+</sup> cells (n = 3 mice; each data point represents the average of 3 representative fields).</p><p id="P83">(B) Representative images of proximal and distal regions of the SI and colon from AAV-PHP.S-hSYN1-XFP-infected mice. Dotted lines demarcate the rugae (folds) in the proximal colon.</p><p id="P84">(C) Representative images of cross-sections and myenteric and submucosal plexuses in ChAT-Cre and TH-Cre mice infected with AAV-PHP.S-hSYN1-DIO-XFP.</p><p id="P85">(D and E) Density of neurons in the myenteric plexus and submucosal plexus of ChAT-Cre and TH-Cre mice normalized to the number of crypts (n = 3 mice; each data point represents the average of 3 representative fields). ChAT-Cre vs. TH-Cre mice were compared using two-way ANOVA with Sidak&#x02019;s correction for multiple comparisons for the SI and LI independently. Comparison of different regions in the SI of ChAT-Cre or TH-Cre mice was analyzed using one-way ANOVAs with Tukey&#x02019;s correction for multiple comparisons.</p><p id="P86">See also <a href="#SD1" rid="SD1" class=" supplementary-material">Figures S1</a>&#x02013;<a href="#SD1" rid="SD1" class=" supplementary-material">S5</a> and <a href="#SD6" rid="SD6" class=" supplementary-material">Video S1</a>. Source data can be found at <a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/ENS%20quantification" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/ENS%20quantification</a>.</p></div></div></div><p id="P11">To explore the general architecture of the ENS, we transduced wild-type mice with a single intravenous (i.v.) injection of a pool of AAV-PHP.S packaging multiple fluorescent proteins (AAV-PHP.S-hSYN1-XFP), which broadly labeled enteric neurons in the gut and enabled us to distinguish cells by distinct colors resulting from stochastic transduction with different combinations of XFPs (<a href="/pmc/articles/PMC11132177/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>). We quantified the number of neurons and ganglia as well as the ganglion size (i.e., the number of neurons in each ganglion) in the myenteric and submucosal plexuses of seven regions of the SI and two regions of the colon (<a href="#SD1" rid="SD1" class=" supplementary-material">Figures S5A</a>&#x02013;<a href="#SD1" rid="SD1" class=" supplementary-material">S5F</a>). Regions were approximately 1 cm in length, and the tissue was sampled every 2&#x02013;3 cm. We saw that, in the SI, the numbers of neurons and ganglia generally increased toward the distal portion of the myenteric plexus, while the converse was true for the submucosal plexus (i.e., lower numbers in distal than proximal regions) (<a href="#SD1" rid="SD1" class=" supplementary-material">Figures S5A</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">S5B</a>). Additionally, the size of the ganglia (i.e., the number of neurons per ganglion) increased in the distal region of the SI myenteric plexus, a feature not observed in the submucosal plexus (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S5C</a>). While neuronal numbers were similar in the proximal and distal regions of the colonic plexuses (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S5D</a>), the number of myenteric ganglia increased (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S5E</a>) while the size of each ganglion decreased in the distal colon (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S5F</a>). Interestingly, submucosal neurons in the proximal colon localized to natural folds in the tissue (<a href="/pmc/articles/PMC11132177/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1B</span></a>, dashed lines, bottom, second from right).</p><p id="P12" class="p p-last">To visualize ChAT<sup>+</sup> and TH<sup>+</sup> neurons, we employed mouse lines in which Cre recombinase (Cre) is expressed under the control of the respective gene promoter and engineered viral constructs with the transgene in a double-floxed inverted orientation (DIO) so that the transgene is flipped and expressed in a Cre-dependent manner. After transducing ChAT-Cre or TH-Cre mice with AAV-PHP.S-hSYN1-DIO-XFP, we observed that both neuronal populations occupy spatially distinct layers of the GI tract, with ChAT<sup>+</sup> neurons primarily located in the myenteric plexus and TH<sup>+</sup> neurons more abundant in the submucosal plexus (<a href="/pmc/articles/PMC11132177/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1C</span></a>). Quantifying this effect, we found more ChAT<sup>+</sup> than TH<sup>+</sup> neurons in all assayed regions of the myenteric plexus (<a href="/pmc/articles/PMC11132177/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1D</span></a>), although the density of TH<sup>+</sup> myenteric neurons increased distally (<a href="/pmc/articles/PMC11132177/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1D</span></a>; 10-fold increase from SI-1 vs. SI-7; 2-fold increase from SI-7 vs. SI-10/13/15). In the SI, by contrast, there were more TH<sup>+</sup> than ChAT<sup>+</sup> submucosal neurons (<a href="/pmc/articles/PMC11132177/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1E</span></a>). In addition to providing these insights into ENS architecture, this approach for whole-tissue imaging without the need for antibody labeling (which has limited penetration to deeper layers) should be broadly useful for profiling other neuronal and non-neuronal cell types in the gut.</p></div><div id="S4" class="sec"><h3 id="S4title">Activation of gut-associated neurons reshapes the gut microbiome</h3><p id="P13" class="p p-first">The unique spatial organization of ChAT<sup>+</sup> and TH<sup>+</sup> neurons we observed suggests potentially distinct functions, which we decided to investigate through specific activation of each neuronal population. First, we examined the specificity of AAV-PHP.S-hSYN1 by staining gut-extrinsic PNS ganglia for TH and found no transduction of TH<sup>+</sup> cells in the DRGs or jugular-nodose ganglia (<a href="#SD1" rid="SD1" class=" supplementary-material">Figures S2B</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">S2C</a>). ChAT<sup>+</sup> neurons are absent in these peripheral ganglia (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S2C</a>).<sup><a href="#R37" rid="R37" class=" bibr popnode">37</a>,<a href="#R38" rid="R38" class=" bibr popnode">38</a></sup> Prior research has shown that AAV-PHP.S-hSYN1 transduces the prevertebral sympathetic ganglia, which are known to innervate the gut,<sup><a href="#R39" rid="R39" class=" bibr popnode">39</a></sup> but these ganglia also lack ChAT<sup>+</sup> neurons.<sup><a href="#R40" rid="R40" class=" bibr popnode">40</a></sup> In fact, the vast majority of neurons in the prevertebral sympathetic ganglia are TH<sup>+</sup>.<sup><a href="#R40" rid="R40" class=" bibr popnode">40</a>,<a href="#R41" rid="R41" class=" bibr popnode">41</a></sup> Therefore, for the remainder of the manuscript, we will use the term &#x0201c;gut-associated&#x0201d; to refer to ChAT<sup>+</sup> neurons in the ENS or TH<sup>+</sup> neurons in the ENS plus innervating prevertebral sympathetic ganglia.</p><p id="P14">For cell-specific neuronal activation, we employed a Cre-dependent genetic construct encoding an activating designer receptor exclusively activated by designer drugs (DREADD), named hM3Dq, which is a modified neurotransmitter receptor designed to induce neuronal activation when exposed to compound 21 (C21), a &#x0201c;designer drug&#x0201d; specific to this receptor.<sup><a href="#R42" rid="R42" class=" bibr popnode">42</a></sup> We validated functional gene delivery and expression using intestinal explants from a ChAT-Cre mouse transduced with the activating DREADD and a construct encoding the calcium sensor GCaMP6f, observing a gradual increase in fluorescence consistent with a calcium transient following administration of C21 (<a href="#SD1" rid="SD1" class=" supplementary-material">Figure S5G</a>; <a href="#SD6" rid="SD6" class=" supplementary-material">Video S1</a>).</p><p id="P15">We reasoned that neuronal activation in the gut may impact the composition and community structure of the gut microbiome. Accordingly, we transduced ChAT-Cre or TH-Cre mice with either virus carrying the activating DREADD (AAV-PHP.S-hSYN1-DIO-hM3Dq-mRuby2) or a control virus expressing only the fluorescent reporter protein (AAV-PHP.S-hSYN1-DIO-mRuby2). We performed shotgun metagenomics on a longitudinal series of fecal samples collected prior to and following ChAT<sup>+</sup> or TH<sup>+</sup> neuron activation by C21 (on days 2, 6, and 10 of C21 administration) as well as contents of the terminal cecum collected on day 10 (<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>). In ChAT<sup>+</sup>-activated mice, Faith&#x02019;s phylogenetic diversity (i.e., alpha diversity) decreased dramatically in the day 10 fecal and cecal samples (<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a>), with many microbial taxa less abundant (<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figures 2I</span></a>&#x02013;<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>2K</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S6</a>, and <a href="#SD1" rid="SD1" class=" supplementary-material">S7</a>). In contrast, TH<sup>+</sup>-activated mice displayed phylogenetic diversity similar to controls throughout the experiment (<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a>). Using weighted UniFrac distances and principal-coordinate analysis (PCoA) to determine the composition of microbial communities (i.e., beta diversity), we observed a distinction between ChAT<sup>+</sup>-activated and control animals in both feces and cecal contents, a shift that was absent in samples from TH<sup>+</sup>-activated mice and controls (<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figures 2C</span></a>&#x02013;<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>2H</span></a>). Over the experimental time course, Verrucomicrobia became significantly enriched in ChAT<sup>+</sup>-activated mice (<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2I</span></a>). To explore differentially abundant bacterial taxa, we used linear discriminant analysis effect size (LEfSe)<sup><a href="#R43" rid="R43" class=" bibr popnode">43</a></sup> and generated cladograms depicting the phylogenetic relationships of differentially abundant taxa (<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figures 2J</span></a>&#x02013;<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>2M</span></a>). This analysis revealed that the bacterial species <em>Akkermansia muciniphila</em> drove the increase in Verrucomicrobia we observed in ChAT<sup>+</sup>-activated mice (<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figures 2N</span></a> and <a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and2O</span></span><span>2O</span></a>).</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F2" co-legend-rid="lgnd_F2"><a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140264990444272"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0003.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0003.jpg" src="/pmc/articles/PMC11132177/bin/nihms-1988700-f0003.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140264990444272"><a target="object" rel="noopener" href="/pmc/articles/PMC11132177/figure/F2/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F2"><div><a class="figpopup" href="/pmc/articles/PMC11132177/figure/F2/" target="figure" rid-figpopup="F2" rid-ob="ob-F2">Figure 2.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Gut-associated ChAT<sup>+</sup> and TH<sup>+</sup> neuronal activation alters the gut microbiome</strong><p id="P87">(A) Experimental paradigm. Cre-dependent hM3Dq was virally administered to either ChAT-Cre or TH-Cre mice. After 3&#x02013;4 weeks of expression, C21 was injected daily for 10 days to induce specific neuronal activation. Feces was sampled the day prior to the first C21 injection and on days 2, 6, and 10 of C21 administration, and tissue and cecal contents were collected 1 h after the last injection.</p><p id="P88">(B) Faith&#x02019;s phylogenetic diversity of feces and cecal contents over 10 days of neuronal activation in ChAT<sup>+</sup> and TH<sup>+</sup> mice. Feces was collected pre-experiment (1 day before the first injection) and on days 2, 6, and 10. Cecal contents were collected at the experimental endpoint on day 10. **p &#x0003c; 0.01, ***p &#x0003c; 0.001, determined by Kruskal-Wallis one-way ANOVA.</p><p id="P89">(C&#x02013;H) Weighted UniFrac principal-coordinate analysis (PCoA) of activated vs. control in ChAT<sup>+</sup> and TH<sup>+</sup> mice. Statistics were performed in QIIME2 as in Bolyen et al.<sup><a href="#R44" rid="R44" class=" bibr popnode">44</a></sup></p><p id="P90">(I) Stacked bar graph showing phylum-level changes in relative abundance on day 6 and day 10 of injection for feces and day 10 for cecal contents.</p><p id="P91">(J&#x02013;M) Linear discriminant analysis effect size (LEfSe) of the cecal microbiome. Cladograms show differential phylogenetic clusters and family-level differences in activated and control (J and K) ChAT<sup>+</sup> and (L and M) TH<sup>+</sup> mice. Cutoff: log<sub>10</sub>(LDA score) &#x0003e; 2 or &#x0003c; &#x02212;2.</p><p id="P92">(N and O) Changes in relative abundance of <em>A. muciniphila</em> in feces and cecal contents of (N) ChAT<sup>+</sup> and (O) TH<sup>+</sup> mice (n = 11&#x02013;14 mice per group per time point). Red, control; green, hM3Dq-activated. *p &#x0003c; 0.05, **p &#x0003c; 0.01, determined by multiple t tests with Holm-Sidak correction for multiple comparisons.</p><p id="P93">(P&#x02013;S) Beta diversity of bacterial gene families and pathways in the (P and R) cecum and (Q and S) post-9 feces of control and activated mice. The direction and length of arrows indicate their influence in separating control and activated groups. Colors represent gene families and pathways (annotated in <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S7</a>). See also <a href="#SD1" rid="SD1" class=" supplementary-material">Figures S6</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">S7</a>. Source can be found at <a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/metagenomics" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/metagenomics</a>.</p></div></div></div><p id="P16" class="p p-last">In addition to identifying microbial species, metagenomic analysis can reveal gene families and pathways that are differentially abundant in the microbiome. ChAT<sup>+</sup>-activated mice, but not TH<sup>+</sup>-activated mice, showed changes in beta diversity of both gene families and pathways, with shifts evident in the cecal contents and feces collected 9 days after activation (<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figures 2P</span></a>&#x02013;<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>2S</span></a>). The most distinguishing features were highly represented in the control group and downregulated in ChAT<sup>+</sup>-activated mice and were mainly associated with bacterial processes, such as nucleotide biosynthesis and metabolism and protein translation and transport (<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figures 2P</span></a>&#x02013;<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>2S</span></a>, <a href="#SD1" rid="SD1" class=" supplementary-material">S7C</a>, and <a href="#SD1" rid="SD1" class=" supplementary-material">S7D</a>). This downregulation is consistent with the decrease in bacterial alpha diversity we observed in ChAT<sup>+</sup>-activated mice (<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2B</span></a>). We conclude that neuronal activation actively reshapes the gut microbiome at community, species, and genetic levels, with considerable differences between the effects of ChAT<sup>+</sup> and TH<sup>+</sup> neurons.</p></div><div id="S5" class="sec"><h3 id="S5title">Neuronal stimulation impacts the gut metabolome</h3><p id="P17" class="p p-first">Given the intimate and intertwined mouse and microbial co-metabolism, the changes in the microbial metagenome we observed in response to neuronal activation led us to predict that there would also be alterations in the profile of gut metabolites. We therefore performed untargeted metabolomics using liquid chromatography-tandem mass spectrometry (LC-MS/MS) to assay molecular changes in cecal contents and feces following neuronal activation in the gut. In both ChAT<sup>+</sup>- and TH<sup>+</sup>-activated neurons, compared with unactivated controls (no DREADD), we observed a strong separation of metabolome profiles in cecal samples taken 1 h following the last C21 injection (<a href="/pmc/articles/PMC11132177/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figures 3A</span></a> and <a href="/pmc/articles/PMC11132177/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and3B).</span></span><span>3B</span></a>). Thus, targeted activation of ChAT<sup>+</sup> and TH<sup>+</sup> gut-associated neurons appears to strongly influence the gut metabolome.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F3" co-legend-rid="lgnd_F3"><a href="/pmc/articles/PMC11132177/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140264990431904"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0004.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0004.jpg" src="/pmc/articles/PMC11132177/bin/nihms-1988700-f0004.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140264990431904"><a target="object" rel="noopener" href="/pmc/articles/PMC11132177/figure/F3/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F3"><div><a class="figpopup" href="/pmc/articles/PMC11132177/figure/F3/" target="figure" rid-figpopup="F3" rid-ob="ob-F3">Figure 3.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Gut-associated ChAT<sup>+</sup> or TH<sup>+</sup> neuronal activation alters host and microbe-derived luminal metabolites</strong><p id="P94">(A and B) Canberra PCoA of the cell-free, luminal metabolome of cecal contents from control (red) and activated (green) ChAT<sup>+</sup> and TH<sup>+</sup> mice. Statistical analyses were performed in QIIME2 as in Bolyen et al.<sup><a href="#R44" rid="R44" class=" bibr popnode">44</a></sup></p><p id="P95">(C and D) Metabolic networks constructed from identified cecal metabolites in TH<sup>+</sup> and ChAT<sup>+</sup> mice. Each node is colored by its upregulation (green) or downregulation (red) in the activated group and is labeled with an ID number corresponding to annotation, mass-to-charge ratio, retention time, fold change, and significance value in <a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>.</p><p id="P96">(E) Fold changes of specific bile acids identified as upregulated (green bars) or downregulated (red bars) in activated ChAT<sup>+</sup> mice.</p><p id="P97">(F) Annotations of bile acids highlighted in (E). Metabolite IDs are colored according to annotation as primary (blue) or secondary (orange) bile acids. Metabolite IDs are specific to each sample (n = 12&#x02013;14 for each group analyzed). *p &#x0003c; 0.05, **p &#x0003c; 0.01, ***p &#x0003c; 0.001, ****p &#x0003c; 0.0001.</p><p id="P98">See also <a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>.</p></div></div></div><p id="P18">To contextualize these data, we applied the Global Natural Products Social Molecular Networking (GNPS) tool,<sup><a href="#R45" rid="R45" class=" bibr popnode">45</a></sup> an openaccess MS repository and analysis pipeline. GNPS revealed metabolic networks of both annotated and unannotated molecules in the cecal contents of ChAT<sup>+</sup>-activated and TH<sup>+</sup>-activated mice (<a href="/pmc/articles/PMC11132177/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figures 3C</span></a> and <a href="/pmc/articles/PMC11132177/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and3D),</span></span><span>3D</span></a>), allowing us to identify metabolites with differential abundance between control and activated samples. Activation of TH<sup>+</sup> neurons strongly increased metabolites whose closest spectral matches were linoelaidic acid (ID: 626), oleanolic acid methyl ester (ID: 378), and coproporphyrin I (ID: 739). Metabolites that spectrally resembled xanthine (ID: 259), genistein (ID: 846), and <em>trans</em>-ferulic acid (ID: 707) were decreased upon activation of TH<sup>+</sup> neurons (<a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>).</p><p id="P19" class="p p-last">In both ChAT<sup>+</sup>-activated and TH<sup>+</sup>-activated mice, the molecular networks largely consisted of level 3 annotations (based on the Metabolomics Standards Initiative [MSI]<sup><a href="#R46" rid="R46" class=" bibr popnode">46</a></sup>) of compounds belonging to the bile acid molecular family and their conjugates as well as unannotated analogs (<a href="/pmc/articles/PMC11132177/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figures 3C</span></a> and <a href="/pmc/articles/PMC11132177/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and3D).</span></span><span>3D</span></a>). Primary bile acids are chemicals derived from host (mouse) cholesterol biosynthesis, which are subsequently co-metabolized by gut bacteria into secondary bile acids.<sup><a href="#R47" rid="R47" class=" bibr popnode">47</a>,<a href="#R48" rid="R48" class=" bibr popnode">48</a></sup> Interestingly, metabolites with a closest spectral match to the primary bile acid cholic acid (IDs: 108, 114, 215, 219, 221, 224, and 259) were significantly enriched in the cecum of ChAT<sup>+</sup>-activated mice (<a href="/pmc/articles/PMC11132177/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figures 3D</span></a>&#x02013;<a href="/pmc/articles/PMC11132177/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>3F</span></a>; <a href="#SD2" rid="SD2" class=" supplementary-material">Table S1</a>). Additional metabolites that spectrally resemble tauro-conjugated primary bile acids, such as taurocholic acid (IDs: 234 and 248) and taurohyocholic acid (ID: 235), trended upwards. Conversely, features matching the spectra of secondary bile acids and bile acid metabolites, such as ursodeoxycholic acid (ID: 13), deoxycholic acid (ID: 100), beta-hyodeoxycholic acid (IDs: 1 and 143) and 12-ketodeoxycholic acid (IDs: 19 and 138) were decreased in ChAT<sup>+</sup>-activated mice (<a href="/pmc/articles/PMC11132177/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>Figures 3D</span></a>&#x02013;<a href="/pmc/articles/PMC11132177/figure/F3/" target="figure" class="fig-table-link figpopup" rid-figpopup="F3" rid-ob="ob-F3" co-legend-rid="lgnd_F3"><span>3F</span></a>). These data suggest that activation of ChAT<sup>+</sup> neurons may modulate, either directly or indirectly, primary bile acid secretion and/or metabolism to secondary bile acids, which have been implicated in a number of metabolic and immunologic functions, as discussed below.</p></div><div id="S6" class="sec"><h3 id="S6title">Neuronal subpopulations differentially shape the gut luminal proteome</h3><p id="P20" class="p p-first">Proteins from the mouse, gut microbes, and diet converge and interact in the GI tract.<sup><a href="#R49" rid="R49" class=" bibr popnode">49</a></sup> We performed untargeted label-free proteomics by LC-MS/MS of cell-free supernatants of the cecal contents from ChAT<sup>+</sup>-activated and TH<sup>+</sup>-activated mice and controls collected 1 h following the final C21 treatment (<a href="/pmc/articles/PMC11132177/figure/F2/" target="figure" class="fig-table-link figpopup" rid-figpopup="F2" rid-ob="ob-F2" co-legend-rid="lgnd_F2"><span>Figure 2A</span></a>). Consistent with the increase in cecal bile acid metabolites that we observed in ChAT<sup>+</sup>-activated mice, we report an increased abundance of Niemann-Pick C1-like 1 protein (NPC1L1) in the cecum of these mice (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4A</span></a>). NPC1L1 is expressed on the apical surface of enterocytes and is integral to the absorption of free cholesterol, the precursor of bile acids, from the lumen.<sup><a href="#R50" rid="R50" class=" bibr popnode">50</a></sup> Goblet cell-related proteins, specifically Mucin-19 (MUC19) and Zymogen granule 16 (ZG16), a protein localized to secretory granules,<sup><a href="#R51" rid="R51" class=" bibr popnode">51</a></sup> also trended upward following ChAT<sup>+</sup> neuronal activation (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4A</span></a>). Conversely, one of the most highly downregulated proteins was an aldehyde dehydrogenase (<a id="__tag_1149457094" class="tag_hotlink" href="/protein/Q3U367" ref="/protein/Q3U367"><span class="tag-json" style="display:none">{"type":"entrez-protein","attrs":{"text":"Q3U367","term_id":"123796953","term_text":"Q3U367"}}</span>Q3U367</a>) encoded by the <em>Aldh9a1</em> gene, which is involved in the catalytic conversion of putrescine to &#x003b3;-aminobutyric acid (GABA).<sup><a href="#R52" rid="R52" class=" bibr popnode">52</a></sup> While GABA is the primary inhibitory neurotransmitter in the CNS, little is known about its role in the ENS. The most significantly upregulated proteins in cecal contents of ChAT<sup>+</sup>-activated mice were pancreatic digestive enzymes, including chymopasin (CTRL), chymotrypsinogen B1 (CTRB1), and pancreatic lipase-related protein 2 (PNLIPRP2) (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4A</span></a>). Accordingly, network analysis of upregulated proteins revealed that KEGG pathways associated with digestion represent the majority of the network (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4B</span></a>). This is consistent with evidence that cholinergic, viscerofugal neurons send signals from the GI tract to other organs of the digestive system, including the pancreas.<sup><a href="#R4" rid="R4" class=" bibr popnode">4</a></sup> Cholinergic innervation of the pancreas plays a significant role in regulating pancreatic functions, such as the secretion of digestive enzymes and insulin release.<sup><a href="#R53" rid="R53" class=" bibr popnode">53</a></sup></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F4" co-legend-rid="lgnd_F4"><a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140264990416352"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0005.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0005.jpg" src="/pmc/articles/PMC11132177/bin/nihms-1988700-f0005.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140264990416352"><a target="object" rel="noopener" href="/pmc/articles/PMC11132177/figure/F4/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F4"><div><a class="figpopup" href="/pmc/articles/PMC11132177/figure/F4/" target="figure" rid-figpopup="F4" rid-ob="ob-F4">Figure 4.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>Gut-associated ChAT<sup>+</sup> or TH<sup>+</sup> neuronal activation alters host and microbe-derived luminal proteins</strong><p id="P99">(A) Volcano plot of differentially expressed host proteins identified in the cecal contents of ChAT<sup>+</sup>-activated (n = 8) vs. ChAT<sup>+</sup> control (n = 9) mice 1 h after final C21 administration.</p><p id="P100">(B) STRING network analysis of host proteins that were more abundant in ChAT<sup>+</sup>-activated mice (p<sub>nom.</sub> &#x0003c; 0.2).</p><p id="P101">(C) Proteomics volcano plot of TH<sup>+</sup>-activated vs. TH<sup>+</sup> control mice (n = 7 mice per group).</p><p id="P102">(D) STRING network analysis of upregulated host proteins in TH<sup>+</sup>-activated mice (p<sub>nom.</sub> &#x0003c; 0.2).</p><p id="P103">(E and F) Unipept metaproteomics analysis of upregulated microbial proteins (fold change &#x0003e; 2, p<sub>nom.</sub> &#x0003c; 0.2) in TH<sup>+</sup>-activated and ChAT<sup>+</sup>-activated mice (n = 7&#x02013;9 mice per group).</p><p id="P104">Source data for (A) can be found at <a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/blob/main/proteomics/CHAT_proteomics_volcano.txt" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/blob/main/proteomics/CHAT_proteomics_volcano.txt</a>. Source data for (C) can be found at <a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/blob/main/proteomics/TH_proteomics_volcano.txt" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/blob/main/proteomics/TH_proteomics_volcano.txt</a>. Source data for (E) and (F) can be found at <a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/blob/main/proteomics/metaproteomics/Microbiome_associated_proteins.xlsx" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/blob/main/proteomics/metaproteomics/Microbiome_associated_proteins.xlsx</a>.</p></div></div></div><p id="P21">Peripheral activation of TH<sup>+</sup> gut-associated neurons also altered the luminal proteome of the cecum. Notably, 88% (52 of 59) of the differentially abundant proteins (p<sub>adj.</sub> &#x0003c; 0.25) were distinct from those identified in ChAT<sup>+</sup>-activated mice. The overall direction of the effect was also reversed: ~90% of differentially abundant cecal proteins in TH<sup>+</sup>-activated mice were upregulated (53 of 59) compared with ~18% in ChAT<sup>+</sup>-activated mice (20 of 112), suggesting that activation of distinct neuronal subsets is associated with opposing changes in GI function. We observed signatures of increased protein-protein interactions in cecal contents of TH<sup>+</sup>-activated mice, evidenced by more network nodes and connections (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4D</span></a>). Filamin B (FLNB) and spectrin &#x003b2; chain, non-erythrocytic 1 (SPTBN1) were two of the most significantly enriched proteins following TH<sup>+</sup> neuron activation (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4C</span></a>). Both are associated with the intestinal brush border and membrane vesicles.<sup><a href="#R54" rid="R54" class=" bibr popnode">54</a>,<a href="#R55" rid="R55" class=" bibr popnode">55</a></sup> Accordingly, coatomer proteins also trended upward (COPA and COPB2) (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4C</span></a>), and vesicle-mediated transport was one of the major protein networks altered (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4D</span></a>). Other upregulated protein interaction networks were associated with metabolic pathways, ribosomal activity, and the immune system (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4D</span></a>). For example, the immune-related proteins immunoglobulin heavy constant alpha (IGHA) (in ChAT<sup>+</sup>-activated), immunoglobulin heavy constant gamma 2C (IGHG2C), and complement component 3 (in TH<sup>+</sup>-activated) trended upward (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figures 4A</span></a> and <a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and4C</span></span><span>4C</span></a>).</p><p id="P22" class="p p-last">Perhaps the most intriguing observation was the strong depletion of acidic mammalian chitinase (CHIA) upon activation of TH<sup>+</sup> neurons (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4C</span></a>). Chitin is a natural polysaccharide that is a major component of fungal cell walls,<sup><a href="#R56" rid="R56" class=" bibr popnode">56</a></sup> but intestinal chitinases are poorly studied in mice. This result prompted us to query the pan-proteomics dataset against a microbial protein database, which revealed that the decrease in CHIA abundance following TH<sup>+</sup> neuron activation was accompanied by a large bloom in fungus-associated peptides in the microbiome (~59% of peptides mapped to any microbe) (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4E</span></a>). In contrast, fungal peptides represented only ~0.4% of enriched peptides in the lumen of ChAT<sup>+</sup>-activated mice (<a href="/pmc/articles/PMC11132177/figure/F4/" target="figure" class="fig-table-link figpopup" rid-figpopup="F4" rid-ob="ob-F4" co-legend-rid="lgnd_F4"><span>Figure 4F</span></a>). Unfortunately, we were unable to corroborate these proteomics data with metagenomics since the DNA extraction method we used was not optimized for fungi. However, these findings suggest that the reduced chitinase production of activated TH<sup>+</sup> cells is directly associated with a dramatic increase in fungal proteins, which, if experimentally validated in the future, would represent a circuit by which the gut-associated neurons of mice regulate fungal load in the gut.</p></div><div id="S7" class="sec"><h3 id="S7title">Activation of ChAT<sup>+</sup> and TH<sup>+</sup> neurons alters the intestinal transcriptome</h3><p id="P23" class="p p-first">Given the changes to the gut microbiome, proteome, and metabolome that we observed, we were interested in the tissuelevel impact of neuronal activation on the intestinal transcriptome. We therefore profiled gene expression with QuantSeq, a quantitative 3&#x02032; mRNA sequencing technology, in 1 cm of tissue from the distal SI and proximal colon harvested 1 h after the last C21 injection. Rapid and transient expression of immediate-early genes (IEGs) is widely used as a measure of increased neuronal activity,<sup><a href="#R57" rid="R57" class=" bibr popnode">57</a></sup> and the IEGs <em>Fos</em>, <em>Egr1</em>, <em>Jun</em>, and <em>Klf2</em> were among the most significantly upregulated transcripts we identified in the SI and colon of both ChAT<sup>+</sup>- and TH<sup>+</sup>-activated mice (<a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figures 5A</span></a>&#x02013;<a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>5D</span></a>). These IEGs are also known to be upregulated during growth and differentiation of highly active cell types, such as immune cells,<sup><a href="#R58" rid="R58" class=" bibr popnode">58</a>,<a href="#R59" rid="R59" class=" bibr popnode">59</a></sup> smooth muscle cells,<sup><a href="#R60" rid="R60" class=" bibr popnode">60</a></sup> and intestinal epithelial cells.<sup><a href="#R61" rid="R61" class=" bibr popnode">61</a></sup></p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F5" co-legend-rid="lgnd_F5"><a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140264984161728"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0006.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0006.jpg" src="/pmc/articles/PMC11132177/bin/nihms-1988700-f0006.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140264984161728"><a target="object" rel="noopener" href="/pmc/articles/PMC11132177/figure/F5/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F5"><div><a class="figpopup" href="/pmc/articles/PMC11132177/figure/F5/" target="figure" rid-figpopup="F5" rid-ob="ob-F5">Figure 5.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>ChAT <sup>+</sup> and TH<sup>+</sup> activation-mediated transcriptomic changes</strong><p id="P105">(A&#x02013;D) DEGs in DREADD-activated vs. control (A) ChAT <sup>+</sup> distal SI, (B) ChAT<sup>+</sup> proximal colon, (C) TH<sup>+</sup> distal SI, and (D) TH<sup>+</sup> proximal colon. Dashed vertical lines, fold change (FC) &#x000b1; 1.5; dashed horizontal lines, p<sub>adj.</sub> &#x0003c; 0.05. Transcripts of IEGs are highlighted in green and annotated. (n = 10 mice per group).</p><p id="P106">(E) FC of upregulated IEGs (p<sub>adj.</sub> &#x0003c; 0.05) as defined previously.<sup><a href="#R57" rid="R57" class=" bibr popnode">57</a></sup></p><p id="P107">(F&#x02013;I) GSEA of Gene Ontology (GO) terms for (F) ChAT<sup>+</sup> distal SI, (G) ChAT<sup>+</sup> proximal colon, (H) TH<sup>+</sup> distal SI, and (I) TH<sup>+</sup> proximal colon. See also <a href="#SD3" rid="SD3" class=" supplementary-material">Tables S2</a> and <a href="#SD4" rid="SD4" class=" supplementary-material">S3</a>. Source Can be found at <a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/RNAseq" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/RNAseq</a>.</p></div></div></div><p id="P24">In the distal SI, we found similar numbers of differentially expressed genes (DEGs; p<sub>adj.</sub> &#x0003c; 0.05) in ChAT<sup>+</sup>-activated mice (162 DEGs) and TH<sup>+</sup>-activated mice (165 DEGs) (<a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figures 5A</span></a> and <a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and5C).</span></span><span>5C</span></a>). The direction of regulation differed, however, with ~73% of DEGs upregulated upon ChAT<sup>+</sup> activation (118 upregulated and 44 downregulated) and ~58% of DEGs downregulated upon TH<sup>+</sup> activation (69 upregulated and 96 downregulated). IEGs followed this overall pattern, with 29 upregulated in the distal SI of ChAT<sup>+</sup>-activated mice but only two upregulated in TH<sup>+</sup>-activated mice (<a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5E</span></a>) and three (i.e., <em>Hbegf</em>, <em>Soca3</em>, and <em>Mcl1</em>) downregulated (<a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5C</span></a>). Similar proportions of DEGs were upregulated in the proximal colon of both ChAT<sup>+</sup>-activated (169 upregulated and 84 downregulated) and TH<sup>+</sup>-activated mice (130 upregulated and 62 downregulated) (<a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figures 5B</span></a> and <a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and5D).</span></span><span>5D</span></a>). Given the enrichment in fungal proteins and reduction in the level of the CHIA protein in the TH<sup>+</sup>-activated mice, we explored potential immune responses to fungi but found no obvious inflammatory signals compared with control mice (<a href="#SD4" rid="SD4" class=" supplementary-material">Table S3</a>).</p><p id="P25" class="p p-last">To gain insight into the cellular functions of DEGs, we used gene set enrichment analysis (GSEA) (<a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figures 5F</span></a>&#x02013;<a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>5I</span></a>; <a href="#SD3" rid="SD3" class=" supplementary-material">Table S2</a>). Notably, the most highly enriched gene ontology (GO) term for the distal SI of ChAT<sup>+</sup>-activated mice was &#x0201c;regulation of smooth muscle cell proliferation&#x0201d; (<a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5F</span></a>), whereas in TH<sup>+</sup>-activated mice, it was &#x0201c;response to bacteria&#x0201d; (<a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5H</span></a>), consistent with the increase in immune-related responses suggested by our proteomics dataset. In the proximal colon, we observed similar GO pathways in ChAT<sup>+</sup>-activated and TH<sup>+</sup>-activated mice (<a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figures 5G</span></a> and <a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and5I),</span></span><span>5I</span></a>), suggesting that transcriptomic signatures may depend on the context of the activated neurons. In the SI, ChAT<sup>+</sup> neurons predominantly border muscle cells in the myenteric plexus, while TH<sup>+</sup> neurons neighbor epithelial and immune cells, which respond to bacteria in the submucosal plexus of the distal SI (<a href="/pmc/articles/PMC11132177/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figures 1C</span></a>&#x02013;<a href="/pmc/articles/PMC11132177/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>1E</span></a>). In the colon, both neuronal subsets are abundant in the myenteric plexus (<a href="/pmc/articles/PMC11132177/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1D</span></a>). In both myenteric and submucosal plexuses, we saw a wider breadth of pathways upregulated by activation of ChAT<sup>+</sup> neurons than TH<sup>+</sup> neurons, with ChAT<sup>+</sup> neuronal activation impacting diverse cellular functions in the GI tract, involving endothelial, epithelial, immune, and adipose cells (<a href="/pmc/articles/PMC11132177/figure/F5/" target="figure" class="fig-table-link figpopup" rid-figpopup="F5" rid-ob="ob-F5" co-legend-rid="lgnd_F5"><span>Figure 5G</span></a>; <a href="#SD3" rid="SD3" class=" supplementary-material">Table S2</a>).</p></div><div id="S8" class="sec sec-last"><h3 id="S8title">Differential functional GI outcomes of activation of ChAT<sup>+</sup> and TH<sup>+</sup> neurons</h3><p id="P26" class="p p-first-last">Motivated by the complexity of responses we observed following activation of neuronal populations in the gut, we decided to assay functional GI outcomes. Both ChAT<sup>+</sup> and TH<sup>+</sup> neuronal populations are known to be important for motility and secretory function,<sup><a href="#R17" rid="R17" class=" bibr popnode">17</a>,<a href="#R27" rid="R27" class=" bibr popnode">27</a></sup> but they have never been specifically modulated to study GI physiology in a freely behaving mammal. Activation of either ChAT<sup>+</sup> or TH<sup>+</sup> gut-associated neurons resulted in faster whole-gut transit time, increased fecal pellet output, and mass of cecal contents compared with control mice (<a href="/pmc/articles/PMC11132177/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figures 6A</span></a>&#x02013;<a href="/pmc/articles/PMC11132177/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>6C</span></a>). Fecal pellets from ChAT<sup>+</sup>-activated but not TH<sup>+</sup>-activated mice had increased water content, which is consistent with reports in the literature of involvement of ChAT<sup>+</sup> enteric neurons in fluid secretion (<a href="/pmc/articles/PMC11132177/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figures 6D</span></a>&#x02013;<a href="/pmc/articles/PMC11132177/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>6F</span></a>).<sup><a href="#R15" rid="R15" class=" bibr popnode">15</a>,<a href="#R21" rid="R21" class=" bibr popnode">21</a>,<a href="#R22" rid="R22" class=" bibr popnode">22</a></sup> This distinction is particularly notable given the higher concentration of TH<sup>+</sup> neurons than ChAT<sup>+</sup> neurons in most regions of the submucosal plexus (<a href="/pmc/articles/PMC11132177/figure/F1/" target="figure" class="fig-table-link figpopup" rid-figpopup="F1" rid-ob="ob-F1" co-legend-rid="lgnd_F1"><span>Figure 1</span></a>), which is largely responsible for fluid secretion and absorption.<sup><a href="#R1" rid="R1" class=" bibr popnode">1</a></sup> Daily administration of C21 for 9 days to control mice (no DREADD) did not cause any obvious health impairment, and the mice maintained body weight throughout the experimental period (<a href="#SD1" rid="SD1" class=" supplementary-material">Figures S6C</a> and <a href="#SD1" rid="SD1" class=" supplementary-material">S6D</a>). TH<sup>+</sup>-activated mice also maintained body weight, but ChAT<sup>+</sup>-activated animals experienced slight weight loss that likely reflects the diarrhea-like phenotype over 9 consecutive days. To further examine gut motility in the absence of extrinsic innervation, we analyzed propulsive colonic migrating motor complexes (CMMCs) in an <em>ex vivo</em> system. Activation of ChAT<sup>+</sup> neurons by C21 administration resulted in more frequent migration of motor complexes (<a href="/pmc/articles/PMC11132177/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figures 6G</span></a>, <a href="/pmc/articles/PMC11132177/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -0.5em;">,6I,</span></span><span>6I</span></a>, and <a href="/pmc/articles/PMC11132177/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span style="position: relative;text-decoration:none;">&#x200B;<span class="figpopup-sensitive-area" style="left: -1.5em;">and6J;</span></span><span>6J</span></a>; <a href="#SD5" rid="SD5" class=" supplementary-material">Table S4</a>), whereas activation of TH<sup>+</sup> neurons had no effect on CMMCs (<a href="/pmc/articles/PMC11132177/figure/F6/" target="figure" class="fig-table-link figpopup" rid-figpopup="F6" rid-ob="ob-F6" co-legend-rid="lgnd_F6"><span>Figure 6H</span></a>; <a href="#SD5" rid="SD5" class=" supplementary-material">Table S4</a>). The discrepancy between the <em>in vivo</em> and <em>ex vivo</em> results we observed may be due to activation of TH<sup>+</sup> neurons in the sympathetic prevertebral ganglia that project to the gut.<sup><a href="#R30" rid="R30" class=" bibr popnode">30</a></sup> Overall, these data reveal that ChAT<sup>+</sup> but not TH<sup>+</sup> neurons in the gut mediate intestinal fluid balance and <em>ex vivo</em> colonic motility.</p><!--fig ft0--><!--fig mode=article f1--><div class="fig iconblock whole_rhythm" id="F6" co-legend-rid="lgnd_F6"><a href="/pmc/articles/PMC11132177/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6"><!--fig/graphic|fig/alternatives/graphic mode="anchored" m1--><div class="figure" data-largeobj="" data-largeobj-link-rid="largeobj_idm140264984142976"><img loading="lazy" class="fig-image" alt="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0007.jpg" title="An external file that holds a picture, illustration, etc.&#10;Object name is nihms-1988700-f0007.jpg" src="/pmc/articles/PMC11132177/bin/nihms-1988700-f0007.jpg" /></div><div class="largeobj-link align_right" id="largeobj_idm140264984142976"><a target="object" rel="noopener" href="/pmc/articles/PMC11132177/figure/F6/?report=objectonly">Open in a separate window</a></div></a><div class="icnblk_cntnt" id="lgnd_F6"><div><a class="figpopup" href="/pmc/articles/PMC11132177/figure/F6/" target="figure" rid-figpopup="F6" rid-ob="ob-F6">Figure 6.</a></div><!--caption a7--><div class="caption"><!--caption a8--><strong>GI physiology differences in ChAT<sup>+</sup> vs. TH<sup>+</sup> mice following activation</strong><p id="P108">(A) Activation-mediated changes in whole-gut transit time in ChAT<sup>+</sup> and TH<sup>+</sup> mice.</p><p id="P109">(B) Activation-mediated changes in fecal pellet output in ChAT<sup>+</sup> and TH<sup>+</sup> mice.</p><p id="P110">(C) Activation-mediated changes in normalized cecal content mass in ChAT<sup>+</sup> and TH<sup>+</sup> mice.</p><p id="P111">(D and E) Fecal pellet water content in (D) ChAT<sup>+</sup> and (E) TH<sup>+</sup> mice over 2 h following C21 activation, with a least-squares nonlinear regression displaying a 95% confidence interval.</p><p id="P112">(F) Average fecal pellet water content in ChAT<sup>+</sup> and TH<sup>+</sup> mice following activation.</p><p id="P113">(A&#x02013;F) (n = 10&#x02013;11 mice per group). *p &#x0003c; 0.05, ***p &#x0003c; 0.001, ****p &#x0003c; 0.0001, determined by 2-way ANOVA with Sidak&#x02032;s method for multiple comparisons.</p><p id="P1110">(G and H) Frequency of <em>ex vivo</em> CMMCs from (G) ChAT<sup>+</sup> and (H) TH<sup>+</sup> mice over 30 min following activation (n = 3&#x02013;6 mice per group). **p &#x0003c; 0.01, ***p &#x0003c; 0.001, determined by 2-way ANOVA with Sidak&#x02032;s method for multiple comparisons.</p><p id="P1115">(I and J) Heatmaps showing frequency of CMMCs over 400 s following activation in <em>ex vivo</em> preparations from (I) ChAT<sup>+</sup> control and (J) ChAT<sup>+</sup> DREADD-administered mice.</p><p id="P1112">See also <a href="#SD1" rid="SD1" class=" supplementary-material">Figure S6</a> and <a href="#SD4" rid="SD4" class=" supplementary-material">Table S4</a>.</p></div></div></div></div></div><div id="S9" class="tsec sec"><h2 class="head no_bottom_margin" id="S9title">DISCUSSION</h2><p id="P27" class="p p-first">While early pioneers of neuroscience in the 20<sup>th</sup> century focused on the ENS as a model, more recent research has centered on the brain, and our understanding of the CNS has outpaced that of other neuronal systems in the body. As a result, basic knowledge of many aspects of neuronal architecture and function within the gut remains rudimentary.<sup><a href="#R15" rid="R15" class=" bibr popnode">15</a>,<a href="#R21" rid="R21" class=" bibr popnode">21</a>,<a href="#R22" rid="R22" class=" bibr popnode">22</a></sup> Here, using a viral delivery system with enhanced tropism for the ENS, we mapped the distribution of ChAT<sup>+</sup> and TH<sup>+</sup> neurons across the mouse GI tract and assayed the complex effects of their peripheral activation on physiology and function. Although the DREADD-based activation paradigm we use in this study is inherently artificial, the results reveal strikingly different roles for neuronal populations, with nearly every feature characterized (spatial distribution, metagenomic, metabolic, transcriptional, and proteomic profiles, and even physiological output) unique to cell type.</p><p id="P28">The viral vector we used, AAV-PHP.S, can transduce other neuronal subtypes in the PNS, such as those in the DRGs, and, with a strong ubiquitous promoter, induce transgene expression.<sup><a href="#R28" rid="R28" class=" bibr popnode">28</a></sup> To limit this off-target effect, we utilized a weaker promoter with increased ENS specificity and focused our analyses on GI tissue and lumen. By thus excluding most known extrinsic innervation pathways, we uncover cell-type-specific effects of gut-associated neuronal activation that are independent of signaling from the brain.</p><p id="P29">Exposure to the external environment charges the intestines with myriad responsibilities, including absorption and digestion of dietary nutrients, exclusion of xenobiotics, protection from enteric infection, and partnership with the gut microbiome. Deletion of ChAT in enteric neurons leads to microbiome dysbiosis,<sup><a href="#R62" rid="R62" class=" bibr popnode">62</a></sup> and we observed differences in the compositional profile (both metagenomic and proteomic) of the gut microbiome specific to activation of ChAT<sup>+</sup> neurons. Notably, we found an expansion of Verrucomicrobia driven by <em>A. muciniphila</em>, which has been implicated in human diseases such as obesity,<sup><a href="#R63" rid="R63" class=" bibr popnode">63</a>,<a href="#R64" rid="R64" class=" bibr popnode">64</a></sup> multiple sclerosis,<sup><a href="#R65" rid="R65" class=" bibr popnode">65</a>,<a href="#R66" rid="R66" class=" bibr popnode">66</a></sup> and seizures.<sup><a href="#R67" rid="R67" class=" bibr popnode">67</a></sup>
<em>A. muciniphila</em> metabolizes host-derived mucus as a nutrient source,<sup><a href="#R68" rid="R68" class=" bibr popnode">68</a></sup> consistent with the increase in luminal mucin proteins and digestive enzymes we observed in ChAT<sup>+</sup>-activated mice. A particularly interesting host-microbial interaction emerged from activation of TH<sup>+</sup> cells, which led to a dramatic decrease in CHIA protein expression and a concomitant bloom in fungi, suggesting that neuronal circuits can regulate fungal populations in the gut. If validated, this intriguing host-microbial interaction could have implications for health. Our study does not, however, reveal the mechanism(s) by which ENS activation reshapes the gut microbial community structure, which may involve altered colonic motility, changes in mucus production, modulation of mucosal immune responses, and/or shifts in metabolism and nutrient availability.</p><p id="P30">The human gut microbiome possesses as much metabolic capacity as the liver; it is therefore no surprise that changes to both mouse gut physiology and the microbiome have major influences on the gut metabolome. In a striking example of mutualism, we report widespread changes to the pool of intestinal bile acids, molecules produced via host-microbial co-metabolism. Activation of ChAT<sup>+</sup> neurons, but not TH<sup>+</sup> neurons, impacted expression of NPC1L1, which is involved in cholesterol transport. In mammals, cholesterol is the substrate for production of primary bile acids, which are then metabolized exclusively by the gut microbiome into secondary bile acids. Bile acids play critical roles in fat absorption,<sup><a href="#R69" rid="R69" class=" bibr popnode">69</a></sup> gut motility,<sup><a href="#R70" rid="R70" class=" bibr popnode">70</a></sup> hormonal signaling<sup><a href="#R71" rid="R71" class=" bibr popnode">71</a></sup> and immune functions,<sup><a href="#R72" rid="R72" class=" bibr popnode">72</a></sup> and neurological conditions.<sup><a href="#R73" rid="R73" class=" bibr popnode">73</a></sup> Expression of bile salt hydrolases and hydratases increases the fitness of both commensal and pathogenic bacteria.<sup><a href="#R74" rid="R74" class=" bibr popnode">74</a>&#x02013;<a href="#R78" rid="R78" class=" bibr popnode">78</a></sup> While additional work is required to determine how the ENS affects the levels and constitution of the bile acid pool, understanding the processes that regulate synthesis of secondary bile acids may have implications for organ systems throughout the body.</p><p id="P31">Our study complements recent single-cell RNA sequencing (RNA-seq) studies of the ENS neuronal transcriptome<sup><a href="#R79" rid="R79" class=" bibr popnode">79</a></sup> by giving us the ability to selectively activate specific enteric neurons and explore the dynamic interplay between cells of various lineages in the gut. Importantly, the transcriptomic changes we observed may be a consequence of direct or indirect effects of neuronal activation. Indeed, induced activation of ChAT<sup>+</sup> or TH<sup>+</sup> neurons rapidly changed GI transit and fluid secretion patterns, which are only a fraction of the processes that may feed back on epithelial or immune cells, altering their gene expression profiles. Further single-cell analysis may help dissect the roles of the various intestinal cells that collaborate to coordinate gut functions.</p><p id="P32" class="p">The ENS adapts and responds to incredibly diverse molecular cues from the environment and must do so throughout the entire length and surface area of the intestines&#x02014;the largest and most extensive internal organ with a rich network of neurons termed the &#x0201c;second brain&#x0201d;.<sup><a href="#R80" rid="R80" class=" bibr popnode">80</a></sup> Exposure to molecules from the diet or the microbiome may modulate ENS function, along with signals from outside the gut, such as the circulatory system. Curiously, many disorders of the brain are also associated with GI symptoms.<sup><a href="#R81" rid="R81" class=" bibr popnode">81</a>&#x02013;<a href="#R85" rid="R85" class=" bibr popnode">85</a></sup> While mechanisms linking the gut and the brain and their consequences for health are an active area of study, the impact of neuronal activation within the ENS has largely been unexplored. Here, we establish an experimental system that allows controlled activation of intrinsic and extrinsic neurons of the gut, separated from inputs from the brain, and demonstrate broad changes in the gut environment and its physiology that differ by activated neuronal population. The extensive datasets on activation of two major gut-associated neuronal populations that we generated should serve as a resource for further studies on the interconnected biological systems governing the complex relationship between gut physiology and the microbiome. Future deployment of this approach could enable mapping of neuronal connections into and out of the gut, providing insights into how the ENS networks with tissues throughout the body and advancing growing research into the many functions of the GI tract, an endeavor with important consequences for human health.</p><div id="S10" class="sec sec-last"><h3 id="S10title">Limitations of the study</h3><p id="P33" class="p p-first-last">It has been shown recently that a similar strategy using AAV-PHP.S-hSYN1 in ChAT-Cre mice also labels ChAT<sup>+</sup> neurons in cardiac ganglia and that activation of cholinergic neurons reduces heart rate and blood pressure.<sup><a href="#R86" rid="R86" class=" bibr popnode">86</a>,<a href="#R87" rid="R87" class=" bibr popnode">87</a></sup> Cardiac afferent neurons signal through the vagus nerve and jugular-nodose ganglia to the brain and in sensory pathways through the DRGs and spinal cord.<sup><a href="#R88" rid="R88" class=" bibr popnode">88</a></sup> Though there is no known direct route for signaling from cardiac ganglia to gut-associated neurons, it is possible that the gut may be impacted by an indirect route involving the CNS. In the future, further refinement of AAVs through directed evolution may generate serotypes with exclusive tropism for the ENS and allow full separation of the functions of intrinsic and extrinsic activation of gut-associated neuronal subsets. Since the hSYN1 promoter we used has been shown to drive expression only in neurons,<sup><a href="#R89" rid="R89" class=" bibr popnode">89</a>,<a href="#R90" rid="R90" class=" bibr popnode">90</a></sup> we did not characterize other cell types, such as enteroendocrine cells (EECs), which have been shown recently to form synapses with enteric neurons and contribute to sensory transmission from the gut to the brain through the vagus nerve. Since these cells turn over every few days<sup><a href="#R91" rid="R91" class=" bibr popnode">91</a></sup> and have not been reported to express TH or ChAT, we think it unlikely that they are a major contributor to the activation-induced phenotypes we observed, but we cannot completely rule out their involvement.</p></div></div><div id="S11" class="tsec sec"><h2 class="head no_bottom_margin" id="S11title">STAR&#x02605;METHODS</h2><div id="S12" class="sec sec-first"><h3 id="S12title">RESOURCE AVAILABILITY</h3><p id="P34" class="p p-first">Lead contact Further information and requests for resources and reagents should be directed to and will be fulfilled by the lead contact, Sarkis K. Mazmanian (<a href="mailto:dev@null" data-email="ude.hcetlac@sikras" class="oemail">ude.hcetlac@sikras</a>).</p><div id="S13" class="sec"><p></p><h4 id="S13title" class="inline">Materials availability </h4><p id="P35" class="p p-first-last">No new reagents were generated in this study.</p></div><div id="S14" class="sec sec-last"><p></p><h4 id="S14title" class="inline">Data and code availability </h4><ul class="unordered" style="list-style-type:disc" id="L2"><li><div id="P36">Microbial sequencing data have been deposited at the European Bioinformatics Institute (ERP131523), Metabolomic data at UCSD MassIVE repository (MSV000084550), proteomic data at UCSD MassIVE repository (MSV000087917), and QuantSeq data at NCBI GEO repository (<a id="__tag_1149457095" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE180961" ref="/geo/query/acc.cgi?acc=GSE180961"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE180961","term_id":"180961"}}</span>GSE180961</a>) and are publicly available as of the date of publication. All other experimental data used to generate the figures reported in this paper can be found at (DOI:<a href="//doi.org/10.5281/zenodo.10525220" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">10.5281/zenodo.10525220</a>, <a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/</a>) and are publicly available as of the date of publication.</div></li><li><div id="P37">This paper does not report original code.</div></li><li><div id="P38">Any additional information required to reanalyze the data reported in this paper is available from the lead contact upon request.</div></li></ul></div></div><div id="S15" class="sec"><h3 id="S15title">EXPERIMENTAL MODEL AND STUDY PARTICIPANT DETAILS</h3><div id="S16" class="sec sec-first"><p></p><h4 id="S16title" class="inline">Mice </h4><p id="P39" class="p p-first-last">All mouse experiments were performed in accordance with the NIH Guide for the Care and Use of Laboratory Animals using protocols approved by the Institutional Animal Care and Use Committee at the California Institute of Technology. Mice were fed <em>ad libitum</em> for the entire duration of experiments. Homozygous TH-Cre (gift to V.G. from Ted Ebendal, B6.129X1-Thtm1(cre)Te/Kieg<sup><a href="#R92" rid="R92" class=" bibr popnode">92</a></sup>) and ChAT-Cre (Jackson Laboratories, Bar Harbor, ME- Stock# 028861, RRID:IMSR_JAX:028861) mice were bred to wild-type mice to yield the male and female heterozygous Cre-mice used for our studies. Wild-type specific pathogen free (SPF) C57BL/6 (Jackson Laboratories, Bar Harbor, ME- Stock #000664, RRID:IMSR_JAX:000664) males and females were used for breeding and experiments. Mice at 6&#x02013;8 weeks of age were used for experiments.</p></div><div id="S17" class="sec sec-last"><p></p><h4 id="S17title" class="inline">Virus production </h4><p id="P40" class="p p-first-last">Virus was produced using the methods described in Challis et al., 2019<sup><a href="#R39" rid="R39" class=" bibr popnode">39</a></sup> and <a href="//doi.org/10.17504/protocols.io.bzn6p5he" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">dx.doi.org/10.17504/protocols.io.bzn6p5he</a>. Briefly, human embryonic kidney (HEK293T) cells were triple-transfected with pUCmini-iCAP-AAV-PHP.S, pHelper plasmid, and one of the following pAAV genomes: hSYN1-tdTomato, hSYN1-mRuby2, hSYN1-DIO-mRuby2, hSYN1-mNeonGreen, CAG-mNeonGreen, hSYN1-DIO-mNeonGreen, hSYN1-mTurquoise2, hSYN1-DIO-mTurquoise2, hSYN1-DIO-hM3Dq-mRuby2, CAG-GCaMP6f. Cells were grown in DMEM + Glutamax + Pyruvate (Gibco, Gaithersburg, MD- Stock# 10569&#x02013;010) + 5% FBS + non-essential amino acids (Gibco, Gaithersburg, MD- Stock# 11140&#x02013;050) + penicillin-streptomycin (Gibco, Gaithersburg, MD- Stock# 15070&#x02013;063). Virus was precipitated from cells and supernatant with an 8% PEG solution (w/v), and purified by ultracentrifugation using 15%, 25%, 40%, 60% stacked iodixanol gradients.</p></div></div><div id="S18" class="sec"><h3 id="S18title">METHOD DETAILS</h3><div id="S19" class="sec sec-first"><p></p><h4 id="S19title" class="inline">Systemic delivery of AAV </h4><p id="P41" class="p p-first-last">Mice were anesthetized using 2% isoflurane. Virus was titered to 1&#x000d7;10<sup>12</sup> vg, resuspended in a volume of 100&#x003bc;Lwith sterile PBS, and injected retro-orbitally as described in <a href="//doi.org/10.17504/protocols.io.bzn6p5he" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">dx.doi.org/10.17504/protocols.io.bzn6p5he</a>.</p></div><div id="S20" class="sec"><p></p><h4 id="S20title" class="inline">Neuronal activation of the GI tract </h4><p id="P42" class="p p-first-last">See <a href="//doi.org/10.17504/protocols.io.bzp5p5q6" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">dx.doi.org/10.17504/protocols.io.bzp5p5q6</a>. TH-Cre and ChAT-Cre mice were used for these experiments. &#x0201c;Activated&#x0201d; mice were infected with AAV-PHP.S-hSYN1-DIO-hM3Dq-mRuby2 and &#x0201c;Control Mice&#x0201d; were infected with AAV-PHP.S-hSYN1-DIO-mRuby2. This was to control for both AAV-PHP.S-mediated expression and the effects of Compound 21 dihydrochloride (C21) (HelloBio, Princeton, NJ- HB6124). C21 was injected intraperitoneally (i.p.) at a dose of 3 mg/kg for 10 consecutive days to both groups of mice. Mice for timecourse experiments were single-housed in sterile cages with autoclaved water following the first C21 administration. Injections of C21 were administered at the same time every day (10a.m.). Mice were sacrificed 1 h after the day 10 injection.</p></div><div id="S21" class="sec"><p></p><h4 id="S21title" class="inline">Tissue preparation, immunohistochemistry, imaging, and quantification </h4><p id="P43" class="p p-first-last">Procedures are described in <a href="//doi.org/10.17504/protocols.io.bzp6p5re" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">dx.doi.org/10.17504/protocols.io.bzp6p5re</a>. 100 mg/kg of pentobarbital (Euthasol - Virbac, Carros, France) was administered i.p., and tissues were perfused with 30 mL of phosphate-buffered saline (PBS) and then cold 4% paraformaldehyde (PFA) in PBS. GI tract was post-fixed in 4% PFA overnight at 4&#x000b0;C, and stored in PBS +0.025% sodium azide. Tissues that underwent subsequent immunohistochemistry were made transparent by the passive CLARITY technique (PACT) as in Treweek et al., 2015.<sup><a href="#R33" rid="R33" class=" bibr popnode">33</a></sup> Briefly, perfused and fixed tissues were embedded with polymerized 4% (w/v) acrylamide, and lipids were eliminated using 8% (w/v) SDS solution. Jugular-nodose ganglia and dorsal root ganglia tissues were cryoprotected in 10% then 30% sucrose in PBS for 1 day each. Tissues were embedded and flash frozen in OCT and cryostat sectioned into 40 &#x003bc;M sections. Spinal cord and brain tissues were vibratome sectioned into 50 &#x003bc;M sections. Tissues were blocked in 3% donkey serum and permeabilized with PBS +0.3% Triton (PBST). Primary antibodies were incubated in PBST for 48 h and washed with PBST for 24 h (replacing the wash solution 3 times). Tissues were next incubated in secondary antibodies (and DAPI) for 24 h and washed in PBS for 48 h, intermittently replacing the wash solution with fresh PBS. Primary antibodies used were rabbit anti-PGP9.5 (1:300; Millipore Cat# AB1761-I, RRID:AB_2868444), rabbit anti-tyrosine hydroxylase (1:500, Abcam Cat# ab112, RRID:AB_297840), rabbit anti-choline acetyltransferase, (1:250, Abcam Cat# ab178850, RRID:AB_2721842), and mouse anti-NeuN (1:300, Abcam Cat# ab104224, RRID:AB_10711040). Secondary antibodies used were donkey anti-rabbit Alexa 568 (Thermo Fisher Scientific Cat# A10042, RRID:AB_2534017), goat anti-rabbit Alexa 647 (Thermo Fisher Scientific Cat# A-21245 (also A21245), RRID:AB_2535813), and goat anti-mouse Alexa 594 (Thermo Fisher Scientific Cat# A-11032, RRID:AB_2534091). GI tissues imaged for virally-expressed, endogenous fluorescence were made transparent using a sorbitol-based optical clearing method, Sca<em>l</em>eS as in Hama et al., 2015.<sup><a href="#R32" rid="R32" class=" bibr popnode">32</a></sup> Tissues were mounted in method-respective mounting media (RIMS and Scales S4) on a glass slide with a 0.5mm spacer (iSpacer, SunJin Lab Co.). Images were acquired on Zeiss LSM 780 or 880, and microscope, laser settings, contrast, and gamma remained constant across images that were directly compared. All confocal images were taken with the following objectives: Fluar 5 &#x000d7; 0.25 M27 Plan-Apochromat, 10 &#x000d7; 0.45 M27 (working distance 2.0 mm) and Plan-Apochromat 25 &#x000d7; 0.8 Imm Corr DIC M27 multiimmersion. Neurons in each ganglion were counted by counting cells that were of a distinct color. Colonic ganglia were defined as distinct if separated by a width of 3 or more neurons.</p></div><div id="S22" class="sec"><p></p><h4 id="S22title" class="inline">GCaMP6f fluorescence in <em>ex vivo</em> intestinal preparations </h4><p id="P44" class="p p-first-last">As described in <a href="//doi.org/10.17504/protocols.io.bzqap5se" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">dx.doi.org/10.17504/protocols.io.bzqap5se</a>, small intestinal tissue was quickly harvested from ChAT-Cre mice, flushed, and placed in oxygenated (95% O<sub>2</sub>, 5% CO<sub>2</sub>), ice-cold Krebs-Henseleit solution for 1 h followed by 15 min at room temperature. A segment was cut along the mesenteric attachment and pinned flat (mucosa facing down) on a Sylgard-lined recording chamber (Warner Instruments, PH1) in oxygenated Krebs-Henseleit solution. C21 was added at 10nM and GCaMP6f fluorescence was detected on an upright microscope (Zeiss, Oberkochen, Germany- Examiner D1).</p></div><div id="S23" class="sec"><p></p><h4 id="S23title" class="inline">Metagenomics </h4><p id="P45" class="p p-first-last">Procedures are described in <a href="//doi.org/10.17504/protocols.io.bzqep5te" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">dx.doi.org/10.17504/protocols.io.bzqep5te</a>.</p></div><div id="S24" class="sec"><p></p><h4 id="S24title" class="inline">Fecal collection </h4><p id="P46" class="p p-first-last">AAV-PHP.S-hSYN1-DIO-hM3Dq-mRuby2 (10<sup>12</sup> vg) was delivered systemically to TH-Cre and ChAT-Cre mice. 3&#x02013;4 week after infection, C21 (3 mg/kg) was administered daily for 10 consecutive days. Fecal pellets were collected in sterile containers one day before the initial C21 dose, and on day 2, 6, and 10 of injections.</p></div><div id="S25" class="sec"><p></p><h4 id="S25title" class="inline">Fecal sample DNA extraction and library preparation </h4><p id="P47" class="p p-first">DNA was extracted with the Qiagen MagAttract PowerSoil DNA kit as previously described in Marotz et al., 2017.<sup><a href="#R115" rid="R115" class=" bibr popnode">115</a></sup>This protocol is optimized for an input quantity of 1 ng DNA per reaction. Prior to library preparation, input DNA was transferred to a 384-well plate and quantified using a PicoGreen fluorescence assay (ThermoFisher, Inc). Input DNA was then normalized to 1 ng in a volume of 3.5 &#x003bc;L of molecular-grade water using an Echo 550 acoustic liquid-handling robot (Labcyte, Inc). Enzyme mixes for fragmentation, end repair and A-tailing, ligation, and PCR were prepared and added in approximately 1:8 scale volumes using a Mosquito HV micropipetting robot (TTP Labtech). Fragmentation was performed at 37&#x000b0;C for 20 min, followed by end repair and A-tailing at 65&#x000b0;C for 30 min.</p><p id="P48">Sequencing adapters and barcode indices were added in two steps, following the iTru adapter protocol.<sup><a href="#R116" rid="R116" class=" bibr popnode">116</a></sup> Universal &#x0201c;stub&#x0201d; adapter molecules and ligase mix were first added to the end-repaired DNA using the Mosquito HV robot and ligation performed at 20&#x000b0;C for 1 h. Unligated adapters and adapter dimers were then removed using AMPure XP magnetic beads and a BlueCat purification robot (BlueCat Bio). 7.5 &#x003bc;L magnetic bead solution was added to the total adapter-ligated sample volume, washed twice with 70% EtOH, and then resuspended in 7 &#x003bc;L molecular-grade water.</p><p id="P49" class="p p-last">Next, individual i7 and i5 were added to the adapter-ligated samples using the Echo 550 robot. Because this liquid handler individually addresses wells, and we used the full set of 384 unique error-correcting i7 and i5 indices, we were able to generate each plate of 384 libraries without repeating any barcodes, eliminating the problem of sequence misassignment due to barcode swapping.<sup><a href="#R117" rid="R117" class=" bibr popnode">117</a>,<a href="#R118" rid="R118" class=" bibr popnode">118</a></sup> To ensure that libraries generated on different plates could be pooled if necessary, and to safeguard against the possibility of contamination due to sample carryover between runs, we also iterated the assignment of i7 to i5 indices each run, such that each unique i7:i5 index combination was repeated only once every 147,456 libraries. 4.5 &#x003bc;L of eluted bead-washed ligated samples was added to 5.5 &#x003bc;L of PCR master mix and PCR-amplified for 15 cycles. The amplified and indexed libraries were then purified again using magnetic beads and the BlueCat robot, resuspended in 10 &#x003bc;L water, and 9 &#x003bc;L of final purified library transferred to a 384-well plate using the Mosquito HV liquid-handling robot for library quantitation, sequencing, and storage. 384 samples were then normalized based on a PicoGreen fluorescence assay.</p></div><div id="S26" class="sec"><p></p><h4 id="S26title" class="inline">Shallow shotgun metagenome sequencing and diversity analysis </h4><p id="P50" class="p p-first-last">The Illumina data for each HiSeq lane was uploaded to Qiita, a tool with standardized pipelines for processing and analyzing metagenomic data.<sup><a href="#R119" rid="R119" class=" bibr popnode">119</a></sup> Adapter sequences were removed from the reads using the Atropos v.1.1.15 (RRID:SCR_023962, <a href="https://github.com/jdidion/atropos" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/jdidion/atropos</a>)<sup><a href="#R120" rid="R120" class=" bibr popnode">120</a></sup> command (from the qp-shogun 0.1.5 pipeline) and the trimmed sequences were downloaded from Qiita. The reads for each sample were filtered of any potential mouse contamination using Bowtie2 v.2&#x02013;2.2.3<sup><a href="#R94" rid="R94" class=" bibr popnode">94</a></sup> (RRID:SCR_016368, <a href="https://bowtie-bio.sourceforge.net/bowtie2/index.shtml" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a>). The filtered reads were then aligned to the Web of Life (WoL) reference phylogeny<sup><a href="#R121" rid="R121" class=" bibr popnode">121</a></sup> with Bowtie2 using an adapted SHOGUN pipeline.<sup><a href="#R122" rid="R122" class=" bibr popnode">122</a></sup> The WoL contains 10,575 bacterial and archaeal genomes, with each genome representing an operational taxonomic unit (OTU). Sequencing reads that did not map to a single reference genome as well as reads that mapped to multiple genomes were not included in the analysis. If an OTU had a relative abundance less than 0.01% in a given sample, the OTU was not included for that sample. Additionally, OTUs with fewer than 5 assigned reads were not considered. The samples were rarefied to a depth of 12,750 reads and those with fewer than the rarefaction depth were excluded. The QIIME2 v.2019.7<sup><a href="#R44" rid="R44" class=" bibr popnode">44</a></sup> (RRID:SCR_021258, <a href="https://qiime2.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://qiime2.org/</a>) DEICODE plugin was used to calculate the Aitchison distances, a compositional beta diversity metric, and perform Robust Aitchison PCA to create biplots that visualize relationships between features and samples.<sup><a href="#R123" rid="R123" class=" bibr popnode">123</a></sup> The QIIME2 diversity plugin was used to calculate the other alpha- and beta-diversity metrics used in this study.</p></div><div id="S27" class="sec"><p></p><h4 id="S27title" class="inline">Metagenomics-based functional profiling </h4><p id="P51" class="p p-first-last">The filtered reads were also analyzed using HUMAnN2 v2.8.1<sup><a href="#R95" rid="R95" class=" bibr popnode">95</a></sup> (RRID:SCR_016280, <a href="https://huttenhower.sph.harvard.edu/humann2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://huttenhower.sph.harvard.edu/humann2</a>) to establish functional profiles for the samples. HUMAnN2 is a pipeline that begins by using MetaPhlAn2 to compile custom databases of reference genomes based on the species detected in a sample.<sup><a href="#R96" rid="R96" class=" bibr popnode">96</a></sup> HUMAnN2 then maps the filtered reads onto these custom databases and the reads that do not map to any of the references are then subjected to a translated search against UniProt Reference Clusters or UniRef<sup><a href="#R97" rid="R97" class=" bibr popnode">97</a></sup> (RRID:SCR_010646, <a href="https://www.uniprot.org/help/uniref" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.uniprot.org/help/uniref</a>). Here, the UniRef90 database was used for the translated search and installed according to the HUMAnN2 documentation. The results from both the search performed using the custom reference genome database and the search against the UniRef90 database were combined and the gene families identified in each sample were reported in units of reads per kilobase (RPKs) to account for gene length. HUMAnN2 also compared the gene families found in a sample with the MetaCyc pathways database<sup><a href="#R124" rid="R124" class=" bibr popnode">124</a></sup> (RRID:SCR_007778, <a href="https://metacyc.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://metacyc.org/</a>) and output a table reporting the pathway abundances found in each sample. After rarefying gene family tables to a depth of 166,000 RPKs and using a depth of 22,600 for pathway abundances, the QIIME2 diversity and DEICODE plugins were used to calculate alpha- and beta-diversity metrics. The metagenomics data from this study are available from. (<a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/metagenomics" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/metagenomics</a>).</p></div><div id="S28" class="sec"><p></p><h4 id="S28title" class="inline">Metabolomics </h4><p id="P52" class="p p-first-last">Procedures are described in <a href="//doi.org/10.17504/protocols.io.bzqfp5tn" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">dx.doi.org/10.17504/protocols.io.bzqfp5tn</a>.</p></div><div id="S29" class="sec"><p></p><h4 id="S29title" class="inline">Sample preparation </h4><p id="P53" class="p p-first-last">Frozen cecal samples were transported on dry ice for metabolomics analysis. The samples were weighed and an extraction solvent (1:1 methanol to water with an internal standard of 1 &#x003bc;M sulfamethazine) was added at a 1:10 mg to microliter ratio. The samples were then homogenized using a TissueLyser II (Qiagen) for 5 min at 25 Hz followed by a 15 min centrifugation at 14,000 rpm. 120 &#x003bc;L of supernatant was transferred to a 96-well DeepWell plate (Eppendorf) and lyophilized using a CentriVap Benchtop Vacuum Concentrator (Labconco) and stored at &#x02212;80&#x000b0;C. At the time of data acquisition, the lyophilized plates were resuspended in a 1:1 methanol to water solvent spiked with 1 &#x003bc;M of sulfadimethoxine. The plates were vortexed for 2 min, centrifuged at 14,000 rpm for 15 min and 120 &#x003bc;L of the supernatant was transferred to a 96-well autosampler plate (Eppendorf). Plates were stored at 4&#x000b0;C prior to LCMS analysis.</p></div><div id="S30" class="sec"><p></p><h4 id="S30title" class="inline">Data acquisition </h4><p id="P54" class="p p-first-last">The untargeted metabolomics analysis was completed using an ultra-high performance liquid chromatography system (Thermo Dionex Ultimate 3000 UHPLC) coupled to an ultra-high resolution quadrupole time of flight (qTOF) mass spectrometer (Bruker Daltonics MaXis HD). A Phenomenex Kinetex column (C18 1.7 &#x003bc;m, 2.1 mm &#x000d7; 50 mm) was used for chromatographic separation. An injection volume of 5 &#x003bc;L was used for each sample and a flow-rate of 0.500 mL was used throughout the analysis. The mobile phase consisted of solvent A: 100% LC-MS grade water spiked with 0.1% formic acid and solvent B: 100% LC-MS grade acetonitrile spiked with 0.1% formic acid. The chromatographic gradient was as follows: 0.0&#x02013;1.0 min, 5% B; 1.0&#x02013;9.0 min, 5&#x02013;100% B; 9.0&#x02013;11.0 min, 100% B; 11.0&#x02013;11.5 min, 100&#x02013;5% B; 11.5&#x02013;12.5 min, 5% B. Data was collected with electrospray ionization in positive mode, and was saved as.d file folders.</p></div><div id="S31" class="sec"><p></p><h4 id="S31title" class="inline">Data processing </h4><p id="P55" class="p p-first-last">The raw.d data files were converted to mzXML format using Bruker Compass DataAnalysis 4.1 software. The resulting.mzXML file, the original.d file folders, and basic prep information sheet are stored in the UC San Diego MassIVE data repository under the accession number MSV000084550. For MS1 level feature detection, the open-source software MZmine version 2.51 (RRID:SCR_012040, <a href="https://mzmine.github.io/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mzmine.github.io/</a>) was used. The parameters used are as follows: 1) Mass Detection (Centroid, Noise Level MS1 1E3, MS2 1E2); 2) ADAP Chromatogram Builder (Min Group size in # of scans = 3, Group Intensity Threshold = 3E3, Min Highest Intensity = 1E3, m/z tolerance 0.01 m/z or 10.0 ppm); 3) Chromatogram Deconvolution (Local Minimum Search&#x0003e;Chromatographic Threshold 0.01%, Minimum in RT range 0.50 min, &#x0003c;Minimum Relative Height 0.01%, Minimum Absolute Height 3E3, Min Ratio of Peak Top/Edge 2, Peak Duration Range 0.05&#x02013;0.50 min; m/z Calculation Auto, m/z range for MS2 pairing 0.01 Da, and RT Range for MS2 Pairing 0.1 min); Isotopic Peaks Grouper (m/z Tolerance 0.01 m/z or 10.0 ppm, Retention Time Tolerance 0.3 min, Maximum Charge 4, Representative Ion Most Intense); Join Aligner (m/z Tolerance 0.01 m/z or 10.0 ppm, Weight for m/z 75, Retention Time Tolerance 0.3 min, Weight for RT 25); Gap-Filling Peak Finder (Intensity Tolerance 20%, m/z Tolerance 0.005 m/z or 10.0 ppm, Retention Time Tolerance 0.2 min). The resulting feature table was saved as a.csv file and.mgf file for use in GNPS and MetaboAnalyst.</p></div><div id="S32" class="sec"><p></p><h4 id="S32title" class="inline">Molecular networking and statistical analysis </h4><p id="P56" class="p p-first">Molecular networking was performed using the feature networking tool available through the Global Natural Products Social Molecular Networking (GNPS, RRID:SCR_019012, <a href="https://gnps.ucsd.edu/ProteoSAFe/static/gnps-splash.jsp" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://gnps.ucsd.edu/ProteoSAFe/static/gnps-splash.jsp</a>) portal:<a href="https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://gnps.ucsd.edu/ProteoSAFe/index.jsp?params=%7B%22workflow%22:%22FEATURE-BASED-MOLECULAR-NETWORKING%22,%22library_on_server%22:%22d.speclibs;%22%7D</a>.</p><p id="P57" class="p p-last">The annotations obtained using this workflow fall under MSI level 2 or 3 and were used for feature analysis.<sup><a href="#R46" rid="R46" class=" bibr popnode">46</a></sup> Briefly, level 2 compounds are putatively annotated, meaning they are not identified using chemical reference standards but rather based on physical properties and/or spectral similarities to available spectral libraries (publicly available and purchased NIST17 CID). Level 3 compounds are putatively characterized classes of compounds identified similarly to level 2 compounds. The feature-based molecular networking workflow on GNPS<sup><a href="#R125" rid="R125" class=" bibr popnode">125</a></sup> was utilized in order to analyze the spectra associated with the feature tables produced using the open source software Mzmine version 2.51<sup><a href="#R98" rid="R98" class=" bibr popnode">98</a></sup> (RRID:SCR_012040, <a href="https://mzmine.github.io/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mzmine.github.io/</a>). The.mgf and.csv outputs from MZmine v2.51 were used to run the workflow. The GNPS workflow parameters used were as follows: Precursor Ion Mass 0.02 Da, Fragment Ion Mass Tolerance 0.02 Da, Min Pairs Cos 0.7, Minimum Matched Fragments 6, Maximum Shift Between Precursors 500 Da, Network TopK 10, Maximum Connected Component Size (Beta) 100, and the files were row sum normalized. Default parameters were used for the rest of the settings. Visualizations and statistical analyses were performed using QIIME 2 v.2019.10<sup><a href="#R44" rid="R44" class=" bibr popnode">44</a></sup> (RRID:SCR_021258), MetaboAnalyst<sup><a href="#R99" rid="R99" class=" bibr popnode">99</a></sup> (RRID:SCR_015539, <a href="https://www.metaboanalyst.ca/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.metaboanalyst.ca/</a>) and Cytoscape v3.7.2<sup><a href="#R100" rid="R100" class=" bibr popnode">100</a></sup> (RRID:SCR_003032, <a href="https://cytoscape.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://cytoscape.org/</a>). The metabolomics data from this study are available from. (<a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/metabolomics" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/metabolomics</a>).</p></div><div id="S33" class="sec"><p></p><h4 id="S33title" class="inline">Proteome preparation </h4><p id="P58" class="p p-first-last">Procedures are described in <a href="//doi.org/10.17504/protocols.io.bzqcp5sw" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">dx.doi.org/10.17504/protocols.io.bzqcp5sw</a>.</p></div><div id="S34" class="sec"><p></p><h4 id="S34title" class="inline">Protein extraction </h4><p id="P59" class="p p-first-last">Mice were sacrificed 1 h after the final C21 administration and cecal contents were isolated and resuspended in 400 &#x003bc;L of phosphate buffered solution and centrifuged at 20,000 xg to spin down cells and lysate. Protein was isolated from the resulting supernatant using Wessel-Fl&#x000fc;gge&#x02019;s methanol/chloroform extraction method (Wessel and Fl&#x000fc;gge, 1984). Briefly, MeOH and chloroform were added to the samples at a 4:1 and 1:1 ratio, respectively. Next, dH<sub>2</sub>O was added at a 3:1 ratio, samples were vortexed and centrifuged at 20,000 xg. Resulting precipitated protein was collected and washed with MeOH. Precipitated protein was centrifuged and left to air dry, and stored at &#x02212;20&#x000b0;C until protein digestion.</p></div><div id="S35" class="sec"><p></p><h4 id="S35title" class="inline">In-solution protein digestion and desalting </h4><p id="P60" class="p p-first-last">Precipitated protein samples were denatured in 40 &#x003bc;L of 8M Urea (100 mM Tris-HCl pH 8.5). To reduce disulfide bonds, 1.25 &#x003bc;L of 100 mM Tris(2-carboxyethyl)Phosphine was added and incubated at room temperature (RT) for 20 min. Then 1.8 &#x003bc;L of 250 mM iodoacetamide was added and incubated at RT in the dark to alkylate cysteines. The first step of digestion was initiated by adding 1 &#x003bc;L of 0.1 &#x003bc;g/&#x003bc;L of lysyl endopeptidase. After 4 h of incubation, the urea concentration was adjusted to 2 M by adding 120 &#x003bc;L of 100 mM Tris-HCl pH 8.5. The second step of digestion was done by adding 2.5 &#x003bc;L of 2&#x003bc;g/&#x003bc;L trypsin plus 1.6 &#x003bc;L of 100 mM CaCl<sub>2</sub> and incubating overnight in the dark. Formic acid was added to stop trypsin digestion. Digested peptides were desalted by HPLC using a C8 peptide microtrap (Microm Bioresources), lyophilized, and diluted to 200 ng/&#x003bc;L in 0.2% formic acid prior to LC-MS/MS analysis.</p></div><div id="S36" class="sec"><p></p><h4 id="S36title" class="inline">LC-MS/MS </h4><p id="P61" class="p p-first-last">Samples were analyzed on a Q Exactive HF Orbitrap mass spectrometer coupled to an EASY nLC 1200 liquid chromatographic system (Thermo Scientific, San Jose, CA). Approximately 200 ng of peptides were loaded on a 50 &#x003bc;m I.D. &#x000d7; 25 cm column with a 10 &#x003bc;m electrospray tip (PicoFrit from New Objective, Woburn, MA) in-house-packed with ReproSil-Pur C18-AQ 1.9 &#x003bc;m (Dr. Maisch, Ammerbuch, Germany). Solvent A consisted of 2% MeCN in 0.2% FA and solvent B consisted of 80% MeCN in 0.2% FA. A non-linear 60 min gradient from 2% B to 40% B was used to separate the peptides for analysis. The mass spectrometer was operated in a data-dependent mode, with MS1 scans collected from 400 to 1650 m/z at 60,000 resolution and MS/MS scans collected from 200 to 2000 m/z at 30,000 resolution. Dynamic exclusion of 45 s was used. The top 12 most abundant peptides with charge states between 2 and 5 were selected for fragmentation with normalized collision energy of 28.</p></div><div id="S37" class="sec"><p></p><h4 id="S37title" class="inline">Peptide and protein identification </h4><p id="P62" class="p p-first-last">Thermo.raw files were converted to.ms1 and.ms2 files using RawConverter 1.1.0.18 operating in data dependent mode and selecting for monoisotopic m/z. Tandem mass spectra (.ms2 files) were identified by a database search method using the Integrated Proteomics Pipeline 6.5.4 (IP2, Integrated Proteomics Applications, Inc., <a href="http://www.integratedproteomics.com" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.integratedproteomics.com</a>). Briefly, databases containing forward and reverse (decoy)<sup><a href="#R126" rid="R126" class=" bibr popnode">126</a>,<a href="#R127" rid="R127" class=" bibr popnode">127</a></sup> peptide sequences were generated from <em>in silico</em> trypsin digestion of either the mouse proteome (UniProt; Oct. 2, 2019)<sup><a href="#R101" rid="R101" class=" bibr popnode">101</a></sup> or protein sequences derived from large comprehensive public repositories (ComPIL 2.0).<sup><a href="#R102" rid="R102" class=" bibr popnode">102</a></sup> Tandem mass spectra were matched to peptide sequences using the ProLuCID/SEQUEST (1.4)<sup><a href="#R103" rid="R103" class=" bibr popnode">103</a>,<a href="#R104" rid="R104" class=" bibr popnode">104</a></sup> software package. The validity of spectrum-peptide matches was assessed using the SEQUEST-defined parameters XCorr (cross-correlation score) and DeltaCN (normalized difference in cross-correlation scores) in the DTASelect2 (2.1.4)<sup><a href="#R105" rid="R105" class=" bibr popnode">105</a>,<a href="#R106" rid="R106" class=" bibr popnode">106</a></sup> software package. Search settings were configured as follows: (1) 5 ppm precursor ion mass tolerance, (2) 10 ppm fragment ion mass tolerance, (3) 1% peptide false discovery rate, (4) 2 peptide per protein minimum, (5) 600&#x02013;6000 Da precursor mass window, (6) 2 differential modifications per peptide maximum (methionine oxidation: M+15.994915 Da), (7) unlimited static modifications per peptide (cysteine carbamidomethylation: C+57.02146 Da), and (8) the search space included half- and fully-tryptic (cleavage C-terminal to K and R residues) peptide candidates with unlimited (mouse database, custom metagenomic shotgun database) or 2 missed cleavage events (ComPIL 2.0).</p></div><div id="S38" class="sec"><p></p><h4 id="S38title" class="inline">Differential analysis of detected proteins using peptide-spectrum matches (spectral counts) </h4><p id="P63" class="p p-first-last">Detected proteins were grouped by sequence similarity into &#x0201c;clusters&#x0201d; using CD-HIT 4.8.1 (RRID:SCR_007105, <a href="http://weizhong-lab.ucsd.edu/cd-hit/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://weizhong-lab.ucsd.edu/cd-hit/</a>)<sup><a href="#R107" rid="R107" class=" bibr popnode">107</a>,<a href="#R108" rid="R108" class=" bibr popnode">108</a>,<a href="#R128" rid="R128" class=" bibr popnode">128</a></sup> at the following similarity cut-offs: 65%, 75%, 85%, and 95%. The following is an example command line input: &#x0201c;cd-hit -i fastafile.fasta -o outputfile -c 0.65 -g 1 -d 0&#x0201d;. Tandem mass spectra identified as peptides (peptide spectrum matches, PSMs) were mapped to CD-HIT generated clusters. PSMs mapping to &#x0003e;1 cluster were discarded. Cluster-PSM tables were generated and differential analysis was performed in DESeq2 (1.25.13, RRID:SCR_015687, <a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://bioconductor.org/packages/release/bioc/html/DESeq2.html</a>).<sup><a href="#R109" rid="R109" class=" bibr popnode">109</a></sup> Briefly, count data (PSMs) were modeled using the negative binomial distribution, and the mean-variance relationship was estimated. Variance was estimated using an information sharing approach whereby a single feature&#x02019;s (or cluster&#x02019;s) variance was estimated by taking into account variances of other clusters measured in the same experiment. Feature significance calling and ranking were performed using estimated effect sizes. Multiple testing correction was performed by the Benjamini-Hochberg method within the DESeq2 package. Volcano plots were generated in Prism (GraphPad).</p></div><div id="S39" class="sec"><p></p><h4 id="S39title" class="inline">Differential analysis of detected proteins using ion intensity (precursor intensity) </h4><p id="P64" class="p p-first">Detected proteins were grouped into &#x0201c;clusters&#x0201d; by sequence similarity using CD-HIT 4.8.1<sup><a href="#R107" rid="R107" class=" bibr popnode">107</a>,<a href="#R108" rid="R108" class=" bibr popnode">108</a>,<a href="#R128" rid="R128" class=" bibr popnode">128</a></sup> at the following similarity cutoffs: 65%, 75%, 85%, and 95%. The following is an example command line input: &#x0201c;cd-hit &#x02212;i fastafile.fasta &#x02212;o outputfile &#x02212;c 0.65 &#x02212;g 1 &#x02212;d 0&#x0201d;. Using the Census software package<sup><a href="#R129" rid="R129" class=" bibr popnode">129</a></sup> (Integrated Proteomics Pipeline 6.5.4), peptide ion intensities were calculated from.ms1 files. Peptide ion intensities were assigned to their parent peptide, then parent peptides were mapped to their appropriate CD-HIT generated clusters. Ion intensities belonging to parent peptides that mapped to &#x0003e;1 CD-HIT cluster were discarded. Clusterion intensity tables were generated.</p><p id="P65" class="p p-last">Ion intensity data were analyzed using the Differential Enrichment analysis of Proteomics data DEP package (RRID:SCR_023090, <a href="https://bioconductor.org/packages/release/bioc/html/DEP.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://bioconductor.org/packages/release/bioc/html/DEP.html</a>)<sup><a href="#R110" rid="R110" class=" bibr popnode">110</a></sup> operating in R. Intensity values were automatically Log2 transformed in DEP. The cluster list was subsequently filtered with the &#x02018;filter_proteins&#x02019; function such that clusters with missing values above a 65% threshold were discarded. Remaining intensities were further transformed by the &#x02018;normalize_vsn&#x02019; function.<sup><a href="#R130" rid="R130" class=" bibr popnode">130</a></sup> Missing data in remaining clusters were imputed using a mixed approach. Clusters where either the control or treatment group contained only null entries were classified as &#x02018;missing not at random&#x02019; (MNAR) and imputed with 0 values. All other groups were treated as &#x02018;missing at random&#x02019; (MAR) and imputed using the maximum likelihood method (&#x02018;MLE&#x02019;).<sup><a href="#R131" rid="R131" class=" bibr popnode">131</a></sup> Note that for a given cluster, missing values for treatment groups were imputed separately by treatment group. Differential expression analyses were performed on filled-in cluster-ion intensity tables using the &#x02018;test_diff&#x02019; function<sup><a href="#R132" rid="R132" class=" bibr popnode">132</a></sup> and multiple testing correction was performed using the &#x02018;add_rejections&#x02019; function.</p></div><div id="S40" class="sec"><p></p><h4 id="S40title" class="inline">Network analysis using the STRING database </h4><p id="P66" class="p p-first-last">Upregulated proteins with a nominal p value &#x0003c;0.2 were searched against protein-protein interactions in the STRING database<sup><a href="#R111" rid="R111" class=" bibr popnode">111</a></sup> (RRID:SCR_005223, <a href="http://www.string-db.org" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://www.string-db.org</a>) where high confidence interactions are selected for. Briefly, the STRING database sources protein-protein interactions from primary databases consisting of genomic context predictions, high-throughput lab experiments, (conserved) co-expression, automated text mining, and previous knowledge in databases.<sup><a href="#R111" rid="R111" class=" bibr popnode">111</a></sup></p></div><div id="S41" class="sec"><p></p><h4 id="S41title" class="inline">Metaproteome analysis using unipept </h4><p id="P67" class="p p-first-last">Upregulated tryptic, microbial peptide sequences, with fold change and nominal p value cutoffs of &#x0003e;2 and &#x0003c;0.2, respectively, were input into Unipept (<a href="http://unipept.ugent.be" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://unipept.ugent.be</a>),<sup><a href="#R133" rid="R133" class=" bibr popnode">133</a>,<a href="#R134" rid="R134" class=" bibr popnode">134</a></sup> equating leucine and isoleucine and filtering duplicate peptides. Briefly, Unipept indexes tryptic peptide sequences from the UniProtKB and details peptides with NCBI&#x02019;s taxonomic database. Lowest common ancestor was calculated for each tryptic peptide. The proteomics data from this study are available from. (<a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/proteomics" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/proteomics</a>).</p></div><div id="S42" class="sec"><p></p><h4 id="S42title" class="inline">3&#x02032; mRNA sequencing </h4><p id="P68" class="p p-first-last">Procedures are described in <a href="//doi.org/10.17504/protocols.io.bzqbp5sn" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">dx.doi.org/10.17504/protocols.io.bzqbp5sn</a>.</p></div><div id="S43" class="sec"><p></p><h4 id="S43title" class="inline">Tissue collection and RNA extraction </h4><p id="P69" class="p p-first-last">Mice were cervically dislocated and the GI tract was removed. 1 cm of tissue above and below the cecum were dissected and cleaned to represent tissue from the distal SI and proximal colon, respectively. Tissue was homogenized in TRIzol (ThermoFisher Scientific, Waltham. MA- Cat. No. 15596018) solution using a bead-based homogenizing method, and total RNA was extracted using chloroform per manufacturer&#x02019;s instructions.</p></div><div id="S44" class="sec"><p></p><h4 id="S44title" class="inline">Library preparation, sequencing, and analysis </h4><p id="P70" class="p p-first">The cDNA libraries were prepared using the QuantSeq 3&#x02032;mRNA-Seq Library Prep Kit FWD for Illumina (Lexogen, Greenland, NH) supplemented with UMI (unique molecular index) as per the manufacturer&#x02019;s instructions. Briefly, total RNA was reverse transcribed using oligo (dT) primers. The second cDNA strand was synthesized by random priming, in a manner such that DNA polymerase is efficiently stopped when reaching the next hybridized random primer, so only the fragment closest to the 3&#x02032; end gets captured for later indexed adapter ligation and PCR amplification. UMIs were incorporated to the first 6 bases of each read, followed by 4 bases of spacer sequences. UMIs are used to eliminate possible PCR duplicates in sequencing datasets and therefore facilitate unbiased gene expression profiling. The basic principle behind the UMI deduplication step is to collapse reads with identical mapping coordinates and UMI sequences. This step helps increase the accuracy of sequencing read counts for downstream analysis of gene expression levels. The processed libraries were assessed for size distribution and concentration using the Bioanalyzer High Sensitivity DNA Kit (Agilent Technologies, Santa Clara, CA- Cat. No. 5067&#x02013;4626 and &#x02212;4627). Pooled libraries were diluted to 2 nM in EB buffer (Qiagen, Hilden, Germany, Cat. No. 19086) and then denatured using the Illumina protocol. The libraries were pooled and diluted to 2 nM using 10 mM Tris-HCl, pH 8.5 and then denatured using the Illumina protocol. The denatured libraries were diluted to 10 p.m. with pre-chilled hybridization buffer and loaded onto an Illumina MiSeq v3 flow cell for 150 cycles using a single-read recipe according to the manufacturer&#x02019;s instructions. Single-end 75 bp reads (max 4.5M reads) were obtained. De-multiplexed sequencing reads were generated using Illumina BaseSpace.</p><p id="P71" class="p p-last">UMI specific workflows that were developed and distributed by Lexogen were used to extract reads that are free from PCR artifacts (i.e., deduplication). First, the umi2index tool was used to add the 6 nucleotide UMI sequence to the identifier of each read and trim the UMI from the start of each read. This generated a new FASTQ file, which was then processed through trimming and alignment. Second, after the quality and polyA trimming by BBDuk<sup><a href="#R112" rid="R112" class=" bibr popnode">112</a></sup> (Bestus Bioinformaticus Duk, RRID:SCR_016969, <a href="https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbduk-guide/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbduk-guide/</a>) and alignment by HISAT2 (version 2.1.0, RRID:SCR_015530, <a href="https://daehwankimlab.github.io/hisat2/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://daehwankimlab.github.io/hisat2/</a>),<sup><a href="#R113" rid="R113" class=" bibr popnode">113</a></sup> the mapped reads were collapsed according to the UMI sequence of each read. Reads were collapsed if they had the same mapping coordinates (CIGAR string) and identical UMI sequences. Collapsing reads in this manner removes PCR duplicates. Read counts were calculated using HTSeq (RRID:SCR_005514, <a href="https://htseq.readthedocs.io/en/release_0.9.1/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://htseq.readthedocs.io/en/release_0.9.1/</a>)<sup><a href="#R114" rid="R114" class=" bibr popnode">114</a></sup> by supplementing Ensembl gene annotation (GRCm38.78). Raw read counts were run through ShinySeq to obtain differentially expressed genes and downstream gene ontology analyses.<sup><a href="#R135" rid="R135" class=" bibr popnode">135</a></sup> The RNAseq data from this study are available from. (<a href="https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/RNAseq" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/mazmanianlab/Griffiths_Yoo_et_al/tree/main/RNAseq</a>).</p></div><div id="S45" class="sec"><p></p><h4 id="S45title" class="inline">Whole gut transit time, fecal water content, and fecal output </h4><p id="P72" class="p p-first">Procedures are described in <a href="//doi.org/10.17504/protocols.io.36wgq3p1xlk5/v1" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">dx.doi.org/10.17504/protocols.io.36wgq3p1xlk5/v1</a>.</p><p id="P73" class="p p-last">6% (w/v) carmine red (Sigma-Aldrich, St. Louis, MO) with 0.5% methylcellulose (Sigma-Aldrich) was dissolved and autoclaved prior to use. ChAT-Cre and TH-Cre mice were administered C21 (3 mg/kg) intraperitoneally, and subsequently orally gavaged with 150 &#x003bc;L of carmine red solution. Mice were single-housed with no bedding for the duration of the experiment, and animals were not fasted beforehand. Over the 5 h following gavage, the time of expulsion was recorded for each fecal pellet, and each pellet was collected in pre-weighed, 1.5 mL microcentrifuge tube. Each pellet collected was checked for the presence of carmine red, and the time of initial carmine red pellet expulsion was recorded as GI transit time. The mass of collected fecal pellets was determined, and pellets were left to dry in an 80&#x000b0;C oven for 2 days before weighing the desiccated pellets and calculating the pellets&#x02019; initial water content. Fecal output rate for each mouse was calculated as the total number of pellets expelled during the 5 h time course post-C21 administration divided by the time the last fecal pellet was expelled.</p></div><div id="S46" class="sec sec-last"><p></p><h4 id="S46title" class="inline">Colonic migrating motor complexes in <em>ex vivo</em> intestinal preparations </h4><p id="P74" class="p p-first">Procedures are described in <a href="//doi.org/10.17504/protocols.io.n92ldm61nL5b/v1" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">dx.doi.org/10.17504/protocols.io.n92ldm61nL5b/v1</a>.</p><p id="P75" class="p p-last">Intact colons were dissected from cervically-dislocated ChAT-Cre and TH-Cre mice, flushed and placed in pre-oxygenated (95% O<sub>2</sub>, 5% CO<sub>2</sub>) Krebs-Henseleit solution at RT. Proximal and distal ends were cannulated to 2 mm diameter tubes and secured in the center of an organ bath with continuously oxygenated Krebs-Henseleit solution at 37&#x000b0;C. Syringe pumps were connected to the inlet and outlet tubes to maintain a flow of solution at a rate of 500 &#x003bc;L/min through the colon. The system was allowed to equilibrate for 30 min before recording. Baseline recordings were taken for 30 min, then the Krebs solution in the organ bath was briefly removed, mixed with C21 to a final concentration of 2 &#x003bc;M, and replaced in the organ bath. Recordings were taken for another 30 min.</p></div></div><div id="S47" class="sec sec-last"><h3 id="S47title">QUANTIFICATION AND STATISTICAL ANALYSIS</h3><p id="P76" class="p p-first-last">Statistical methodologies and software used for performing various types of multi-omic analysis in this work are cited where appropriate in the STAR Methods text. The p value calculations for viral transduction, microbiome differences, gastrointestinal function, and animal welfare were done using Prism GraphPad v.9.2.0. The specific statistical test used for each figure is denoted in the figure legends. Error bars represent the standard error of the mean unless otherwise stated.</p></div></div><div id="unreferenced-ft" class="tsec sec"><div class="unreferenced">​
                    <!--table ft1--><!--table-wrap mode="anchored" t5--><div class="table-wrap anchored whole_rhythm" id="T1"><!--caption a7--><div class="caption"><p id="P114">KEY RESOURCES TABLE</p></div><div class="xtable"><table frame="hsides" rules="groups" class="rendered small default_table"><thead><tr><th align="left" valign="middle" rowspan="1" colspan="1">REAGENT or RESOURCE</th><th align="left" valign="middle" rowspan="1" colspan="1">SOURCE</th><th align="left" valign="middle" rowspan="1" colspan="1">IDENTIFIER</th></tr></thead><tbody><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">Antibodies</td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">rabbit anti-PGP9.5</td><td align="left" valign="top" rowspan="1" colspan="1">Millipore</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# AB1761-I, RRID:AB_2868444</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">rabbit anti-tyrosine hydroxylase</td><td align="left" valign="top" rowspan="1" colspan="1">Abcam</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# ab112, RRID:AB_297840</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">rabbit anti-choline acetyltransferase</td><td align="left" valign="top" rowspan="1" colspan="1">Abcam</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# ab178850, RRID:AB_2721842</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">mouse anti-NeuN</td><td align="left" valign="top" rowspan="1" colspan="1">Abcam</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# ab104224, RRID:AB_10711040</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">donkey anti-rabbit Alexa 568</td><td align="left" valign="top" rowspan="1" colspan="1">Thermo Fisher Scientific</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# A10042, RRID:AB_2534017</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">goat anti-rabbit Alexa 647</td><td align="left" valign="top" rowspan="1" colspan="1">Thermo Fisher Scientific</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# A-21245, RRID:AB_2535813</td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">goat anti-mouse Alexa 594</td><td align="left" valign="top" rowspan="1" colspan="1">Thermo Fisher Scientific</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# A-11032, RRID:AB_2534091</td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">Chemicals, peptides, and recombinant proteins</td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">Compound 21 dihydrochloride (C21)</td><td align="left" valign="top" rowspan="1" colspan="1">HelloBio</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# HB6124</td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">Critical commercial assays</td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">MagAttract PowerSoil DNA kit</td><td align="left" valign="top" rowspan="1" colspan="1">Qiagen</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# 27100&#x02013;4-EP</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">PicoGreen fluorescence assay</td><td align="left" valign="top" rowspan="1" colspan="1">ThermoFisher</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# P7589</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">QuantSeq 3&#x02032;mRNA-Seq Library Prep Kit FWD for Illumina</td><td align="left" valign="top" rowspan="1" colspan="1">Lexogen</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# 015</td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">Bioanalyzer High Sensitivity DNA Kit</td><td align="left" valign="top" rowspan="1" colspan="1">Agilent Technologies</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# 5067&#x02013;4626 and &#x02212;4627</td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">Deposited data</td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Metagenomic data</td><td align="left" valign="top" rowspan="1" colspan="1">This paper</td><td align="left" valign="top" rowspan="1" colspan="1">EBI accession: ERP131523</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Metabolomic data</td><td align="left" valign="top" rowspan="1" colspan="1">This paper</td><td align="left" valign="top" rowspan="1" colspan="1">UCSD MassIVE: MSV000084550</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Proteomic data</td><td align="left" valign="top" rowspan="1" colspan="1">This paper</td><td align="left" valign="top" rowspan="1" colspan="1">UCSD MassIVE: MSV000087917</td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">QuantSeq data</td><td align="left" valign="top" rowspan="1" colspan="1">This paper</td><td align="left" valign="top" rowspan="1" colspan="1">GEO: <a id="__tag_1149457096" class="tag_hotlink" href="/geo/query/acc.cgi?acc=GSE180961" ref="/geo/query/acc.cgi?acc=GSE180961"><span class="tag-json" style="display:none">{"type":"entrez-geo","attrs":{"text":"GSE180961","term_id":"180961"}}</span>GSE180961</a></td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">Experimental models: Cell lines</td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">HEK293T/17</td><td align="left" valign="top" rowspan="1" colspan="1">ATCC</td><td align="left" valign="top" rowspan="1" colspan="1">Cat# CRL-11268, RRID:CVCL_1926</td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">Experimental models: Organisms/strains</td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Mouse: B6.129X1-Thtm1(cre)Te/Kieg (TH-cre)</td><td align="left" valign="top" rowspan="1" colspan="1">Gift from Ted Ebendal<sup><a href="#R92" rid="R92" class=" bibr popnode">92</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:MGI:3487234</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Mouse: B6J.129S6-Chattm2(cre)Lowl/MwarJ (ChAT-cre)</td><td align="left" valign="top" rowspan="1" colspan="1">Jackson Laboratories</td><td align="left" valign="top" rowspan="1" colspan="1">ME- Stock# 028861, RRID:IMSR_JAX:028861</td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">Mouse: C57BL/6</td><td align="left" valign="top" rowspan="1" colspan="1">Jackson Laboratories</td><td align="left" valign="top" rowspan="1" colspan="1">ME- Stock# 000664, RRID:IMSR_JAX:000664</td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">Recombinant DNA</td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Plasmid: AAV-PHP.S</td><td align="left" valign="top" rowspan="1" colspan="1">Gradinaru Lab</td><td align="left" valign="top" rowspan="1" colspan="1">Addgene plasmid# 103006, RRID:Addgene_103006</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Plasmid: hSYN1-tdTomato</td><td align="left" valign="top" rowspan="1" colspan="1">This study</td><td align="left" valign="top" rowspan="1" colspan="1">Adapted from Addgene plasmid# 99116, RRID: Addgene_99116 and plasmid# 99126, RRID:Addgene_99126</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Plasmid: hSYN1-mRuby2</td><td align="left" valign="top" rowspan="1" colspan="1">Gradinaru Lab</td><td align="left" valign="top" rowspan="1" colspan="1">Addgene plasmid# 99126, RRID:Addgene_99126</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Plasmid: hSYN1-DIO-mRuby2</td><td align="left" valign="top" rowspan="1" colspan="1">This study</td><td align="left" valign="top" rowspan="1" colspan="1">Adapted from Addgene plasmid# 99126, RRID:Addgene_99126</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Plasmid: CAG-mNeonGreen</td><td align="left" valign="top" rowspan="1" colspan="1">Gradinaru Lab</td><td align="left" valign="top" rowspan="1" colspan="1">Addgene plasmid# 99134, RRID:Addgene_99134</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Plasmid: hSYN1-mNeonGreen</td><td align="left" valign="top" rowspan="1" colspan="1">Gradinaru Lab</td><td align="left" valign="top" rowspan="1" colspan="1">Addgene plasmid# 99135, RRID:Addgene_99135</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Plasmid: hSYN1-DIO-mNeonGreen</td><td align="left" valign="top" rowspan="1" colspan="1">This study</td><td align="left" valign="top" rowspan="1" colspan="1">Adapted from Addgene plasmid# 99125, RRID:Addgene_99135</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Plasmid: hSYN1-mTurquoise2</td><td align="left" valign="top" rowspan="1" colspan="1">Gradinaru Lab</td><td align="left" valign="top" rowspan="1" colspan="1">Addgene plasmid# 99125, RRID: Addgene_99125</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Plasmid: hSYN1-DIO-mTurquoise2</td><td align="left" valign="top" rowspan="1" colspan="1">This study</td><td align="left" valign="top" rowspan="1" colspan="1">Adapted from Addgene plasmid# 99125, RRID: Addgene_99125</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Plasmid: hSYN1-DIO-hM3Dq-mRuby2</td><td align="left" valign="top" rowspan="1" colspan="1">This study</td><td align="left" valign="top" rowspan="1" colspan="1">Adapted from Addgene plasmid# 50474 RRID:Addgene_50474 and Addgene plasmid# 99126, RRID:Addgene_99126</td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">Plasmid: CAG-GCaMP6f</td><td align="left" valign="top" rowspan="1" colspan="1">This study</td><td align="left" valign="top" rowspan="1" colspan="1">Adapted from Addgene plasmid# 100837 RRID:Addgene_100837</td></tr><tr style="border-bottom: solid 1px"><td align="left" valign="top" rowspan="1" colspan="1">Software and algorithms</td><td align="left" valign="top" rowspan="1" colspan="1"></td><td align="left" valign="top" rowspan="1" colspan="1"></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">FIJI</td><td align="left" valign="top" rowspan="1" colspan="1">Schindelin et al.<sup><a href="#R93" rid="R93" class=" bibr popnode">93</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_002285, <a href="https://imagej.net/software/fiji" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://imagej.net/software/fiji</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">R</td><td align="left" valign="top" rowspan="1" colspan="1">R Project for Statistical Computing</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_001905, <a href="https://www.r-project.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.r-project.org/</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Qiita</td><td align="left" valign="top" rowspan="1" colspan="1">Bolyen et al.<sup><a href="#R44" rid="R44" class=" bibr popnode">44</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://www.qiime2.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.qiime2.org/</a>
</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Bowtie2</td><td align="left" valign="top" rowspan="1" colspan="1">Langmead and Salzberg<sup><a href="#R94" rid="R94" class=" bibr popnode">94</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_016368, <a href="http://bowtie-bio.sourceforge.net/bowtie2/index.shtml" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://bowtie-bio.sourceforge.net/bowtie2/index.shtml</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">HUMAnN2 v2.8.1</td><td align="left" valign="top" rowspan="1" colspan="1">Franzosa et al.<sup><a href="#R95" rid="R95" class=" bibr popnode">95</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_016280, <a href="https://huttenhower.sph.harvard.edu/humann2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://huttenhower.sph.harvard.edu/humann2</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">QIIME2 v.2019.10</td><td align="left" valign="top" rowspan="1" colspan="1">Bolyen et al.<sup><a href="#R44" rid="R44" class=" bibr popnode">44</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_021258</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">MetaPhlAn2</td><td align="left" valign="top" rowspan="1" colspan="1">Truong et al.<sup><a href="#R96" rid="R96" class=" bibr popnode">96</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_004915, <a href="https://huttenhower.sph.harvard.edu/metaphlan2" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://huttenhower.sph.harvard.edu/metaphlan2</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">UniRef</td><td align="left" valign="top" rowspan="1" colspan="1">Suzek et al.<sup><a href="#R97" rid="R97" class=" bibr popnode">97</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_010646, <a href="https://www.uniprot.org/help/uniref" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.uniprot.org/help/uniref</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Mzmine version 2.51</td><td align="left" valign="top" rowspan="1" colspan="1">Pluskal et al.<sup><a href="#R98" rid="R98" class=" bibr popnode">98</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_012040, <a href="https://mzmine.github.io/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://mzmine.github.io/</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">MetaboAnalyst</td><td align="left" valign="top" rowspan="1" colspan="1">Xia et al.<sup><a href="#R99" rid="R99" class=" bibr popnode">99</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_015539, <a href="https://www.metaboanalyst.ca/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.metaboanalyst.ca/</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Cytoscape v3.7.2</td><td align="left" valign="top" rowspan="1" colspan="1">Shannon et al.<sup><a href="#R100" rid="R100" class=" bibr popnode">100</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_003032, <a href="https://cytoscape.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://cytoscape.org/</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Global Natural Products Social Molecular Networking (GNPS)</td><td align="left" valign="top" rowspan="1" colspan="1">Wang et al.<sup><a href="#R45" rid="R45" class=" bibr popnode">45</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_019012, <a href="https://gnps.ucsd.edu/ProteoSAFe/static/gnps-splash.jsp" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://gnps.ucsd.edu/ProteoSAFe/static/gnps-splash.jsp</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">UniProt</td><td align="left" valign="top" rowspan="1" colspan="1">The Uniprot Consortium<sup><a href="#R101" rid="R101" class=" bibr popnode">101</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID: SCR_002380, <a href="https://www.uniprot.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.uniprot.org/</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">ComPIL 2.0</td><td align="left" valign="top" rowspan="1" colspan="1">Park et al.<sup><a href="#R102" rid="R102" class=" bibr popnode">102</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">
<a href="https://github.com/robinparky/prolucidComPIL" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://github.com/robinparky/prolucidComPIL</a>
</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">ProLuCID/SEQUEST (1.4)</td><td align="left" valign="top" rowspan="1" colspan="1">Xu et al.<sup><a href="#R103" rid="R103" class=" bibr popnode">103</a>,<a href="#R104" rid="R104" class=" bibr popnode">104</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://fields.scripps.edu/yates" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://fields.scripps.edu/yates</a>
</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">DTASelect2 (2.1.4)</td><td align="left" valign="top" rowspan="1" colspan="1">Cociorva et al.<sup><a href="#R105" rid="R105" class=" bibr popnode">105</a></sup> &#x00026; Tabb et al.<sup><a href="#R106" rid="R106" class=" bibr popnode">106</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">
<a href="http://fields.scripps.edu/yates" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://fields.scripps.edu/yates</a>
</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">CD-HIT 4.8.1</td><td align="left" valign="top" rowspan="1" colspan="1">Fu et al.<sup><a href="#R107" rid="R107" class=" bibr popnode">107</a></sup> &#x00026; Li et al.<sup><a href="#R108" rid="R108" class=" bibr popnode">108</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_007105, <a href="http://weizhong-lab.ucsd.edu/cd-hit/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">http://weizhong-lab.ucsd.edu/cd-hit/</a>)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">DESeq2</td><td align="left" valign="top" rowspan="1" colspan="1">Love et al.<sup><a href="#R109" rid="R109" class=" bibr popnode">109</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">1.25.13, RRID:SCR_015687, <a href="https://bioconductor.org/packages/release/bioc/html/DESeq2.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://bioconductor.org/packages/release/bioc/html/DESeq2.html</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">Differential Enrichment analysis of Proteomics data DEP package</td><td align="left" valign="top" rowspan="1" colspan="1">Zhang et al.<sup><a href="#R110" rid="R110" class=" bibr popnode">110</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_023090, <a href="https://bioconductor.org/packages/release/bioc/html/DEP.html" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://bioconductor.org/packages/release/bioc/html/DEP.html</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">STRING database</td><td align="left" valign="top" rowspan="1" colspan="1">Szklarczyk et al.<sup><a href="#R111" rid="R111" class=" bibr popnode">111</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_005223, <a href="https://www.string-db.org/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.string-db.org/</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">BBDuk (BBTools)</td><td align="left" valign="top" rowspan="1" colspan="1">Bushnell et al.<sup><a href="#R112" rid="R112" class=" bibr popnode">112</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">Bestus Bioinformaticus Duk, RRID:SCR_016969, <a href="https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbduk-guide/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://jgi.doe.gov/data-and-tools/bbtools/bb-tools-user-guide/bbduk-guide/</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">HISAT2</td><td align="left" valign="top" rowspan="1" colspan="1">Kim et al.<sup><a href="#R113" rid="R113" class=" bibr popnode">113</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">version 2.1.0, RRID:SCR_015530, <a href="https://daehwankimlab.github.io/hisat2/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://daehwankimlab.github.io/hisat2/</a></td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">HTSeq</td><td align="left" valign="top" rowspan="1" colspan="1">Anders et al.<sup><a href="#R114" rid="R114" class=" bibr popnode">114</a></sup></td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_005514, <a href="https://htseq.readthedocs.io/en/release_0.9.1/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://htseq.readthedocs.io/en/release_0.9.1/</a>)</td></tr><tr><td align="left" valign="top" rowspan="1" colspan="1">GraphPad Prism v9.2.0</td><td align="left" valign="top" rowspan="1" colspan="1">GraphPad Software</td><td align="left" valign="top" rowspan="1" colspan="1">RRID:SCR_002798, <a href="https://www.graphpad.com/" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CURI" target="_blank">https://www.graphpad.com/</a></td></tr></tbody></table></div><div class="largeobj-link align_right" id="largeobj_idm140264984140368"><a target="object" rel="noopener" href="/pmc/articles/PMC11132177/table/T1/?report=objectonly">Open in a separate window</a></div></div><div class="boxed-text-box whole_rhythm hide-overflow" id="BX1"><div class="caption"><h3 id="caption-a.t.h.atitle">Highlights</h3></div><ul class="unordered" style="list-style-type:disc" id="L4"><li><div id="P115">ChAT<sup>+</sup> and TH<sup>+</sup> neurons show distinct spatial organization within the ENS of mice</div></li><li><div id="P116">ChAT<sup>+</sup> and TH<sup>+</sup> neurons differentially shape the composition of the gut microbiome</div></li><li><div id="P117">Gut-associated TH<sup>+</sup> neurons regulate fungal colonization of the gut</div></li><li><div id="P118">Gut-associated ChAT<sup>+</sup> neuron activation promotes fluid secretion and colonic motility</div></li></ul></div></div></div><div id="SM1" class="tsec sec"><a id="supplementary-material-sec"></a><h2 class="head no_bottom_margin" id="SM1title">Supplementary Material</h2><!--/article/body/sec/--><div class="sec suppmat" id="SD1"><h4>1</h4><div class="sup-box half_rhythm" id="d66e1975"><a href="/pmc/articles/PMC11132177/bin/NIHMS1988700-supplement-1.pdf" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(53M, pdf)</sup></div></div><div class="sec suppmat" id="SD2"><h4>2</h4><div class="sup-box half_rhythm" id="d66e1978"><a href="/pmc/articles/PMC11132177/bin/NIHMS1988700-supplement-2.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(15K, xlsx)</sup></div></div><div class="sec suppmat" id="SD3"><h4>3</h4><div class="sup-box half_rhythm" id="d66e1981"><a href="/pmc/articles/PMC11132177/bin/NIHMS1988700-supplement-3.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(90K, xlsx)</sup></div></div><div class="sec suppmat" id="SD4"><h4>4</h4><div class="sup-box half_rhythm" id="d66e1984"><a href="/pmc/articles/PMC11132177/bin/NIHMS1988700-supplement-4.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(14K, xlsx)</sup></div></div><div class="sec suppmat" id="SD5"><h4>5</h4><div class="sup-box half_rhythm" id="d66e1987"><a href="/pmc/articles/PMC11132177/bin/NIHMS1988700-supplement-5.xlsx" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(28K, xlsx)</sup></div></div><div class="sec suppmat" id="SD6"><h4>6</h4><div class="sup-box half_rhythm" id="d66e1990"><a href="/pmc/articles/PMC11132177/bin/NIHMS1988700-supplement-6.mp4" data-ga-action="click_feat_suppl">Click here to view.</a><sup>(8.6M, mp4)</sup></div></div></div><div id="S48" class="tsec sec"><h2 class="head no_bottom_margin" id="S48title">ACKNOWLEDGMENTS</h2><div class="sec"><p id="P77">We thank members of the Mazmanian laboratory and Dr. Jonathan Hoang for discussions throughout the research project and critical reading of the manuscript and Dr. Catherine Oikonomou for invaluable manuscript editing. We thank Dr. Andres Collazo and Caltech&#x02019;s Biological Imaging Facility for training and access to microscopy capabilities and the Caltech Proteome Exploration Laboratory for access to LC-MS/MS capabilities. We thank Prof. Elisa Hill-Yardin for providing MATLAB scripts for CMMC heatmaps. This research was funded in part by Aligning Science Across Parkinson&#x02019;s (ASAP-020495 and ASAP-000375) through the Michael J. Fox Foundation for Parkinson&#x02019;s Research (MJFF). For the purpose of open access, the authors have applied a CC BY 4.0 public copyright license to all Author Accepted Manuscripts arising from this submission. S.K.M. was supported by grants from the Heritage Medical Research Institute, Emerald Foundation, Caltech Center for Environmental and Microbial Interactions (CEMI), National Institutes of Health (GM007616 and DK078938), and Department of Defense (PD160030). P.C.D. was supported by grants from the National Institute of Diabetes and Digestive and Kidney Diseases (R01DK136117 and U24DK133658).</p></div></div><div id="fn-group-a.s.b" class="tsec sec"><h2 class="head no_bottom_margin" id="fn-group-a.s.btitle">Footnotes</h2><!--back/fn-group--><div class="fm-sec half_rhythm small"><p class="fn sec" id="FN2"><p id="P78" class="p p-first">SUPPLEMENTAL INFORMATION</p><p id="P79" class="p p-last">Supplemental information can be found online at <a href="//doi.org/10.1016/j.celrep.2024.113953" data-ga-action="click_feat_suppl" ref="reftype=extlink-doi&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CBody&amp;TO=External%7CLink%7CDOI" target="_blank">https://doi.org/10.1016/j.celrep.2024.113953</a>.</p></p><p class="fn sec" id="FN3"><p id="P80" class="p p-first">DECLARATION OF INTERESTS</p><p id="P81" class="p p-last">B.B.Y. declares financial interests in Nuanced Health, which is not related to the present study. S.K.M. declares financial interests in Axial Therapeutics and Nuanced Health, which is not related to the present study. P.C.D. is an advisor and holds equity in Cybele and Sirenas and is a scientific co-founder and advisor and holds equity to Ometa, Enveda, and Arome with prior approval by University of California, San Diego. He also consulted for DSM animal health in 2023. R.K. is a scientific advisory board member and consultant for BiomeSense, Inc.; has equity, and receives income. He is a scientific advisory board member of and has equity in GenCirq. He is a consultant and scientific advisory board member for DayTwo and receives income. He has equity in and acts as a consultant for Cybele. He is a co-founder of Biota, Inc. and has equity. He is a cofounder of Micronoma, has equity, and is a scientific advisory board member. The terms of these arrangements have been reviewed and approved by the University of California, San Diego in accordance with its conflict of interest policies.</p></p></div></div><div id="ref-list-a.s.c" class="tsec sec"><h2 class="head no_bottom_margin" id="ref-list-a.s.ctitle">REFERENCES</h2><div class="ref-list-sec sec" id="reference-list"><div class="ref-cit-blk half_rhythm" id="R1">1. <span class="mixed-citation">Furness JB (2008). <span class="ref-journal">The Enteric Nervous System</span> (Wiley; ). <a href="https://books.google.com/books?id=pvkpdNHhI6cC" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://books.google.com/books?id=pvkpdNHhI6cC</a>. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=The+Enteric+Nervous+System&amp;author=JB+Furness&amp;publication_year=2008&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R2">2. <span class="mixed-citation">Rao M, and Gershon MD (2016). <span class="ref-title">The bowel and beyond: the enteric nervous system in neurological disorders</span>. <span class="ref-journal">Nat. Rev. Gastroenterol. Hepatol</span>
<span class="ref-vol">13</span>, 517&#x02013;528. 10.1038/nrgastro.2016.107.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5005185/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27435372" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrgastro.2016.107" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Gastroenterol.+Hepatol&amp;title=The+bowel+and+beyond:+the+enteric+nervous+system+in+neurological+disorders&amp;author=M+Rao&amp;author=MD+Gershon&amp;volume=13&amp;publication_year=2016&amp;pages=517-528&amp;pmid=27435372&amp;doi=10.1038/nrgastro.2016.107&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R3">3. <span class="mixed-citation">Grundy D, and Brookes S (2011). <span class="ref-journal">Neural Control of Gastrointestinal Function</span> (Morgan &#x00026; Claypool Life Science Publishers; ). <a href="https://books.google.com/books?id=NV2BMJk6XHwC" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://books.google.com/books?id=NV2BMJk6XHwC</a>. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Neural+Control+of+Gastrointestinal+Function&amp;author=D+Grundy&amp;author=S+Brookes&amp;publication_year=2011&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R4">4. <span class="mixed-citation">Furness JB, Callaghan BP, Rivera LR, and Cho HJ (2014). <span class="ref-title">The enteric nervous system and gastrointestinal innervation: integrated local and central control</span>. <span class="ref-journal">Adv. Exp. Med. Biol</span>
<span class="ref-vol">817</span>, 39&#x02013;71. 10.1007/978-1-4939-0897-4_3.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/24997029" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2F978-1-4939-0897-4_3" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv.+Exp.+Med.+Biol&amp;title=The+enteric+nervous+system+and+gastrointestinal+innervation:+integrated+local+and+central+control&amp;author=JB+Furness&amp;author=BP+Callaghan&amp;author=LR+Rivera&amp;author=HJ+Cho&amp;volume=817&amp;publication_year=2014&amp;pages=39-71&amp;pmid=24997029&amp;doi=10.1007/978-1-4939-0897-4_3&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R5">5. <span class="mixed-citation">Schneider S, Wright CM, and Heuckeroth RO (2019). <span class="ref-title">Unexpected Roles for the Second Brain: Enteric Nervous System as Master Regulator of Bowel Function</span>. <span class="ref-journal">Annu. Rev. Physiol</span>
<span class="ref-vol">81</span>, 235&#x02013;259. 10.1146/annurev-physiol-021317-121515.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/30379617" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1146%2Fannurev-physiol-021317-121515" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Physiol&amp;title=Unexpected+Roles+for+the+Second+Brain:+Enteric+Nervous+System+as+Master+Regulator+of+Bowel+Function&amp;author=S+Schneider&amp;author=CM+Wright&amp;author=RO+Heuckeroth&amp;volume=81&amp;publication_year=2019&amp;pages=235-259&amp;pmid=30379617&amp;doi=10.1146/annurev-physiol-021317-121515&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R6">6. <span class="mixed-citation">Gabanyi I, Muller PA, Feighery L, Oliveira TY, Costa-Pinto FA, and Mucida D (2016). <span class="ref-title">Neuro-immune Interactions Drive Tissue Programming in Intestinal Macrophages</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">164</span>, 378&#x02013;391. 10.1016/j.cell.2015.12.023.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4733406/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26777404" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2015.12.023" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Neuro-immune+Interactions+Drive+Tissue+Programming+in+Intestinal+Macrophages&amp;author=I+Gabanyi&amp;author=PA+Muller&amp;author=L+Feighery&amp;author=TY+Oliveira&amp;author=FA+Costa-Pinto&amp;volume=164&amp;publication_year=2016&amp;pages=378-391&amp;pmid=26777404&amp;doi=10.1016/j.cell.2015.12.023&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R7">7. <span class="mixed-citation">Muller PA, Koscs&#x000f3; B, Rajani GM, Stevanovic K, Berres ML, Hashimoto D, Mortha A, Leboeuf M, Li XM, Mucida D, et al. (2014). <span class="ref-title">Crosstalk between muscularis macrophages and enteric neurons regulates gastrointestinal motility</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">158</span>, 300&#x02013;313. 10.1016/j.cell.2014.04.050.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4149228/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25036630" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2014.04.050" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Crosstalk+between+muscularis+macrophages+and+enteric+neurons+regulates+gastrointestinal+motility&amp;author=PA+Muller&amp;author=B+Koscs&#x000f3;&amp;author=GM+Rajani&amp;author=K+Stevanovic&amp;author=ML+Berres&amp;volume=158&amp;publication_year=2014&amp;pages=300-313&amp;pmid=25036630&amp;doi=10.1016/j.cell.2014.04.050&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R8">8. <span class="mixed-citation">Bravo JA, Forsythe P, Chew MV, Escaravage E, Savignac HM, Dinan TG, Bienenstock J, and Cryan JF (2011). <span class="ref-title">Ingestion of Lactobacillus strain regulates emotional behavior and central GABA receptor expression in a mouse via the vagus nerve</span>. <span class="ref-journal">Proc Natl Acad Sci</span>
<span class="ref-vol">108</span>, 16050&#x02013;16055. 10.1073/pnas.1102999108.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3179073/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21876150" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1102999108" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci&amp;title=Ingestion+of+Lactobacillus+strain+regulates+emotional+behavior+and+central+GABA+receptor+expression+in+a+mouse+via+the+vagus+nerve&amp;author=JA+Bravo&amp;author=P+Forsythe&amp;author=MV+Chew&amp;author=E+Escaravage&amp;author=HM+Savignac&amp;volume=108&amp;publication_year=2011&amp;pages=16050-16055&amp;pmid=21876150&amp;doi=10.1073/pnas.1102999108&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R9">9. <span class="mixed-citation">Kaelberer MM, Buchanan KL, Klein ME, Barth BB, Montoya MM, Shen X, and Boh&#x000f3;rquez DV (2018). <span class="ref-title">A gut-brain neural circuit for nutrient sensory transduction</span>. <span class="ref-journal">Science</span>
<span class="ref-vol">361</span>, eaat5236. 10.1126/science.aat5236.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6417812/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30237325" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscience.aat5236" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Science&amp;title=A+gut-brain+neural+circuit+for+nutrient+sensory+transduction&amp;author=MM+Kaelberer&amp;author=KL+Buchanan&amp;author=ME+Klein&amp;author=BB+Barth&amp;author=MM+Montoya&amp;volume=361&amp;publication_year=2018&amp;pages=eaat5236&amp;pmid=30237325&amp;doi=10.1126/science.aat5236&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R10">10. <span class="mixed-citation">Jarret A, Jackson R, Duizer C, Healy ME, Zhao J, Rone JM, Bielecki P, Sefik E, Roulis M, Rice T, et al. (2020). <span class="ref-title">Enteric Nervous System-Derived IL-18 Orchestrates Mucosal Barrier Immunity</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">180</span>, 50&#x02013;63.e12. 10.1016/j.cell.2019.12.016.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7339937/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31923399" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2019.12.016" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Enteric+Nervous+System-Derived+IL-18+Orchestrates+Mucosal+Barrier+Immunity&amp;author=A+Jarret&amp;author=R+Jackson&amp;author=C+Duizer&amp;author=ME+Healy&amp;author=J+Zhao&amp;volume=180&amp;publication_year=2020&amp;pages=50-63.e12&amp;pmid=31923399&amp;doi=10.1016/j.cell.2019.12.016&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R11">11. <span class="mixed-citation">Seillet C, Luong K, Tellier J, Jacquelot N, Shen RD, Hickey P, Wimmer VC, Whitehead L, Rogers K, Smyth GK, et al. (2020). <span class="ref-title">The neuropeptide VIP confers anticipatory mucosal immunity by regulating ILC3 activity</span>. <span class="ref-journal">Nat. Immunol</span>
<span class="ref-vol">21</span>, 168&#x02013;177. 10.1038/s41590-019-0567-y.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/31873294" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41590-019-0567-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Immunol&amp;title=The+neuropeptide+VIP+confers+anticipatory+mucosal+immunity+by+regulating+ILC3+activity&amp;author=C+Seillet&amp;author=K+Luong&amp;author=J+Tellier&amp;author=N+Jacquelot&amp;author=RD+Shen&amp;volume=21&amp;publication_year=2020&amp;pages=168-177&amp;pmid=31873294&amp;doi=10.1038/s41590-019-0567-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R12">12. <span class="mixed-citation">Talbot J, Hahn P, Kroehling L, Nguyen H, Li D, and Littman DR (2020). <span class="ref-title">Feeding-dependent VIP neuron&#x02013;ILC3 circuit regulates the intestinal barrier</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">579</span>, 575&#x02013;580. 10.1038/s41586-020-2039-9.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7135938/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32050257" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41586-020-2039-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Feeding-dependent+VIP+neuron&#x02013;ILC3+circuit+regulates+the+intestinal+barrier&amp;author=J+Talbot&amp;author=P+Hahn&amp;author=L+Kroehling&amp;author=H+Nguyen&amp;author=D+Li&amp;volume=579&amp;publication_year=2020&amp;pages=575-580&amp;pmid=32050257&amp;doi=10.1038/s41586-020-2039-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R13">13. <span class="mixed-citation">Lai NY, Musser MA, Pinho-Ribeiro FA, Baral P, Jacobson A, Ma P, Potts DE, Chen Z, Paik D, Soualhi S, et al. (2020). <span class="ref-title">Gut-Innervating Nociceptor Neurons Regulate Peyer&#x02019;s Patch Microfold Cells and SFB Levels to Mediate Salmonella Host Defense</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">180</span>, 33&#x02013;49.e22. 10.1016/j.cell.2019.11.014.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6954329/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31813624" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2019.11.014" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Gut-Innervating+Nociceptor+Neurons+Regulate+Peyer&#x02019;s+Patch+Microfold+Cells+and+SFB+Levels+to+Mediate+Salmonella+Host+Defense&amp;author=NY+Lai&amp;author=MA+Musser&amp;author=FA+Pinho-Ribeiro&amp;author=P+Baral&amp;author=A+Jacobson&amp;volume=180&amp;publication_year=2020&amp;pages=33-49.e22&amp;pmid=31813624&amp;doi=10.1016/j.cell.2019.11.014&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R14">14. <span class="mixed-citation">Matheis F, Muller PA, Graves CL, Gabanyi I, Kerner ZJ, Costa-Borges D, Ahrends T, Rosenstiel P, and Mucida D (2020). <span class="ref-title">Adrenergic Signaling in Muscularis Macrophages Limits Infection-Induced Neuronal Loss</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">180</span>, 64&#x02013;78.e16. 10.1016/j.cell.2019.12.002.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7271821/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31923400" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2019.12.002" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Adrenergic+Signaling+in+Muscularis+Macrophages+Limits+Infection-Induced+Neuronal+Loss&amp;author=F+Matheis&amp;author=PA+Muller&amp;author=CL+Graves&amp;author=I+Gabanyi&amp;author=ZJ+Kerner&amp;volume=180&amp;publication_year=2020&amp;pages=64-78.e16&amp;pmid=31923400&amp;doi=10.1016/j.cell.2019.12.002&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R15">15. <span class="mixed-citation">Nezami BG, and Srinivasan S (2010). <span class="ref-title">Enteric nervous system in the small intestine: pathophysiology and clinical implications</span>. <span class="ref-journal">Curr. Gastroenterol. Rep</span>
<span class="ref-vol">12</span>, 358&#x02013;365. 10.1007/s11894-010-0129-9.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3752592/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20725870" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs11894-010-0129-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Gastroenterol.+Rep&amp;title=Enteric+nervous+system+in+the+small+intestine:+pathophysiology+and+clinical+implications&amp;author=BG+Nezami&amp;author=S+Srinivasan&amp;volume=12&amp;publication_year=2010&amp;pages=358-365&amp;pmid=20725870&amp;doi=10.1007/s11894-010-0129-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R16">16. <span class="mixed-citation">Qu ZD, Thacker M, Castelucci P, Bagy&#x000e1;nszki M, Epstein ML, and Furness JB (2008). <span class="ref-title">Immunohistochemical analysis of neuron types in the mouse small intestine</span>. <span class="ref-journal">Cell Tissue Res</span>
<span class="ref-vol">334</span>, 147&#x02013;161. 10.1007/s00441-008-0684-7.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/18855018" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00441-008-0684-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Tissue+Res&amp;title=Immunohistochemical+analysis+of+neuron+types+in+the+mouse+small+intestine&amp;author=ZD+Qu&amp;author=M+Thacker&amp;author=P+Castelucci&amp;author=M+Bagy&#x000e1;nszki&amp;author=ML+Epstein&amp;volume=334&amp;publication_year=2008&amp;pages=147-161&amp;pmid=18855018&amp;doi=10.1007/s00441-008-0684-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R17">17. <span class="mixed-citation">Furness JB (2012). <span class="ref-title">The enteric nervous system and neurogastroenterology</span>. <span class="ref-journal">Nat. Rev. Gastroenterol. Hepatol</span>
<span class="ref-vol">9</span>, 286&#x02013;294. 10.1038/nrgastro.2012.32.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/22392290" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnrgastro.2012.32" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Gastroenterol.+Hepatol&amp;title=The+enteric+nervous+system+and+neurogastroenterology&amp;author=JB+Furness&amp;volume=9&amp;publication_year=2012&amp;pages=286-294&amp;pmid=22392290&amp;doi=10.1038/nrgastro.2012.32&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R18">18. <span class="mixed-citation">Hennig GW, Gould TW, Koh SD, Corrigan RD, Heredia DJ, Shonnard MC, and Smith TK (2015). <span class="ref-title">Use of Genetically Encoded Calcium Indicators (GECIs) Combined with Advanced Motion Tracking Techniques to Examine the Behavior of Neurons and Glia in the Enteric Nervous System of the Intact Murine Colon</span>. <span class="ref-journal">Front. Cell. Neurosci</span>
<span class="ref-vol">9</span>, 436. 10.3389/fncel.2015.00436.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4639702/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26617487" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffncel.2015.00436" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Cell.+Neurosci&amp;title=Use+of+Genetically+Encoded+Calcium+Indicators+(GECIs)+Combined+with+Advanced+Motion+Tracking+Techniques+to+Examine+the+Behavior+of+Neurons+and+Glia+in+the+Enteric+Nervous+System+of+the+Intact+Murine+Colon&amp;author=GW+Hennig&amp;author=TW+Gould&amp;author=SD+Koh&amp;author=RD+Corrigan&amp;author=DJ+Heredia&amp;volume=9&amp;publication_year=2015&amp;pages=436&amp;pmid=26617487&amp;doi=10.3389/fncel.2015.00436&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R19">19. <span class="mixed-citation">Niesler B, Kuerten S, Demir IE, and Sch&#x000e4;fer KH (2021). <span class="ref-title">Disorders of the enteric nervous system &#x02014; a holistic view</span>. <span class="ref-journal">Nat. Rev. Gastroenterol. Hepatol</span>
<span class="ref-vol">18</span>, 393&#x02013;410. 10.1038/s41575-020-00385-2.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/33514916" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41575-020-00385-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Rev.+Gastroenterol.+Hepatol&amp;title=Disorders+of+the+enteric+nervous+system+&#x02014;+a+holistic+view&amp;author=B+Niesler&amp;author=S+Kuerten&amp;author=IE+Demir&amp;author=KH+Sch&#x000e4;fer&amp;volume=18&amp;publication_year=2021&amp;pages=393-410&amp;pmid=33514916&amp;doi=10.1038/s41575-020-00385-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R20">20. <span class="mixed-citation">Lott EL, and Jones EB (2023). <span class="ref-journal">Cholinergic Toxicity. In StatPearls</span> (StatPearls Publishing; ). [<a href="https://pubmed.ncbi.nlm.nih.gov/30969605" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?title=Cholinergic+Toxicity.+In+StatPearls&amp;author=EL+Lott&amp;author=EB+Jones&amp;publication_year=2023&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R21">21. <span class="mixed-citation">Monane M, Avorn J, Beers MH, and Everitt DE (1993). <span class="ref-title">Anticholinergic drug use and bowel function in nursing home patients</span>. <span class="ref-journal">Arch. Intern. Med</span>
<span class="ref-vol">153</span>, 633&#x02013;638.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/8439226" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arch.+Intern.+Med&amp;title=Anticholinergic+drug+use+and+bowel+function+in+nursing+home+patients&amp;author=M+Monane&amp;author=J+Avorn&amp;author=MH+Beers&amp;author=DE+Everitt&amp;volume=153&amp;publication_year=1993&amp;pages=633-638&amp;pmid=8439226&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R22">22. <span class="mixed-citation">Fung C, Koussoulas K, Unterweger P, Allen AM, Bornstein JC, and Foong JPP (2018). <span class="ref-title">Cholinergic Submucosal Neurons Display Increased Excitability Following in Vivo Cholera Toxin Exposure in Mouse Ileum</span>. <span class="ref-journal">Front. Physiol</span>
<span class="ref-vol">9</span>, 260. 10.3389/fphys.2018.00260.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5871806/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29618987" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffphys.2018.00260" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Physiol&amp;title=Cholinergic+Submucosal+Neurons+Display+Increased+Excitability+Following+in+Vivo+Cholera+Toxin+Exposure+in+Mouse+Ileum&amp;author=C+Fung&amp;author=K+Koussoulas&amp;author=P+Unterweger&amp;author=AM+Allen&amp;author=JC+Bornstein&amp;volume=9&amp;publication_year=2018&amp;pages=260&amp;pmid=29618987&amp;doi=10.3389/fphys.2018.00260&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R23">23. <span class="mixed-citation">Wang H, Foong JPP, Harris NL, and Bornstein JC (2022). <span class="ref-title">Enteric neuroimmune interactions coordinate intestinal responses in health and disease</span>. <span class="ref-journal">Mucosal Immunol</span>
<span class="ref-vol">15</span>, 27&#x02013;39. 10.1038/s41385-021-00443-1.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8732275/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34471248" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41385-021-00443-1" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mucosal+Immunol&amp;title=Enteric+neuroimmune+interactions+coordinate+intestinal+responses+in+health+and+disease&amp;author=H+Wang&amp;author=JPP+Foong&amp;author=NL+Harris&amp;author=JC+Bornstein&amp;volume=15&amp;publication_year=2022&amp;pages=27-39&amp;pmid=34471248&amp;doi=10.1038/s41385-021-00443-1&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R24">24. <span class="mixed-citation">Li ZS, Schmauss C, Cuenca A, Ratcliffe E, and Gershon MD (2006). <span class="ref-title">Physiological modulation of intestinal motility by enteric dopaminergic neurons and the D2 receptor: analysis of dopamine receptor expression, location, development, and function in wild-type and knock-out mice</span>. <span class="ref-journal">J. Neurosci</span>
<span class="ref-vol">26</span>, 2798&#x02013;2807. 10.1523/JNEUROSCI.4720-05.2006.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6675162/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16525059" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1523%2FJNEUROSCI.4720-05.2006" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurosci&amp;title=Physiological+modulation+of+intestinal+motility+by+enteric+dopaminergic+neurons+and+the+D2+receptor:+analysis+of+dopamine+receptor+expression,+location,+development,+and+function+in+wild-type+and+knock-out+mice&amp;author=ZS+Li&amp;author=C+Schmauss&amp;author=A+Cuenca&amp;author=E+Ratcliffe&amp;author=MD+Gershon&amp;volume=26&amp;publication_year=2006&amp;pages=2798-2807&amp;pmid=16525059&amp;doi=10.1523/JNEUROSCI.4720-05.2006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R25">25. <span class="mixed-citation">Baumuratov AS, Antony PMA, Ostaszewski M, He F, Salamanca L, Antunes L, Weber J, Longhino L, Derkinderen P, Koopman WJH, and Diederich NJ (2016). <span class="ref-title">Enteric neurons from Parkinson&#x02019;s disease patients display ex vivo aberrations in mitochondrial structure</span>. <span class="ref-journal">Sci. Rep</span>
<span class="ref-vol">6</span>, 33117. 10.1038/srep33117.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5021970/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27624977" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fsrep33117" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep&amp;title=Enteric+neurons+from+Parkinson&#x02019;s+disease+patients+display+ex+vivo+aberrations+in+mitochondrial+structure&amp;author=AS+Baumuratov&amp;author=PMA+Antony&amp;author=M+Ostaszewski&amp;author=F+He&amp;author=L+Salamanca&amp;volume=6&amp;publication_year=2016&amp;pages=33117&amp;pmid=27624977&amp;doi=10.1038/srep33117&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R26">26. <span class="mixed-citation">McQuade RM, Singleton LM, Wu H, Lee S, Constable R, Di Natale M, Ringuet MT, Berger JP, Kauhausen J, Parish CL, et al. (2021). <span class="ref-title">The association of enteric neuropathy with gut phenotypes in acute and progressive models of Parkinson&#x02019;s disease</span>. <span class="ref-journal">Sci. Rep</span>
<span class="ref-vol">11</span>, 7934. 10.1038/s41598-021-86917-5.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8041759/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/33846426" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41598-021-86917-5" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Rep&amp;title=The+association+of+enteric+neuropathy+with+gut+phenotypes+in+acute+and+progressive+models+of+Parkinson&#x02019;s+disease&amp;author=RM+McQuade&amp;author=LM+Singleton&amp;author=H+Wu&amp;author=S+Lee&amp;author=R+Constable&amp;volume=11&amp;publication_year=2021&amp;pages=7934&amp;pmid=33846426&amp;doi=10.1038/s41598-021-86917-5&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R27">27. <span class="mixed-citation">Mittal R, Debs LH, Patel AP, Nguyen D, Patel K, O&#x02019;Connor G, Grati M, Mittal J, Yan D, Eshraghi AA, et al. (2017). <span class="ref-title">Neurotransmitters: The Critical Modulators Regulating Gut-Brain Axis</span>. <span class="ref-journal">J. Cell. Physiol</span>
<span class="ref-vol">232</span>, 2359&#x02013;2372. 10.1002/jcp.25518.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5772764/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27512962" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fjcp.25518" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Cell.+Physiol&amp;title=Neurotransmitters:+The+Critical+Modulators+Regulating+Gut-Brain+Axis&amp;author=R+Mittal&amp;author=LH+Debs&amp;author=AP+Patel&amp;author=D+Nguyen&amp;author=K+Patel&amp;volume=232&amp;publication_year=2017&amp;pages=2359-2372&amp;pmid=27512962&amp;doi=10.1002/jcp.25518&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R28">28. <span class="mixed-citation">Chan KY, Jang MJ, Yoo BB, Greenbaum A, Ravi N, Wu WL, S&#x000e1;nchez-Guardado L, Lois C, Mazmanian SK, Deverman BE, and Gradinaru V (2017). <span class="ref-title">Engineered AAVs for efficient noninvasive gene delivery to the central and peripheral nervous systems</span>. <span class="ref-journal">Nat. Neurosci</span>
<span class="ref-vol">20</span>, 1172&#x02013;1179. 10.1038/nn.4593.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5529245/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28671695" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnn.4593" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Neurosci&amp;title=Engineered+AAVs+for+efficient+noninvasive+gene+delivery+to+the+central+and+peripheral+nervous+systems&amp;author=KY+Chan&amp;author=MJ+Jang&amp;author=BB+Yoo&amp;author=A+Greenbaum&amp;author=N+Ravi&amp;volume=20&amp;publication_year=2017&amp;pages=1172-1179&amp;pmid=28671695&amp;doi=10.1038/nn.4593&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R29">29. <span class="mixed-citation">Wess J, Nakajima K, and Jain S (2013). <span class="ref-title">Novel designer receptors to probe GPCR signaling and physiology</span>. <span class="ref-journal">Trends Pharmacol. Sci</span>
<span class="ref-vol">34</span>, 385&#x02013;392. 10.1016/j.tips.2013.04.006.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3758874/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23769625" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.tips.2013.04.006" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Trends+Pharmacol.+Sci&amp;title=Novel+designer+receptors+to+probe+GPCR+signaling+and+physiology&amp;author=J+Wess&amp;author=K+Nakajima&amp;author=S+Jain&amp;volume=34&amp;publication_year=2013&amp;pages=385-392&amp;pmid=23769625&amp;doi=10.1016/j.tips.2013.04.006&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R30">30. <span class="mixed-citation">Furness JB (2006). <span class="ref-title">The organisation of the autonomic nervous system: peripheral connections</span>. <span class="ref-journal">Auton. Neurosci</span>
<span class="ref-vol">130</span>, 1&#x02013;5. 10.1016/j.autneu.2006.05.003.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16798102" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.autneu.2006.05.003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Auton.+Neurosci&amp;title=The+organisation+of+the+autonomic+nervous+system:+peripheral+connections&amp;author=JB+Furness&amp;volume=130&amp;publication_year=2006&amp;pages=1-5&amp;pmid=16798102&amp;doi=10.1016/j.autneu.2006.05.003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R31">31. <span class="mixed-citation">Hamnett R, Dershowitz LB, Sampathkumar V, Wang Z, Gomez-Frittelli J, De Andrade V, Kasthuri N, Druckmann S, and Kaltschmidt JA (2022). <span class="ref-title">Regional cytoarchitecture of the adult and developing mouse enteric nervous system</span>. <span class="ref-journal">Curr. Biol</span>
<span class="ref-vol">32</span>, 4483&#x02013;4492.e5. 10.1016/j.cub.2022.08.030.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9613618/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/36070775" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cub.2022.08.030" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr.+Biol&amp;title=Regional+cytoarchitecture+of+the+adult+and+developing+mouse+enteric+nervous+system&amp;author=R+Hamnett&amp;author=LB+Dershowitz&amp;author=V+Sampathkumar&amp;author=Z+Wang&amp;author=J+Gomez-Frittelli&amp;volume=32&amp;publication_year=2022&amp;pages=4483-4492.e5&amp;pmid=36070775&amp;doi=10.1016/j.cub.2022.08.030&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R32">32. <span class="mixed-citation">Hama H, Hioki H, Namiki K, Hoshida T, Kurokawa H, Ishidate F, Kaneko T, Akagi T, Saito T, Saido T, and Miyawaki A (2015). <span class="ref-title">ScaleS: an optical clearing palette for biological imaging</span>. <span class="ref-journal">Nat. Neurosci</span>
<span class="ref-vol">18</span>, 1518&#x02013;1529. 10.1038/nn.4107.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/26368944" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnn.4107" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Neurosci&amp;title=ScaleS:+an+optical+clearing+palette+for+biological+imaging&amp;author=H+Hama&amp;author=H+Hioki&amp;author=K+Namiki&amp;author=T+Hoshida&amp;author=H+Kurokawa&amp;volume=18&amp;publication_year=2015&amp;pages=1518-1529&amp;pmid=26368944&amp;doi=10.1038/nn.4107&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R33">33. <span class="mixed-citation">Treweek JB, Chan KY, Flytzanis NC, Yang B, Deverman BE, Greenbaum A, Lignell A, Xiao C, Cai L, Ladinsky MS, et al. (2015). <span class="ref-title">Whole-body tissue stabilization and selective extractions via tissue-hydrogel hybrids for high-resolution intact circuit mapping and phenotyping</span>. <span class="ref-journal">Nat. Protoc</span>
<span class="ref-vol">10</span>, 1860&#x02013;1896. 10.1038/nprot.2015.122.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4917295/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26492141" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnprot.2015.122" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Protoc&amp;title=Whole-body+tissue+stabilization+and+selective+extractions+via+tissue-hydrogel+hybrids+for+high-resolution+intact+circuit+mapping+and+phenotyping&amp;author=JB+Treweek&amp;author=KY+Chan&amp;author=NC+Flytzanis&amp;author=B+Yang&amp;author=BE+Deverman&amp;volume=10&amp;publication_year=2015&amp;pages=1860-1896&amp;pmid=26492141&amp;doi=10.1038/nprot.2015.122&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R34">34. <span class="mixed-citation">Yang B, Treweek JB, Kulkarni RP, Deverman BE, Chen CK, Lubeck E, Shah S, Cai L, and Gradinaru V (2014). <span class="ref-title">Single-cell phenotyping within transparent intact tissue through whole-body clearing</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">158</span>, 945&#x02013;958. 10.1016/j.cell.2014.07.017.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4153367/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25088144" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2014.07.017" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=Single-cell+phenotyping+within+transparent+intact+tissue+through+whole-body+clearing&amp;author=B+Yang&amp;author=JB+Treweek&amp;author=RP+Kulkarni&amp;author=BE+Deverman&amp;author=CK+Chen&amp;volume=158&amp;publication_year=2014&amp;pages=945-958&amp;pmid=25088144&amp;doi=10.1016/j.cell.2014.07.017&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R35">35. <span class="mixed-citation">Deverman BE, Pravdo PL, Simpson BP, Kumar SR, Chan KY, Banerjee A, Wu WL, Yang B, Huber N, Pasca SP, and Gradinaru V (2016). <span class="ref-title">Cre-dependent selection yields AAV variants for widespread gene transfer to the adult brain</span>. <span class="ref-journal">Nat. Biotechnol</span>
<span class="ref-vol">34</span>, 204&#x02013;209. 10.1038/nbt.3440.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5088052/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26829320" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnbt.3440" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol&amp;title=Cre-dependent+selection+yields+AAV+variants+for+widespread+gene+transfer+to+the+adult+brain&amp;author=BE+Deverman&amp;author=PL+Pravdo&amp;author=BP+Simpson&amp;author=SR+Kumar&amp;author=KY+Chan&amp;volume=34&amp;publication_year=2016&amp;pages=204-209&amp;pmid=26829320&amp;doi=10.1038/nbt.3440&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R36">36. <span class="mixed-citation">Haenraets K, Foster E, Johannssen H, Kandra V, Frezel N, Steffen T, Jaramillo V, Paterna JC, Zeilhofer HU, and Wildner H (2017). <span class="ref-title">Spinal nociceptive circuit analysis with recombinant adeno-associated viruses: the impact of serotypes and promoters</span>. <span class="ref-journal">J. Neurochem</span>
<span class="ref-vol">142</span>, 721&#x02013;733. 10.1111/jnc.14124.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/28700081" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fjnc.14124" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurochem&amp;title=Spinal+nociceptive+circuit+analysis+with+recombinant+adeno-associated+viruses:+the+impact+of+serotypes+and+promoters&amp;author=K+Haenraets&amp;author=E+Foster&amp;author=H+Johannssen&amp;author=V+Kandra&amp;author=N+Frezel&amp;volume=142&amp;publication_year=2017&amp;pages=721-733&amp;pmid=28700081&amp;doi=10.1111/jnc.14124&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R37">37. <span class="mixed-citation">Jakob MO, Kofoed-Branzk M, Deshpande D, Murugan S, and Klose CSN (2021). <span class="ref-title">An Integrated View on Neuronal Subsets in the Peripheral Nervous System and Their Role in Immunoregulation</span>. <span class="ref-journal">Front. Immunol</span>
<span class="ref-vol">12</span>, 679055. 10.3389/fimmu.2021.679055.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8312561/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34322118" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffimmu.2021.679055" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Immunol&amp;title=An+Integrated+View+on+Neuronal+Subsets+in+the+Peripheral+Nervous+System+and+Their+Role+in+Immunoregulation&amp;author=MO+Jakob&amp;author=M+Kofoed-Branzk&amp;author=D+Deshpande&amp;author=S+Murugan&amp;author=CSN+Klose&amp;volume=12&amp;publication_year=2021&amp;pages=679055&amp;pmid=34322118&amp;doi=10.3389/fimmu.2021.679055&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R38">38. <span class="mixed-citation">Tavares-Ferreira D, Shiers S, Ray PR, Wangzhou A, Jeevakumar V, Sankaranarayanan I, Cervantes AM, Reese JC, Chamessian A, Copits BA, et al. (2022). <span class="ref-title">Spatial transcriptomics of dorsal root ganglia identifies molecular signatures of human nociceptors</span>. <span class="ref-journal">Sci. Transl. Med</span>
<span class="ref-vol">14</span>, eabj8186. 10.1126/scitranslmed.abj8186.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC9272153/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/35171654" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1126%2Fscitranslmed.abj8186" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Sci.+Transl.+Med&amp;title=Spatial+transcriptomics+of+dorsal+root+ganglia+identifies+molecular+signatures+of+human+nociceptors&amp;author=D+Tavares-Ferreira&amp;author=S+Shiers&amp;author=PR+Ray&amp;author=A+Wangzhou&amp;author=V+Jeevakumar&amp;volume=14&amp;publication_year=2022&amp;pages=eabj8186&amp;pmid=35171654&amp;doi=10.1126/scitranslmed.abj8186&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R39">39. <span class="mixed-citation">Challis RC, Ravindra Kumar S, Chan KY, Challis C, Beadle K, Jang MJ, Kim HM, Rajendran PS, Tompkins JD, Shivkumar K, et al. (2019). <span class="ref-title">Systemic AAV vectors for widespread and targeted gene delivery in rodents</span>. <span class="ref-journal">Nat. Protoc</span>
<span class="ref-vol">14</span>, 379&#x02013;414. 10.1038/s41596-018-0097-3.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/30626963" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41596-018-0097-3" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Protoc&amp;title=Systemic+AAV+vectors+for+widespread+and+targeted+gene+delivery+in+rodents&amp;author=RC+Challis&amp;author=S+Ravindra+Kumar&amp;author=KY+Chan&amp;author=C+Challis&amp;author=K+Beadle&amp;volume=14&amp;publication_year=2019&amp;pages=379-414&amp;pmid=30626963&amp;doi=10.1038/s41596-018-0097-3&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R40">40. <span class="mixed-citation">Kaestner CL, Smith EH, Peirce SG, and Hoover DB (2019). <span class="ref-title">Immunohistochemical analysis of the mouse celiac ganglion: An integrative relay station of the peripheral nervous system</span>. <span class="ref-journal">J. Comp. Neurol</span>
<span class="ref-vol">527</span>, 2742&#x02013;2760. 10.1002/cne.24705.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6722036/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31021409" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fcne.24705" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Comp.+Neurol&amp;title=Immunohistochemical+analysis+of+the+mouse+celiac+ganglion:+An+integrative+relay+station+of+the+peripheral+nervous+system&amp;author=CL+Kaestner&amp;author=EH+Smith&amp;author=SG+Peirce&amp;author=DB+Hoover&amp;volume=527&amp;publication_year=2019&amp;pages=2742-2760&amp;pmid=31021409&amp;doi=10.1002/cne.24705&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R41">41. <span class="mixed-citation">Browning KN, and Travagli RA (2014). <span class="ref-title">Central nervous system control of gastrointestinal motility and secretion and modulation of gastrointestinal functions</span>. <span class="ref-journal">Compr. Physiol</span>
<span class="ref-vol">4</span>, 1339&#x02013;1368. 10.1002/cphy.c130055.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4858318/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25428846" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fcphy.c130055" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Compr.+Physiol&amp;title=Central+nervous+system+control+of+gastrointestinal+motility+and+secretion+and+modulation+of+gastrointestinal+functions&amp;author=KN+Browning&amp;author=RA+Travagli&amp;volume=4&amp;publication_year=2014&amp;pages=1339-1368&amp;pmid=25428846&amp;doi=10.1002/cphy.c130055&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R42">42. <span class="mixed-citation">Thompson KJ, Khajehali E, Bradley SJ, Navarrete JS, Huang XP, Slocum S, Jin J, Liu J, Xiong Y, Olsen RHJ, et al. (2018). <span class="ref-title">DREADD Agonist 21 Is an Effective Agonist for Muscarinic-Based DREADDs in Vitro and in Vivo. ACS Pharmacol</span>. <span class="ref-journal">Transl. Sci</span>
<span class="ref-vol">1</span>, 61&#x02013;72. 10.1021/acsptsci.8b00012. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6407913/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30868140" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facsptsci.8b00012" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Transl.+Sci&amp;title=DREADD+Agonist+21+Is+an+Effective+Agonist+for+Muscarinic-Based+DREADDs+in+Vitro+and+in+Vivo.+ACS+Pharmacol&amp;author=KJ+Thompson&amp;author=E+Khajehali&amp;author=SJ+Bradley&amp;author=JS+Navarrete&amp;author=XP+Huang&amp;volume=1&amp;publication_year=2018&amp;pages=61-72&amp;doi=10.1021/acsptsci.8b00012&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R43">43. <span class="mixed-citation">Segata N, Izard J, Waldron L, Gevers D, Miropolsky L, Garrett WS, and Huttenhower C (2011). <span class="ref-title">Metagenomic biomarker discovery and explanation</span>. <span class="ref-journal">Genome Biol</span>
<span class="ref-vol">12</span>, R60. 10.1186/gb-2011-12-6-r60.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3218848/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21702898" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fgb-2011-12-6-r60" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Metagenomic+biomarker+discovery+and+explanation&amp;author=N+Segata&amp;author=J+Izard&amp;author=L+Waldron&amp;author=D+Gevers&amp;author=L+Miropolsky&amp;volume=12&amp;publication_year=2011&amp;pages=R60&amp;pmid=21702898&amp;doi=10.1186/gb-2011-12-6-r60&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R44">44. <span class="mixed-citation">Bolyen E, Rideout JR, Dillon MR, Bokulich NA, Abnet CC, Al-Ghalith GA, Alexander H, Alm EJ, Arumugam M, Asnicar F, et al. (2019). <span class="ref-title">Reproducible, interactive, scalable and extensible microbiome data science using QIIME 2</span>. <span class="ref-journal">Nat. Biotechnol</span>
<span class="ref-vol">37</span>, 852&#x02013;857. 10.1038/s41587-019-0209-9.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7015180/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31341288" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41587-019-0209-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol&amp;title=Reproducible,+interactive,+scalable+and+extensible+microbiome+data+science+using+QIIME+2&amp;author=E+Bolyen&amp;author=JR+Rideout&amp;author=MR+Dillon&amp;author=NA+Bokulich&amp;author=CC+Abnet&amp;volume=37&amp;publication_year=2019&amp;pages=852-857&amp;pmid=31341288&amp;doi=10.1038/s41587-019-0209-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R45">45. <span class="mixed-citation">Wang M, Carver JJ, Phelan VV, Sanchez LM, Garg N, Peng Y, Nguyen DD, Watrous J, Kapono CA, Luzzatto-Knaan T, et al. (2016). <span class="ref-title">Sharing and community curation of mass spectrometry data with Global Natural Products Social Molecular Networking</span>. <span class="ref-journal">Nat. Biotechnol</span>
<span class="ref-vol">34</span>, 828&#x02013;837. 10.1038/nbt.3597.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5321674/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27504778" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnbt.3597" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Biotechnol&amp;title=Sharing+and+community+curation+of+mass+spectrometry+data+with+Global+Natural+Products+Social+Molecular+Networking&amp;author=M+Wang&amp;author=JJ+Carver&amp;author=VV+Phelan&amp;author=LM+Sanchez&amp;author=N+Garg&amp;volume=34&amp;publication_year=2016&amp;pages=828-837&amp;pmid=27504778&amp;doi=10.1038/nbt.3597&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R46">46. <span class="mixed-citation">Sumner LW, Amberg A, Barrett D, Beale MH, Beger R, Daykin CA, Fan TWM, Fiehn O, Goodacre R, Griffin JL, et al. (2007). <span class="ref-title">Proposed minimum reporting standards for chemical analysis Chemical Analysis Working Group (CAWG) Metabolomics Standards Initiative (MSI)</span>. <span class="ref-journal">Metabolomics</span>
<span class="ref-vol">3</span>, 211&#x02013;221. 10.1007/s11306-007-0082-2.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3772505/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24039616" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs11306-007-0082-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Metabolomics&amp;title=Proposed+minimum+reporting+standards+for+chemical+analysis+Chemical+Analysis+Working+Group+(CAWG)+Metabolomics+Standards+Initiative+(MSI)&amp;author=LW+Sumner&amp;author=A+Amberg&amp;author=D+Barrett&amp;author=MH+Beale&amp;author=R+Beger&amp;volume=3&amp;publication_year=2007&amp;pages=211-221&amp;pmid=24039616&amp;doi=10.1007/s11306-007-0082-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R47">47. <span class="mixed-citation">Aries V, Crowther JS, Drasar BS, and Hill MJ (1969). <span class="ref-title">Degradation of bile salts by human intestinal bacteria</span>. <span class="ref-journal">Gut</span>
<span class="ref-vol">10</span>, 575&#x02013;576. 10.1136/gut.10.7.575.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1552963/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/5806939" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1136%2Fgut.10.7.575" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Degradation+of+bile+salts+by+human+intestinal+bacteria&amp;author=V+Aries&amp;author=JS+Crowther&amp;author=BS+Drasar&amp;author=MJ+Hill&amp;volume=10&amp;publication_year=1969&amp;pages=575-576&amp;pmid=5806939&amp;doi=10.1136/gut.10.7.575&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R48">48. <span class="mixed-citation">Sakai K, Makino T, Kawai Y, and Mutai M (1980). <span class="ref-title">Intestinal microflora and bile acids. Effect of bile acids on the distribution of microflora and bile acid in the digestive tract of the rat</span>. <span class="ref-journal">Microbiol. Immunol</span>
<span class="ref-vol">24</span>, 187&#x02013;196. 10.1111/j.1348-0421.1980.tb00578.x.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/6447830" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1348-0421.1980.tb00578.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Microbiol.+Immunol&amp;title=Intestinal+microflora+and+bile+acids.+Effect+of+bile+acids+on+the+distribution+of+microflora+and+bile+acid+in+the+digestive+tract+of+the+rat&amp;author=K+Sakai&amp;author=T+Makino&amp;author=Y+Kawai&amp;author=M+Mutai&amp;volume=24&amp;publication_year=1980&amp;pages=187-196&amp;pmid=6447830&amp;doi=10.1111/j.1348-0421.1980.tb00578.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R49">49. <span class="mixed-citation">Albenberg LG, and Wu GD (2014). <span class="ref-title">Diet and the intestinal microbiome: associations, functions, and implications for health and disease</span>. <span class="ref-journal">Gastroenterology</span>
<span class="ref-vol">146</span>, 1564&#x02013;1572. 10.1053/j.gastro.2014.01.058.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4216184/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/24503132" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1053%2Fj.gastro.2014.01.058" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gastroenterology&amp;title=Diet+and+the+intestinal+microbiome:+associations,+functions,+and+implications+for+health+and+disease&amp;author=LG+Albenberg&amp;author=GD+Wu&amp;volume=146&amp;publication_year=2014&amp;pages=1564-1572&amp;pmid=24503132&amp;doi=10.1053/j.gastro.2014.01.058&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R50">50. <span class="mixed-citation">Jia L, Betters JL, and Yu L (2011). <span class="ref-title">Niemann-pick C1-like 1 (NPC1L1) protein in intestinal and hepatic cholesterol transport</span>. <span class="ref-journal">Annu. Rev. Physiol</span>
<span class="ref-vol">73</span>, 239&#x02013;259. 10.1146/annurev-physiol-012110-142233.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3965667/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20809793" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1146%2Fannurev-physiol-012110-142233" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Annu.+Rev.+Physiol&amp;title=Niemann-pick+C1-like+1+(NPC1L1)+protein+in+intestinal+and+hepatic+cholesterol+transport&amp;author=L+Jia&amp;author=JL+Betters&amp;author=L+Yu&amp;volume=73&amp;publication_year=2011&amp;pages=239-259&amp;pmid=20809793&amp;doi=10.1146/annurev-physiol-012110-142233&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R51">51. <span class="mixed-citation">Rodr&#x000ed;guez-Pi&#x000f1;eiro AM, Bergstr&#x000f6;m JH, Ermund A, Gustafsson JK, Sch&#x000fc;tte A, Johansson MEV, and Hansson GC (2013). <span class="ref-title">Studies of mucus in mouse stomach, small intestine, and colon. II. Gastrointestinal mucus proteome reveals Muc2 and Muc5ac accompanied by a set of core proteins</span>. <span class="ref-journal">Am. J. Physiol. Gastrointest. Liver Physiol</span>
<span class="ref-vol">305</span>, G348&#x02013;G356. 10.1152/ajpgi.00047.2013.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3761249/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23832517" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1152%2Fajpgi.00047.2013" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Physiol.+Gastrointest.+Liver+Physiol&amp;title=Studies+of+mucus+in+mouse+stomach,+small+intestine,+and+colon.+II.+Gastrointestinal+mucus+proteome+reveals+Muc2+and+Muc5ac+accompanied+by+a+set+of+core+proteins&amp;author=AM+Rodr&#x000ed;guez-Pi&#x000f1;eiro&amp;author=JH+Bergstr&#x000f6;m&amp;author=A+Ermund&amp;author=JK+Gustafsson&amp;author=A+Sch&#x000fc;tte&amp;volume=305&amp;publication_year=2013&amp;pages=G348-G356&amp;pmid=23832517&amp;doi=10.1152/ajpgi.00047.2013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R52">52. <span class="mixed-citation">Matsushima S, Hori S, and Matsuda M (1986). <span class="ref-title">Conversion of 4-aminobutyraldehyde to gamma-aminobutyric acid in striatum treated with semicarbazide and kainic acid</span>. <span class="ref-journal">Neurochem. Res</span>
<span class="ref-vol">11</span>, 1313&#x02013;1319. 10.1007/BF00966125.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/3785548" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2FBF00966125" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neurochem.+Res&amp;title=Conversion+of+4-aminobutyraldehyde+to+gamma-aminobutyric+acid+in+striatum+treated+with+semicarbazide+and+kainic+acid&amp;author=S+Matsushima&amp;author=S+Hori&amp;author=M+Matsuda&amp;volume=11&amp;publication_year=1986&amp;pages=1313-1319&amp;pmid=3785548&amp;doi=10.1007/BF00966125&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R53">53. <span class="mixed-citation">Li W, Yu G, Liu Y, and Sha L (2019). <span class="ref-title">Intrapancreatic Ganglia and Neural Regulation of Pancreatic Endocrine Secretion</span>. <span class="ref-journal">Front. Neurosci</span>
<span class="ref-vol">13</span>, 21. 10.3389/fnins.2019.00021.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6391893/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30842720" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffnins.2019.00021" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Neurosci&amp;title=Intrapancreatic+Ganglia+and+Neural+Regulation+of+Pancreatic+Endocrine+Secretion&amp;author=W+Li&amp;author=G+Yu&amp;author=Y+Liu&amp;author=L+Sha&amp;volume=13&amp;publication_year=2019&amp;pages=21&amp;pmid=30842720&amp;doi=10.3389/fnins.2019.00021&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R54">54. <span class="mixed-citation">Donowitz M, Singh S, Salahuddin FF, Hogema BM, Chen Y, Gucek M, Cole RN, Ham A, Zachos NC, Kovbasnjuk O, et al. (2007). <span class="ref-title">Proteome of murine jejunal brush border membrane vesicles</span>. <span class="ref-journal">J. Proteome Res</span>
<span class="ref-vol">6</span>, 4068&#x02013;4079. 10.1021/pr0701761.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17845021" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fpr0701761" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res&amp;title=Proteome+of+murine+jejunal+brush+border+membrane+vesicles&amp;author=M+Donowitz&amp;author=S+Singh&amp;author=FF+Salahuddin&amp;author=BM+Hogema&amp;author=Y+Chen&amp;volume=6&amp;publication_year=2007&amp;pages=4068-4079&amp;pmid=17845021&amp;doi=10.1021/pr0701761&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R55">55. <span class="mixed-citation">McConnell RE, Benesh AE, Mao S, Tabb DL, and Tyska MJ (2011). <span class="ref-title">Proteomic analysis of the enterocyte brush border</span>. <span class="ref-journal">Am. J. Physiol. Gastrointest. Liver Physiol</span>
<span class="ref-vol">300</span>, G914&#x02013;G926. 10.1152/ajpgi.00005.2011.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3094140/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21330445" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1152%2Fajpgi.00005.2011" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Am.+J.+Physiol.+Gastrointest.+Liver+Physiol&amp;title=Proteomic+analysis+of+the+enterocyte+brush+border&amp;author=RE+McConnell&amp;author=AE+Benesh&amp;author=S+Mao&amp;author=DL+Tabb&amp;author=MJ+Tyska&amp;volume=300&amp;publication_year=2011&amp;pages=G914-G926&amp;pmid=21330445&amp;doi=10.1152/ajpgi.00005.2011&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R56">56. <span class="mixed-citation">Latg&#x000e9; JP (2007). <span class="ref-title">The cell wall: a carbohydrate armour for the fungal cell</span>. <span class="ref-journal">Mol. Microbiol</span>
<span class="ref-vol">66</span>, 279&#x02013;290. 10.1111/j.1365-2958.2007.05872.x.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17854405" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1111%2Fj.1365-2958.2007.05872.x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Microbiol&amp;title=The+cell+wall:+a+carbohydrate+armour+for+the+fungal+cell&amp;author=JP+Latg&#x000e9;&amp;volume=66&amp;publication_year=2007&amp;pages=279-290&amp;pmid=17854405&amp;doi=10.1111/j.1365-2958.2007.05872.x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R57">57. <span class="mixed-citation">Wu YE, Pan L, Zuo Y, Li X, and Hong W (2017). <span class="ref-title">Detecting Activated Cell Populations Using Single-Cell RNA-Seq</span>. <span class="ref-journal">Neuron</span>
<span class="ref-vol">96</span>, 313&#x02013;329.e6. 10.1016/j.neuron.2017.09.026.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/29024657" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.neuron.2017.09.026" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Neuron&amp;title=Detecting+Activated+Cell+Populations+Using+Single-Cell+RNA-Seq&amp;author=YE+Wu&amp;author=L+Pan&amp;author=Y+Zuo&amp;author=X+Li&amp;author=W+Hong&amp;volume=96&amp;publication_year=2017&amp;pages=313-329.e6&amp;pmid=29024657&amp;doi=10.1016/j.neuron.2017.09.026&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R58">58. <span class="mixed-citation">Bahrami S, and Drabl&#x000f8;s F (2016). <span class="ref-title">Gene regulation in the immediate-early response process</span>. <span class="ref-journal">Adv. Biol. Regul</span>
<span class="ref-vol">62</span>, 37&#x02013;49. 10.1016/j.jbior.2016.05.001.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/27220739" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.jbior.2016.05.001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Adv.+Biol.+Regul&amp;title=Gene+regulation+in+the+immediate-early+response+process&amp;author=S+Bahrami&amp;author=F+Drabl&#x000f8;s&amp;volume=62&amp;publication_year=2016&amp;pages=37-49&amp;pmid=27220739&amp;doi=10.1016/j.jbior.2016.05.001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R59">59. <span class="mixed-citation">Ramirez-Carrozzi VR, Braas D, Bhatt DM, Cheng CS, Hong C, Doty KR, Black JC, Hoffmann A, Carey M, and Smale ST (2009). <span class="ref-title">A unifying model for the selective regulation of inducible transcription by CpG islands and nucleosome remodeling</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">138</span>, 114&#x02013;128. 10.1016/j.cell.2009.04.020.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2712736/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19596239" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2009.04.020" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=A+unifying+model+for+the+selective+regulation+of+inducible+transcription+by+CpG+islands+and+nucleosome+remodeling&amp;author=VR+Ramirez-Carrozzi&amp;author=D+Braas&amp;author=DM+Bhatt&amp;author=CS+Cheng&amp;author=C+Hong&amp;volume=138&amp;publication_year=2009&amp;pages=114-128&amp;pmid=19596239&amp;doi=10.1016/j.cell.2009.04.020&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R60">60. <span class="mixed-citation">Miano JM, Vlasic N, Tota RR, and Stemerman MB (1993). <span class="ref-title">Smooth muscle cell immediate-early gene and growth factor activation follows vascular injury. A putative in vivo mechanism for autocrine growth</span>. <span class="ref-journal">Arterioscler. Thromb</span>
<span class="ref-vol">13</span>, 211&#x02013;219. 10.1161/01.atv.13.2.211.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/8427857" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1161%2F01.atv.13.2.211" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Arterioscler.+Thromb&amp;title=Smooth+muscle+cell+immediate-early+gene+and+growth+factor+activation+follows+vascular+injury.+A+putative+in+vivo+mechanism+for+autocrine+growth&amp;author=JM+Miano&amp;author=N+Vlasic&amp;author=RR+Tota&amp;author=MB+Stemerman&amp;volume=13&amp;publication_year=1993&amp;pages=211-219&amp;pmid=8427857&amp;doi=10.1161/01.atv.13.2.211&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R61">61. <span class="mixed-citation">Flandez M, Guilmeau S, Blache P, and Augenlicht LH (2008). <span class="ref-title">KLF4 regulation in intestinal epithelial cell maturation</span>. <span class="ref-journal">Exp. Cell Res</span>
<span class="ref-vol">314</span>, 3712&#x02013;3723. 10.1016/j.yexcr.2008.10.004.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2652355/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18977346" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.yexcr.2008.10.004" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Exp.+Cell+Res&amp;title=KLF4+regulation+in+intestinal+epithelial+cell+maturation&amp;author=M+Flandez&amp;author=S+Guilmeau&amp;author=P+Blache&amp;author=LH+Augenlicht&amp;volume=314&amp;publication_year=2008&amp;pages=3712-3723&amp;pmid=18977346&amp;doi=10.1016/j.yexcr.2008.10.004&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R62">62. <span class="mixed-citation">Johnson CD, Barlow-Anacker AJ, Pierre JF, Touw K, Erickson CS, Furness JB, Epstein ML, and Gosain A (2018). <span class="ref-title">Deletion of choline acetyltransferase in enteric neurons results in postnatal intestinal dysmotility and dysbiosis</span>. <span class="ref-journal">FASEB J Off Publ Fed Am Soc Exp Biol</span>
<span class="ref-vol">32</span>, 4744&#x02013;4752. 10.1096/fj.201701474RR. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6103169/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29570391" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1096%2Ffj.201701474RR" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FASEB+J+Off+Publ+Fed+Am+Soc+Exp+Biol&amp;title=Deletion+of+choline+acetyltransferase+in+enteric+neurons+results+in+postnatal+intestinal+dysmotility+and+dysbiosis&amp;author=CD+Johnson&amp;author=AJ+Barlow-Anacker&amp;author=JF+Pierre&amp;author=K+Touw&amp;author=CS+Erickson&amp;volume=32&amp;publication_year=2018&amp;pages=4744-4752&amp;doi=10.1096/fj.201701474RR&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R63">63. <span class="mixed-citation">Everard A, Belzer C, Geurts L, Ouwerkerk JP, Druart C, Bindels LB, Guiot Y, Derrien M, Muccioli GG, Delzenne NM, et al. (2013). <span class="ref-title">Cross-talk between Akkermansia muciniphila and intestinal epithelium controls diet-induced obesity</span>. <span class="ref-journal">Proc Natl Acad Sci</span>
<span class="ref-vol">110</span>, 9066&#x02013;9071. 10.1073/pnas.1219451110.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3670398/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23671105" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1219451110" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci&amp;title=Cross-talk+between+Akkermansia+muciniphila+and+intestinal+epithelium+controls+diet-induced+obesity&amp;author=A+Everard&amp;author=C+Belzer&amp;author=L+Geurts&amp;author=JP+Ouwerkerk&amp;author=C+Druart&amp;volume=110&amp;publication_year=2013&amp;pages=9066-9071&amp;pmid=23671105&amp;doi=10.1073/pnas.1219451110&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R64">64. <span class="mixed-citation">Plovier H, Everard A, Druart C, Depommier C, Van Hul M, Geurts L, Chilloux J, Ottman N, Duparc T, Lichtenstein L, et al. (2017). <span class="ref-title">A purified membrane protein from Akkermansia muciniphila or the pasteurized bacterium improves metabolism in obese and diabetic mice</span>. <span class="ref-journal">Nat. Med</span>
<span class="ref-vol">23</span>, 107&#x02013;113. 10.1038/nm.4236.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/27892954" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnm.4236" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Med&amp;title=A+purified+membrane+protein+from+Akkermansia+muciniphila+or+the+pasteurized+bacterium+improves+metabolism+in+obese+and+diabetic+mice&amp;author=H+Plovier&amp;author=A+Everard&amp;author=C+Druart&amp;author=C+Depommier&amp;author=M+Van+Hul&amp;volume=23&amp;publication_year=2017&amp;pages=107-113&amp;pmid=27892954&amp;doi=10.1038/nm.4236&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R65">65. <span class="mixed-citation">Cekanaviciute E, Yoo BB, Runia TF, Debelius JW, Singh S, Nelson CA, Kanner R, Bencosme Y, Lee YK, Hauser SL, et al. (2017). <span class="ref-title">Gut bacteria from multiple sclerosis patients modulate human T cells and exacerbate symptoms in mouse models</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. USA</span>
<span class="ref-vol">114</span>, 10713&#x02013;10718. 10.1073/pnas.1711235114.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5635915/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28893978" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.1711235114" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=Gut+bacteria+from+multiple+sclerosis+patients+modulate+human+T+cells+and+exacerbate+symptoms+in+mouse+models&amp;author=E+Cekanaviciute&amp;author=BB+Yoo&amp;author=TF+Runia&amp;author=JW+Debelius&amp;author=S+Singh&amp;volume=114&amp;publication_year=2017&amp;pages=10713-10718&amp;pmid=28893978&amp;doi=10.1073/pnas.1711235114&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R66">66. <span class="mixed-citation">Jangi S, Gandhi R, Cox LM, Li N, von Glehn F, Yan R, Patel B, Mazzola MA, Liu S, Glanz BL, et al. (2016). <span class="ref-title">Alterations of the human gut microbiome in multiple sclerosis</span>. <span class="ref-journal">Nat. Commun</span>
<span class="ref-vol">7</span>, 12015. 10.1038/ncomms12015.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4931233/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27352007" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fncomms12015" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun&amp;title=Alterations+of+the+human+gut+microbiome+in+multiple+sclerosis&amp;author=S+Jangi&amp;author=R+Gandhi&amp;author=LM+Cox&amp;author=N+Li&amp;author=F+von+Glehn&amp;volume=7&amp;publication_year=2016&amp;pages=12015&amp;pmid=27352007&amp;doi=10.1038/ncomms12015&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R67">67. <span class="mixed-citation">Olson CA, Vuong HE, Yano JM, Liang QY, Nusbaum DJ, and Hsiao EY (2018). <span class="ref-title">The Gut Microbiota Mediates the Anti-Seizure Effects of the Ketogenic Diet</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">173</span>, 1728&#x02013;1741.e13. 10.1016/j.cell.2018.04.027.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6003870/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29804833" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2018.04.027" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+Gut+Microbiota+Mediates+the+Anti-Seizure+Effects+of+the+Ketogenic+Diet&amp;author=CA+Olson&amp;author=HE+Vuong&amp;author=JM+Yano&amp;author=QY+Liang&amp;author=DJ+Nusbaum&amp;volume=173&amp;publication_year=2018&amp;pages=1728-1741.e13&amp;pmid=29804833&amp;doi=10.1016/j.cell.2018.04.027&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R68">68. <span class="mixed-citation">Derrien M, Vaughan EE, Plugge CM, and de Vos WM (2004). <span class="ref-title">Akkermansia muciniphila gen.nov., sp. nov., a human intestinal mucin-degrading bacterium</span>. <span class="ref-journal">Int. J. Syst. Evol. Microbiol</span>
<span class="ref-vol">54</span>, 1469&#x02013;1476. 10.1099/ijs.0.02873-0.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15388697" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1099%2Fijs.0.02873-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Int.+J.+Syst.+Evol.+Microbiol&amp;title=Akkermansia+muciniphila+gen.nov.,+sp.+nov.,+a+human+intestinal+mucin-degrading+bacterium&amp;author=M+Derrien&amp;author=EE+Vaughan&amp;author=CM+Plugge&amp;author=WM+de+Vos&amp;volume=54&amp;publication_year=2004&amp;pages=1469-1476&amp;pmid=15388697&amp;doi=10.1099/ijs.0.02873-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R69">69. <span class="mixed-citation">de Aguiar Vallim TQ, Tarling EJ, and Edwards PA (2013). <span class="ref-title">Pleiotropic roles of bile acids in metabolism</span>. <span class="ref-journal">Cell Metabol</span>
<span class="ref-vol">17</span>, 657&#x02013;669. 10.1016/j.cmet.2013.03.013. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3654004/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23602448" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cmet.2013.03.013" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell+Metabol&amp;title=Pleiotropic+roles+of+bile+acids+in+metabolism&amp;author=TQ+de+Aguiar+Vallim&amp;author=EJ+Tarling&amp;author=PA+Edwards&amp;volume=17&amp;publication_year=2013&amp;pages=657-669&amp;doi=10.1016/j.cmet.2013.03.013&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R70">70. <span class="mixed-citation">Kirwan WO, Smith AN, Mitchell WD, Falconer JD, and Eastwood MA (1975). <span class="ref-title">Bile acids and colonic motility in the rabbit and the human</span>. <span class="ref-journal">Gut</span>
<span class="ref-vol">16</span>, 894&#x02013;902. 10.1136/gut.16.11.894.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1413128/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/1193418" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1136%2Fgut.16.11.894" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gut&amp;title=Bile+acids+and+colonic+motility+in+the+rabbit+and+the+human&amp;author=WO+Kirwan&amp;author=AN+Smith&amp;author=WD+Mitchell&amp;author=JD+Falconer&amp;author=MA+Eastwood&amp;volume=16&amp;publication_year=1975&amp;pages=894-902&amp;pmid=1193418&amp;doi=10.1136/gut.16.11.894&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R71">71. <span class="mixed-citation">Watanabe M, Houten SM, Mataki C, Christoffolete MA, Kim BW, Sato H, Messaddeq N, Harney JW, Ezaki O, Kodama T, et al. (2006). <span class="ref-title">Bile acids induce energy expenditure by promoting intracellular thyroid hormone activation</span>. <span class="ref-journal">Nature</span>
<span class="ref-vol">439</span>, 484&#x02013;489. 10.1038/nature04330.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/16400329" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnature04330" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nature&amp;title=Bile+acids+induce+energy+expenditure+by+promoting+intracellular+thyroid+hormone+activation&amp;author=M+Watanabe&amp;author=SM+Houten&amp;author=C+Mataki&amp;author=MA+Christoffolete&amp;author=BW+Kim&amp;volume=439&amp;publication_year=2006&amp;pages=484-489&amp;pmid=16400329&amp;doi=10.1038/nature04330&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R72">72. <span class="mixed-citation">Fiorucci S, Biagioli M, Zampella A, and Distrutti E (2018). <span class="ref-title">Bile Acids Activated Receptors Regulate Innate Immunity</span>. <span class="ref-journal">Front. Immunol</span>
<span class="ref-vol">9</span>, 1853. 10.3389/fimmu.2018.01853.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6099188/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30150987" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffimmu.2018.01853" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Immunol&amp;title=Bile+Acids+Activated+Receptors+Regulate+Innate+Immunity&amp;author=S+Fiorucci&amp;author=M+Biagioli&amp;author=A+Zampella&amp;author=E+Distrutti&amp;volume=9&amp;publication_year=2018&amp;pages=1853&amp;pmid=30150987&amp;doi=10.3389/fimmu.2018.01853&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R73">73. <span class="mixed-citation">McMillin M, and DeMorrow S (2016). <span class="ref-title">Effects of bile acids on neurological function and disease</span>. <span class="ref-journal">FASEB J Off Publ Fed Am Soc Exp Biol</span>
<span class="ref-vol">30</span>, 3658&#x02013;3668. 10.1096/fj.201600275R. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5067249/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/27468758" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1096%2Ffj.201600275R" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=FASEB+J+Off+Publ+Fed+Am+Soc+Exp+Biol&amp;title=Effects+of+bile+acids+on+neurological+function+and+disease&amp;author=M+McMillin&amp;author=S+DeMorrow&amp;volume=30&amp;publication_year=2016&amp;pages=3658-3668&amp;doi=10.1096/fj.201600275R&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R74">74. <span class="mixed-citation">Begley M, Sleator RD, Gahan CGM, and Hill C (2005). <span class="ref-title">Contribution of three bile-associated loci, bsh, pva, and btlB, to gastrointestinal persistence and bile tolerance of Listeria monocytogenes</span>. <span class="ref-journal">Infect. Immun</span>
<span class="ref-vol">73</span>, 894&#x02013;904. 10.1128/IAI.73.2.894-904.2005.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC546953/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/15664931" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FIAI.73.2.894-904.2005" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect.+Immun&amp;title=Contribution+of+three+bile-associated+loci,+bsh,+pva,+and+btlB,+to+gastrointestinal+persistence+and+bile+tolerance+of+Listeria+monocytogenes&amp;author=M+Begley&amp;author=RD+Sleator&amp;author=CGM+Gahan&amp;author=C+Hill&amp;volume=73&amp;publication_year=2005&amp;pages=894-904&amp;pmid=15664931&amp;doi=10.1128/IAI.73.2.894-904.2005&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R75">75. <span class="mixed-citation">Delpino MV, Marchesini MI, Estein SM, Comerci DJ, Cassataro J, Fossati CA, and Baldi PC (2007). <span class="ref-title">A bile salt hydrolase of Brucella abortus contributes to the establishment of a successful infection through the oral route in mice</span>. <span class="ref-journal">Infect. Immun</span>
<span class="ref-vol">75</span>, 299&#x02013;305. 10.1128/IAI.00952-06.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1828384/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/17088355" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2FIAI.00952-06" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Infect.+Immun&amp;title=A+bile+salt+hydrolase+of+Brucella+abortus+contributes+to+the+establishment+of+a+successful+infection+through+the+oral+route+in+mice&amp;author=MV+Delpino&amp;author=MI+Marchesini&amp;author=SM+Estein&amp;author=DJ+Comerci&amp;author=J+Cassataro&amp;volume=75&amp;publication_year=2007&amp;pages=299-305&amp;pmid=17088355&amp;doi=10.1128/IAI.00952-06&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R76">76. <span class="mixed-citation">Hofmann AF, and Eckmann L (2006). <span class="ref-title">How bile acids confer gut mucosal protection against bacteria</span>. <span class="ref-journal">Proc. Natl. Acad. Sci. USA</span>
<span class="ref-vol">103</span>, 4333&#x02013;4334. 10.1073/pnas.0600780103.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC1450168/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/16537368" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.0600780103" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc.+Natl.+Acad.+Sci.+USA&amp;title=How+bile+acids+confer+gut+mucosal+protection+against+bacteria&amp;author=AF+Hofmann&amp;author=L+Eckmann&amp;volume=103&amp;publication_year=2006&amp;pages=4333-4334&amp;pmid=16537368&amp;doi=10.1073/pnas.0600780103&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R77">77. <span class="mixed-citation">Jones BV, Begley M, Hill C, Gahan CGM, and Marchesi JR (2008). <span class="ref-title">Functional and comparative metagenomic analysis of bile salt hydrolase activity in the human gut microbiome</span>. <span class="ref-journal">Proc Natl Acad Sci</span>
<span class="ref-vol">105</span>, 13580&#x02013;13585. 10.1073/pnas.0804437105.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2533232/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18757757" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1073%2Fpnas.0804437105" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proc+Natl+Acad+Sci&amp;title=Functional+and+comparative+metagenomic+analysis+of+bile+salt+hydrolase+activity+in+the+human+gut+microbiome&amp;author=BV+Jones&amp;author=M+Begley&amp;author=C+Hill&amp;author=CGM+Gahan&amp;author=JR+Marchesi&amp;volume=105&amp;publication_year=2008&amp;pages=13580-13585&amp;pmid=18757757&amp;doi=10.1073/pnas.0804437105&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R78">78. <span class="mixed-citation">Sannasiddappa TH, Lund PA, and Clarke SR (2017). <span class="ref-title">In Vitro Antibacterial Activity of Unconjugated and Conjugated Bile Salts on Staphylococcus aureus</span>. <span class="ref-journal">Front. Microbiol</span>
<span class="ref-vol">8</span>, 1581. 10.3389/fmicb.2017.01581.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5572772/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28878747" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffmicb.2017.01581" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Microbiol&amp;title=In+Vitro+Antibacterial+Activity+of+Unconjugated+and+Conjugated+Bile+Salts+on+Staphylococcus+aureus&amp;author=TH+Sannasiddappa&amp;author=PA+Lund&amp;author=SR+Clarke&amp;volume=8&amp;publication_year=2017&amp;pages=1581&amp;pmid=28878747&amp;doi=10.3389/fmicb.2017.01581&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R79">79. <span class="mixed-citation">Drokhlyansky E, Smillie CS, Van Wittenberghe N, Ericsson M, Griffin GK, Eraslan G, Dionne D, Cuoco MS, Goder-Reiser MN, Sharova T, et al. (2020). <span class="ref-title">The Human and Mouse Enteric Nervous System at Single-Cell Resolution</span>. <span class="ref-journal">Cell</span>
<span class="ref-vol">182</span>, 1606&#x02013;1622.e23. 10.1016/j.cell.2020.08.003.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8358727/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32888429" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.cell.2020.08.003" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Cell&amp;title=The+Human+and+Mouse+Enteric+Nervous+System+at+Single-Cell+Resolution&amp;author=E+Drokhlyansky&amp;author=CS+Smillie&amp;author=N+Van+Wittenberghe&amp;author=M+Ericsson&amp;author=GK+Griffin&amp;volume=182&amp;publication_year=2020&amp;pages=1606-1622.e23&amp;pmid=32888429&amp;doi=10.1016/j.cell.2020.08.003&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R80">80. <span class="mixed-citation">Gershon MD (1999). <span class="ref-title">The Enteric Nervous System: A Second Brain</span>. <span class="ref-journal">Hospital Practice</span>
<span class="ref-vol">34</span>, 31&#x02013;52. [<a href="https://pubmed.ncbi.nlm.nih.gov/10418549" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Hospital+Practice&amp;title=The+Enteric+Nervous+System:+A+Second+Brain&amp;author=MD+Gershon&amp;volume=34&amp;publication_year=1999&amp;pages=31-52&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R81">81. <span class="mixed-citation">Bhavsar AS, Verma S, Lamba R, Lall CG, Koenigsknecht V, and Rajesh A (2013). <span class="ref-title">Abdominal manifestations of neurologic disorders</span>. <span class="ref-journal">Radiographics</span>
<span class="ref-vol">33</span>, 135&#x02013;153. 10.1148/rg.331125097.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/23322834" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1148%2Frg.331125097" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Radiographics&amp;title=Abdominal+manifestations+of+neurologic+disorders&amp;author=AS+Bhavsar&amp;author=S+Verma&amp;author=R+Lamba&amp;author=CG+Lall&amp;author=V+Koenigsknecht&amp;volume=33&amp;publication_year=2013&amp;pages=135-153&amp;pmid=23322834&amp;doi=10.1148/rg.331125097&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R82">82. <span class="mixed-citation">Cersosimo MG, Raina GB, Pecci C, Pellene A, Calandra CR, Guti&#x000e9;rrez C, Micheli FE, and Benarroch EE (2013). <span class="ref-title">Gastrointestinal manifestations in Parkinson&#x02019;s disease: prevalence and occurrence before motor symptoms</span>. <span class="ref-journal">J. Neurol</span>
<span class="ref-vol">260</span>, 1332&#x02013;1338. 10.1007/s00415-012-6801-2.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/23263478" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1007%2Fs00415-012-6801-2" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Neurol&amp;title=Gastrointestinal+manifestations+in+Parkinson&#x02019;s+disease:+prevalence+and+occurrence+before+motor+symptoms&amp;author=MG+Cersosimo&amp;author=GB+Raina&amp;author=C+Pecci&amp;author=A+Pellene&amp;author=CR+Calandra&amp;volume=260&amp;publication_year=2013&amp;pages=1332-1338&amp;pmid=23263478&amp;doi=10.1007/s00415-012-6801-2&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R83">83. <span class="mixed-citation">Del Giudice E, Staiano A, Capano G, Romano A, Florimonte L, Miele E, Ciarla C, Campanozzi A, and Crisanti AF (1999). <span class="ref-title">Gastrointestinal manifestations in children with cerebral palsy</span>. <span class="ref-journal">Brain Dev</span>
<span class="ref-vol">21</span>, 307&#x02013;311. 10.1016/s0387-7604(99)00025-x.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/10413017" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fs0387-7604(99)00025-x" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Brain+Dev&amp;title=Gastrointestinal+manifestations+in+children+with+cerebral+palsy&amp;author=E+Del+Giudice&amp;author=A+Staiano&amp;author=G+Capano&amp;author=A+Romano&amp;author=L+Florimonte&amp;volume=21&amp;publication_year=1999&amp;pages=307-311&amp;pmid=10413017&amp;doi=10.1016/s0387-7604(99)00025-x&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R84">84. <span class="mixed-citation">Pfeiffer RF (2003). <span class="ref-title">Gastrointestinal dysfunction in Parkinson&#x02019;s disease</span>. <span class="ref-journal">Lancet Neurol</span>
<span class="ref-vol">2</span>, 107&#x02013;116. 10.1016/s1474-4422(03)00307-7.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12849267" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fs1474-4422(03)00307-7" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Lancet+Neurol&amp;title=Gastrointestinal+dysfunction+in+Parkinson&#x02019;s+disease&amp;author=RF+Pfeiffer&amp;volume=2&amp;publication_year=2003&amp;pages=107-116&amp;pmid=12849267&amp;doi=10.1016/s1474-4422(03)00307-7&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R85">85. <span class="mixed-citation">Valicenti-McDermott MD, McVicar K, Cohen HJ, Wershil BK, and Shinnar S (2008). <span class="ref-title">Gastrointestinal symptoms in children with an autism spectrum disorder and language regression</span>. <span class="ref-journal">Pediatr. Neurol</span>
<span class="ref-vol">39</span>, 392&#x02013;398. 10.1016/j.pediatrneurol.2008.07.019.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/19027584" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.pediatrneurol.2008.07.019" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Pediatr.+Neurol&amp;title=Gastrointestinal+symptoms+in+children+with+an+autism+spectrum+disorder+and+language+regression&amp;author=MD+Valicenti-McDermott&amp;author=K+McVicar&amp;author=HJ+Cohen&amp;author=BK+Wershil&amp;author=S+Shinnar&amp;volume=39&amp;publication_year=2008&amp;pages=392-398&amp;pmid=19027584&amp;doi=10.1016/j.pediatrneurol.2008.07.019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R86">86. <span class="mixed-citation">Rajendran PS, Challis RC, Fowlkes CC, Hanna P, Tompkins JD, Jordan MC, Hiyari S, Gabris-Weber BA, Greenbaum A, Chan KY, et al. (2019). <span class="ref-title">Identification of peripheral neural circuits that regulate heart rate using optogenetic and viral vector strategies</span>. <span class="ref-journal">Nat. Commun</span>
<span class="ref-vol">10</span>, 1944. 10.1038/s41467-019-09770-1.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6486614/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31028266" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-019-09770-1" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun&amp;title=Identification+of+peripheral+neural+circuits+that+regulate+heart+rate+using+optogenetic+and+viral+vector+strategies&amp;author=PS+Rajendran&amp;author=RC+Challis&amp;author=CC+Fowlkes&amp;author=P+Hanna&amp;author=JD+Tompkins&amp;volume=10&amp;publication_year=2019&amp;pages=1944&amp;pmid=31028266&amp;doi=10.1038/s41467-019-09770-1&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R87">87. <span class="mixed-citation">Roy A, Guatimosim S, Prado VF, Gros R, and Prado MAM (2015). <span class="ref-title">Cholinergic activity as a new target in diseases of the heart</span>. <span class="ref-journal">Mol. Med</span>
<span class="ref-vol">20</span>, 527&#x02013;537. 10.2119/molmed.2014.00125.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4365064/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25222914" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2119%2Fmolmed.2014.00125" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Mol.+Med&amp;title=Cholinergic+activity+as+a+new+target+in+diseases+of+the+heart&amp;author=A+Roy&amp;author=S+Guatimosim&amp;author=VF+Prado&amp;author=R+Gros&amp;author=MAM+Prado&amp;volume=20&amp;publication_year=2015&amp;pages=527-537&amp;pmid=25222914&amp;doi=10.2119/molmed.2014.00125&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R88">88. <span class="mixed-citation">Mohanta SK, Yin C, Weber C, Godinho-Silva C, Veiga-Fernandes H, Xu QJ, Chang RB, and Habenicht AJR (2023). <span class="ref-title">Cardiovascular Brain Circuits</span>. <span class="ref-journal">Circ. Res</span>
<span class="ref-vol">132</span>, 1546&#x02013;1565. 10.1161/CIRCRESAHA.123.322791.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC10231443/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/37228235" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1161%2FCIRCRESAHA.123.322791" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Circ.+Res&amp;title=Cardiovascular+Brain+Circuits&amp;author=SK+Mohanta&amp;author=C+Yin&amp;author=C+Weber&amp;author=C+Godinho-Silva&amp;author=H+Veiga-Fernandes&amp;volume=132&amp;publication_year=2023&amp;pages=1546-1565&amp;pmid=37228235&amp;doi=10.1161/CIRCRESAHA.123.322791&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R89">89. <span class="mixed-citation">Finneran DJ, Njoku IP, Flores-Pazarin D, Ranabothu MR, Nash KR, Morgan D, and Gordon MN (2021). <span class="ref-title">Toward Development of Neuron Specific Transduction After Systemic Delivery of Viral Vectors</span>. <span class="ref-journal">Front. Neurol</span>
<span class="ref-vol">12</span>, 685802. 10.3389/fneur.2021.685802.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC8426581/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/34512509" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.3389%2Ffneur.2021.685802" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Front.+Neurol&amp;title=Toward+Development+of+Neuron+Specific+Transduction+After+Systemic+Delivery+of+Viral+Vectors&amp;author=DJ+Finneran&amp;author=IP+Njoku&amp;author=D+Flores-Pazarin&amp;author=MR+Ranabothu&amp;author=KR+Nash&amp;volume=12&amp;publication_year=2021&amp;pages=685802&amp;pmid=34512509&amp;doi=10.3389/fneur.2021.685802&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R90">90. <span class="mixed-citation">K&#x000fc;gler S, Kilic E, and B&#x000e4;hr M (2003). <span class="ref-title">Human synapsin 1 gene promoter confers highly neuron-specific long-term transgene expression from an adenoviral vector in the adult rat brain depending on the transduced area</span>. <span class="ref-journal">Gene Ther</span>
<span class="ref-vol">10</span>, 337&#x02013;347. 10.1038/sj.gt.3301905.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12595892" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fsj.gt.3301905" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Gene+Ther&amp;title=Human+synapsin+1+gene+promoter+confers+highly+neuron-specific+long-term+transgene+expression+from+an+adenoviral+vector+in+the+adult+rat+brain+depending+on+the+transduced+area&amp;author=S+K&#x000fc;gler&amp;author=E+Kilic&amp;author=M+B&#x000e4;hr&amp;volume=10&amp;publication_year=2003&amp;pages=337-347&amp;pmid=12595892&amp;doi=10.1038/sj.gt.3301905&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R91">91. <span class="mixed-citation">Moran GW, Leslie FC, Levison SE, Worthington J, and McLaughlin JT (2008). <span class="ref-title">Enteroendocrine cells: neglected players in gastrointestinal disorders?</span>
<span class="ref-journal">Ther Adv Gastroenterol</span>
<span class="ref-vol">1</span>, 51&#x02013;60. 10.1177/1756283X08093943. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3002486/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/21180514" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1177%2F1756283X08093943" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Ther+Adv+Gastroenterol&amp;title=Enteroendocrine+cells:+neglected+players+in+gastrointestinal+disorders?&amp;author=GW+Moran&amp;author=FC+Leslie&amp;author=SE+Levison&amp;author=J+Worthington&amp;author=JT+McLaughlin&amp;volume=1&amp;publication_year=2008&amp;pages=51-60&amp;doi=10.1177/1756283X08093943&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R92">92. <span class="mixed-citation">Lindeberg J, Usoskin D, Bengtsson H, Gustafsson A, Kylberg A, S&#x000f6;derstr&#x000f6;m S, and Ebendal T (2004). <span class="ref-title">Transgenic expression of Cre recombinase from the tyrosine hydroxylase locus</span>. <span class="ref-journal">Genesis</span>
<span class="ref-vol">40</span>, 67&#x02013;73. 10.1002/gene.20065.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/15452869" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fgene.20065" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genesis&amp;title=Transgenic+expression+of+Cre+recombinase+from+the+tyrosine+hydroxylase+locus&amp;author=J+Lindeberg&amp;author=D+Usoskin&amp;author=H+Bengtsson&amp;author=A+Gustafsson&amp;author=A+Kylberg&amp;volume=40&amp;publication_year=2004&amp;pages=67-73&amp;pmid=15452869&amp;doi=10.1002/gene.20065&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R93">93. <span class="mixed-citation">Schindelin J, Arganda-Carreras I, Frise E, Kaynig V, Longair M, Pietzsch T, Preibisch S, Rueden C, Saalfeld S, Schmid B, et al. (2012). <span class="ref-title">Fiji: an open-source platform for biological-image analysis</span>. <span class="ref-journal">Nat. Methods</span>
<span class="ref-vol">9</span>, 676&#x02013;682. 10.1038/nmeth.2019.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3855844/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22743772" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth.2019" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Fiji:+an+open-source+platform+for+biological-image+analysis&amp;author=J+Schindelin&amp;author=I+Arganda-Carreras&amp;author=E+Frise&amp;author=V+Kaynig&amp;author=M+Longair&amp;volume=9&amp;publication_year=2012&amp;pages=676-682&amp;pmid=22743772&amp;doi=10.1038/nmeth.2019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R94">94. <span class="mixed-citation">Langmead B, and Salzberg SL (2012). <span class="ref-title">Fast gapped-read alignment with Bowtie 2</span>. <span class="ref-journal">Nat. Methods</span>
<span class="ref-vol">9</span>, 357&#x02013;359. 10.1038/nmeth.1923.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3322381/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/22388286" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth.1923" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Fast+gapped-read+alignment+with+Bowtie+2&amp;author=B+Langmead&amp;author=SL+Salzberg&amp;volume=9&amp;publication_year=2012&amp;pages=357-359&amp;pmid=22388286&amp;doi=10.1038/nmeth.1923&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R95">95. <span class="mixed-citation">Franzosa EA, McIver LJ, Rahnavard G, Thompson LR, Schirmer M, Weingart G, Lipson KS, Knight R, Caporaso JG, Segata N, and Huttenhower C (2018). <span class="ref-title">Species-level functional profiling of metagenomes and metatranscriptomes</span>. <span class="ref-journal">Nat. Methods</span>
<span class="ref-vol">15</span>, 962&#x02013;968. 10.1038/s41592-018-0176-y.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6235447/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30377376" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41592-018-0176-y" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Species-level+functional+profiling+of+metagenomes+and+metatranscriptomes&amp;author=EA+Franzosa&amp;author=LJ+McIver&amp;author=G+Rahnavard&amp;author=LR+Thompson&amp;author=M+Schirmer&amp;volume=15&amp;publication_year=2018&amp;pages=962-968&amp;pmid=30377376&amp;doi=10.1038/s41592-018-0176-y&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R96">96. <span class="mixed-citation">Truong DT, Franzosa EA, Tickle TL, Scholz M, Weingart G, Pasolli E, Tett A, Huttenhower C, and Segata N (2015). <span class="ref-title">MetaPhlAn2 for enhanced metagenomic taxonomic profiling</span>. <span class="ref-journal">Nat. Methods</span>
<span class="ref-vol">12</span>, 902&#x02013;903. 10.1038/nmeth.3589.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/26418763" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth.3589" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=MetaPhlAn2+for+enhanced+metagenomic+taxonomic+profiling&amp;author=DT+Truong&amp;author=EA+Franzosa&amp;author=TL+Tickle&amp;author=M+Scholz&amp;author=G+Weingart&amp;volume=12&amp;publication_year=2015&amp;pages=902-903&amp;pmid=26418763&amp;doi=10.1038/nmeth.3589&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R97">97. <span class="mixed-citation">Suzek BE, Huang H, McGarvey P, Mazumder R, and Wu CH (2007). <span class="ref-title">UniRef: comprehensive and non-redundant UniProt reference clusters</span>. <span class="ref-journal">Bioinforma Oxf Engl</span>
<span class="ref-vol">23</span>, 1282&#x02013;1288. 10.1093/bioinformatics/btm098. [<a href="https://pubmed.ncbi.nlm.nih.gov/17379688" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtm098" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinforma+Oxf+Engl&amp;title=UniRef:+comprehensive+and+non-redundant+UniProt+reference+clusters&amp;author=BE+Suzek&amp;author=H+Huang&amp;author=P+McGarvey&amp;author=R+Mazumder&amp;author=CH+Wu&amp;volume=23&amp;publication_year=2007&amp;pages=1282-1288&amp;doi=10.1093/bioinformatics/btm098&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R98">98. <span class="mixed-citation">Pluskal T, Castillo S, Villar-Briones A, and Ore&#x00161;i&#x0010d; M (2010). <span class="ref-title">MZmine 2: Modular framework for processing, visualizing, and analyzing mass spectrometry-based molecular profile data</span>. <span class="ref-journal">BMC Bioinf</span>
<span class="ref-vol">11</span>, 395. 10.1186/1471-2105-11-395. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2918584/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/20650010" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2F1471-2105-11-395" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Bioinf&amp;title=MZmine+2:+Modular+framework+for+processing,+visualizing,+and+analyzing+mass+spectrometry-based+molecular+profile+data&amp;author=T+Pluskal&amp;author=S+Castillo&amp;author=A+Villar-Briones&amp;author=M+Ore&#x00161;i&#x0010d;&amp;volume=11&amp;publication_year=2010&amp;pages=395&amp;doi=10.1186/1471-2105-11-395&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R99">99. <span class="mixed-citation">Xia J, Psychogios N, Young N, and Wishart DS (2009). <span class="ref-title">MetaboAnalyst: a web server for metabolomic data analysis and interpretation</span>. <span class="ref-journal">Nucleic Acids Res</span>
<span class="ref-vol">37</span>, W652&#x02013;W660, Web Server issue). 10.1093/nar/gkp356.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2703878/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/19429898" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkp356" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=MetaboAnalyst:+a+web+server+for+metabolomic+data+analysis+and+interpretation&amp;author=J+Xia&amp;author=N+Psychogios&amp;author=N+Young&amp;author=DS+Wishart&amp;volume=37&amp;publication_year=2009&amp;pages=W652-W660&amp;pmid=19429898&amp;doi=10.1093/nar/gkp356&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R100">100. <span class="mixed-citation">Shannon P, Markiel A, Ozier O, Baliga NS, Wang JT, Ramage D, Amin N, Schwikowski B, and Ideker T (2003). <span class="ref-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</span>. <span class="ref-journal">Genome Res</span>
<span class="ref-vol">13</span>, 2498&#x02013;2504. 10.1101/gr.1239303.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC403769/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/14597658" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1101%2Fgr.1239303" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Res&amp;title=Cytoscape:+a+software+environment+for+integrated+models+of+biomolecular+interaction+networks&amp;author=P+Shannon&amp;author=A+Markiel&amp;author=O+Ozier&amp;author=NS+Baliga&amp;author=JT+Wang&amp;volume=13&amp;publication_year=2003&amp;pages=2498-2504&amp;pmid=14597658&amp;doi=10.1101/gr.1239303&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R101">101. <span class="mixed-citation">Consortium UniProt (2019). <span class="ref-title">UniProt: a worldwide hub of protein knowledge</span>. <span class="ref-journal">Nucleic Acids Res</span>
<span class="ref-vol">47</span>, D506&#x02013;D515. 10.1093/nar/gky1049.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6323992/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30395287" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgky1049" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=UniProt:+a+worldwide+hub+of+protein+knowledge&amp;author=UniProt+Consortium&amp;volume=47&amp;publication_year=2019&amp;pages=D506-D515&amp;pmid=30395287&amp;doi=10.1093/nar/gky1049&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R102">102. <span class="mixed-citation">Park SKR, Jung T, Thuy-Boun PS, Wang AY, Yates JR 3rd, and Wolan DW (2019). <span class="ref-title">ComPIL 2.0: An Updated Comprehensive Metaproteomics Database</span>. <span class="ref-journal">J. Proteome Res</span>
<span class="ref-vol">18</span>, 616&#x02013;622. 10.1021/acs.jproteome.8b00722.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7767584/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30525664" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.jproteome.8b00722" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res&amp;title=ComPIL+2.0:+An+Updated+Comprehensive+Metaproteomics+Database&amp;author=SKR+Park&amp;author=T+Jung&amp;author=PS+Thuy-Boun&amp;author=AY+Wang&amp;author=JR+Yates&amp;volume=18&amp;publication_year=2019&amp;pages=616-622&amp;pmid=30525664&amp;doi=10.1021/acs.jproteome.8b00722&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R103">103. <span class="mixed-citation">Xu T, Venable JD, Park SK, Cociorva D, Lu B, Liao L, Wohlschlegel J, Hewel J, and Yates JR (2006). <span class="ref-title">ProLuCID, a fast and sensitive tandem mass spectra-based protein identification program</span>. <span class="ref-journal">Molecular &#x00026; Cellular Proteomics</span>
<span class="ref-vol">5</span>, S174. <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Molecular+&#x00026;+Cellular+Proteomics&amp;title=ProLuCID,+a+fast+and+sensitive+tandem+mass+spectra-based+protein+identification+program&amp;author=T+Xu&amp;author=JD+Venable&amp;author=SK+Park&amp;author=D+Cociorva&amp;author=B+Lu&amp;volume=5&amp;publication_year=2006&amp;pages=S174&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R104">104. <span class="mixed-citation">Xu T, Park SK, Venable JD, Wohlschlegel JA, Diedrich JK, Cociorva D, Lu B, Liao L, Hewel J, Han X, et al. (2015). <span class="ref-title">ProLuCID: An improved SEQUEST-like algorithm with enhanced sensitivity and specificity</span>. <span class="ref-journal">J. Proteonomics</span>
<span class="ref-vol">129</span>, 16&#x02013;24. 10.1016/j.jprot.2015.07.001. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4630125/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/26171723" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1016%2Fj.jprot.2015.07.001" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Proteonomics&amp;title=ProLuCID:+An+improved+SEQUEST-like+algorithm+with+enhanced+sensitivity+and+specificity&amp;author=T+Xu&amp;author=SK+Park&amp;author=JD+Venable&amp;author=JA+Wohlschlegel&amp;author=JK+Diedrich&amp;volume=129&amp;publication_year=2015&amp;pages=16-24&amp;doi=10.1016/j.jprot.2015.07.001&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R105">105. <span class="mixed-citation">Cociorva D, L Tabb D, and Yates JR (2007). <span class="ref-title">Validation of tandem mass spectrometry database search results using DTASelect</span>. <span class="ref-journal">Curr Protoc Bioinforma</span> Chapter <span class="ref-vol">13</span>, Unit 13.4.
 10.1002/0471250953.bi1304s16. [<a href="https://pubmed.ncbi.nlm.nih.gov/18428785" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2F0471250953.bi1304s16" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Curr+Protoc+Bioinforma&amp;title=Validation+of+tandem+mass+spectrometry+database+search+results+using+DTASelect&amp;author=D+Cociorva&amp;author=D+L+Tabb&amp;author=JR+Yates&amp;volume=13&amp;publication_year=2007&amp;doi=10.1002/0471250953.bi1304s16&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R106">106. <span class="mixed-citation">Tabb DL, McDonald WH, and Yates JR (2002). <span class="ref-title">DTASelect and Contrast: Tools for Assembling and Comparing Protein Identifications from Shotgun Proteomics</span>. <span class="ref-journal">J. Proteome Res</span>
<span class="ref-vol">1</span>, 21&#x02013;26. 10.1021/pr015504q.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC2811961/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/12643522" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fpr015504q" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res&amp;title=DTASelect+and+Contrast:+Tools+for+Assembling+and+Comparing+Protein+Identifications+from+Shotgun+Proteomics&amp;author=DL+Tabb&amp;author=WH+McDonald&amp;author=JR+Yates&amp;volume=1&amp;publication_year=2002&amp;pages=21-26&amp;pmid=12643522&amp;doi=10.1021/pr015504q&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R107">107. <span class="mixed-citation">Fu L, Niu B, Zhu Z, Wu S, and Li W (2012). <span class="ref-title">CD-HIT: accelerated for clustering the next-generation sequencing data</span>. <span class="ref-journal">Bioinforma Oxf Engl</span>
<span class="ref-vol">28</span>, 3150&#x02013;3152. 10.1093/bioinformatics/bts565. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3516142/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/23060610" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbts565" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinforma+Oxf+Engl&amp;title=CD-HIT:+accelerated+for+clustering+the+next-generation+sequencing+data&amp;author=L+Fu&amp;author=B+Niu&amp;author=Z+Zhu&amp;author=S+Wu&amp;author=W+Li&amp;volume=28&amp;publication_year=2012&amp;pages=3150-3152&amp;doi=10.1093/bioinformatics/bts565&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R108">108. <span class="mixed-citation">Li W, Jaroszewski L, and Godzik A (2001). <span class="ref-title">Clustering of highly homologous sequences to reduce the size of large protein databases</span>. <span class="ref-journal">Bioinforma Oxf Engl</span>
<span class="ref-vol">17</span>, 282&#x02013;283. 10.1093/bioinformatics/17.3.282. [<a href="https://pubmed.ncbi.nlm.nih.gov/11294794" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2F17.3.282" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinforma+Oxf+Engl&amp;title=Clustering+of+highly+homologous+sequences+to+reduce+the+size+of+large+protein+databases&amp;author=W+Li&amp;author=L+Jaroszewski&amp;author=A+Godzik&amp;volume=17&amp;publication_year=2001&amp;pages=282-283&amp;doi=10.1093/bioinformatics/17.3.282&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R109">109. <span class="mixed-citation">Love MI, Huber W, and Anders S (2014). <span class="ref-title">Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2</span>. <span class="ref-journal">Genome Biol</span>
<span class="ref-vol">15</span>, 550. 10.1186/s13059-014-0550-8.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4302049/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25516281" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs13059-014-0550-8" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Genome+Biol&amp;title=Moderated+estimation+of+fold+change+and+dispersion+for+RNA-seq+data+with+DESeq2&amp;author=MI+Love&amp;author=W+Huber&amp;author=S+Anders&amp;volume=15&amp;publication_year=2014&amp;pages=550&amp;pmid=25516281&amp;doi=10.1186/s13059-014-0550-8&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R110">110. <span class="mixed-citation">Zhang X, Smits AH, van Tilburg GB, Ovaa H, Huber W, and Vermeulen M (2018). <span class="ref-title">Proteome-wide identification of ubiquitin interactions using UbIA-MS</span>. <span class="ref-journal">Nat. Protoc</span>
<span class="ref-vol">13</span>, 530&#x02013;550. 10.1038/nprot.2017.147.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/29446774" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnprot.2017.147" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Protoc&amp;title=Proteome-wide+identification+of+ubiquitin+interactions+using+UbIA-MS&amp;author=X+Zhang&amp;author=AH+Smits&amp;author=GB+van+Tilburg&amp;author=H+Ovaa&amp;author=W+Huber&amp;volume=13&amp;publication_year=2018&amp;pages=530-550&amp;pmid=29446774&amp;doi=10.1038/nprot.2017.147&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R111">111. <span class="mixed-citation">Szklarczyk D, Gable AL, Lyon D, Junge A, Wyder S, Huerta-Cepas J, Simonovic M, Doncheva NT, Morris JH, Bork P, et al. (2019). <span class="ref-title">STRING v11: protein-protein association networks with increased coverage, supporting functional discovery in genome-wide experimental datasets</span>. <span class="ref-journal">Nucleic Acids Res</span>
<span class="ref-vol">47</span>, D607&#x02013;D613. 10.1093/nar/gky1131.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6323986/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30476243" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgky1131" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=STRING+v11:+protein-protein+association+networks+with+increased+coverage,+supporting+functional+discovery+in+genome-wide+experimental+datasets&amp;author=D+Szklarczyk&amp;author=AL+Gable&amp;author=D+Lyon&amp;author=A+Junge&amp;author=S+Wyder&amp;volume=47&amp;publication_year=2019&amp;pages=D607-D613&amp;pmid=30476243&amp;doi=10.1093/nar/gky1131&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R112">112. <span class="mixed-citation">Bushnell B, Rood J, and Singer E (2017). <span class="ref-title">BBMerge - Accurate paired shotgun read merging via overlap</span>. <span class="ref-journal">PLoS One</span>
<span class="ref-vol">12</span>, e0185056. 10.1371/journal.pone.0185056.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5657622/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29073143" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1371%2Fjournal.pone.0185056" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PLoS+One&amp;title=BBMerge+-+Accurate+paired+shotgun+read+merging+via+overlap&amp;author=B+Bushnell&amp;author=J+Rood&amp;author=E+Singer&amp;volume=12&amp;publication_year=2017&amp;pages=e0185056&amp;pmid=29073143&amp;doi=10.1371/journal.pone.0185056&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R113">113. <span class="mixed-citation">Kim D, Langmead B, and Salzberg SL (2015). <span class="ref-title">HISAT: a fast spliced aligner with low memory requirements</span>. <span class="ref-journal">Nat. Methods</span>
<span class="ref-vol">12</span>, 357&#x02013;360. 10.1038/nmeth.3317.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4655817/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25751142" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth.3317" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=HISAT:+a+fast+spliced+aligner+with+low+memory+requirements&amp;author=D+Kim&amp;author=B+Langmead&amp;author=SL+Salzberg&amp;volume=12&amp;publication_year=2015&amp;pages=357-360&amp;pmid=25751142&amp;doi=10.1038/nmeth.3317&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R114">114. <span class="mixed-citation">Anders S, Pyl PT, and Huber W (2015). <span class="ref-title">HTSeq&#x02013;a Python framework to work with high-throughput sequencing data</span>. <span class="ref-journal">Bioinforma Oxf Engl</span>
<span class="ref-vol">31</span>, 166&#x02013;169. 10.1093/bioinformatics/btu638. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4287950/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25260700" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtu638" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinforma+Oxf+Engl&amp;title=HTSeq&#x02013;a+Python+framework+to+work+with+high-throughput+sequencing+data&amp;author=S+Anders&amp;author=PT+Pyl&amp;author=W+Huber&amp;volume=31&amp;publication_year=2015&amp;pages=166-169&amp;doi=10.1093/bioinformatics/btu638&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R115">115. <span class="mixed-citation">Marotz C, Amir A, Humphrey G, Gaffney J, Gogul G, and Knight R (2017). <span class="ref-title">DNA extraction for streamlined metagenomics of diverse environmental samples</span>. <span class="ref-journal">Biotechniques</span>
<span class="ref-vol">62</span>, 290&#x02013;293. 10.2144/000114559.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/28625159" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.2144%2F000114559" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Biotechniques&amp;title=DNA+extraction+for+streamlined+metagenomics+of+diverse+environmental+samples&amp;author=C+Marotz&amp;author=A+Amir&amp;author=G+Humphrey&amp;author=J+Gaffney&amp;author=G+Gogul&amp;volume=62&amp;publication_year=2017&amp;pages=290-293&amp;pmid=28625159&amp;doi=10.2144/000114559&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R116">116. <span class="mixed-citation">Glenn TC, Nilsen RA, Kieran TJ, Sanders JG, Bayona-V&#x000e1;squez NJ, Finger JW, Pierson TW, Bentley KE, Hoffberg SL, Louha S, et al. (2019). <span class="ref-title">Adapterama I: universal stubs and primers for 384 unique dual-indexed or 147,456 combinatorially-indexed Illumina libraries (iTru &#x00026; iNext)</span>. <span class="ref-journal">PeerJ</span>
<span class="ref-vol">7</span>, e7755. 10.7717/peerj.7755.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6791352/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31616586" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7717%2Fpeerj.7755" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Adapterama+I:+universal+stubs+and+primers+for+384+unique+dual-indexed+or+147,456+combinatorially-indexed+Illumina+libraries+(iTru+&#x00026;+iNext)&amp;author=TC+Glenn&amp;author=RA+Nilsen&amp;author=TJ+Kieran&amp;author=JG+Sanders&amp;author=NJ+Bayona-V&#x000e1;squez&amp;volume=7&amp;publication_year=2019&amp;pages=e7755&amp;pmid=31616586&amp;doi=10.7717/peerj.7755&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R117">117. <span class="mixed-citation">Costello M, Fleharty M, Abreu J, Farjoun Y, Ferriera S, Holmes L, Granger B, Green L, Howd T, Mason T, et al. (2018). <span class="ref-title">Characterization and remediation of sample index swaps by non-redundant dual indexing on massively parallel sequencing platforms</span>. <span class="ref-journal">BMC Genom</span>
<span class="ref-vol">19</span>, 332. 10.1186/s12864-018-4703-0. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5941783/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/29739332" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs12864-018-4703-0" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Genom&amp;title=Characterization+and+remediation+of+sample+index+swaps+by+non-redundant+dual+indexing+on+massively+parallel+sequencing+platforms&amp;author=M+Costello&amp;author=M+Fleharty&amp;author=J+Abreu&amp;author=Y+Farjoun&amp;author=S+Ferriera&amp;volume=19&amp;publication_year=2018&amp;pages=332&amp;doi=10.1186/s12864-018-4703-0&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R118">118. <span class="mixed-citation">Sinha R, Stanley GM, Gulati GS, Ezran C, Travaglini KJ, Wei E, Chan CKF, Nabhan AN, Su T, Morganti RM, et al. (2017). <span class="ref-title">Index switching causes &#x0201c;spreading-of-signal&#x0201d; among multiplexed samples in Illumina HiSeq 4000 DNA sequencing</span>. Preprint at
<span class="ref-journal">bioRxiv</span>. 10.1101/125724. <a href="https://api.semanticscholar.org/CorpusID:10764771" data-ga-action="click_feat_suppl" ref="reftype=extlink&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=External%7CLink%7CURI" target="_blank">https://api.semanticscholar.org/CorpusID:10764771</a>. [<a href="//doi.org/10.1101%2F125724" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=bioRxiv&amp;title=Index+switching+causes+&#x0201c;spreading-of-signal&#x0201d;+among+multiplexed+samples+in+Illumina+HiSeq+4000+DNA+sequencing&amp;author=R+Sinha&amp;author=GM+Stanley&amp;author=GS+Gulati&amp;author=C+Ezran&amp;author=KJ+Travaglini&amp;publication_year=2017&amp;doi=10.1101/125724&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R119">119. <span class="mixed-citation">Gonzalez A, Navas-Molina JA, Kosciolek T, McDonald D, V&#x000e1;zquez-Baeza Y, Ackermann G, DeReus J, Janssen S, Swafford AD, Orchanian SB, et al. (2018). <span class="ref-title">Qiita: rapid, web-enabled microbiome meta-analysis</span>. <span class="ref-journal">Nat. Methods</span>
<span class="ref-vol">15</span>, 796&#x02013;798. 10.1038/s41592-018-0141-9.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6235622/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30275573" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41592-018-0141-9" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Qiita:+rapid,+web-enabled+microbiome+meta-analysis&amp;author=A+Gonzalez&amp;author=JA+Navas-Molina&amp;author=T+Kosciolek&amp;author=D+McDonald&amp;author=Y+V&#x000e1;zquez-Baeza&amp;volume=15&amp;publication_year=2018&amp;pages=796-798&amp;pmid=30275573&amp;doi=10.1038/s41592-018-0141-9&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R120">120. <span class="mixed-citation">Didion JP, Martin M, and Collins FS (2017). <span class="ref-title">Atropos: specific, sensitive, and speedy trimming of sequencing reads</span>. <span class="ref-journal">PeerJ</span>
<span class="ref-vol">5</span>, e3720. 10.7717/peerj.3720.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC5581536/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/28875074" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.7717%2Fpeerj.3720" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=PeerJ&amp;title=Atropos:+specific,+sensitive,+and+speedy+trimming+of+sequencing+reads&amp;author=JP+Didion&amp;author=M+Martin&amp;author=FS+Collins&amp;volume=5&amp;publication_year=2017&amp;pages=e3720&amp;pmid=28875074&amp;doi=10.7717/peerj.3720&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R121">121. <span class="mixed-citation">Zhu Q, Mai U, Pfeiffer W, Janssen S, Asnicar F, Sanders JG, Belda-Ferre P, Al-Ghalith GA, Kopylova E, McDonald D, et al. (2019). <span class="ref-title">Phylogenomics of 10,575 genomes reveals evolutionary proximity between domains Bacteria and Archaea</span>. <span class="ref-journal">Nat. Commun</span>
<span class="ref-vol">10</span>, 5477. 10.1038/s41467-019-13443-4.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6889312/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31792218" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41467-019-13443-4" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Commun&amp;title=Phylogenomics+of+10,575+genomes+reveals+evolutionary+proximity+between+domains+Bacteria+and+Archaea&amp;author=Q+Zhu&amp;author=U+Mai&amp;author=W+Pfeiffer&amp;author=S+Janssen&amp;author=F+Asnicar&amp;volume=10&amp;publication_year=2019&amp;pages=5477&amp;pmid=31792218&amp;doi=10.1038/s41467-019-13443-4&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R122">122. <span class="mixed-citation">Hillmann B, Al-Ghalith GA, Shields-Cutler RR, Zhu Q, Gohl DM, Beckman KB, Knight R, and Knights D (2018). <span class="ref-title">Evaluating the Infor mation Content of Shallow Shotgun Metagenomics</span>. <span class="ref-journal">mSystems</span>, <span class="ref-vol">3</span>. 10.1128/msystems.00069-18. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6234283/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30443602" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2Fmsystems.00069-18" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=Evaluating+the+Infor+mation+Content+of+Shallow+Shotgun+Metagenomics&amp;author=B+Hillmann&amp;author=GA+Al-Ghalith&amp;author=RR+Shields-Cutler&amp;author=Q+Zhu&amp;author=DM+Gohl&amp;volume=3&amp;publication_year=2018&amp;doi=10.1128/msystems.00069-18&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R123">123. <span class="mixed-citation">Martino C, Morton JT, Marotz CA, Thompson LR, Tripathi A, Knight R, and Zengler K (2019). <span class="ref-title">A Novel Sparse Compositional Technique Reveals Microbial Perturbations</span>. <span class="ref-journal">mSystems</span>
<span class="ref-vol">4</span>. 10.1128/msystems.00016-19. <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6372836/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/30801021" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1128%2Fmsystems.00016-19" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=mSystems&amp;title=A+Novel+Sparse+Compositional+Technique+Reveals+Microbial+Perturbations&amp;author=C+Martino&amp;author=JT+Morton&amp;author=CA+Marotz&amp;author=LR+Thompson&amp;author=A+Tripathi&amp;volume=4&amp;publication_year=2019&amp;doi=10.1128/msystems.00016-19&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R124">124. <span class="mixed-citation">Caspi R, Billington R, Keseler IM, Kothari A, Krummenacker M, Midford PE, Ong WK, Paley S, Subhraveti P, and Karp PD (2020). <span class="ref-title">The MetaCyc database of metabolic pathways and enzymes - a 2019 update</span>. <span class="ref-journal">Nucleic Acids Res</span>
<span class="ref-vol">48</span>, D445&#x02013;D453. 10.1093/nar/gkz862.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6943030/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31586394" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkz862" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=The+MetaCyc+database+of+metabolic+pathways+and+enzymes+-+a+2019+update&amp;author=R+Caspi&amp;author=R+Billington&amp;author=IM+Keseler&amp;author=A+Kothari&amp;author=M+Krummenacker&amp;volume=48&amp;publication_year=2020&amp;pages=D445-D453&amp;pmid=31586394&amp;doi=10.1093/nar/gkz862&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R125">125. <span class="mixed-citation">Nothias LF, Petras D, Schmid R, D&#x000fc;hrkop K, Rainer J, Sarvepalli A, Protsyuk I, Ernst M, Tsugawa H, Fleischauer M, et al. (2020). <span class="ref-title">Feature-based molecular networking in the GNPS analysis environment</span>. <span class="ref-journal">Nat. Methods</span>
<span class="ref-vol">17</span>, 905&#x02013;908. 10.1038/s41592-020-0933-6.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC7885687/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/32839597" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fs41592-020-0933-6" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Feature-based+molecular+networking+in+the+GNPS+analysis+environment&amp;author=LF+Nothias&amp;author=D+Petras&amp;author=R+Schmid&amp;author=K+D&#x000fc;hrkop&amp;author=J+Rainer&amp;volume=17&amp;publication_year=2020&amp;pages=905-908&amp;pmid=32839597&amp;doi=10.1038/s41592-020-0933-6&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R126">126. <span class="mixed-citation">Elias JE, and Gygi SP (2007). <span class="ref-title">Target-decoy search strategy for increased confidence in large-scale protein identifications by mass spectrometry</span>. <span class="ref-journal">Nat. Methods</span>
<span class="ref-vol">4</span>, 207&#x02013;214. 10.1038/nmeth1019.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/17327847" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth1019" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=Target-decoy+search+strategy+for+increased+confidence+in+large-scale+protein+identifications+by+mass+spectrometry&amp;author=JE+Elias&amp;author=SP+Gygi&amp;volume=4&amp;publication_year=2007&amp;pages=207-214&amp;pmid=17327847&amp;doi=10.1038/nmeth1019&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R127">127. <span class="mixed-citation">Peng J, Elias JE, Thoreen CC, Licklider LJ, and Gygi SP (2003). <span class="ref-title">Evaluation of Multidimensional Chromatography Coupled with Tandem Mass Spectrometry (LC/LC&#x02013;MS/MS) for Large-Scale Protein Analysis: The Yeast Proteome</span>. <span class="ref-journal">J. Proteome Res</span>
<span class="ref-vol">2</span>, 43&#x02013;50. 10.1021/pr025556v.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/12643542" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Fpr025556v" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res&amp;title=Evaluation+of+Multidimensional+Chromatography+Coupled+with+Tandem+Mass+Spectrometry+(LC/LC&#x02013;MS/MS)+for+Large-Scale+Protein+Analysis:+The+Yeast+Proteome&amp;author=J+Peng&amp;author=JE+Elias&amp;author=CC+Thoreen&amp;author=LJ+Licklider&amp;author=SP+Gygi&amp;volume=2&amp;publication_year=2003&amp;pages=43-50&amp;pmid=12643542&amp;doi=10.1021/pr025556v&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R128">128. <span class="mixed-citation">Li W, and Godzik A (2006). <span class="ref-title">Cd-hit: a fast program for clustering and comparing large sets of protein or nucleotide sequences</span>. <span class="ref-journal">Bioinforma Oxf Engl</span>
<span class="ref-vol">22</span>, 1658&#x02013;1659. 10.1093/bioinformatics/btl158. [<a href="https://pubmed.ncbi.nlm.nih.gov/16731699" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtl158" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinforma+Oxf+Engl&amp;title=Cd-hit:+a+fast+program+for+clustering+and+comparing+large+sets+of+protein+or+nucleotide+sequences&amp;author=W+Li&amp;author=A+Godzik&amp;volume=22&amp;publication_year=2006&amp;pages=1658-1659&amp;doi=10.1093/bioinformatics/btl158&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R129">129. <span class="mixed-citation">Park SK, Venable JD, Xu T, and Yates JR (2008). <span class="ref-title">A quantitative analysis software tool for mass spectrometry&#x02013;based proteomics</span>. <span class="ref-journal">Nat. Methods</span>
<span class="ref-vol">5</span>, 319&#x02013;322. 10.1038/nmeth.1195.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC3509211/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/18345006" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1038%2Fnmeth.1195" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nat.+Methods&amp;title=A+quantitative+analysis+software+tool+for+mass+spectrometry&#x02013;based+proteomics&amp;author=SK+Park&amp;author=JD+Venable&amp;author=T+Xu&amp;author=JR+Yates&amp;volume=5&amp;publication_year=2008&amp;pages=319-322&amp;pmid=18345006&amp;doi=10.1038/nmeth.1195&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R130">130. <span class="mixed-citation">Huber W, von Heydebreck A, S&#x000fc;ltmann H, Poustka A, and Vingron M (2002). <span class="ref-title">Variance stabilization applied to microarray data calibration and to the quantification of differential expression</span>. <span class="ref-journal">Bioinforma Oxf Engl</span>
<span class="ref-vol">18</span> (<span class="ref-iss">Suppl 1</span>), S96&#x02013;S104. 10.1093/bioinformatics/18.suppl_1.s96. [<a href="https://pubmed.ncbi.nlm.nih.gov/12169536" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2F18.suppl_1.s96" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinforma+Oxf+Engl&amp;title=Variance+stabilization+applied+to+microarray+data+calibration+and+to+the+quantification+of+differential+expression&amp;author=W+Huber&amp;author=A+von+Heydebreck&amp;author=H+S&#x000fc;ltmann&amp;author=A+Poustka&amp;author=M+Vingron&amp;volume=18&amp;issue=Suppl+1&amp;publication_year=2002&amp;pages=S96-S104&amp;doi=10.1093/bioinformatics/18.suppl_1.s96&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R131">131. <span class="mixed-citation">Gatto L, and Lilley KS (2012). <span class="ref-title">MSnbase-an R/Bioconductor package for isobaric tagged mass spectrometry data visualization, processing and quantitation</span>. <span class="ref-journal">Bioinforma Oxf Engl</span>
<span class="ref-vol">28</span>, 288&#x02013;289. 10.1093/bioinformatics/btr645. [<a href="https://pubmed.ncbi.nlm.nih.gov/22113085" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fbioinformatics%2Fbtr645" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Bioinforma+Oxf+Engl&amp;title=MSnbase-an+R/Bioconductor+package+for+isobaric+tagged+mass+spectrometry+data+visualization,+processing+and+quantitation&amp;author=L+Gatto&amp;author=KS+Lilley&amp;volume=28&amp;publication_year=2012&amp;pages=288-289&amp;doi=10.1093/bioinformatics/btr645&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R132">132. <span class="mixed-citation">Ritchie ME, Phipson B, Wu D, Hu Y, Law CW, Shi W, and Smyth GK (2015). <span class="ref-title">limma powers differential expression analyses for RNAsequencing and microarray studies</span>. <span class="ref-journal">Nucleic Acids Res</span>
<span class="ref-vol">43</span>, e47. 10.1093/nar/gkv007.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC4402510/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/25605792" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1093%2Fnar%2Fgkv007" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Nucleic+Acids+Res&amp;title=limma+powers+differential+expression+analyses+for+RNAsequencing+and+microarray+studies&amp;author=ME+Ritchie&amp;author=B+Phipson&amp;author=D+Wu&amp;author=Y+Hu&amp;author=CW+Law&amp;volume=43&amp;publication_year=2015&amp;pages=e47&amp;pmid=25605792&amp;doi=10.1093/nar/gkv007&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R133">133. <span class="mixed-citation">Gurdeep Singh R, Tanca A, Palomba A, Van der Jeugt F, Verschaffelt P, Uzzau S, Martens L, Dawyndt P, and Mesuere B (2019). <span class="ref-title">Unipept 4.0: Functional Analysis of Metaproteome Data</span>. <span class="ref-journal">J. Proteome Res</span>
<span class="ref-vol">18</span>, 606&#x02013;615. 10.1021/acs.jproteome.8b00716.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/30465426" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1021%2Facs.jproteome.8b00716" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=J.+Proteome+Res&amp;title=Unipept+4.0:+Functional+Analysis+of+Metaproteome+Data&amp;author=R+Gurdeep+Singh&amp;author=A+Tanca&amp;author=A+Palomba&amp;author=F+Van+der+Jeugt&amp;author=P+Verschaffelt&amp;volume=18&amp;publication_year=2019&amp;pages=606-615&amp;pmid=30465426&amp;doi=10.1021/acs.jproteome.8b00716&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R134">134. <span class="mixed-citation">Mesuere B, Debyser G, Aerts M, Devreese B, Vandamme P, and Dawyndt P (2015). <span class="ref-title">The Unipept metaproteomics analysis pipeline</span>. <span class="ref-journal">Proteomics</span>
<span class="ref-vol">15</span>, 1437&#x02013;1442. 10.1002/pmic.201400361.
 [<a href="https://pubmed.ncbi.nlm.nih.gov/25477242" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1002%2Fpmic.201400361" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=Proteomics&amp;title=The+Unipept+metaproteomics+analysis+pipeline&amp;author=B+Mesuere&amp;author=G+Debyser&amp;author=M+Aerts&amp;author=B+Devreese&amp;author=P+Vandamme&amp;volume=15&amp;publication_year=2015&amp;pages=1437-1442&amp;pmid=25477242&amp;doi=10.1002/pmic.201400361&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div><div class="ref-cit-blk half_rhythm" id="R135">135. <span class="mixed-citation">Sundararajan Z, Knoll R, Hombach P, Becker M, Schultze JL, and Ulas T (2019). <span class="ref-title">Shiny-Seq: advanced guided transcriptome analysis</span>. <span class="ref-journal">BMC Res. Notes</span>
<span class="ref-vol">12</span>, 432. 10.1186/s13104-019-4471-1.
 <span class="nowrap">[<a class="int-reflink" href="/pmc/articles/PMC6637470/">PMC free article</a>]</span> [<a href="https://pubmed.ncbi.nlm.nih.gov/31319888" ref="reftype=pubmed&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Entrez%7CPubMed%7CRecord">PubMed</a>] [<a href="//doi.org/10.1186%2Fs13104-019-4471-1" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CCrosslink%7CDOI">CrossRef</a>] <span class="nowrap">[<a href="https://scholar.google.com/scholar_lookup?journal=BMC+Res.+Notes&amp;title=Shiny-Seq:+advanced+guided+transcriptome+analysis&amp;author=Z+Sundararajan&amp;author=R+Knoll&amp;author=P+Hombach&amp;author=M+Becker&amp;author=JL+Schultze&amp;volume=12&amp;publication_year=2019&amp;pages=432&amp;pmid=31319888&amp;doi=10.1186/s13104-019-4471-1&amp;" target="_blank" rel="noopener noreferrer" ref="reftype=other&amp;article-id=11132177&amp;issue-id=463401&amp;journal-id=319&amp;FROM=Article%7CCitationRef&amp;TO=Content%20Provider%7CLink%7CGoogle%20Scholar">Google Scholar</a>]</span></span></div></div></div></div><!--post-content--><div id="body-link-poppers"><span></span></div></div>
            
        </section>
    </article>
    <aside class="usa-width-one-fourth usa-layout-docs-sidenav pmc-sidebar">
         
  

<div class="scroller">

    
        <section>
                <h6>Other Formats</h6>
                <ul class="pmc-sidebar__formats">
                  <li class="pdf-link other_item"><a href="/pmc/articles/PMC11132177/pdf/nihms-1988700.pdf" class="int-view">PDF (3.4M)</a></li>
                </ul>
        </section>
    
    <section>
        <h6>Actions</h6>
        <ul class="pmc-sidebar__actions">
            <li>
                <button role="button" class="citation-button citation-dialog-trigger ctxp"
                        aria-label="Open dialog with citation text in different styles" data-ga-category="save_share" data-ga-action="cite" data-ga-label="open"
                        data-all-citations-url="/pmc/resources/citations/11132177/"
                        data-citation-style="nlm"
                        data-download-format-link="/pmc/resources/citations/11132177/export/"
                >
                    <span class="button-label">Cite</span>
                </button>
            </li>
            <li>
                
                    

<link type="text/css" href="ncbi-overlay-block/src/overlay-block.css">
<div class="collections-button-container" data-article-id="11132177" data-article-db="pmc">
  <button class="collections-button collections-dialog-trigger"
          aria-label="Save article in MyNCBI collections."
          data-ga-category="collections_button"
          data-ga-action="click"
          data-ga-label="collections_button"
          data-collections-open-dialog-enabled="false"
          data-collections-open-dialog-url="https://www.ncbi.nlm.nih.gov/account?back_url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11132177%2F%23open-collections-dialog"
          data-in-collections="false">
      <span class="button-label">Collections</span>
  </button>
  <div class="overlay" role="dialog">
  <div id="collections-action-dialog"
       class="dialog collections-dialog"
       aria-hidden="true">
    <div class="title">Add to Collections</div>
    <div class="collections-action-panel action-panel">
      


<form id="collections-action-dialog-form"
      class="collections-action-panel-form action-panel-content action-form action-panel-smaller-selectors"
      data-existing-collections-url="/pmc/list-existing-collections/"
      data-add-to-existing-collection-url="/pmc/add-to-existing-collection/"
      data-create-and-add-to-new-collection-url="/pmc/create-and-add-to-new-collection/"
      data-myncbi-max-collection-name-length="100"
      data-collections-root-url="https://www.ncbi.nlm.nih.gov/myncbi/collections/">

  <input type="hidden" name="csrfmiddlewaretoken" value="pqU7rL8SzYlKJ3KnG6tPtxSidQIUtuVmlFmqyr8eOUO6T30PmV1OWab9aIRSjHX7">

  

  <div class="choice-group" role="radiogroup">
    <ul class="radio-group-items">
      <li>
        <input type="radio"
               id="collections-action-dialog-new-aside "
               class="collections-new"
               name="collections"
               value="new"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_new">
        <label for="collections-action-dialog-new-aside ">Create a new collection</label>
      </li>
      <li>
        <input type="radio"
               id="collections-action-dialog-existing-aside "
               class="collections-existing"
               name="collections"
               value="existing"
               checked="true"
               data-ga-category="collections_button"
               data-ga-action="click"
               data-ga-label="collections_radio_existing">
        <label for="collections-action-dialog-existing-aside ">Add to an existing collection</label>
      </li>
    </ul>
  </div>

  <div class="controls-wrapper">
    <div class="action-panel-control-wrap new-collections-controls">
      <label for="collections-action-dialog-add-to-new" class="action-panel-label required-field-asterisk">
        Name your collection:
      </label>
      <input
        type="text"
        name="add-to-new-collection"
        id="collections-action-dialog-add-to-new"
        class="collections-action-add-to-new"
        pattern="[^&quot;&amp;=&lt;&gt;/]*" title="The following characters are not allowed in the Name field: &quot;&amp;=&lt;&gt;/"
        maxlength=""
        data-ga-category="collections_button"
        data-ga-action="create_collection"
        data-ga-label="non_favorties_collection">
      <div class="collections-new-name-too-long usa-input-error-message selection-validation-message">
        Name must be less than  characters
      </div>
    </div>
    <div class="action-panel-control-wrap existing-collections-controls">
      <label for="collections-action-dialog-add-to-existing" class="action-panel-label">
        Choose a collection:
      </label>
      <select id="collections-action-dialog-add-to-existing"
              class="action-panel-selector collections-action-add-to-existing"
              data-ga-category="collections_button"
              data-ga-action="select_collection"
              data-ga-label="($('.collections-action-add-to-existing').val() === 'Favorites') ? 'Favorites' : 'non_favorites_collection'">
      </select>
      <div class="collections-retry-load-on-error usa-input-error-message selection-validation-message">
        Unable to load your collection due to an error<br>
        <a href="#">Please try again</a>
      </div>
    </div>
  </div>

  <div class="action-panel-actions">
    <button class="action-panel-submit"
            type="submit"
            data-loading-label="Adding..."
            data-pinger-ignore
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="add">
      Add
    </button>
    <button class="action-panel-cancel"
            aria-label="Close 'Add to Collections' panel"
            ref="linksrc=close_collections_panel"
            aria-controls="collections-action-panel"
            aria-expanded="false"
            data-ga-category="collections_button"
            data-ga-action="click"
            data-ga-label="cancel">
      Cancel
    </button>
  </div>
</form>
    </div>
  </div>
</div>
</div>
                
            </li>

        </ul>
    </section>
    
        <section class="social-sharing">
            <h6>Share</h6>
            <ul class="pmc-sidebar__share">
                <li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://twitter.com/intent/tweet?url=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11132177%2F&amp;text=Peripheral%20neuronal%20activation%20shapes%20the%20microbiome%20and%20alters%20gut%20physiology" alt="Share on Twitter"><i class="fa fa-twitter fa-stack-1x">&#160;</i></a></li> 
<li><a class="fa-stack fa-lg" target="_blank" rel="noopener noreferrer" role="button" href="https://www.facebook.com/sharer/sharer.php?u=https%3A%2F%2Fwww.ncbi.nlm.nih.gov%2Fpmc%2Farticles%2FPMC11132177%2F" alt="Share on Facebook"><i class="fa fa-facebook fa-stack-1x">&#160;</i></a></li>
                <li>
                    <div class="share-permalink">
                        <button class="trigger"  alt="Show article permalink" aria-expanded="false" aria-haspopup="true">
                            <i class="fa-stack fa-lg" >
                                <i class="fa fa-link fa-stack-1x">&nbsp;</i>
                            </i>
                        </button>
                        <div class="dropdown dropdown-container" hidden>
                              <div class="title">
                                Permalink
                              </div>
                              <div class="content">
                                  <input type="text" class="permalink-text" value="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC11132177/" aria-label="Article permalink"><button class="permalink-copy-button usa-button-primary" title="Copy article permalink" data-ga-category="save_share" data-ga-action="link" data-ga-label="copy_link">
                                      <span class="button-title">Copy</span>
                                  </button>
                              </div>
                        </div>
                    </div>
                </li>
            </ul>
        </section>
    
    <section>
        <h6>RESOURCES</h6>
        <ul class="pmc-sidebar__resources">
        
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_similar_articles"
                            data-ga-label="/pmc/articles/PMC11132177/"
                            class="usa-accordion-button"
                            aria-controls="similar-articles-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_similar_articles"
                            data-action-close="close_similar_articles"
                    >
                        Similar articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/similar-article-links/38517896/"
                            

                         class="usa-accordion-content pmc-sidebar__resources--citations" id="similar-articles-accordion-aside" aria-hidden="true">
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_cited_by"
                            data-ga-label="/pmc/articles/PMC11132177/"
                            class="usa-accordion-button"
                            aria-controls="cited-by-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_cited_by"
                            data-action-close="close_cited_by"
                    >
                        Cited by other articles
                    </button>
                    <div
                            
                                data-source-url="/pmc/resources/cited-by-links/38517896/"
                            
                            class="usa-accordion-content pmc-sidebar__resources--citations"
                            id="cited-by-accordion-aside"
                            aria-hidden="true"
                    >
                        
                    </div>
                </div>
            </li>
            <li>
                <div class="usa-accordion">
                    <button
                            data-ga-category="resources_accordion"
                            data-ga-action="open_NCBI_links"
                            data-ga-label="/pmc/articles/PMC11132177/"
                            class="usa-accordion-button"
                            aria-controls="links-accordion-aside"
                            aria-expanded="false"
                            data-action-open="open_NCBI_links"
                            data-action-close="close_NCBI_link"
                    >
                        Links to NCBI Databases
                    </button>
                    <div data-source-url="/pmc/resources/db-links/11132177/" class="usa-accordion-content" id="links-accordion-aside" aria-hidden="true"></div>
                </div>
            </li>

            
        
        </ul>
    </section>

 </div>

    </aside>
</main>

    <div class="overlay" role="dialog" aria-label="Citation Dialog">
  <div class="dialog citation-dialog">
    <button class="close-overlay" tabindex="1">[x]</button>
    <div class="title">Cite</div>
    <div class="citation-text-block">
  <div class="citation-text"></div>
  <div class="citation-actions">
    <button
      class="copy-button dialog-focus"
      data-ga-category="save_share"
      data-ga-action="cite"
      data-ga-label="copy"
      tabindex="2">
      Copy
    </button>

      <a href="#"
              class="export-button"
              data-ga-category="save_share"
              data-ga-action="cite"
              data-ga-label="download"
              title="Download a file for external citation management software"
              tabindex="3">
          <span class="title">Download .nbib</span>
          <span class="title-mobile">.nbib</span>
      </a>


    

<div class="citation-style-selector-wrapper">
  <label class="selector-label">Format:</label>
  <select aria-label="Format" class="citation-style-selector" tabindex="4" >
    
      <option data-style-url-name="ama"
              value="AMA"
              >
        AMA
      </option>
    
      <option data-style-url-name="apa"
              value="APA"
              >
        APA
      </option>
    
      <option data-style-url-name="mla"
              value="MLA"
              >
        MLA
      </option>
    
      <option data-style-url-name="nlm"
              value="NLM"
              selected="selected">
        NLM
      </option>
    
  </select>
</div>
  </div>
</div>
  </div>
</div>

     <!-- ========== BEGIN FOOTER ========== -->
 <footer>
      <section class="icon-section">
        <div id="icon-section-header" class="icon-section_header">Follow NCBI</div>
        <div class="grid-container container">
          <div class="icon-section_container">
            <a class="footer-icon" id="footer_twitter" href="https://twitter.com/ncbi" aria-label="Twitter"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11 {
                      fill: #737373;
                    }
                  </style>
                </defs>
                <title>Twitter</title>
                <path class="cls-11"
                  d="M250.11,105.48c-7,3.14-13,3.25-19.27.14,8.12-4.86,8.49-8.27,11.43-17.46a78.8,78.8,0,0,1-25,9.55,39.35,39.35,0,0,0-67,35.85,111.6,111.6,0,0,1-81-41.08A39.37,39.37,0,0,0,81.47,145a39.08,39.08,0,0,1-17.8-4.92c0,.17,0,.33,0,.5a39.32,39.32,0,0,0,31.53,38.54,39.26,39.26,0,0,1-17.75.68,39.37,39.37,0,0,0,36.72,27.3A79.07,79.07,0,0,1,56,223.34,111.31,111.31,0,0,0,116.22,241c72.3,0,111.83-59.9,111.83-111.84,0-1.71,0-3.4-.1-5.09C235.62,118.54,244.84,113.37,250.11,105.48Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_facebook" href="https://www.facebook.com/ncbi.nlm" aria-label="Facebook"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>Facebook</title>
                <path class="cls-11"
                  d="M210.5,115.12H171.74V97.82c0-8.14,5.39-10,9.19-10h27.14V52l-39.32-.12c-35.66,0-42.42,26.68-42.42,43.77v19.48H99.09v36.32h27.24v109h45.41v-109h35Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_linkedin"
              href="https://www.linkedin.com/company/ncbinlm"
              aria-label="LinkedIn"><svg data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <title>LinkedIn</title>
                <path class="cls-11"
                  d="M101.64,243.37H57.79v-114h43.85Zm-22-131.54h-.26c-13.25,0-21.82-10.36-21.82-21.76,0-11.65,8.84-21.15,22.33-21.15S101.7,78.72,102,90.38C102,101.77,93.4,111.83,79.63,111.83Zm100.93,52.61A17.54,17.54,0,0,0,163,182v61.39H119.18s.51-105.23,0-114H163v13a54.33,54.33,0,0,1,34.54-12.66c26,0,44.39,18.8,44.39,55.29v58.35H198.1V182A17.54,17.54,0,0,0,180.56,164.44Z">
                </path>
              </svg></a>
            <a class="footer-icon" id="footer_github" href="https://github.com/ncbi" aria-label="GitHub"><svg
                data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 300 300">
                <defs>
                  <style>
                    .cls-11,
                    .cls-12 {
                      fill: #737373;
                    }

                    .cls-11 {
                      fill-rule: evenodd;
                    }
                  </style>
                </defs>
                <title>GitHub</title>
                <path class="cls-11"
                  d="M151.36,47.28a105.76,105.76,0,0,0-33.43,206.1c5.28,1,7.22-2.3,7.22-5.09,0-2.52-.09-10.85-.14-19.69-29.42,6.4-35.63-12.48-35.63-12.48-4.81-12.22-11.74-15.47-11.74-15.47-9.59-6.56.73-6.43.73-6.43,10.61.75,16.21,10.9,16.21,10.9,9.43,16.17,24.73,11.49,30.77,8.79,1-6.83,3.69-11.5,6.71-14.14C108.57,197.1,83.88,188,83.88,147.51a40.92,40.92,0,0,1,10.9-28.39c-1.1-2.66-4.72-13.42,1-28,0,0,8.88-2.84,29.09,10.84a100.26,100.26,0,0,1,53,0C198,88.3,206.9,91.14,206.9,91.14c5.76,14.56,2.14,25.32,1,28a40.87,40.87,0,0,1,10.89,28.39c0,40.62-24.74,49.56-48.29,52.18,3.79,3.28,7.17,9.71,7.17,19.58,0,14.15-.12,25.54-.12,29,0,2.82,1.9,6.11,7.26,5.07A105.76,105.76,0,0,0,151.36,47.28Z">
                </path>
                <path class="cls-12"
                  d="M85.66,199.12c-.23.52-1.06.68-1.81.32s-1.2-1.06-.95-1.59,1.06-.69,1.82-.33,1.21,1.07.94,1.6Zm-1.3-1">
                </path>
                <path class="cls-12"
                  d="M90,203.89c-.51.47-1.49.25-2.16-.49a1.61,1.61,0,0,1-.31-2.19c.52-.47,1.47-.25,2.17.49s.82,1.72.3,2.19Zm-1-1.08">
                </path>
                <path class="cls-12"
                  d="M94.12,210c-.65.46-1.71,0-2.37-.91s-.64-2.07,0-2.52,1.7,0,2.36.89.65,2.08,0,2.54Zm0,0"></path>
                <path class="cls-12"
                  d="M99.83,215.87c-.58.64-1.82.47-2.72-.41s-1.18-2.06-.6-2.7,1.83-.46,2.74.41,1.2,2.07.58,2.7Zm0,0">
                </path>
                <path class="cls-12"
                  d="M107.71,219.29c-.26.82-1.45,1.2-2.64.85s-2-1.34-1.74-2.17,1.44-1.23,2.65-.85,2,1.32,1.73,2.17Zm0,0">
                </path>
                <path class="cls-12"
                  d="M116.36,219.92c0,.87-1,1.59-2.24,1.61s-2.29-.68-2.3-1.54,1-1.59,2.26-1.61,2.28.67,2.28,1.54Zm0,0">
                </path>
                <path class="cls-12"
                  d="M124.42,218.55c.15.85-.73,1.72-2,1.95s-2.37-.3-2.52-1.14.73-1.75,2-2,2.37.29,2.53,1.16Zm0,0"></path>
              </svg></a>
            <a class="footer-icon" id="footer_blog" href="https://ncbiinsights.ncbi.nlm.nih.gov/" aria-label="Blog">
              <svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 40 40"><defs><style>.cls-1{fill:#737373;}</style></defs><path class="cls-1" d="M14,30a4,4,0,1,1-4-4,4,4,0,0,1,4,4Zm11,3A19,19,0,0,0,7.05,15a1,1,0,0,0-1,1v3a1,1,0,0,0,.93,1A14,14,0,0,1,20,33.07,1,1,0,0,0,21,34h3a1,1,0,0,0,1-1Zm9,0A28,28,0,0,0,7,6,1,1,0,0,0,6,7v3a1,1,0,0,0,1,1A23,23,0,0,1,29,33a1,1,0,0,0,1,1h3A1,1,0,0,0,34,33Z"/></svg>
            </a>
          </div>
        </div>
      </section>

      <section class="container-fluid bg-primary">
        <div class="container pt-5">
          <div class="row mt-3">
            <div class="col-lg-3 col-12">
              <p><a class="text-white" href="https://www.nlm.nih.gov/socialmedia/index.html">Connect with NLM</a></p>
              <ul class="list-inline social_media">
                <li class="list-inline-item"><a href="https://twitter.com/NLM_NIH" aria-label="Twitter"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st20 {
                          fill: #FFFFFF;
                        }

                        .st30 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Twitter</title>
                      <g>
                        <g>
                          <g>
                            <path class="st20" d="M192.9,88.1c-5,2.2-9.2,2.3-13.6,0.1c5.7-3.4,6-5.8,8.1-12.3c-5.4,3.2-11.4,5.5-17.6,6.7
                                                c-10.5-11.2-28.1-11.7-39.2-1.2c-7.2,6.8-10.2,16.9-8,26.5c-22.3-1.1-43.1-11.7-57.2-29C58,91.6,61.8,107.9,74,116
                                                c-4.4-0.1-8.7-1.3-12.6-3.4c0,0.1,0,0.2,0,0.4c0,13.2,9.3,24.6,22.3,27.2c-4.1,1.1-8.4,1.3-12.5,0.5c3.6,11.3,14,19,25.9,19.3
                                                c-11.6,9.1-26.4,13.2-41.1,11.5c12.7,8.1,27.4,12.5,42.5,12.5c51,0,78.9-42.2,78.9-78.9c0-1.2,0-2.4-0.1-3.6
                                                C182.7,97.4,189.2,93.7,192.9,88.1z"></path>
                          </g>
                        </g>
                        <circle class="st30" cx="124.4" cy="128.8" r="108.2"></circle>
                      </g>
                    </svg></a></li>
                <li class="list-inline-item"><a href="https://www.facebook.com/nationallibraryofmedicine"
                    aria-label="Facebook" rel="noopener noreferrer" target="_blank">
                    <svg version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" x="0px"
                      y="0px" viewBox="0 0 249 249" style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <style type="text/css">
                        .st10 {
                          fill: #FFFFFF;
                        }

                        .st110 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }
                      </style>
                      <title>SM-Facebook</title>
                      <g>
                        <g>
                          <path class="st10" d="M159,99.1h-24V88.4c0-5,3.3-6.2,5.7-6.2h16.8V60l-24.4-0.1c-22.1,0-26.2,16.5-26.2,27.1v12.1H90v22.5h16.9
                                                      v67.5H135v-67.5h21.7L159,99.1z"></path>
                        </g>
                      </g>
                      <circle class="st110" cx="123.6" cy="123.2" r="108.2"></circle>
                    </svg>
                  </a></li>
                <li class="list-inline-item"><a href="https://www.youtube.com/user/NLMNIH" aria-label="Youtube"
                    target="_blank" rel="noopener noreferrer"><svg version="1.1" xmlns="http://www.w3.org/2000/svg"
                      xmlns:xlink="http://www.w3.org/1999/xlink" x="0px" y="0px" viewBox="0 0 249 249"
                      style="enable-background:new 0 0 249 249;" xml:space="preserve">
                      <title>SM-Youtube</title>
                      <style type="text/css">
                        .st4 {
                          fill: none;
                          stroke: #FFFFFF;
                          stroke-width: 8;
                          stroke-miterlimit: 10;
                        }

                        .st5 {
                          fill: #FFFFFF;
                        }
                      </style>
                      <circle class="st4" cx="124.2" cy="123.4" r="108.2"></circle>
                      <g transform="translate(0,-952.36218)">
                        <path class="st5"
                          d="M88.4,1037.4c-10.4,0-18.7,8.3-18.7,18.7v40.1c0,10.4,8.3,18.7,18.7,18.7h72.1c10.4,0,18.7-8.3,18.7-18.7
                                            v-40.1c0-10.4-8.3-18.7-18.7-18.7H88.4z M115.2,1058.8l29.4,17.4l-29.4,17.4V1058.8z"></path>
                      </g>
                    </svg></a></li>
              </ul>
            </div>
            <div class="col-lg-3 col-12">
              <p class="address_footer text-white">National Library of Medicine<br>
                <a href="https://www.google.com/maps/place/8600+Rockville+Pike,+Bethesda,+MD+20894/@38.9959508,-77.101021,17z/data=!3m1!4b1!4m5!3m4!1s0x89b7c95e25765ddb:0x19156f88b27635b8!8m2!3d38.9959508!4d-77.0988323"
                  class="text-white" target="_blank" rel="noopener noreferrer">8600 Rockville Pike<br>
                  Bethesda, MD 20894</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a href="https://www.nlm.nih.gov/web_policies.html" class="text-white">Web Policies</a><br>
                <a href="https://www.nih.gov/institutes-nih/nih-office-director/office-communications-public-liaison/freedom-information-act-office"
                  class="text-white">FOIA</a><br>
                <a href="https://www.hhs.gov/vulnerability-disclosure-policy/index.html" class="text-white" id="vdp">HHS Vulnerability Disclosure</a></p>
            </div>
            <div class="col-lg-3 col-12 centered-lg">
              <p><a class="supportLink text-white" href="https://support.nlm.nih.gov/">Help</a><br>
                <a href="https://www.nlm.nih.gov/accessibility.html" class="text-white">Accessibility</a><br>
                <a href="https://www.nlm.nih.gov/careers/careers.html" class="text-white">Careers</a></p>
            </div>
          </div>
          <div class="row">
            <div class="col-lg-12 centered-lg">
              <nav class="bottom-links">
                <ul class="mt-3">
                  <li>
                    <a class="text-white" href="//www.nlm.nih.gov/">NLM</a>
                  </li>
                  <li>
                    <a class="text-white"
                  href="https://www.nih.gov/">NIH</a>
                  </li>
                  <li>
                    <a class="text-white" href="https://www.hhs.gov/">HHS</a>
                  </li>
                  <li>
                    <a
                  class="text-white" href="https://www.usa.gov/">USA.gov</a>
                  </li>
                </ul>
              </nav>
            </div>
          </div>
        </div>
      </section>
    </footer>
 <!-- ========== END FOOTER ========== -->
  <!-- javascript to inject NWDS meta tags. Note: value of nwds_version is updated by "npm version" command -->
 
  <script type="text/javascript">
    var nwds_version = "1.1.9-2";

    var meta_nwds_ver = document.createElement('meta');
    meta_nwds_ver.name = 'ncbi_nwds_ver';
    meta_nwds_ver.content = nwds_version;
    document.getElementsByTagName('head')[0].appendChild(meta_nwds_ver);

    var meta_nwds = document.createElement('meta');
    meta_nwds.name = 'ncbi_nwds';
    meta_nwds.content = 'yes';
    document.getElementsByTagName('head')[0].appendChild(meta_nwds);

	var alertsUrl = "/core/alerts/alerts.js";
	if (typeof ncbiBaseUrl !== 'undefined') {
		alertsUrl = ncbiBaseUrl + alertsUrl;
	}
  </script>



  
    <!-- JavaScript -->
    <script src="/pmc/static/CACHE/js/output.0f72d6a64937.js"></script>
  
  
    <script src="https://code.jquery.com/jquery-3.5.0.min.js"
          integrity="sha256-xNzN2a4ltkB44Mc/Jz3pT4iU1cmeR0FkXs4pru/JxaQ="
          crossorigin="anonymous">
    </script>
    <script>
        var fallbackJquery = "/pmc/static/base/js/jquery-3.5.0.min.js";
        window.jQuery || document.write("<script src=" + fallbackJquery + ">\x3C/script>")
    </script>
  

  <script src="/pmc/static/CACHE/js/output.a212a9fcf845.js"></script>
<script src="/pmc/static/CACHE/js/output.7999321d1aac.js"></script>
<script src="/pmc/static/CACHE/js/output.7ca436b2ea51.js"></script>
<script src="/pmc/static/CACHE/js/output.f8422046fbe0.js"></script>
<script src="/pmc/static/CACHE/js/output.ff40c7d85ff8.js"></script>
<script src="/pmc/static/CACHE/js/output.a6a84a0ad361.js"></script>

<script type="text/javascript" src="/pmc/static/bundles/base/base.038d36e4e31104b40a19.js" ></script>

    <script type="text/javascript">
        if(typeof jQuery !=='undefined') {
            jQuery.migrateMute = true;
        }
    </script>
    <script type="text/javascript" src="https://code.jquery.com/jquery-migrate-1.4.1.js"></script>
    <script type="text/javascript" src="/core/jig/1.15.2/js/jig.nojquery.min.js">//</script><script type="text/javascript" src="/corehtml/pmc/js/common.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/NcbiTagServer.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/crb.min.js?_=3.18">//</script><script type="text/javascript" src="/corehtml/pmc/js/jactions.min.js?_=3.18">//</script><script type="text/javascript">window.name="mainwindow";</script>

    <script type="text/javascript">var exports = {};</script>
    <script src="/pmc/static/CACHE/js/output.340a3b9cce7f.js"></script>
<script src="/pmc/static/CACHE/js/output.228f96f3298e.js"></script>
    <script type="text/javascript" src="/pmc/static/bundles/article/article.b7dd20e527283dd68f45.js" ></script>
    <script type="text/javascript">
        window.ncbi.pmc.articlePage.init({ pageURL: '/pmc/articles/PMC11132177/', citeCookieName: 'pmc-cf'});
    </script>


  
  
  <script  type="text/javascript" src="https://www.ncbi.nlm.nih.gov/core/pinger/pinger.js"> </script>


  
      
  

</body>
</html>
